<html>
<body>
<H2>
Discordant Protein and mRNA Expression in Lung Adenocarcinomas <A NAME="RFN3"></A><SUP><A HREF="#FN3">*</A></SUP>
</H2>

<STRONG>
</NOBR><NOBR>Guoan Chen<SUP><IMG SRC="/math/Dagger.gif" ALT="{ddagger}" BORDER="0"></SUP></NOBR>, 
<NOBR>Tarek G. Gharib<SUP><IMG SRC="/math/Dagger.gif" ALT="{ddagger}" BORDER="0"></SUP></NOBR>, 
<NOBR>Chiang-Ching Huang<SUP><IMG SRC="/math/sect.gif" ALT="&#167;" BORDER="0"></SUP></NOBR>, 
<NOBR>Jeremy M. G. Taylor<SUP><IMG SRC="/math/sect.gif" ALT="&#167;" BORDER="0"></SUP></NOBR>, 
<NOBR>David E. Misek<SUP>&#182;</SUP></NOBR>, 
<NOBR>Sharon L. R. Kardia<SUP>||</SUP></NOBR>, 
<NOBR>Thomas J. Giordano<SUP>**</SUP></NOBR>, 
<NOBR>Mark D. Iannettoni<SUP><IMG SRC="/math/Dagger.gif" ALT="{ddagger}" BORDER="0"></SUP></NOBR>, 
<NOBR>Mark B. Orringer<SUP><IMG SRC="/math/Dagger.gif" ALT="{ddagger}" BORDER="0"></SUP></NOBR>, 
<NOBR>Samir M. Hanash<SUP>&#182;</SUP></NOBR> and 
<NOBR>David G. Beer<SUP><IMG SRC="/math/Dagger.gif" ALT="{ddagger}" BORDER="0"></SUP><SUP>,<A HREF="#COR1"><IMG SRC="/math/link//Dagger.gif" ALT="{ddagger}" BORDER="0"><IMG SRC="/math/link//Dagger.gif" ALT="{ddagger}" BORDER="0"></A></SUP></NOBR>
</STRONG><P>
<FONT SIZE=-1>
<SUP><IMG SRC="/math/Dagger.gif" ALT="{ddagger}" BORDER="0"></SUP> Department of Surgery, University of Michigan, Ann Arbor, Michigan 48109<BR>
<SUP><IMG SRC="/math/sect.gif" ALT="&#167;" BORDER="0"></SUP> Department of Biostatistics, University of Michigan, Ann Arbor, Michigan 48109<BR>
<SUP>||</SUP> Department of Epidemiology, University of Michigan, Ann Arbor, Michigan 48109<BR>
<SUP>**</SUP> Department of Pathology,University of Michigan, Ann Arbor, Michigan 48109<BR>
<SUP>&#182;</SUP> Department of Pediatrics, University of Michigan, Ann Arbor, Michigan 48109
</FONT><P>




<P>
<A NAME="ABS"><!-- null --></A>
<BR CLEAR=right><TABLE WIDTH=100% BGCOLOR=e1e1e1 CELLPADDING=0 CELLSPACING=0>
 <TR><TD ALIGN=left VALIGN=middle WIDTH=5% BGCOLOR=ffffff><IMG WIDTH=10 HEIGHT=21 HSPACE=5 ALT=" "
SRC="/icons/toc/rarrow.gif"></TD>
 <TH ALIGN=left VALIGN=middle WIDTH=95%><FONT SIZE=+2>&nbsp;&nbsp;
ABSTRACT
</FONT></TH></TR></TABLE>
<TABLE ALIGN=right CELLPADDING=5 BORDER><TR><TH
ALIGN=left><FONT SIZE=-1>
<A HREF="#top"><IMG WIDTH=11 HEIGHT=9 BORDER=0 HSPACE=5 ALT=" "
SRC="/icons/toc/uarrow.gif">TOP<BR></A>
<IMG WIDTH=11 HEIGHT=9 BORDER=0 HSPACE=5 ALT=" "
SRC="/icons/toc/dot.gif"><FONT COLOR=464c53>ABSTRACT</FONT><BR>
<A  HREF="#SEC1"><IMG WIDTH=11 HEIGHT=9 BORDER=0 HSPACE=5 ALT=" "
SRC="/icons/toc/darrow.gif">EXPERIMENTAL PROCEDURES<BR></A>
<A  HREF="#SEC2"><IMG WIDTH=11 HEIGHT=9 BORDER=0 HSPACE=5 ALT=" "
SRC="/icons/toc/darrow.gif">RESULTS<BR></A>
<A  HREF="#SEC3"><IMG WIDTH=11 HEIGHT=9 BORDER=0 HSPACE=5 ALT=" "
SRC="/icons/toc/darrow.gif">DISCUSSION<BR></A>
<A  HREF="#BIBL"><IMG WIDTH=11 HEIGHT=9 BORDER=0 HSPACE=5 ALT=" "
SRC="/icons/toc/darrow.gif">REFERENCES<BR></A>
</FONT></TH></TR></TABLE>
&nbsp;<BR>


The relationship between gene expression measured at the mRNA<SUP> </SUP>level and the corresponding protein level is not well characterized<SUP> </SUP>in human cancer. In this study, we compared mRNA and protein<SUP> </SUP>expression for a cohort of genes in the same lung adenocarcinomas.<SUP> </SUP>The abundance of 165 protein spots representing 98 individual<SUP> </SUP>genes was analyzed in 76 lung adenocarcinomas and nine non-neoplastic<SUP> </SUP>lung tissues using two-dimensional polyacrylamide gel electrophoresis.<SUP> </SUP>Specific polypeptides were identified using matrix-assisted<SUP> </SUP>laser desorption/ionization mass spectrometry. For the same<SUP> </SUP>85 samples, mRNA levels were determined using oligonucleotide<SUP> </SUP>microarrays, allowing a comparative analysis of mRNA and protein<SUP> </SUP>expression among the 165 protein spots. Twenty-eight of the<SUP> </SUP>165 protein spots (17%) or 21 of 98 genes (21.4%) had a statistically<SUP> </SUP>significant correlation between protein and mRNA expression<SUP> </SUP>(<I>r</I> &gt; 0.2445; <I>p</I> &lt; 0.05); however, among all 165 proteins<SUP> </SUP>the correlation coefficient values (<I>r</I>) ranged from -0.467 to<SUP> </SUP>0.442. Correlation coefficient values were not related to protein<SUP> </SUP>abundance. Further, no significant correlation between mRNA<SUP> </SUP>and protein expression was found (<I>r</I> = -0.025) if the average<SUP> </SUP>levels of mRNA or protein among all samples were applied across<SUP> </SUP>the 165 protein spots (98 genes). The mRNA/protein correlation<SUP> </SUP>coefficient also varied among proteins with multiple isoforms,<SUP> </SUP>indicating potentially separate isoform-specific mechanisms<SUP> </SUP>for the regulation of protein abundance. Among the 21 genes<SUP> </SUP>with a significant correlation between mRNA and protein, five<SUP> </SUP>genes differed significantly between stage I and stage III lung<SUP> </SUP>adenocarcinomas. Using a quantitative analysis of mRNA and protein<SUP> </SUP>expression within the same lung adenocarcinomas, we showed that<SUP> </SUP>only a subset of the proteins exhibited a significant correlation<SUP> </SUP>with mRNA abundance.<SUP> </SUP><P>

<HR NOSHADE SIZE=1 WIDTH="50%" ALIGN="CENTER">


Lung cancer is the leading cause of cancer death for both men<SUP> </SUP>and women in the United States. Adenocarcinomas of the lung<SUP> </SUP>comprise <IMG SRC="/math/sim.gif" ALT="~" BORDER="0">40% of all new cases of non-small cell lung cancer<SUP> </SUP>and are now the most common histologic type. Functional genomics,<SUP> </SUP>broadly defined as the comprehensive analysis of genes and their<SUP> </SUP>products, have become a recent focus of the life sciences (<A HREF="#R1">1</A>).<SUP> </SUP>Application of these approaches to lung adenocarcinomas has<SUP> </SUP>the potential to aid in the identification of high risk patients<SUP> </SUP>with resectable early stage lung cancer that may benefit from<SUP> </SUP>adjuvant therapy, as well as to identify new therapeutic targets.<SUP> </SUP>In human lung cancer, however, little is currently understood<SUP> </SUP>regarding the relationship between gene expression as determined<SUP> </SUP>by measuring mRNA levels and the corresponding abundance of<SUP> </SUP>the protein products.<SUP> </SUP><P>

A number of powerful techniques for analysis of gene expression<SUP> </SUP>have been used including differential display (<A HREF="#R2">2</A>), serial analysis<SUP> </SUP>of gene expression (<A HREF="#R3">3</A>), DNA microarrays (<A HREF="#R4">4</A>), and proteomics<SUP> </SUP>via two-dimensional polyacrylamide gel electrophoresis and mass<SUP> </SUP>spectrometry (<A HREF="#R5">5</A>). Bioinformatics tools have also been developed<SUP> </SUP>to help determine quantitative mRNA/protein expression profiles<SUP> </SUP>of all types of cells and tissues (<A HREF="#R6">6</A>) and now can be applied<SUP> </SUP>to benign and malignant tumors. DNA microarrays (cDNA and oligonucleotide)<SUP> </SUP>permit the parallel assessment of thousands of genes and have<SUP> </SUP>been utilized in gene expression monitoring (<A HREF="#R7">7</A>), polymorphism<SUP> </SUP>analysis (<A HREF="#R8">8</A>), and DNA sequencing (<A HREF="#R9">9</A>). Recent studies have focused<SUP> </SUP>on classification or identification of subgroups of lung tumors<SUP> </SUP>using DNA microarrays (<A HREF="#R10">10</A>, <A HREF="#R11">11</A>). The use of mRNA expression patterns<SUP> </SUP>by themselves, however, is insufficient for understanding the<SUP> </SUP>expression of protein products, as additional post-transcriptional<SUP> </SUP>mechanisms, including protein translation, post-translational<SUP> </SUP>modification, and degradation, may influence the level of a<SUP> </SUP>protein present in a given cell or tissue. Proteomic analyses,<SUP> </SUP>a complementary technology to DNA microarrays for monitoring<SUP> </SUP>gene expression, involves protein separation and quantitative<SUP> </SUP>assessment of protein spots using 2D <A NAME="RFN1"></A><SUP><A HREF="#FN1">1</A></SUP> -PAGE and protein identification<SUP> </SUP>using mass spectrometry. By combining proteomic and transcriptional<SUP> </SUP>analyses of the same samples, however, it may be possible to<SUP> </SUP>understand the complex mechanisms influencing protein expression<SUP> </SUP>in human cancer.<SUP> </SUP><P>

In this study, we determined mRNA and protein levels for 165<SUP> </SUP>proteins (98 genes) in 76 lung adenocarcinomas and nine non-neoplastic<SUP> </SUP>lung tissues. Protein levels were determined using quantitative<SUP> </SUP>2D-PAGE analysis, and the separated protein polypeptides were<SUP> </SUP>identified using matrix-assisted laser desorption/ionization<SUP> </SUP>mass spectrometry (MALDI-MS). The corresponding mRNA levels<SUP> </SUP>for the identified proteins within the same samples were determined<SUP> </SUP>using oligonucleotide microarrays. Correlation analyses showed<SUP> </SUP>that protein abundance is likely a reflection of the transcription<SUP> </SUP>for a subset of proteins, but translation and post-translational<SUP> </SUP>modifications also appear to influence the expression levels<SUP> </SUP>of many individual proteins in lung adenocarcinomas.<SUP> </SUP><P>

<A NAME="SEC1"><!-- null --></A>
<BR CLEAR=right><TABLE WIDTH=100% BGCOLOR=e1e1e1 CELLPADDING=0 CELLSPACING=0>
 <TR><TD ALIGN=left VALIGN=middle WIDTH=5% BGCOLOR=ffffff><IMG WIDTH=10 HEIGHT=21 HSPACE=5 ALT=" "
SRC="/icons/toc/rarrow.gif"></TD>
 <TH ALIGN=left VALIGN=middle WIDTH=95%><FONT SIZE=+2>&nbsp;&nbsp;
EXPERIMENTAL PROCEDURES
</FONT></TH></TR></TABLE>
<TABLE ALIGN=right CELLPADDING=5 BORDER><TR><TH
ALIGN=left><FONT SIZE=-1>
<A HREF="#top"><IMG WIDTH=11 HEIGHT=9 BORDER=0 HSPACE=5 ALT=" "
SRC="/icons/toc/uarrow.gif">TOP<BR></A>
<A  HREF="#ABS"><IMG WIDTH=11 HEIGHT=9 BORDER=0 HSPACE=5 ALT=" "
SRC="/icons/toc/uarrow.gif">ABSTRACT<BR></A>
<IMG WIDTH=11 HEIGHT=9 BORDER=0 HSPACE=5 ALT=" "
SRC="/icons/toc/dot.gif"><FONT COLOR=464c53>EXPERIMENTAL PROCEDURES</FONT><BR>
<A  HREF="#SEC2"><IMG WIDTH=11 HEIGHT=9 BORDER=0 HSPACE=5 ALT=" "
SRC="/icons/toc/darrow.gif">RESULTS<BR></A>
<A  HREF="#SEC3"><IMG WIDTH=11 HEIGHT=9 BORDER=0 HSPACE=5 ALT=" "
SRC="/icons/toc/darrow.gif">DISCUSSION<BR></A>
<A  HREF="#BIBL"><IMG WIDTH=11 HEIGHT=9 BORDER=0 HSPACE=5 ALT=" "
SRC="/icons/toc/darrow.gif">REFERENCES<BR></A>
</FONT></TH></TR></TABLE>
&nbsp;<BR>

<STRONG><EM>Tissues&#151;</EM></STRONG><BR>

Fifty-seven stage I and 19 stage III lung adenocarcinomas, as<SUP> </SUP>well as nine non-neoplastic lung tissue samples, were used for<SUP> </SUP>protein and mRNA analyses. Patient consent was obtained, and<SUP> </SUP>the project was approved by the Institutional Review Board.<SUP> </SUP>All tissues were obtained after resection at the University<SUP> </SUP>of Michigan Health System between May 1991 and July 1998. Tissues<SUP> </SUP>were all snap-frozen in liquid nitrogen and then stored at -80&#176;C.<SUP> </SUP>The patients included 46 females and 30 males ranging in age<SUP> </SUP>from 40.9 to 84.6 (average 63.8) years. Most patients (66/76)<SUP> </SUP>demonstrated a positive smoking history. Sixty-one tumor samples<SUP> </SUP>were classified as bronchial-derived, 14 were classified as<SUP> </SUP>bronchoalveolar, and one had both features. Eighteen tumor samples<SUP> </SUP>were classified as well differentiated, 38 were classified as<SUP> </SUP>moderate, and 19 were classified as poorly differentiated adenocarcinomas.<SUP> </SUP>Hematoxylin-stained cryostat sections (5 &#181;m), prepared<SUP> </SUP>from the same tumor pieces to be utilized for protein and mRNA<SUP> </SUP>isolation, were evaluated by a pathologist and compared with<SUP> </SUP>hematoxylin- and eosin-stained sections made from paraffin blocks<SUP> </SUP>of the same tumors. Specimens were excluded from analysis if<SUP> </SUP>they showed unclear or mixed histology (<I>e.g.</I> adenosquamous),<SUP> </SUP>tumor cellularity less than 70%, potential metastatic origin<SUP> </SUP>as indicated by previous tumor history, extensive lymphocytic<SUP> </SUP>infiltration, or fibrosis or if the patient had received prior<SUP> </SUP>chemotherapy or radiotherapy.<SUP> </SUP><P>

<STRONG><EM>Oligonucleotide Array Hybridization&#151;</EM></STRONG><BR>

The HuGeneFL oligonucleotide arrays (Affymetrix, Santa Clara,<SUP> </SUP>CA) containing 6800 genes were used in this study. Total RNA<SUP> </SUP>was isolated from all samples using Trizol reagent (Invitrogen).<SUP> </SUP>The resulting RNA was then subjected to further purification<SUP> </SUP>using RNeasy spin columns (Qiagen). Preparation of cRNA, hybridization,<SUP> </SUP>and scanning of the HuGeneFL arrays were performed according<SUP> </SUP>to the manufacturer&#146;s protocol (Affymetrix, Santa Clara,<SUP> </SUP>CA). Data analysis was performed using GeneChip 4.0 software.<SUP> </SUP>The gene expression profile of each tumor was normalized to<SUP> </SUP>the median gene expression profile for the entire sample. Details<SUP> </SUP>of data trimming and normalization are described elsewhere (<A HREF="#R11">11</A>).<SUP> </SUP><P>

<STRONG><EM>2D-PAGE and Quantitative Protein Analysis&#151;</EM></STRONG><BR>

Tissue for both protein and mRNA isolation came from contiguous<SUP> </SUP>areas of each sample. Protein separation using 2D-PAGE, silver<SUP> </SUP>staining, and digitization were performed as described previously<SUP> </SUP>(<A HREF="#R12">12</A>, <A HREF="#R13">13</A>). Our 2D-PAGE system allows us to run 20 gels at one<SUP> </SUP>time (one batch). Spot detection and quantification were accomplished<SUP> </SUP>utilizing Bio Image Visage System software (Bioimage Corp.,<SUP> </SUP>Ann Arbor, MI). The integrated intensity of each spot was calculated<SUP> </SUP>as the measured optical density units <FONT FACE="arial,helvetica">x</FONT> mm<SUP>2</SUP>. Of the total possible<SUP> </SUP>2000 spots detectable on each gel, 820 spots on the gel of each<SUP> </SUP>sample were matched using a Gel-ed match program with the same<SUP> </SUP>spots on a chosen "master" gel. In each sample, 250 ubiquitously<SUP> </SUP>expressed reference spots were used to adjust for variations<SUP> </SUP>between gels, such as that created by subtle differences in<SUP> </SUP>protein loading or gel staining. Slight differences because<SUP> </SUP>of batch were corrected after spot-size quantification.<SUP> </SUP><P>

<STRONG><EM>Mass Spectrometry and 2D Western Blotting&#151;</EM></STRONG><BR>

Preparative 2D gels were run using extracts from A549 lung adenocarcinoma<SUP> </SUP>cells (obtained from ATCC) and using the identical experimental<SUP> </SUP>conditions as the analytical 2D gels, except 30% more protein<SUP> </SUP>was loaded. The resolved protein gels were silver-stained using<SUP> </SUP>successive incubations in 0.02% sodium thiosulfate for 2 min,<SUP> </SUP>0.1% silver nitrate for 40 min, and 0.014% formaldehyde plus<SUP> </SUP>2% sodium carbonate for 10 min. For protein identification,<SUP> </SUP>protein polypeptides underwent trypsin digestion followed by<SUP> </SUP>MALDI-MS using a MALDI-TOF Voyager-DE mass spectrometer (Perseptive<SUP> </SUP>Biosystems, Framingham, MA). The masses were compared with known<SUP> </SUP>trypsin digest databases using the MS-FIT database (University<SUP> </SUP>of California, San Francisco; prospector.ucsf.edu/ucsfhtml3.2/msfit.htm).<SUP> </SUP>Some of the polypeptides included in the analysis had been identified<SUP> </SUP>prior to this study on the basis of sequencing (<A HREF="#R14">14</A>). The identified<SUP> </SUP>protein spots used in this paper are shown in <A HREF="#F1">Fig. 1<I>A</I></A>. The method<SUP> </SUP>for 2D-PAGE Western blot verification was as described previously<SUP> </SUP>(<A HREF="#R15">15</A>). The 2D Western blots of GRP58 and Op18 are shown in <A HREF="#F1">Fig.<SUP> </SUP>1, <I>C</I> and <I>E</I></A>; the others, such as GRP78, GRP75, HSP70, HSC70,<SUP> </SUP>KRT8, KRT18, KRT19, Vimentin, ApoJ, 14&#150;3-3, Annexin I,<SUP> </SUP>Annexin II, PGP9.5, DJ-1, GST-pi, and PGAM, are described elsewhere.<SUP> </SUP><A NAME="RFN2"></A><SUP><A HREF="#FN2">2</A></SUP><SUP> </SUP><P>

<A NAME="F1"><!-- null --></A>
<BR CLEAR=all><CENTER><TABLE WIDTH=95% CELLPADDING=0 
CELLSPACING=0><TR BGCOLOR=e1e1e1><TD><TABLE CELLPADDING=2 CELLSPACING=2>
<TR BGCOLOR=e1e1e1><TD ALIGN=center VALIGN=top BGCOLOR=ffffff>
<A HREF="/cgi/content/full/1/4/304/F1"><IMG HSPACE=10 VSPACE=5 BORDER=2 WIDTH=200 HEIGHT=115 SRC="/content/vol1/issue4/images/small/jw0320033001.gif" ALT=" "></A><BR>

<STRONG>View larger version</STRONG> (83K):<BR>
<NOBR><A HREF="/cgi/content/full/1/4/304/F1">[in this window]</A><BR>
<A HREF="/cgi/content-nw/full/1/4/304/F1"
    onClick="startTarget('F1', 590, 452); this.href='/cgi/content-nw/full/1/4/304/F1'"
 	onMouseOver="window.status='View figure in a separate window'; return true"
 	TARGET="F1">[in a new window]</A><BR>&#160;</NOBR>
</TD><TD ALIGN=left VALIGN=top>
F<FONT SIZE=-2>IG</FONT>. 1. <B>A</B>, digital image of a silver-stained 2D-PAGE separation of a stage I lung adenocarcinoma showing protein spots separated by molecular mass (<I>MW</I>) and isoelectric point (<I>PI</I>). Twenty-eight protein spots whose expression levels are correlated with mRNA abundance are indicated by the <I>black arrows</I>. <I>B</I>, the outlined areas of <I>A</I> showing protein GRP58. <I>C</I>, 2D Western blot of GRP58 from the A549 lung adenocarcinoma cell line. <I>D</I>, the outlined areas of <I>A</I> showing the protein isoforms of Op18. <I>E</I>, 2D Western blot of Op18 from A549 cells.<P>


</TD></TR></TABLE>
</TD></TR></TABLE></CENTER>&nbsp;<BR>


<STRONG><EM>Statistical Analysis&#151;</EM></STRONG><BR>

Missing values were replaced with the mean value of the protein<SUP> </SUP>spot. The transform x <IMG SRC="/math/rarr.gif" ALT="-&gt;" BORDER="0"> log (1 + x) was applied to normalize<SUP> </SUP>all protein expression values. The relationship between protein<SUP> </SUP>and mRNA expression levels within the same samples was examined<SUP> </SUP>using the Spearman correlation coefficient analysis (<A HREF="#R16">16</A>). To<SUP> </SUP>identify potentially significant correlations between gene and<SUP> </SUP>protein expression, we used an analytical strategy similar to<SUP> </SUP>SAM (significance analysis of microarrays) (<A HREF="#R17">17</A>), which uses<SUP> </SUP>a permutation technique to determine the significance of changes<SUP> </SUP>in gene expression between different biological states. To obtain<SUP> </SUP>permuted correlation coefficients between gene and protein expression,<SUP> </SUP>genes were exchanged first in such a way that permutated correlation<SUP> </SUP>coefficient were calculated based on pseudo pairs of genes and<SUP> </SUP>proteins. The distribution of permutated correlation coefficients<SUP> </SUP>became stable after 60 permutations. This procedure was then<SUP> </SUP>repeated 60 times to obtain 60 sets of permutated correlation<SUP> </SUP>coefficients. For each of the 60 permutations, the correlations<SUP> </SUP>of genes and proteins were ranked such that <I><IMG SRC="/math/rgr.gif" ALT="{rho}" BORDER="0"><SUB>p</SUB></I>(<I>i</I>) denotes the<SUP> </SUP><I>i</I>th largest correlation coefficient for <I>p</I>th permutation. Hence,<SUP> </SUP>the expected correlation coefficient, <I><IMG SRC="/math/rgr.gif" ALT="{rho}" BORDER="0"><SUB>E</SUB></I>(<I>i</I>), was the average<SUP> </SUP>over the 60 permutations,
<IMG SRC="/content/vol1/issue4/fulltext/304/f1.gif" BORDER="0">. A scatter plot of observed correlations (<I><IMG SRC="/math/rgr.gif" ALT="{rho}" BORDER="0"></I>(<I>i</I>)) <I>versus</I> the expected correlations<SUP> </SUP>is shown in <A HREF="#F2">Fig. 2<I>D</I></A>. For this study, we chose threshold <IMG SRC="/math/Dgr.gif" ALT="{Delta}" BORDER="0"> = 0.115<SUP> </SUP>so that correlation would be considered significant if absolute<SUP> </SUP>value of difference between <I><IMG SRC="/math/rgr.gif" ALT="{rho}" BORDER="0"></I>(<I>i</I>) and <I><IMG SRC="/math/rgr.gif" ALT="{rho}" BORDER="0"><SUB>E</SUB></I>(<I>i</I>) was greater than the<SUP> </SUP>threshold. Twenty-nine (including one with observed correlation<SUP> </SUP>coefficient -0.4672) of 165 pairs of gene and protein expression<SUP> </SUP>were called significant in such criteria, and the permuted data<SUP> </SUP>generated an average of 5.1 falsely significant pairs of gene<SUP> </SUP>and protein expression. This provided an estimated false discovery<SUP> </SUP>rate (the percentage of pairs of gene and protein expression<SUP> </SUP>identified by chance) for our data set.<SUP> </SUP><P>

<A NAME="F2"><!-- null --></A>
<BR CLEAR=all><CENTER><TABLE WIDTH=95% CELLPADDING=0 
CELLSPACING=0><TR BGCOLOR=e1e1e1><TD><TABLE CELLPADDING=2 CELLSPACING=2>
<TR BGCOLOR=e1e1e1><TD ALIGN=center VALIGN=top BGCOLOR=ffffff>
<A HREF="/cgi/content/full/1/4/304/F2"><IMG HSPACE=10 VSPACE=5 BORDER=2 WIDTH=200 HEIGHT=125 SRC="/content/vol1/issue4/images/small/jw0320033002.gif" ALT=" "></A><BR>

<STRONG>View larger version</STRONG> (18K):<BR>
<NOBR><A HREF="/cgi/content/full/1/4/304/F2">[in this window]</A><BR>
<A HREF="/cgi/content-nw/full/1/4/304/F2"
    onClick="startTarget('F2', 590, 474); this.href='/cgi/content-nw/full/1/4/304/F2'"
 	onMouseOver="window.status='View figure in a separate window'; return true"
 	TARGET="F2">[in a new window]</A><BR>&#160;</NOBR>
</TD><TD ALIGN=left VALIGN=top>
F<FONT SIZE=-2>IG</FONT>. 2. <B>A&#150;C</B>, plots showing the correlation between mRNA and protein for the three selected genes Op18, Annexin IV, and GAPD for all 76 lung adenocarcinomas and nine non-neoplastic lung samples (<I>p</I> &lt; 0.05). <I>D</I>, distribution of all 165 Spearman correlation coefficients (<I>r</I>) and verification analysis using SAM. A more detailed description of the method is provided under "Experimental Procedures." Approximately 17% of the 165 proteins demonstrate a significant correlation between mRNA and protein levels as demonstrated by the values shown beyond the outer range of threshold <IMG SRC="/math/Dgr.gif" ALT="{Delta}" BORDER="0"> = 0.115. Normalized protein values were used, thus negative values for some proteins are observed.<P>


</TD></TR></TABLE>
</TD></TR></TABLE></CENTER>&nbsp;<BR>


<A NAME="SEC2"><!-- null --></A>
<BR CLEAR=right><TABLE WIDTH=100% BGCOLOR=e1e1e1 CELLPADDING=0 CELLSPACING=0>
 <TR><TD ALIGN=left VALIGN=middle WIDTH=5% BGCOLOR=ffffff><IMG WIDTH=10 HEIGHT=21 HSPACE=5 ALT=" "
SRC="/icons/toc/rarrow.gif"></TD>
 <TH ALIGN=left VALIGN=middle WIDTH=95%><FONT SIZE=+2>&nbsp;&nbsp;
RESULTS
</FONT></TH></TR></TABLE>
<TABLE ALIGN=right CELLPADDING=5 BORDER><TR><TH
ALIGN=left><FONT SIZE=-1>
<A HREF="#top"><IMG WIDTH=11 HEIGHT=9 BORDER=0 HSPACE=5 ALT=" "
SRC="/icons/toc/uarrow.gif">TOP<BR></A>
<A  HREF="#ABS"><IMG WIDTH=11 HEIGHT=9 BORDER=0 HSPACE=5 ALT=" "
SRC="/icons/toc/uarrow.gif">ABSTRACT<BR></A>
<A  HREF="#SEC1"><IMG WIDTH=11 HEIGHT=9 BORDER=0 HSPACE=5 ALT=" "
SRC="/icons/toc/uarrow.gif">EXPERIMENTAL PROCEDURES<BR></A>
<IMG WIDTH=11 HEIGHT=9 BORDER=0 HSPACE=5 ALT=" "
SRC="/icons/toc/dot.gif"><FONT COLOR=464c53>RESULTS</FONT><BR>
<A  HREF="#SEC3"><IMG WIDTH=11 HEIGHT=9 BORDER=0 HSPACE=5 ALT=" "
SRC="/icons/toc/darrow.gif">DISCUSSION<BR></A>
<A  HREF="#BIBL"><IMG WIDTH=11 HEIGHT=9 BORDER=0 HSPACE=5 ALT=" "
SRC="/icons/toc/darrow.gif">REFERENCES<BR></A>
</FONT></TH></TR></TABLE>
&nbsp;<BR>

<STRONG><EM>Correlation of Individual Proteins and mRNA Expression within Each Tumor&#151;</EM></STRONG><BR>

We have examined quantitatively 165 protein spots on 2D gels<SUP> </SUP>representing 98 genes and compared protein levels with mRNA<SUP> </SUP>levels for a cohort of 85 lung adenocarcinomas and normal lung<SUP> </SUP>samples. Of the 165 protein spots, 69 proteins were represented<SUP> </SUP>by only one known spot on 2D gels for an individual gene, whereas<SUP> </SUP>96 protein spots showed multiple protein products from 29 different<SUP> </SUP>genes. 2D Western blotting verified the proteins identified<SUP> </SUP>by mass spectrometry when specific antibodies were available.<SUP> </SUP>Spearman correlation coefficients of the proteins and their<SUP> </SUP>associated mRNA for each protein spot were generated using all<SUP> </SUP>76 lung adenocarcinomas and nine non-neoplastic lung tissues<SUP> </SUP>(see <A HREF="#T1">Tables I</A> and <A HREF="#T2">II</A>, and see <A HREF="#F1">Figs. 1</A> and <A HREF="#F2">2</A>). The correlation<SUP> </SUP>coefficients (<I>r</I>) ranged from -0.467 to 0.442 (<A HREF="#F2">Fig. 2<I>D</I></A>). A total<SUP> </SUP>of 28 protein spots (21 genes) were found to have a statistically<SUP> </SUP>significant correlation between expression of their protein<SUP> </SUP>and mRNA (<I>r</I> &gt; 0.2445; <I>p</I> &lt; 0.05). This accounts for 17%<SUP> </SUP>(28/165) of the 165 protein spots. Among the 69 genes for which<SUP> </SUP>only a single protein spot was known (<A HREF="#T1">Table I</A>), nine genes (9/69,<SUP> </SUP>13%) were observed to show a statistically significant relationship<SUP> </SUP>between protein and mRNA abundance (<I>r</I> &gt; 0.2445; <I>p</I> &lt; 0.05).<SUP> </SUP>The proteins whose expression levels were correlated with their<SUP> </SUP>mRNA abundance included those involved in signal transduction,<SUP> </SUP>carbohydrate metabolism, apoptosis, protein post-translational<SUP> </SUP>modification, structural proteins, and heat shock proteins (<A HREF="#T3">Table III</A>).<SUP> </SUP><SUP> </SUP><P>

<A NAME="T1"><!-- null --></A>
<BR CLEAR=all><CENTER><TABLE WIDTH=95% CELLPADDING=0
CELLSPACING=0><TR BGCOLOR=e1e1e1><TD><TABLE CELLPADDING=2 CELLSPACING=2>
<TR BGCOLOR=e1e1e1><TD ALIGN=center VALIGN=top BGCOLOR=ffffff>
<STRONG>View this table:</STRONG><BR>
<NOBR><A HREF="/cgi/content/full/1/4/304/T1">[in this window]</A><BR>
<A HREF="/cgi/content-nw/full/1/4/304/T1"
    onClick="startTarget('T1', 500, 400); this.href='/cgi/content-nw/full/1/4/304/T1'"
 	onMouseOver="window.status='View table in a separate window'; return true"
 	TARGET="T1">[in a new window]</A><BR>&#160;</NOBR>
</TD><TD ALIGN=left VALIGN=top>
T<FONT SIZE=-2>ABLE</FONT> I Correlation coefficients of protein and mRNA where only one spot was present on 2D gels<P>

r*, correlation coefficient value &gt; 0.2445; <I>p</I> &lt; 0.05. Values in boldface are significant at <I>p</I> &lt; 0.05.<P>


</TD></TR></TABLE>
</TD></TR></TABLE></CENTER>&nbsp;<BR>


<A NAME="T2"><!-- null --></A>
<BR CLEAR=all><CENTER><TABLE WIDTH=95% CELLPADDING=0
CELLSPACING=0><TR BGCOLOR=e1e1e1><TD><TABLE CELLPADDING=2 CELLSPACING=2>
<TR BGCOLOR=e1e1e1><TD ALIGN=center VALIGN=top BGCOLOR=ffffff>
<STRONG>View this table:</STRONG><BR>
<NOBR><A HREF="/cgi/content/full/1/4/304/T2">[in this window]</A><BR>
<A HREF="/cgi/content-nw/full/1/4/304/T2"
    onClick="startTarget('T2', 500, 400); this.href='/cgi/content-nw/full/1/4/304/T2'"
 	onMouseOver="window.status='View table in a separate window'; return true"
 	TARGET="T2">[in a new window]</A><BR>&#160;</NOBR>
</TD><TD ALIGN=left VALIGN=top>
T<FONT SIZE=-2>ABLE</FONT> II Correlation coefficients of protein and mRNA where multiple isoforms were present on 2D gels<P>

r*, correlation coefficient value &gt; 0.2445; <I>p</I> &lt; 0.05. Values in boldface are significant at <I>p</I> &lt; 0.05.<P>


</TD></TR></TABLE>
</TD></TR></TABLE></CENTER>&nbsp;<BR>


<A NAME="T3"><!-- null --></A>
<BR CLEAR=all><CENTER><TABLE WIDTH=95% CELLPADDING=0
CELLSPACING=0><TR BGCOLOR=e1e1e1><TD><TABLE CELLPADDING=2 CELLSPACING=2>
<TR BGCOLOR=e1e1e1><TD ALIGN=center VALIGN=top BGCOLOR=ffffff>
<STRONG>View this table:</STRONG><BR>
<NOBR><A HREF="/cgi/content/full/1/4/304/T3">[in this window]</A><BR>
<A HREF="/cgi/content-nw/full/1/4/304/T3"
    onClick="startTarget('T3', 500, 400); this.href='/cgi/content-nw/full/1/4/304/T3'"
 	onMouseOver="window.status='View table in a separate window'; return true"
 	TARGET="T3">[in a new window]</A><BR>&#160;</NOBR>
</TD><TD ALIGN=left VALIGN=top>
T<FONT SIZE=-2>ABLE</FONT> III Stage-dependent analysis of protein-mRNA correlation coefficients<P>

r, correlation coefficient. Values in boldface indicate a significant difference between stage I and stage III.<P>


</TD></TR></TABLE>
</TD></TR></TABLE></CENTER>&nbsp;<BR>


<STRONG><EM>Individual Isoforms of the Same Protein Have Different Protein/mRNA Correlation Coefficients&#151;</EM></STRONG><BR>

Of the 165 protein spots, 96 represent protein products of 29<SUP> </SUP>genes with at least two isoforms. Among these 96 protein spots,<SUP> </SUP>19 (19/96 protein spots, 20%) showed a statistically significant<SUP> </SUP>correlation between their protein and mRNA expression (<I>r</I> &gt;<SUP> </SUP>0.2445; <I>p</I> &lt; 0.05) (<A HREF="#T2">Table II</A>) and represented 12 genes (12/29,<SUP> </SUP>41%). Individual isoforms of the same protein demonstrated different<SUP> </SUP>protein/mRNA correlation coefficients. For example, 2D-PAGE/Western<SUP> </SUP>analysis revealed four isoforms of OP18 differing in regards<SUP> </SUP>to isoelectric point but similar in molecular weight. Three<SUP> </SUP>of the four isoforms (spots 1492, 1493, and 1494) showed a statistically<SUP> </SUP>significant correlation between their protein and mRNA abundance<SUP> </SUP>(<I>r</I> = 0.3234, 0.3154, and 0.4003, respectively). The forth isoform<SUP> </SUP>(spot 1488) showed no correlation between protein and mRNA expression<SUP> </SUP>(<I>r</I> = 0.0495). Similarly, just one of five quantified isoforms<SUP> </SUP>of cytokeratin 8 (spot 439) demonstrated a statistically significant<SUP> </SUP>correlation between protein and mRNA abundance (<I>r</I> = 0.3049;<SUP> </SUP><I>p</I> &lt; 0.05) (<A HREF="#T2">Table II</A>).<SUP> </SUP><P>

In addition to differences in the relationship between mRNA<SUP> </SUP>levels and protein expression among separate isoforms, some<SUP> </SUP>genes with very comparable mRNA levels showed a 24-fold difference<SUP> </SUP>in their protein expression. Genes with comparable protein expression<SUP> </SUP>levels also showed up to a 28-fold variance in their mRNA levels.<SUP> </SUP><P>

<STRONG><EM>Lack of Correlation for mRNA and Protein Expression when Using Average Tumor Values across All 165 Protein Spots (98 Genes)&#151;</EM></STRONG><BR>

The relationship between mRNA and protein expression was also<SUP> </SUP>examined by using the average expression values for all samples.<SUP> </SUP>To analyze this relationship using this approach, the average<SUP> </SUP>value for each protein or mRNA was generated using all 85 lung<SUP> </SUP>tissue samples. The range of normalized average protein values<SUP> </SUP>ranged from -0.0646 to 0.0979 (raw value 0.0036 to 4.1947),<SUP> </SUP>and the range for mRNA was from 0 to 15260.5 for all 165 individual<SUP> </SUP>protein spots. The Spearman correlation coefficient for the<SUP> </SUP>whole data set (165 protein spots/98 genes) was -0.025 (<A HREF="#F3">Fig.<SUP> </SUP>3<I>A</I></A>). Even for the 28 protein spots (<A HREF="#F2">Fig. 2<I>D</I></A>) that were found<SUP> </SUP>to have a statistically significant correlation between their<SUP> </SUP>mRNA and protein, use of the average value resulted in a correlation<SUP> </SUP>coefficient value of -0.035, which was not significant (<A HREF="#F3">Fig.<SUP> </SUP>3<I>B</I></A>).<SUP> </SUP><P>

<A NAME="F3"><!-- null --></A>
<BR CLEAR=all><CENTER><TABLE WIDTH=95% CELLPADDING=0 
CELLSPACING=0><TR BGCOLOR=e1e1e1><TD><TABLE CELLPADDING=2 CELLSPACING=2>
<TR BGCOLOR=e1e1e1><TD ALIGN=center VALIGN=top BGCOLOR=ffffff>
<A HREF="/cgi/content/full/1/4/304/F3"><IMG HSPACE=10 VSPACE=5 BORDER=2 WIDTH=145 HEIGHT=200 SRC="/content/vol1/issue4/images/small/jw0320033003.gif" ALT=" "></A><BR>

<STRONG>View larger version</STRONG> (19K):<BR>
<NOBR><A HREF="/cgi/content/full/1/4/304/F3">[in this window]</A><BR>
<A HREF="/cgi/content-nw/full/1/4/304/F3"
    onClick="startTarget('F3', 469, 640); this.href='/cgi/content-nw/full/1/4/304/F3'"
 	onMouseOver="window.status='View figure in a separate window'; return true"
 	TARGET="F3">[in a new window]</A><BR>&#160;</NOBR>
</TD><TD ALIGN=left VALIGN=top>
F<FONT SIZE=-2>IG</FONT>. 3. <B>The overall correlation of mRNA and protein levels across all 165 protein spots (<I>A</I>) and across 28 protein spots that contained individual <I>r</I> values larger than 0.244 (<I>B</I>) are shown.</B> Each protein or mRNA mean value was calculated based on all 76 lung adenocarcinomas and nine non-neoplastic lung samples using quantitative 2D-PAGE and Affymetrix oligonucleotide microarrays. The Spearman correlation coefficients for the two data sets (<I>A</I> and <I>B</I>) were -0.025 and -0.035, respectively, indicating a lack of correlation if mean values for mRNA and protein for all samples is used.<P>


</TD></TR></TABLE>
</TD></TR></TABLE></CENTER>&nbsp;<BR>


<STRONG><EM>Lack of a Relationship between Protein/mRNA Correlation Coefficients and Average Protein Abundance&#151;</EM></STRONG><BR>

To determine whether an absolute protein level might influence<SUP> </SUP>the correlation with mRNA, the mean value of each protein (relative<SUP> </SUP>abundance) and the Spearman protein/mRNA correlation coefficients<SUP> </SUP>among all 85 samples were examined. No relationship between<SUP> </SUP>the protein abundance and the correlation coefficients was observed<SUP> </SUP>(<I>r</I> = 0.039; <I>p</I> &gt; 0.05). A detailed analysis of separate subsets<SUP> </SUP>of proteins with differing levels of abundance (less than -0.0014,<SUP> </SUP>larger than -0.0014, or larger than 0.0077) also showed a lack<SUP> </SUP>of correlation between mRNA and protein expression among the<SUP> </SUP>83 (50%), 82 (50%), and 41 (25%) of 165 total protein spots,<SUP> </SUP>respectively (<I>r</I> = 0.016, 0.08, and 0.172, respectively).<SUP> </SUP><P>

<STRONG><EM>Stage-related Changes in the Protein/mRNA Correlation Coefficients&#151;</EM></STRONG><BR>

To determine whether the 21 genes (28 protein spots) showing<SUP> </SUP>a significant correlation between the protein and mRNA expression<SUP> </SUP>among all samples demonstrate changes in this relationship during<SUP> </SUP>tumor progression, the correlations were examined separately<SUP> </SUP>for stage I (<I>n</I> = 57) and stage III (<I>n</I> = 19) lung adenocarcinomas<SUP> </SUP>(<A HREF="#T3">Table III</A>). The number of non-neoplastic lung samples (<I>n</I> =<SUP> </SUP>9) was insufficient for a separate correlation analysis of this<SUP> </SUP>group. Many of the protein spots represent one of several known<SUP> </SUP>protein isoforms for a given gene. The majority of genes (16/21)<SUP> </SUP>did not differ in the protein/mRNA correlation between stage<SUP> </SUP>I and stage III tumors indicating a similar regulatory relationship<SUP> </SUP>between the mRNA and protein spot. GRP-58, PSMC, SOD1, TPI1,<SUP> </SUP>and VIM, however, were found to demonstrate significant differences<SUP> </SUP>in the correlation coefficients between stage I and stage III<SUP> </SUP>lung adenocarcinomas. For GRP-58, PSMC, and VIM the change in<SUP> </SUP>the correlation coefficient was because of a relative increase<SUP> </SUP>in protein expression in stage III tumors. For SOD and TPI the<SUP> </SUP>change resulted from a relative decrease in expression of this<SUP> </SUP>specific protein in stage III tumors.<SUP> </SUP><P>

<A NAME="SEC3"><!-- null --></A>
<BR CLEAR=right><TABLE WIDTH=100% BGCOLOR=e1e1e1 CELLPADDING=0 CELLSPACING=0>
 <TR><TD ALIGN=left VALIGN=middle WIDTH=5% BGCOLOR=ffffff><IMG WIDTH=10 HEIGHT=21 HSPACE=5 ALT=" "
SRC="/icons/toc/rarrow.gif"></TD>
 <TH ALIGN=left VALIGN=middle WIDTH=95%><FONT SIZE=+2>&nbsp;&nbsp;
DISCUSSION
</FONT></TH></TR></TABLE>
<TABLE ALIGN=right CELLPADDING=5 BORDER><TR><TH
ALIGN=left><FONT SIZE=-1>
<A HREF="#top"><IMG WIDTH=11 HEIGHT=9 BORDER=0 HSPACE=5 ALT=" "
SRC="/icons/toc/uarrow.gif">TOP<BR></A>
<A  HREF="#ABS"><IMG WIDTH=11 HEIGHT=9 BORDER=0 HSPACE=5 ALT=" "
SRC="/icons/toc/uarrow.gif">ABSTRACT<BR></A>
<A  HREF="#SEC1"><IMG WIDTH=11 HEIGHT=9 BORDER=0 HSPACE=5 ALT=" "
SRC="/icons/toc/uarrow.gif">EXPERIMENTAL PROCEDURES<BR></A>
<A  HREF="#SEC2"><IMG WIDTH=11 HEIGHT=9 BORDER=0 HSPACE=5 ALT=" "
SRC="/icons/toc/uarrow.gif">RESULTS<BR></A>
<IMG WIDTH=11 HEIGHT=9 BORDER=0 HSPACE=5 ALT=" "
SRC="/icons/toc/dot.gif"><FONT COLOR=464c53>DISCUSSION</FONT><BR>
<A  HREF="#BIBL"><IMG WIDTH=11 HEIGHT=9 BORDER=0 HSPACE=5 ALT=" "
SRC="/icons/toc/darrow.gif">REFERENCES<BR></A>
</FONT></TH></TR></TABLE>
&nbsp;<BR>

Relatively little is known about the regulatory mechanisms controlling<SUP> </SUP>the complex patterns of protein abundance and post-translational<SUP> </SUP>modification in tumors. Most reports concerning the regulation<SUP> </SUP>of protein translation have focused on one or several protein<SUP> </SUP>products (<A HREF="#R18">18</A>). Celis <I>et al.</I> (<A HREF="#R19">19</A>) found a good correlation between<SUP> </SUP>transcript and protein levels among 40 well resolved, abundant<SUP> </SUP>proteins using a proteomic and microarray study of bladder cancer.<SUP> </SUP>By comparing the mRNA and protein expression levels within the<SUP> </SUP>same tumor samples, we found that 17% (28/165) of the protein<SUP> </SUP>spots (21/98 genes) show a statistically significant correlation<SUP> </SUP>between mRNA and protein. These proteins appear to represent<SUP> </SUP>a diverse group of gene products and include those involved<SUP> </SUP>in signal transduction, carbohydrate metabolism, protein modification,<SUP> </SUP>cell structure, heat shock, and apoptosis. These results suggest<SUP> </SUP>that expression of this subset of 165 proteins is likely to<SUP> </SUP>be regulated at the transcriptional level in these tissues.<SUP> </SUP>The majority of the protein isoforms, however, did not correlate<SUP> </SUP>with mRNA levels, and thus their expression is regulated by<SUP> </SUP>other mechanisms. We also observed a subset of proteins that<SUP> </SUP>demonstrated a negative correlation with the mRNA expression<SUP> </SUP>values; for example <IMG SRC="/math/agr.gif" ALT="{alpha}" BORDER="0">-haptoglobin demonstrated a strong negative<SUP> </SUP>correlation with its mRNA expression values. This may reflect<SUP> </SUP>negative feedback on the mRNA or the protein or the presence<SUP> </SUP>of other regulatory influences that are not understood currently.<SUP> </SUP><P>

Post-translational modification or processing will result in<SUP> </SUP>individual protein products of the same gene migrating to different<SUP> </SUP>locations on 2D-PAGE gels (<A HREF="#R20">20</A>). Because the identity of all<SUP> </SUP>possible isoforms for each protein examined has not been characterized<SUP> </SUP>completely, this may influence the correlation analyses performed<SUP> </SUP>in this study. This is partly because of limitations of the<SUP> </SUP>2D-PAGE and mass spectrometry technologies (<A HREF="#R21">21</A>, <A HREF="#R22">22</A>). Potential<SUP> </SUP>inconsistencies between mRNA and protein correlations that have<SUP> </SUP>been reported may also be because of differences, even in the<SUP> </SUP>same gene, in the mechanisms of protein translation among different<SUP> </SUP>cells or as measured in different laboratories (<A HREF="#R23">23</A>).<SUP> </SUP><P>

In this study, we examined 165 protein spots identified in lung<SUP> </SUP>adenocarcinomas. Ninety-six protein spots, representing the<SUP> </SUP>products of 29 genes, contained at least two protein isoforms.<SUP> </SUP>Nineteen of 96 protein spots, representing 12 genes, were shown<SUP> </SUP>to have a statistically significant correlation between their<SUP> </SUP>protein and mRNA expression, suggesting that the levels of these<SUP> </SUP>proteins reflects the transcription of the corresponding genes.<SUP> </SUP>Differences in protein/mRNA correlations were found among the<SUP> </SUP>individual isoforms of a given protein. For example, of the<SUP> </SUP>four OP18 isoforms, three showed a statistically significant<SUP> </SUP>correlation between the protein and mRNA expression levels.<SUP> </SUP>The lack of relationship for the one isoform, however, indicates<SUP> </SUP>that individual protein isoforms of the same gene product can<SUP> </SUP>be regulated differentially. This is not unexpected and likely<SUP> </SUP>reflects other post-translational mechanisms that can influence<SUP> </SUP>isoform abundance in tissues and cancer.<SUP> </SUP><P>

In addition to the analyses of the correlation of mRNA/protein<SUP> </SUP>within the same tumor samples, we also tested the global relationship<SUP> </SUP>between mRNA and the corresponding protein abundance across<SUP> </SUP>all 165 protein spots in the lung samples. A protein and mRNA<SUP> </SUP>average value for each gene was generated using all 85 lung<SUP> </SUP>tissues samples. We observed a very wide range of normalized<SUP> </SUP>average protein and mRNA values. The correlation coefficient<SUP> </SUP>generated using this average value data set was -0.025, and<SUP> </SUP>even for the 28 protein spots that showed a statistically significant<SUP> </SUP>correlation between individual mRNA and proteins, the correlation<SUP> </SUP>value was only -0.035. This suggests that it is not possible<SUP> </SUP>to predict overall protein expression levels based on average<SUP> </SUP>mRNA abundance in lung cancer samples. This conclusion is also<SUP> </SUP>supported by previous results from Anderson and Seilhamer (<A HREF="#R24">24</A>),<SUP> </SUP>who examined 19 genes in human liver cells, and by Gygi <I>et al.</I><SUP> </SUP>(<A HREF="#R25">25</A>), who examined 106 genes in yeast. Both studies found a<SUP> </SUP>lack of correlation between mRNA and protein expression when<SUP> </SUP>average or overall levels were used.<SUP> </SUP><P>

A good correlation was reported when the 11 most abundant proteins<SUP> </SUP>were examined in yeast (<A HREF="#R25">25</A>), suggesting that the level of protein<SUP> </SUP>abundance may be a factor that may influence the correlation<SUP> </SUP>between mRNA and protein. In the present study, a fairly wide<SUP> </SUP>range of mean protein values among 165 protein spots in lung<SUP> </SUP>adenocarcinomas was observed, and the correlation coefficients<SUP> </SUP>also varied from -0.467 to 0.442. A comparison between the mean<SUP> </SUP>value of each protein and the correlation coefficient generated<SUP> </SUP>using all 85 tissue samples did not reveal a strong relationship<SUP> </SUP>between the overall protein abundance and the correlation coefficients<SUP> </SUP>(<I>r</I> = 0.039; <I>p</I> &gt; 0.05). Detailed analysis of different subsets<SUP> </SUP>of protein abundance also failed to show a correlation between<SUP> </SUP>mRNA and protein expression. Thus in contrast to yeast, a relationship<SUP> </SUP>between mRNA/protein correlation coefficient and protein abundance<SUP> </SUP>in human lung adenocarcinomas was not observed.<SUP> </SUP><P>

The results of this study indicate that the level of protein<SUP> </SUP>abundance in lung adenocarcinomas is associated with the corresponding<SUP> </SUP>levels of mRNA in 17% (28 proteins) of the total 165 protein<SUP> </SUP>spots examined. This was substantially higher than the amount<SUP> </SUP>predicted to result by chance alone (which was 5.1) and suggests<SUP> </SUP>that a transcriptional mechanism likely underlies the abundance<SUP> </SUP>of these proteins in lung adenocarcinomas. We also demonstrate<SUP> </SUP>that the expression of individual isoforms of the same protein<SUP> </SUP>may or may not correlate with the mRNA, indicating that separate<SUP> </SUP>and likely post-translational mechanisms account for the regulation<SUP> </SUP>of isoform abundance. These mechanisms may also account for<SUP> </SUP>the differences in the correlation coefficients observed between<SUP> </SUP>stage I and stage III tumors, indicating that specific protein<SUP> </SUP>isoforms show regulatory changes during tumor progression. Further<SUP> </SUP>studies in lung adenocarcinomas will examine the relationship<SUP> </SUP>between the expression of individual protein isoforms and specific<SUP> </SUP>clinical-pathological features of these tumors, such as the<SUP> </SUP>presence of angiolymphatic invasion, and nodal or pleural surface<SUP> </SUP>involvement. The potential to identify specific protein isoforms<SUP> </SUP>associated with biological behavior in lung adenocarcinomas<SUP> </SUP>would be of considerable interest and will add to our understanding<SUP> </SUP>of the regulation of gene products by transcriptional, translational,<SUP> </SUP>and post-translational mechanisms.<SUP> </SUP><P>

<SUP> </SUP><P>


<A NAME="ACK"><!-- null --></A>
<BR CLEAR=right><TABLE WIDTH=100% BGCOLOR=e1e1e1 CELLPADDING=0 CELLSPACING=0>
 <TR><TD ALIGN=left VALIGN=middle WIDTH=5% BGCOLOR=ffffff><IMG WIDTH=10 HEIGHT=21 HSPACE=5 ALT=" "
SRC="/icons/toc/rarrow.gif"></TD>
 <TH ALIGN=left VALIGN=middle WIDTH=95%><FONT SIZE=+2>&nbsp;&nbsp;
ACKNOWLEDGMENTS
</FONT></TH></TR></TABLE>
&nbsp;<BR>

We thank Kerby A. Shedden, Rork D. Kuick, Eric Puravs, Robert<SUP> </SUP>Hinderer, Melissa C. Krause, and Christopher Wood for assistance<SUP> </SUP>in this study.<SUP> </SUP><P>


<A NAME="FN"><!-- null --></A>
<BR CLEAR=right><TABLE WIDTH=100% BGCOLOR=e1e1e1 CELLPADDING=0 CELLSPACING=0>
 <TR><TD ALIGN=left VALIGN=middle WIDTH=5% BGCOLOR=ffffff><IMG WIDTH=10 HEIGHT=21 HSPACE=5 ALT=" "
SRC="/icons/toc/rarrow.gif"></TD>
 <TH ALIGN=left VALIGN=middle WIDTH=95%><FONT SIZE=+2>&nbsp;&nbsp;
FOOTNOTES
</FONT></TH></TR></TABLE>
&nbsp;<BR>
Received, January 21, 2001, and in revised form, March  4, 2002.<P>

<A NAME=""><!-- null --></A>
Published, MCP Papers in Press, March 12, 2001, DOI 10.1074/mcp.M200008-MCP200<SUP> </SUP><P>

<A NAME="FN1"><!-- null --></A>
<SUP><SUP>1</SUP></SUP> The abbreviations used are: 2D, two-dimensional; MALDI-MS, matrix-assisted<SUP> </SUP>laser desorption/ionization mass spectrometry.<SUP> </SUP><A HREF="#RFN1"><IMG BORDER=0 WIDTH=12 HEIGHT=12 ALT="Back"
SRC="/icons/back.gif"></A><P>
<A NAME="FN2"><!-- null --></A>
<SUP><SUP>2</SUP></SUP> Chen <I>et al.</I>, submitted for publication.<SUP> </SUP><A HREF="#RFN2"><IMG BORDER=0 WIDTH=12 HEIGHT=12 ALT="Back"
SRC="/icons/back.gif"></A><P>
<A NAME="FN3"><!-- null --></A>
<SUP>*</SUP> This work was supported by NCI, National Institutes of Health<SUP> </SUP>Grant U19 CA-85953.<SUP> </SUP><A HREF="#RFN3"><IMG BORDER=0 WIDTH=12 HEIGHT=12 ALT="Back"
SRC="/icons/back.gif"></A><P>


<A NAME="COR1"><!-- null --></A>
<SUP><IMG SRC="/math/Dagger.gif" ALT="{ddagger}" BORDER="0"><IMG SRC="/math/Dagger.gif" ALT="{ddagger}" BORDER="0"></SUP> To whom correspondence should be addressed: General Thoracic Surgery, SRB II, B560, Box 0686, University of Michigan Medical School, Ann Arbor, MI 48109-086; E-mail: <span id="em0">dgbeer{at}umich.edu</span><script type="text/javascript"><!--
 var u = "dgbeer", d = "umich.edu"; document.getElementById("em0").innerHTML = '<a href="mailto:' + u + '@' + d + '">' + u + '@' + d + '<\/a>'//--></script><P>

<A NAME="BIBL"><!-- null --></A>
<BR CLEAR=right><TABLE WIDTH=100% BGCOLOR=e1e1e1 CELLPADDING=0 CELLSPACING=0>
 <TR><TD ALIGN=left VALIGN=middle WIDTH=5% BGCOLOR=ffffff><IMG WIDTH=10 HEIGHT=21 HSPACE=5 ALT=" "
SRC="/icons/toc/rarrow.gif"></TD>
 <TH ALIGN=left VALIGN=middle WIDTH=95%><FONT SIZE=+2>&nbsp;&nbsp;
REFERENCES
</FONT></TH></TR></TABLE>
<TABLE ALIGN=right CELLPADDING=5 BORDER><TR><TH
ALIGN=left><FONT SIZE=-1>
<A HREF="#top"><IMG WIDTH=11 HEIGHT=9 BORDER=0 HSPACE=5 ALT=" "
SRC="/icons/toc/uarrow.gif">TOP<BR></A>
<A  HREF="#ABS"><IMG WIDTH=11 HEIGHT=9 BORDER=0 HSPACE=5 ALT=" "
SRC="/icons/toc/uarrow.gif">ABSTRACT<BR></A>
<A  HREF="#SEC1"><IMG WIDTH=11 HEIGHT=9 BORDER=0 HSPACE=5 ALT=" "
SRC="/icons/toc/uarrow.gif">EXPERIMENTAL PROCEDURES<BR></A>
<A  HREF="#SEC2"><IMG WIDTH=11 HEIGHT=9 BORDER=0 HSPACE=5 ALT=" "
SRC="/icons/toc/uarrow.gif">RESULTS<BR></A>
<A  HREF="#SEC3"><IMG WIDTH=11 HEIGHT=9 BORDER=0 HSPACE=5 ALT=" "
SRC="/icons/toc/uarrow.gif">DISCUSSION<BR></A>
<IMG WIDTH=11 HEIGHT=9 BORDER=0 HSPACE=5 ALT=" "
SRC="/icons/toc/dot.gif"><FONT COLOR=464c53>REFERENCES</FONT><BR>
</FONT></TH></TR></TABLE>
&nbsp;<BR>

<OL COMPACT>
<A NAME="R1"><!-- null --></A>
<LI VALUE=1> Ideker, T., Thorsson, V., Ranish, J. A., Christmas, R., Buhler, J., Eng, J. K., Bumgarner, R., Goodlett, D. R., Aebersold, R., Hood, L.
(2001) Integrated genomic and proteomic analyses of a systematically perturbed metabolic network.
<I>Science</I>
<B>292, </B>929
&#150;934<!-- HIGHWIRE ID="1:4:304:1" --><A HREF="/cgi/ijlink?linkType=ABST&journalCode=sci&resid=292/5518/929" ><nobr>[Abstract/<font COLOR="CC0000">Free</font> Full&nbsp;Text]</nobr></A><!-- /HIGHWIRE --><P><A NAME="R2"><!-- null --></A>
<LI VALUE=2> Liang, P., Pardee, A. B.
(1998) Differential display. A general protocol.
<I>Mol. Biotechnol.</I>
<B>10, </B>261
&#150;267<!-- HIGHWIRE ID="1:4:304:2" --><A HREF="/cgi/external_ref?access_num=9951706&link_type=MED" >[Medline]</A><!-- /HIGHWIRE --><P><A NAME="R3"><!-- null --></A>
<LI VALUE=3> Porter, D. A., Krop, I. E., Nasser, S., Sgroi, D., Kaelin, C. M., Marks, J. R., Riggins, G., Polyak, K.
(2001) A sage (serial analysis of gene expression) view of breast tumor progression.
<I>Cancer Res.</I>
<B>61, </B>5697
&#150;5702<!-- HIGHWIRE ID="1:4:304:3" --><A HREF="/cgi/ijlink?linkType=ABST&journalCode=canres&resid=61/15/5697" ><nobr>[Abstract/<font COLOR="CC0000">Free</font> Full&nbsp;Text]</nobr></A><!-- /HIGHWIRE --><P><A NAME="R4"><!-- null --></A>
<LI VALUE=4> Bittner, M., Meltzer, P., Chen, Y., Jiang, Y., Seftor, E., Hendrix, M., Radmacher, M., Simon, R., Yakhini, Z., Ben-Dor, A., Sampas, N., Dougherty, E., Wang, E., Marincola, F., Gooden, C., Lueders, J., Glatfelter, A., Pollock, P., Carpten, J., Gillanders, E., Leja, D., Dietrich, K., Beaudry, C., Berens, M., Alberts, D., Sondak, V.
(2000) Molecular classification of cutaneous malignant melanoma by gene expression profiling.
<I>Nature</I>
<B>406, </B>536
&#150;540<!-- HIGHWIRE ID="1:4:304:4" --><A HREF="/cgi/external_ref?access_num=10.1038/35020115&link_type=DOI" >[CrossRef]</A><A HREF="/cgi/external_ref?access_num=10952317&link_type=MED" >[Medline]</A><!-- /HIGHWIRE --><P><A NAME="R5"><!-- null --></A>
<LI VALUE=5> Fung, E. T., Wright, G. L., Jr., Dalmasso, E. A.
(2000) Proteomic strategies for biomarker identification: progress and challenges.
<I>Curr. Opin. Mol. Ther.</I>
<B>2, </B>643
&#150;650<!-- HIGHWIRE ID="1:4:304:5" --><A HREF="/cgi/external_ref?access_num=11249741&link_type=MED" >[Medline]</A><!-- /HIGHWIRE --><P><A NAME="R6"><!-- null --></A>
<LI VALUE=6> Davidson, D., Baldock, R.
(2001) Bioinformatics beyond sequence: mapping gene function in the embryo.
<I>Nat. Rev. Genet.</I>
<B>2, </B>409
&#150;417<!-- HIGHWIRE ID="1:4:304:6" --><A HREF="/cgi/external_ref?access_num=10.1038/35076500&link_type=DOI" >[CrossRef]</A><A HREF="/cgi/external_ref?access_num=11389457&link_type=MED" >[Medline]</A><!-- /HIGHWIRE --><P><A NAME="R7"><!-- null --></A>
<LI VALUE=7> Chee, M., Yang, R., Hubbell, E., Berno, A., Huang, X. C., Stern, D., Winkler. J., Lockhart, D. J., Morris, M. S., Fodor, S. P.
(1996) Accessing genetic information with high-density DNA arrays.
<I>Science</I>
<B>274, </B>610
&#150;614<!-- HIGHWIRE ID="1:4:304:7" --><A HREF="/cgi/ijlink?linkType=ABST&journalCode=sci&resid=274/5287/610" ><nobr>[Abstract/<font COLOR="CC0000">Free</font> Full&nbsp;Text]</nobr></A><!-- /HIGHWIRE --><P><A NAME="R8"><!-- null --></A>
<LI VALUE=8> Wang, D. G., Fan, J. B., Siao, C. J., Berno, A., Young, P., Sapolsky, R., Ghandour, G., Perkins, N., Winchester, E., Spencer, J., Kruglyak, L., Stein, L., Hsie, L., Topaloglou, T., Hubbell, E., Robinson, E., Mittmann, M., Morris, M. S., Shen, N., Kilburn, D., Rioux, J., Nusbaum, C., Rozen, S., Hudson, T. J., Lander, E. S.
(1998) Large-scale identification, mapping, and genotyping of single-nucleotide polymorphisms in the human genome.
<I>Science</I>
<B>280, </B>1077
&#150;1082<!-- HIGHWIRE ID="1:4:304:8" --><A HREF="/cgi/ijlink?linkType=ABST&journalCode=sci&resid=280/5366/1077" ><nobr>[Abstract/<font COLOR="CC0000">Free</font> Full&nbsp;Text]</nobr></A><!-- /HIGHWIRE --><P><A NAME="R9"><!-- null --></A>
<LI VALUE=9> Pease, A. C., Solas, D., Sullivan, E. J., Cronin, M. T., Holmes, C. P., Fodor, S. P.
(1994) Light-generated oligonucleotide arrays for rapid DNA sequence analysis.
<I>Proc. Natl. Acad. Sci. U. S. A.</I>
<B>91, </B>5022
&#150;5026<!-- HIGHWIRE ID="1:4:304:9" --><A HREF="/cgi/ijlink?linkType=ABST&journalCode=pnas&resid=91/11/5022" >[Abstract]</A><!-- /HIGHWIRE --><P><A NAME="R10"><!-- null --></A>
<LI VALUE=10> Bhattacharjee, A., Richards, W. G., Staunton, J., Li, C., Monti, S., Vasa, P., Ladd, C., Beheshti, J., Bueno, R., Gillette, M., Loda, M., Weber, G., Mark, E. J., Lander, E. S., Wong, W., Johnson, B. E., Golub, T. R., Sugarbaker, D. J., Meyerson, M.
(2001) Classification of human lung carcinomas by mRNA expression profiling reveals distinct adenocarcinoma subclasses.
<I>Proc. Natl. Acad. Sci. U. S. A.</I>
<B>98, </B>13790
&#150;13795<!-- HIGHWIRE ID="1:4:304:10" --><A HREF="/cgi/ijlink?linkType=ABST&journalCode=pnas&resid=98/24/13790" ><nobr>[Abstract/<font COLOR="CC0000">Free</font> Full&nbsp;Text]</nobr></A><!-- /HIGHWIRE --><P><A NAME="R11"><!-- null --></A>
<LI VALUE=11> Giordano, T. J., Shedden, K. A., Schwartz, D. R., Kuick, R., Taylor, J. M. G., Lee, N., Misek, D. E., Greenson, J. K., Kardia, S. L. R., Beer, D. G., Rennert, G., Cho, K. R., Gruber, S. B., Fearon, E. R., Hanash, S.
(2001) Organ-specific molecular classification of lung, colon and ovarian adenocarcinomas using gene expression profiles.
<I>Am. J. Pathol.</I>
<B>159, </B>1231
&#150;1238<!-- HIGHWIRE ID="1:4:304:11" --><A HREF="/cgi/ijlink?linkType=ABST&journalCode=amjpathol&resid=159/4/1231" ><nobr>[Abstract/<font COLOR="CC0000">Free</font> Full&nbsp;Text]</nobr></A><!-- /HIGHWIRE --><P><A NAME="R12"><!-- null --></A>
<LI VALUE=12> Strahler, J. R., Kuick, R., Hanash, S. M.
(1989) in
<I>Protein Structure: A Practical Approach</I> (Creighton, T., ed) pp.65
&#150;92, IRL Press, Oxford<!-- HIGHWIRE ID="1:4:304:12" --><!-- /HIGHWIRE --><P><A NAME="R13"><!-- null --></A>
<LI VALUE=13> Merril, C. R., Dunau, M. L., Goldman, D.
(1981) A rapid sensitive silver stain for polypeptides in polyacrylamide gels.
<I>Anal. Biochem.</I>
<B>101, </B>201
&#150;207<!-- HIGHWIRE ID="1:4:304:13" --><!-- /HIGHWIRE --><P><A NAME="R14"><!-- null --></A>
<LI VALUE=14> Hanash, S. M., Strahler, J. R., Chan, Y., Kuick, R., Teichroew, D., Neel, J. V., Hailat, N., Keim, D. R., Gratiot-Deans, J., Ungar, D., Richardson, B. C.
(1993) Data base analysis of protein expression patterns during T-cell ontogeny and activation.
<I>Proc. Natl. Acad. Sci. U. S. A.</I>
<B>90, </B>3314
&#150;3318<!-- HIGHWIRE ID="1:4:304:14" --><A HREF="/cgi/ijlink?linkType=ABST&journalCode=pnas&resid=90/8/3314" >[Abstract]</A><!-- /HIGHWIRE --><P><A NAME="R15"><!-- null --></A>
<LI VALUE=15> Brichory, F. M., Misek, D. E., Yim, A. M., Krause, M. C., Giordano, T. J., Beer, D. G., Hanash, S. M.
(2001) An immune response manifested by the common occurrence of annexins I and II autoantibodies and high circulating levels of IL-6 in lung cancer.
<I>Proc. Natl. Acad. Sci. U. S. A.</I>
<B>98, </B>9824
&#150;9829<!-- HIGHWIRE ID="1:4:304:15" --><A HREF="/cgi/ijlink?linkType=ABST&journalCode=pnas&resid=98/17/9824" ><nobr>[Abstract/<font COLOR="CC0000">Free</font> Full&nbsp;Text]</nobr></A><!-- /HIGHWIRE --><P><A NAME="R16"><!-- null --></A>
<LI VALUE=16> Lavens-Phillips, S. E., MacGlashan, D. W., Jr.
(2000) The tyrosine kinases p53/56lyn and p72syk are differentially expressed at the protein level but not at the messenger RNA level in nonreleasing human basophils.
<I>Am. J. Respir. Cell Mol. Biol.</I>
<B>23, </B>566
&#150;571<!-- HIGHWIRE ID="1:4:304:16" --><A HREF="/cgi/ijlink?linkType=ABST&journalCode=ajrcmb&resid=23/4/566" ><nobr>[Abstract/<font COLOR="CC0000">Free</font> Full&nbsp;Text]</nobr></A><!-- /HIGHWIRE --><P><A NAME="R17"><!-- null --></A>
<LI VALUE=17> Tusher, V. G., Tibshirani, R., Chu, G.
(2001) Significance analysis of microarrays applied to the ionizing radiation response.
<I>Proc. Natl. Acad. Sci. U. S. A.</I>
<B>98, </B>5116
&#150;5121<!-- HIGHWIRE ID="1:4:304:17" --><A HREF="/cgi/ijlink?linkType=ABST&journalCode=pnas&resid=98/9/5116" ><nobr>[Abstract/<font COLOR="CC0000">Free</font> Full&nbsp;Text]</nobr></A><!-- /HIGHWIRE --><P><A NAME="R18"><!-- null --></A>
<LI VALUE=18> Tew, K. D., Monks, A., Barone, L., Rosser, D., Akerman, G., Montali, J. A., Wheatley, J. B., Schmidt, D. E., Jr.
(1996) Glutathione-associated enzymes in the human cell lines of the National Cancer Institute Drug Screening Program.
<I>Mol. Pharmacol.</I>
<B>50, </B>149
&#150;159<!-- HIGHWIRE ID="1:4:304:18" --><A HREF="/cgi/ijlink?linkType=ABST&journalCode=molpharm&resid=50/1/149" >[Abstract]</A><!-- /HIGHWIRE --><P><A NAME="R19"><!-- null --></A>
<LI VALUE=19> Celis, J. E., Kruhoffer, M., Gromova, I., Frederiksen, C., Ostergaard, M., Thykjaer, T., Gromov, P., Yu, J., Palsdottir, H., Magnusson, N., Orntoft, T. F.
(2000) Gene expression profiling: monitoring transcription and translation products using DNA microarrays and proteomics.
<I>FEBS Lett.</I>
<B>480, </B>2
&#150;16<!-- HIGHWIRE ID="1:4:304:19" --><A HREF="/cgi/external_ref?access_num=10.1016/S0014-5793(00)01771-3&link_type=DOI" >[CrossRef]</A><A HREF="/cgi/external_ref?access_num=10967322&link_type=MED" >[Medline]</A><!-- /HIGHWIRE --><P><A NAME="R20"><!-- null --></A>
<LI VALUE=20> Anderson, N. L., Anderson, N. G.
(1998) Proteome and proteomics: new technologies, new concepts, and new words.
<I>Electrophoresis</I>
<B>19, </B>1853
&#150;1861<!-- HIGHWIRE ID="1:4:304:20" --><A HREF="/cgi/external_ref?access_num=9740045&link_type=MED" >[Medline]</A><!-- /HIGHWIRE --><P><A NAME="R21"><!-- null --></A>
<LI VALUE=21> Gygi, S. P., Corthals, G. L., Zhang, Y., Rochon, Y., Aebersold, R.
(2000) Evaluation of two-dimensional gel electrophoresis-based proteome analysis technology.
<I>Proc. Natl. Acad. Sci. U. S. A.</I>
<B>97, </B>9390
&#150;9395<!-- HIGHWIRE ID="1:4:304:21" --><A HREF="/cgi/ijlink?linkType=ABST&journalCode=pnas&resid=97/17/9390" ><nobr>[Abstract/<font COLOR="CC0000">Free</font> Full&nbsp;Text]</nobr></A><!-- /HIGHWIRE --><P><A NAME="R22"><!-- null --></A>
<LI VALUE=22> Fey, S. J., Larsen, P. M.
(2001) 2D or not 2D. Two-dimensional gel electrophoresis.
<I>Curr. Opin. Chem. Biol.</I>
<B>5, </B>26
&#150;33<!-- HIGHWIRE ID="1:4:304:22" --><A HREF="/cgi/external_ref?access_num=10.1016/S1367-5931(00)00167-8&link_type=DOI" >[CrossRef]</A><A HREF="/cgi/external_ref?access_num=11166644&link_type=MED" >[Medline]</A><!-- /HIGHWIRE --><P><A NAME="R23"><!-- null --></A>
<LI VALUE=23> McBride, S., Walsh, D., Meleady, P., Daly, N., Clynes, M.
(1999) Bromodeoxyuridine induces keratin protein synthesis at a posttranscriptional level in human lung tumor cell lines.
<I>Differentiation</I>
<B>64, </B>185
&#150;193<!-- HIGHWIRE ID="1:4:304:23" --><A HREF="/cgi/external_ref?access_num=10.1046/j.1432-0436.1999.6430185.x&link_type=DOI" >[CrossRef]</A><A HREF="/cgi/external_ref?access_num=10234815&link_type=MED" >[Medline]</A><!-- /HIGHWIRE --><P><A NAME="R24"><!-- null --></A>
<LI VALUE=24> Anderson, L., Seilhamer, J.
(1997) A comparison of selected mRNA and protein abundances in human liver.
<I>Electrophoresis</I>
<B>18, </B>533
&#150;537<!-- HIGHWIRE ID="1:4:304:24" --><A HREF="/cgi/external_ref?access_num=9150937&link_type=MED" >[Medline]</A><!-- /HIGHWIRE --><P><A NAME="R25"><!-- null --></A>
<LI VALUE=25> Gygi, S. P., Rochon, Y., Franza, B. R., Aebersold, R.
(1999) Correlation between protein and mRNA abundance in yeast.
<I>Mol. Cell. Biol.</I>
<B>19, </B>1720
&#150;1730<!-- HIGHWIRE ID="1:4:304:25" --><A HREF="/cgi/ijlink?linkType=ABST&journalCode=mcb&resid=19/3/1720" ><nobr>[Abstract/<font COLOR="CC0000">Free</font> Full&nbsp;Text]</nobr></A><!-- /HIGHWIRE --><P></OL>


		
        
        
	
	
	
	
	





 






<BR CLEAR=ALL>




  <FONT FACE="">
  
  </FONT>
<BR CLEAR=ALL>
<BR CLEAR=ALL>



<A NAME="otherarticles"><!-- null --></A>

<P>
</body></html>