







 
 
  







<HTML>
<HEAD>
<TITLE>JBC --  Lim and Halpain 275 (27): 20578</TITLE>


	<SCRIPT LANGUAGE="JavaScript">
	<!--
	function startTarget(windowname,wid,hei) {
	
	var dotpos = windowname.indexOf(".");
	if (dotpos > -1)
	{
	var tempwn = windowname.substring(0,dotpos) +
		windowname.substring(dotpos + 1, windowname.length);
	windowname = tempwn;
	}
	var sizestring = 'width=' + wid + ',height=' + hei;
	window.open('',windowname,'scrollbars,resizable,' + sizestring + '\'');
	}
	// -->
	</SCRIPT>



	<!-- has inhead tag -->
	 
	 
	<script language="JavaScript">
	<!--
	if ( top != self )
	{
		top.location.href = unescape(window.location.pathname);
	}
	//-->
	</script>
	
	

<meta name="robots" content="nofollow">
</HEAD>
<BODY BGCOLOR="#FFFFFF" 
	
		
			
		
	>



<A NAME="top"><!-- null --></A>

	
	<TABLE CELLPADDING="0" CELLSPACING="0" BORDER="0">
	<TR>
		<TD VALIGN="BOTTOM" NOWRAP>
			<A TARGET="_top" STYLE="text-decoration: none" HREF="/"><IMG BORDER="0" WIDTH="220" HEIGHT="44" ALT="JBC" SRC="/icons/banner/title.gif"></A>
			<A TARGET="_top" HREF="/cgi/adclick?journal=jbc&ad=HWP%20jeans&url=http://highwire.stanford.edu&side=only"><IMG BORDER=2 WIDTH=220 HEIGHT=40 SRC="/ads/gifs/1055197214.gif" ALT="Curious?  www.highwire.org"></A><BR><IMG ALT="" WIDTH="1" HEIGHT="3" SRC="/icons/spacer.gif"><BR>
			
	<TABLE CELLPADDING="2" CELLSPACING="2" WIDTH="450">
	<TR>
		<TD ALIGN="CENTER" VALIGN="MIDDLE" BGCOLOR="#006699" NOWRAP><A TARGET="_top" STYLE="text-decoration: none" HREF="/"><FONT SIZE="-2" COLOR="#FFFFFF" FACE="arial,helvetica">HOME</FONT></A></TD>
	    <TD ALIGN="CENTER" VALIGN="MIDDLE" BGCOLOR="#006699" NOWRAP><A TARGET="_top" STYLE="text-decoration: none" HREF="/help/"><FONT SIZE="-2" COLOR="#FFFFFF" FACE="arial,helvetica">HELP</FONT></A></TD>
	    <TD ALIGN="CENTER" VALIGN="MIDDLE" BGCOLOR="#006699" NOWRAP><A TARGET="_top" STYLE="text-decoration: none" HREF="/cgi/feedback"><FONT SIZE="-2" COLOR="#FFFFFF" FACE="arial,helvetica">FEEDBACK</FONT></A></TD>
	    <TD ALIGN="CENTER" VALIGN="MIDDLE" BGCOLOR="#006699" NOWRAP><A TARGET="_top" STYLE="text-decoration: none" HREF="/subscriptions/"><FONT SIZE="-2" COLOR="#FFFFFF" FACE="arial,helvetica">SUBSCRIPTIONS</FONT></A></TD>
	    <TD ALIGN="CENTER" VALIGN="MIDDLE" BGCOLOR="#006699" NOWRAP><A TARGET="_top" STYLE="text-decoration: none" HREF="/contents-by-date.0.shtml"><FONT SIZE="-2" COLOR="#FFFFFF" FACE="arial,helvetica">ARCHIVE</FONT></A></TD>
	    <TD ALIGN="CENTER" VALIGN="MIDDLE" BGCOLOR="#006699" NOWRAP><A TARGET="_top" STYLE="text-decoration: none" HREF="/search.dtl"><FONT SIZE="-2" COLOR="#FFFFFF" FACE="arial,helvetica">SEARCH</FONT></A></TD>
	    


<TD ALIGN="CENTER" VALIGN="MIDDLE" BGCOLOR="#006699" NOWRAP><A TARGET="_top" STYLE="text-decoration: none" HREF="/content/vol275/issue27/index.shtml"><FONT SIZE="-2" COLOR="#FFFFFF" FACE="arial,helvetica">TABLE OF CONTENTS</FONT></A></TD>




	</TR>
	</TABLE>
	</TD>
		<TD><IMG ALT="" WIDTH="5" HEIGHT="5" SRC="/icons/spacer.gif"></TD>
		<TD VALIGN="BOTTOM">
	
	<TABLE CELLPADDING="0" CELLSPACING="1" BORDER="0" BGCOLOR="#000000"><TR>
	<FORM METHOD="GET" NAME="search" ACTION="/cgi/search"><INPUT TYPE="hidden" NAME="andorexactfulltext" VALUE="and"><INPUT TYPE="hidden" NAME="resourcetype" VALUE="1"><INPUT TYPE="hidden" NAME="disp_type" VALUE="">
	<TD ALIGN="CENTER" VALIGN="MIDDLE"> 
		<TABLE CELLPADDING="1" CELLSPACING="0" BORDER="0" BGCOLOR="#DBDBDB"> 
		<TR>
			<TD BGCOLOR="#006699"><IMG ALT="" WIDTH="2" HEIGHT="2" SRC="/icons/spacer.gif"></TD>
			<TD VALIGN="MIDDLE"  BGCOLOR="#006699" NOWRAP><STRONG><FONT SIZE="-2" FACE="verdana,arial,helvetica,sans-serif" COLOR="#FFFFFF"><EM>QUICK</EM> SEARCH:&nbsp;</FONT></STRONG></TD>
			<TD ALIGN="RIGHT" VALIGN="MIDDLE" BGCOLOR="#006699" NOWRAP><A STYLE="text-decoration:none;color:#FFFFFF" HREF="/search.dtl"><FONT SIZE="-2" FACE="verdana,arial,helvetica" COLOR="#FFFFFF">&nbsp;[advanced]</FONT></A></TD>
			<TD BGCOLOR="#006699"><IMG ALT="" WIDTH="2" HEIGHT="2" SRC="/icons/spacer.gif"></TD>
		</TR><TR>
			<TD BGCOLOR="#DBDBDB"><IMG ALT="" WIDTH="2" HEIGHT="2" SRC="/icons/spacer.gif"></TD>
			<TD BGCOLOR="#DBDBDB" COLSPAN="2">
			<TABLE CELLPADDING="0" CELLSPACING="0" BORDER="0"> 
			<TR>
				<TD ALIGN="CENTER" VALIGN="BOTTOM"><INPUT TYPE="submit" STYLE="text-decoration:none;font-family:verdana,arial,helvetica;font-size:10px;font-weight:bold;color:#000000;background:#FFFFFF;" VALUE="Go"></TD>
				<TH ALIGN="LEFT" COLSPAN="2"><FONT SIZE="-2" FACE="verdana,arial,helvetica,sans-serif" COLOR="#000000">Author:<BR></FONT>
					<INPUT TYPE="text" NAME="author1" SIZE="9" MAXLENGTH="80" VALUE="" STYLE="font-family:verdana,arial,helvetica;font-size:10px;"></TH>
				<TH ALIGN="LEFT" COLSPAN="3"><FONT SIZE="-2" FACE="verdana,arial,helvetica,sans-serif" COLOR="#000000">Keyword(s):<BR></FONT>
					<INPUT TYPE="text" NAME="fulltext" SIZE="16" MAXLENGTH="80" VALUE="" STYLE="font-family:verdana,arial,helvetica;font-size:10px;"></TH>
			</TR><TR>
				<TH><FONT SIZE="-2" FACE="verdana,arial,helvetica,sans-serif" COLOR="#000000">Year:&nbsp;</FONT></TH>
				<TD><INPUT TYPE="text" NAME="pubdate_year" SIZE="4" MAXLENGTH="4" VALUE="" STYLE="font-family:verdana,arial,helvetica;font-size:10px;"></TD>
				<TH ALIGN="RIGHT"><FONT SIZE="-2" FACE="verdana,arial,helvetica,sans-serif" COLOR="#000000">Vol:&nbsp;</FONT></TH>
				<TD><INPUT TYPE="text" NAME="volume" SIZE="4" MAXLENGTH="10" VALUE="" STYLE="font-family:verdana,arial,helvetica;font-size:10px;"></TD>
				<TH><FONT SIZE="-2" FACE="verdana,arial,helvetica,sans-serif" COLOR="#000000">Page:&nbsp;</FONT></TH>
				<TD><INPUT TYPE="text" NAME="firstpage" SIZE="4" MAXLENGTH="10" VALUE="" STYLE="font-family:verdana,arial,helvetica;font-size:10px;"></TD>
			</TR> 
			</TABLE> </TD>
			<TD BGCOLOR="#DBDBDB"><IMG ALT="" WIDTH="2" HEIGHT="2" SRC="/icons/spacer.gif"></TD>
		</TR>
		</TABLE></TD></FORM></TR></TABLE> 
	 
	</TD>
	</TR><TR>
		<TD COLSPAN="3"><IMG ALT="" WIDTH="2" HEIGHT="2" SRC="/icons/spacer.gif"><BR>







	
	

	




	


	
	
	
	 

	


	
	




				<BR><FONT SIZE="-2" FACE="verdana,arial,helvetica">
	<NOBR><STRONG>Institution: Cornell University Medical College</STRONG></NOBR>
	<NOBR><A TARGET="_top" HREF="/cgi/login?uri=%2Fcgi%2Fcontent%2Ffull%2F275%2F27%2F20578">Sign In as Member/Non-Member</A></NOBR>
	
		
		
	
	</FONT>
	
 <!-- showauthstring -->
	<!-- INST -->
	<!-- INDV -->
        <!-- SPONSORED -->
	<!-- USER.PFA -->
	<!-- USER.PFABSAIC -->
	<!-- NOT GUEST -->
	

	
<HR ALIGN="LEFT" SIZE="1" COLOR="#000000" NOSHADE></TD>
	</TR>
	</TABLE>
	



         
                
        



	
		
		
	 
 




		<FONT COLOR="#A70716"></FONT>
	
	<FONT SIZE=-1>Originally published In Press as  <A HREF="/cgi/content/abstract/M001887200v1">doi:10.1074/jbc.M001887200</A> on April 25, 2000</FONT>


	
		
		<P><FONT SIZE=-1>J. Biol. Chem., Vol. 275, Issue 27, 20578-20587, July 7, 2000</FONT><BR>
		
	

 

 
	  
		                    
	


	
		<!-- beginning of outer tables -->
               
                
                <TABLE CELLPADDING=4
                ALIGN=RIGHT
                
                CELLPADDING=5 HSPACE=4 VSPACE=4 BORDER NOWRAP>
               
		<TR><TD ALIGN=LEFT NOWRAP>

		<!-- beginning of inner table -->
				<TABLE BORDER="0" CELLPADDING="0" CELLSPACING="1" NOWRAP>
		
		
	 
	
		
			<TR><TD><IMG WIDTH=9 HEIGHT=11 ALIGN=BASELINE ALT="" BORDER=0 SRC="/icons/toc/rarrowsm.gif"><FONT SIZE="-1">
				<A HREF="/cgi/content/abstract/275/27/20578">Abstract of this Article</A>  (<img src="/icons/shared/toc/free.gif">)
			</FONT></TD></TR>
		
		
	


		
		
		

		
		
		


		
		
		

		
		
		

		
		
		

		
                
                

		
	


		

		
	

		
		

		
		 

		
		 
	 		<TR><TD><IMG WIDTH=9 HEIGHT=11 ALIGN=BASELINE ALT="" BORDER=0 SRC="/icons/toc/rarrowsm.gif"><FONT SIZE="-1">

			
			 

			
			

						
				
				<STRONG><A HREF="/cgi/reprint/275/27/20578">Reprint (PDF) Version of this Article</A>
 				
			

			

		</FONT></TD></TR>
		
		


                
                


		


	



		



		
		

		
		
		
	


		


		

		

		
		
			
			
			
				<TR><TD><IMG WIDTH=9 HEIGHT=11 ALIGN=BASELINE ALT="" BORDER=0 SRC="/icons/toc/rarrowsm.gif"><FONT SIZE="-1">
				<A HREF="/cgi/citemap?id=jbc;275/27/20578">Citation Map</A>
				</FONT></TD></TR>
		


		
		<TR><TD><IMG WIDTH=9 HEIGHT=11 ALIGN=BASELINE ALT="" BORDER=0 SRC="/icons/toc/rarrowsm.gif"><FONT SIZE="-1">
		<A HREF="/cgi/mailafriend?url=http://www.jbc.org/cgi/content/full/275/27/20578&title=Regulated+Association+of+Microtubule-associated+Protein+2+%28MAP2%29+with+Src+and+Grb2%3A+Evidence+for+MAP2+as+a+Scaffolding+Protein">Email this article to a friend</A>
		</FONT></TD></TR>
		


		 
		
		


	
	
	
	
		
		
		
		
	
		
		<TR><TD><IMG WIDTH=9 HEIGHT=11 ALIGN=BASELINE ALT="" BORDER=0 SRC="/icons/toc/rarrowsm.gif"><FONT SIZE="-1">
			Similar articles found in:<BR><IMG WIDTH=9 HEIGHT=9 ALIGN=BASELINE ALT="" BORDER=0 SRC="/icons/spacer.gif"><IMG WIDTH=9 HEIGHT=9 ALIGN=BASELINE ALT="" BORDER=0 SRC="/icons/spacer.gif"><A HREF="/cgi/search?qbe=jbc;M001887200&journalcode=jbc&minscore=5000"><FONT SIZE="-1">JBC Online</FONT></A>
				
				
				<BR><IMG WIDTH=9 HEIGHT=9 ALIGN=BASELINE ALT="" BORDER=0 SRC="/icons/spacer.gif"><IMG WIDTH=9 HEIGHT=9 ALIGN=BASELINE ALT="" BORDER=0 SRC="/icons/spacer.gif"><A HREF="/cgi/external_ref?access_num=10781592&link_type=MED_NBRS"><FONT SIZE="-1">PubMed</FONT></A>
		</FONT></TD></TR>
		
		
	
	
	<TR><TD><IMG WIDTH=9 HEIGHT=11 ALIGN=BASELINE ALT="" BORDER=0 SRC="/icons/toc/rarrowsm.gif"><FONT SIZE="-1">
		<A HREF="/cgi/external_ref?access_num=10781592&link_type=PUBMED">PubMed Citation</A>
	</FONT></TD></TR>
	
	

		
		
		
		
		
		
		
			<TR><TD><IMG WIDTH=9 HEIGHT=11 ALIGN=BASELINE ALT="" BORDER=0 SRC="/icons/toc/rarrowsm.gif"><FONT SIZE="-1">
				This Article has been cited by:
			</FONT></TD></TR>
		

		
		
		
		<TR><TD><FONT SIZE="-1">
			<IMG WIDTH=9 HEIGHT=9 ALIGN=BASELINE ALT="" BORDER=0 SRC="/icons/spacer.gif"><IMG WIDTH=9 HEIGHT=9 ALIGN=BASELINE ALT="" BORDER=0 SRC="/icons/spacer.gif"><A HREF="#otherarticles">other online articles</A>
		</FONT></TD></TR>
		


                
                


                
                

		
		
		
		
		
		<TR><TD><IMG WIDTH=9 HEIGHT=11 ALIGN=BASELINE ALT="" BORDER=0 SRC="/icons/toc/rarrowsm.gif"><FONT SIZE="-1">
		
			Search PubMed for articles by:<BR>
			<IMG WIDTH=9 HEIGHT=9 ALIGN=BASELINE ALT="" BORDER=0 SRC="/icons/spacer.gif"><IMG WIDTH=9 HEIGHT=9 ALIGN=BASELINE ALT="" BORDER=0 SRC="/icons/spacer.gif">
			
				<A HREF="/cgi/external_ref?access_num=Lim+RW&link_type=AUTHORSEARCH">Lim, R. W. L.</A>
				 || 
			
				<A HREF="/cgi/external_ref?access_num=Halpain+S&link_type=AUTHORSEARCH">Halpain, S.</A>
				
			
		
		</FONT></TD></TR>
		

		
		



		



		


		
		
		
		<TR><TD><IMG WIDTH=9 HEIGHT=11 ALIGN=BASELINE ALT="" BORDER=0 SRC="/icons/toc/rarrowsm.gif"><FONT SIZE="-1">
		Alert me when:
		<BR><IMG WIDTH=9 HEIGHT=9 ALIGN=BASELINE ALT="" BORDER=0 SRC="/icons/spacer.gif"><IMG WIDTH=9 HEIGHT=9 ALIGN=BASELINE ALT="" BORDER=0 SRC="/icons/spacer.gif"><A HREF="/cgi/alerts/ctalert?alertType=citedby&addAlert=cited_by&saveAlert=no&cited_by_criteria_resid=jbc;275/27/20578&return_type=article&return_url=%2Fcgi%2Fcontent%2Ffull%2F275%2F27%2F20578"><FONT SIZE="-1">new articles cite this article</FONT></A><BR>
		</FONT></TD></TR>


		


		
		


		
		
		
			<TR><TD><IMG WIDTH=9 HEIGHT=11 ALIGN=BASELINE ALT="" BORDER=0 SRC="/icons/toc/rarrowsm.gif"><FONT SIZE="-1">
			<A HREF="/cgi/citmgr?gca=jbc;275/27/20578">Download to Citation Manager</A>
		</FONT></TD></TR>
		
		

		
		</TABLE> 
		</TD></TR> <!-- end of inner content box -->




		
		
		
		


		
        <!-- 1xxx --><!-- 2xxx --><!-- 3xxx --><!-- 4xxx --><!-- 5xxx -->         
	        <TR><TD ALIGN=LEFT NOWRAP>   <!-- revisions table -->
		

	
	<TABLE BORDER="0" CELLPADDING="0" CELLSPACING="1" NOWRAP>
	<TR><TD><IMG WIDTH=9 HEIGHT=11 ALIGN=BASELINE ALT="" BORDER=0 SRC="/icons/toc/rarrowsm.gif"><FONT SIZE="-1">
	All Versions of this Article:	
	</FONT></TD></TR>
	         
	<TR><TD ALIGN=LEFT NOWRAP><FONT SIZE=-1><IMG WIDTH=27 HEIGHT=11 ALIGN=BASELINE ALT="" BORDER=0 SRC="/icons/spacer.gif">275/27/20578<FONT COLOR="#A70716"> (most recent)</FONT></FONT></TD></TR>
	         

	         
	<TR><TD ALIGN=LEFT NOWRAP><FONT SIZE=-1><IMG WIDTH=27 HEIGHT=11 ALIGN=BASELINE ALT="" BORDER=0 SRC="/icons/spacer.gif"><A HREF="/cgi/content/short/M001887200v1">M001887200v1</A></FONT></TD></TR>
	
	</TABLE>
	         

    
		</TD></TR>
        
        
	</TABLE>	 <!-- end of outer content box -->
	


	









 





	
	
	
	
	
	
	
	
	
		<vardef id="TEXT">

<!--

   Pages created by the Electronic Press Engine from

		Atypon Systems, Inc.

   Visit http://www.atypon.com/

-->





<H2>
<FONT COLOR=A70716 SIZE=-1>

</FONT>

    Regulated Association of Microtubule-associated Protein 2&nbsp;(MAP2</A>) with Src and Grb2: Evidence for MAP2</A> as a Scaffolding
Protein<A HREF="#FN150"><SUP>*</SUP></A>
    <BR>


</H2>



<STRONG>
<NOWRAP>
Rita W.&nbsp;L. 
Lim and<WBR>
Shelley 
Halpain<A HREF="#FN152"><SUP><IMG SRC="/math/12pt/normal/link/Dagger.gif" ALIGN=BASELINE ALT="Dagger" BORDER=0></SUP></A>
</NOWRAP>
</STRONG>
<P>
From the Department of Cell Biology, The Scripps Research
Institute, La Jolla, California 92037
<P>
Received for publication, March 7,&nbsp;2000
<P>






    
	
	    
	
	    
		
    
	
	    
	
    
	
	    
	
    
    
	
	    
	
    







<a name = "Abstract"><!-- comment for mosaic --></a>



<A NAME="Abstract"><!-- null --></A>
<TABLE WIDTH=100% BGCOLOR=E1E1E1 CELLPADDING=0 CELLSPACING=0>
<TR>
<TD ALIGN=LEFT VALIGN=MIDDLE WIDTH=5% BGCOLOR=FFFFFF>&nbsp;<IMG WIDTH=10 HEIGHT=21 HSPACE=5 ALT="" SRC="/icons/toc/rarrow.gif"></TD>
<TH ALIGN=LEFT VALIGN=MIDDLE WIDTH=95%><FONT SIZE=+2>&nbsp;&nbsp;ABSTRACT</FONT></TH>
</TR>
</TABLE>

<TABLE ALIGN=RIGHT CELLPADDING=5 BORDER NOWRAP><R>
<TH ALIGN=LEFT>
<FONT SIZE=-1>



<A HREF="#Top"><IMG WIDTH=11 HEIGHT=9 BORDER=0 HSPACE=5 SRC="/icons/toc/uarrow.gif">TOP</A>


<BR><IMG WIDTH=11 HEIGHT=9 BORDER=0 HSPACE=5 SRC="/icons/toc/dot.gif"><FONT COLOR=46
4C53>ABSTRACT</FONT>


<BR>
<A HREF="#Introduction"><IMG WIDTH=11 HEIGHT=9 BORDER=0 HSPACE=5 SRC="/icons/toc/darrow.gif">INTRODUCTION</A>


<BR>
<A HREF="#SEC1"><IMG WIDTH=11 HEIGHT=9 BORDER=0 HSPACE=5 SRC="/icons/toc/darrow.gif">EXPERIMENTAL PROCEDURES</A>


<BR>
<A HREF="#SEC2"><IMG WIDTH=11 HEIGHT=9 BORDER=0 HSPACE=5 SRC="/icons/toc/darrow.gif">RESULTS</A>


<BR>
<A HREF="#SEC3"><IMG WIDTH=11 HEIGHT=9 BORDER=0 HSPACE=5 SRC="/icons/toc/darrow.gif">DISCUSSION</A>


<BR>
<A HREF="#References"><IMG WIDTH=11 HEIGHT=9 BORDER=0 HSPACE=5 SRC="/icons/toc/darrow.gif">REFERENCES</A>
</FONT></TH>
</TR></TABLE>

<P>Microtubule-associated protein 2&nbsp;(MAP2</A>) and tau,
which is involved in Alzheimer's disease, are major cytoskeletal
proteins<SUP> </SUP>in neurons. These proteins are involved in microtubule
assembly<SUP> </SUP>and stability. To further characterize MAP2</A>, we took a
strategy<SUP> </SUP>of identifying potential MAP2</A> binding partners. The low
molecular<SUP> </SUP>weight MAP2c protein has 11&nbsp;P<I>XX</I>P motifs that are
conserved across<SUP> </SUP>species, and these P<I>XX</I>P motifs could be
potential ligands for<SUP> </SUP>Src homology 3&nbsp;(SH3) domains. We tested for MAP2</A>
interaction with<SUP> </SUP>SH3 domain-containing proteins. All neuronal MAP2</A>
isoforms bound<SUP> </SUP>specifically to the SH3 domains of c-Src and Grb2 in an
<I>in vitro</I><SUP> </SUP>glutathione <I>S</I>-transferase-SH3
pull-down assay. Interactions between<SUP> </SUP>endogenous proteins were
confirmed by co-immunoprecipitation using<SUP> </SUP>brain lysate. All three
proteins were also found co-expressed<SUP> </SUP>in neuronal cell bodies and
dendrites. Surprisingly, the SH3 domain-binding<SUP> </SUP>site was mapped to the
microtubule-binding domain that contains<SUP> </SUP>no P<I>XX</I>P motif. Src
bound primarily the soluble, non-microtubule-associated<SUP> </SUP>MAP2c <I>in
vitro</I>. This specific MAP2/SH3 domain interaction was<SUP> </SUP>inhibited by
phosphorylation of MAP2c by the mitogen-activated<SUP> </SUP>protein kinase
extracellular signal-regulated kinase 2&nbsp;but not<SUP> </SUP>by protein
kinase A.&nbsp;This phosphorylation-regulated association<SUP> </SUP>of MAP2</A> with
proteins of intracellular signal transduction pathways<SUP> </SUP>suggests a
possible link between cellular signaling and neuronal<SUP> </SUP>cytoskeleton,
with MAP2</A> perhaps acting as a molecular scaffold<SUP> </SUP>upon which
cytoskeleton-modifying proteins assemble and dissociate<SUP> </SUP>in response to
neuronal<SUP> </SUP>activity.
<BR><BR CLEAR=ALL>




<TXT>
<A NAME="SEC0"><!-- comment for mosaic --></A>

    
    

    
    

    
    
	
	
	
<A NAME="Introduction"><!-- null --></A>
<TABLE WIDTH=100% BGCOLOR=E1E1E1 CELLPADDING=0 CELLSPACING=0>
<TR>
<TD ALIGN=LEFT VALIGN=MIDDLE WIDTH=5% BGCOLOR=FFFFFF>&nbsp;<IMG WIDTH=10 HEIGHT=21 HSPACE=5 ALT="" SRC="/icons/toc/rarrow.gif"></TD>
<TH ALIGN=LEFT VALIGN=MIDDLE WIDTH=95%><FONT SIZE=+2>&nbsp;&nbsp;INTRODUCTION</FONT></TH>
</TR>
</TABLE>

<TABLE ALIGN=RIGHT CELLPADDING=5 BORDER NOWRAP><R>
<TH ALIGN=LEFT>
<FONT SIZE=-1>



<A HREF="#Top"><IMG WIDTH=11 HEIGHT=9 BORDER=0 HSPACE=5 SRC="/icons/toc/uarrow.gif">TOP</A>


<BR>
<A HREF="#Abstract"><IMG WIDTH=11 HEIGHT=9 BORDER=0 HSPACE=5 SRC="/icons/toc/uarrow.gif">ABSTRACT</A>


<BR><IMG WIDTH=11 HEIGHT=9 BORDER=0 HSPACE=5 SRC="/icons/toc/dot.gif"><FONT COLOR=46
4C53>INTRODUCTION</FONT>


<BR>
<A HREF="#SEC1"><IMG WIDTH=11 HEIGHT=9 BORDER=0 HSPACE=5 SRC="/icons/toc/darrow.gif">EXPERIMENTAL PROCEDURES</A>


<BR>
<A HREF="#SEC2"><IMG WIDTH=11 HEIGHT=9 BORDER=0 HSPACE=5 SRC="/icons/toc/darrow.gif">RESULTS</A>


<BR>
<A HREF="#SEC3"><IMG WIDTH=11 HEIGHT=9 BORDER=0 HSPACE=5 SRC="/icons/toc/darrow.gif">DISCUSSION</A>


<BR>
<A HREF="#References"><IMG WIDTH=11 HEIGHT=9 BORDER=0 HSPACE=5 SRC="/icons/toc/darrow.gif">REFERENCES</A>
</FONT></TH>
</TR></TABLE>

    

<P>In the mammalian nervous system, the neuron-specific
microtubule-associated protein
MAP2<SUP>1</SUP> regulates microtubule
dynamics (<A HREF="#B1">1</A>) and is attributed with<SUP> </SUP>an important role in neurite
outgrowth and dendrite development<SUP> </SUP>(<A HREF="#B2">2-6</A>). MAP2</A> belongs to a family of
heat stable microtubule-associated<SUP> </SUP>proteins found in eukaryotic cells.
Other family members include<SUP> </SUP>MAP4, a ubiquitous protein, and tau, a
neuronal protein involved<SUP> </SUP>in Alzheimer's disease (<A HREF="#B4">4</A>, <A HREF="#B5">5</A>). Members of
this family share<SUP> </SUP>a common microtubule-binding domain (MTBD</A>) consisting
of approximately<SUP> </SUP>100-130 amino acid residues near the carboxyl
terminus of the<SUP> </SUP>molecule. The MTBD</A> contains three or four tandem
imperfect repeats<SUP> </SUP>of 18&nbsp;amino acid residues separated by a variable
inter-repeat<SUP> </SUP>region of approximately 12&nbsp;amino acid residues (<A HREF="#B7">7</A>).
</P>
<P>Recently, tau has been shown to bind the Src family of non-receptor
tyrosine kinases (Src, Lyn, and Fyn (<A HREF="#B8">8</A>)) and protein<SUP> </SUP>phosphatases PP1
and PP2A (<A HREF="#B9">9-11</A>). The interactions among PP1,<SUP> </SUP>PP2A, tau, and
microtubules were postulated to regulate the phosphorylation<SUP> </SUP>state of
tau, and alterations to these interactions could lead<SUP> </SUP>to the
development of tauopathies seen in neurological diseases<SUP> </SUP>such as
Alzheimer's (<A HREF="#B12">12-14</A>). This increasing body of evidence suggests<SUP> </SUP>that
microtubule-based structural proteins like MAP2</A> and tau could<SUP> </SUP>play
other physiological roles in addition to the regulation of<SUP> </SUP>microtubule<SUP> </SUP>assembly.
</P>
<P>Like tau, MAP2</A> is a phosphoprotein, and its cellular activities are
regulated by phosphorylation. The phosphorylation state<SUP> </SUP>of MAP2</A> is
modulated by neural activity <I>in vivo</I> (<A HREF="#B15">15-20</A>). <I>In
vitro</I>,<SUP> </SUP>several serine/threonine protein kinases and phosphatases
are<SUP> </SUP>capable of using MAP2</A> as a substrate (<A HREF="#B21">21-28</A>). Tyrosine kinases<SUP> </SUP>such as insulin receptor kinase, epidermal growth factor receptor<SUP> </SUP>kinase, and Src have been reported to phosphorylate MAP2</A> <I>in
vitro</I><SUP> </SUP>(<A HREF="#B29">29-31</A>). Besides microtubules, several additional proteins
have<SUP> </SUP>been reported to interact with MAP2</A> <I>in vitro</I>, the best
characterized<SUP> </SUP>of which is the type II regulatory subunit of PKA</A>
(<A HREF="#B32">32-34</A>). Other<SUP> </SUP>proteins include F-actin (<A HREF="#B35">35-38</A>), calmodulin (<A HREF="#B39">39</A>), and
the mitochondrial<SUP> </SUP>outer membrane protein porin (<A HREF="#B40">40</A>). The precise
binding sites<SUP> </SUP>on MAP2</A> for these proteins have yet to be identified.
Except for<SUP> </SUP>the MTBD</A> at the carboxyl terminus and the type II regulatory
subunit<SUP> </SUP>of PKA</A> binding region (30&nbsp;amino acids) in the amino terminus of<SUP> </SUP>MAP2</A>, the rest of the molecule (<IMG SRC="/math/12pt/normal/ge.gif" ALIGN=BASELINE ALT="&gt;= ">65%) has no assigned function.<SUP> </SUP>It is
possible that MAP2</A> possesses some as yet uncovered role<SUP> </SUP>in the neuron,
in addition to its contribution to cellular structural<SUP> </SUP>integrity.
</P>
<P>This paper presents evidence of a novel interaction of MAP2</A> with the
non-receptor protein tyrosine kinase c-Src and the adaptor<SUP> </SUP>protein
Grb2. The data support the emerging idea that members<SUP> </SUP>of signal
transduction pathways interact with microtubule-associated<SUP> </SUP>proteins and
could thereby regulate the organization and dynamics<SUP> </SUP>of the<SUP> </SUP>cytoskeleton.
<BR><BR CLEAR=ALL>



<A NAME="SEC1"><!-- null --></A>
<TABLE WIDTH=100% BGCOLOR=E1E1E1 CELLPADDING=0 CELLSPACING=0>
<TR>
<TD ALIGN=LEFT VALIGN=MIDDLE WIDTH=5% BGCOLOR=FFFFFF>&nbsp;<IMG WIDTH=10 HEIGHT=21 HSPACE=5 ALT="" SRC="/icons/toc/rarrow.gif"></TD>
<TH ALIGN=LEFT VALIGN=MIDDLE WIDTH=95%><FONT SIZE=+2>&nbsp;&nbsp;EXPERIMENTAL PROCEDURES</FONT></TH>
</TR>
</TABLE>

<TABLE ALIGN=RIGHT CELLPADDING=5 BORDER NOWRAP><R>
<TH ALIGN=LEFT>
<FONT SIZE=-1>



<A HREF="#Top"><IMG WIDTH=11 HEIGHT=9 BORDER=0 HSPACE=5 SRC="/icons/toc/uarrow.gif">TOP</A>


<BR>
<A HREF="#Abstract"><IMG WIDTH=11 HEIGHT=9 BORDER=0 HSPACE=5 SRC="/icons/toc/uarrow.gif">ABSTRACT</A>


<BR>
<A HREF="#Introduction"><IMG WIDTH=11 HEIGHT=9 BORDER=0 HSPACE=5 SRC="/icons/toc/uarrow.gif">INTRODUCTION</A>


<BR><IMG WIDTH=11 HEIGHT=9 BORDER=0 HSPACE=5 SRC="/icons/toc/dot.gif"><FONT COLOR=46
4C53>EXPERIMENTAL PROCEDURES</FONT>


<BR>
<A HREF="#SEC2"><IMG WIDTH=11 HEIGHT=9 BORDER=0 HSPACE=5 SRC="/icons/toc/darrow.gif">RESULTS</A>


<BR>
<A HREF="#SEC3"><IMG WIDTH=11 HEIGHT=9 BORDER=0 HSPACE=5 SRC="/icons/toc/darrow.gif">DISCUSSION</A>


<BR>
<A HREF="#References"><IMG WIDTH=11 HEIGHT=9 BORDER=0 HSPACE=5 SRC="/icons/toc/darrow.gif">REFERENCES</A>
</FONT></TH>
</TR></TABLE>


    



    



    


<P><I>Materials--</I>

DNA restriction enzymes, PKA</A>, MAPK</A> (extracellular
signal-regulated kinase 2,&nbsp;p42), and phosphotyrosine monoclonal
antibody<SUP> </SUP>P-Tyr-100 were from New England Biolabs. The <I>Pfu</I>
DNA polymerase<SUP> </SUP>and <I>Escherichia coli</I> strains XL1 Blue, XL2
Blue, DH5<IMG SRC="/math/12pt/normal/alpha.gif" ALIGN=BASELINE ALT="alpha ">, JM109,<SUP> </SUP>and BL21(DE3) were from Stratagene. Protein
expression vectors<SUP> </SUP>pET-3a, pET-11a, and pET-32a were from Novagen. The
pGEX-2T vector,<SUP> </SUP>glutathione-Sepharose, SP</A> ion exchange resin,
HRP-conjugated goat<SUP> </SUP>anti-mouse and anti-rabbit antibodies, and thrombin
were from<SUP> </SUP>Amersham Pharmacia Biotech. Monoclonal anti-polyhistidine
antibody,<SUP> </SUP>HIS-1, anti-MAP2 antibody, HM2, PMSF</A>, reduced glutathione,
protein<SUP> </SUP>A-Sepharose, ampicillin, carbenicillin, leupeptin, aprotinin,<SUP> </SUP>pepstatin A, benzamide, benzamidine, <IMG SRC="/math/12pt/normal/epsilon.gif" ALIGN=BASELINE ALT="epsilon ">-caproic acid, GTP, ATP,<SUP> </SUP><font size=-1>L</font>-glutamine, pyruvate, GSK-3<IMG SRC="/math/12pt/normal/beta.gif" ALIGN=BASELINE ALT="beta ">, and lysozyme were from
Sigma. Superfect<SUP> </SUP>reagent was from Qiagen. Paclitaxel was from
Calbiochem. Sypro<SUP> </SUP>Orange was from Bio-Rad. BCA protein assay reagent
was from Pierce.<SUP> </SUP>Nitrocellulose membranes (0.45&nbsp;&#181;m, BA-85) were from
Schleicher<SUP> </SUP>&amp;&nbsp;Schuell. Alkaline phosphatase-conjugated goat anti-rabbit
and<SUP> </SUP>anti-mouse antibodies were from Promega. Alexa-568 conjugated<SUP> </SUP>goat
anti-mouse IgG and Alexa-488 conjugated goat anti-rabbit<SUP> </SUP>IgG were from
Molecular Probes. Rabbit anti-Src sc-18 and rabbit<SUP> </SUP>anti-Grb2 sc-255
antibodies were from Santa Cruz Biotechnology.<SUP> </SUP>Monoclonal
anti-phosphotyrosine antibody 4G10 was from Upstate<SUP> </SUP>Biotechnology.
HRP-conjugated recombinant anti-phosphotyrosine<SUP> </SUP>PY20 (RC20-HRP) and
monoclonal anti-Grb2 antibodies were from<SUP> </SUP>Transduction Laboratories.
Bovine tubulin was from Cytoskeleton<SUP> </SUP>Inc. Talon spin columns (0.5&nbsp;ml)
were from <a href="/cgi/inline?clontech">CLONTECH Laboratories</a>.<SUP> </SUP>[<IMG SRC="/math/12pt/normal/gamma.gif" ALIGN=BASELINE ALT="gamma ">-<SUP>32</SUP>P]ATP was from NEN Life Science<SUP> </SUP>Products.



    

    


<P><I>Construction of Expression Vectors--</I>

All molecular methods
were performed according to Ref. <A HREF="#B41">41</A>. Construction of the expression
plasmid pET3aMAP2c, which encodes<SUP> </SUP>the full-length rat MAP2c protein,
has been described (<A HREF="#B1">1</A>).<SUP> </SUP>Mutant forms of MAP2c were made by PCR cloning
using pET3aMAP2c<SUP> </SUP>or one of the subsequently constructed expression
plasmids described<SUP> </SUP>below as PCR templates. The open reading frames in
all constructs<SUP> </SUP>were verified by automated DNA sequencing conducted at
Scripps<SUP> </SUP>Nucleic Acid Core facility with PE Biosystems instrument 373&nbsp;A<SUP> </SUP>using the <I>Taq</I> DNA polymerase, fluorescence dye detection,
and<SUP> </SUP>dideoxy terminator chemistry (<A HREF="#B42">42</A>). Custom designed primers with<SUP> </SUP>unique restriction enzyme recognition sites were used to amplify<SUP> </SUP>specific regions of MAP2c cDNA and to facilitate ligation of the<SUP> </SUP>amplified PCR fragments into expression vectors. DNA amplifications<SUP> </SUP>were conducted in a model 9600&nbsp;DNA thermal cycler (PE Biosystems).<SUP> </SUP>The
amplification protocol consisted of 30&nbsp;cycles of the following<SUP> </SUP>conditions: denaturation at 95&nbsp;&#176;C for 1&nbsp;min, primer annealing<SUP> </SUP>at
57&nbsp;&#176;C for 1&nbsp;min, and chain extension at 72&nbsp;&#176;C for 1&nbsp;min (5<SUP>&nbsp;</SUP>min
for the last cycle) with recombinant <I>Pfu</I> DNA polymerase
according<SUP> </SUP>to the manufacturer's procedure. The PCR fragments encoding
the<SUP> </SUP>amino acids 1-299, 1-310, 154-467, and 296-467 of MAP2c were
digested<SUP> </SUP>with <I>Nde</I>I and <I>Bam</I>HI and ligated into a
previously <I>Nde</I>I/<I>Bam</I>HI-digested<SUP> </SUP>pET-11a plasmid.
The PCR fragment encoding amino acids 300-400,<SUP> </SUP>the MTBD</A>, was digested
with <I>Nco</I>I and <I>Bam</I>HI and ligated into a<SUP> </SUP>previously
<I>Nco</I>I/<I>Bam</I>HI-digested pET-32a plasmid. The new
constructs<SUP> </SUP>were transformed into cloning host <I>E.&nbsp;coli</I> strain
DH5<IMG SRC="/math/12pt/normal/alpha.gif" ALIGN=BASELINE ALT="alpha ">, XL-1 Blue,<SUP> </SUP>XL-2 Blue, or JM109 and were also transformed into
expression<SUP> </SUP>host <I>E.&nbsp;coli</I> BL21(DE3) by the CaCl<SUB>2</SUB>
transformation method. Stable<SUP> </SUP>transformants were selected on LB plates
supplemented with 50<SUP>&nbsp;</SUP>&#181;g/ml<SUP> </SUP>carbenicillin.
<P>
To make internal deletion MAP2c constructs, separate DNA fragments of
MAP2c were amplified by PCR and then ligated together.<SUP> </SUP>The expression
plasmid <IMG SRC="/math/12pt/normal/Delta.gif" ALIGN=BASELINE ALT="Delta ">134-141 encoding MAP2c, lacking the amino<SUP> </SUP>acids 134-141,
was made by amplifying the DNA sequence encoding<SUP> </SUP>the amino acids 1-133
and 142-467. Both PCR products were gel-purified<SUP> </SUP>and digested with
<I>Bss</I>SI. The 1-133 fragment was further digested<SUP> </SUP>with
<I>Nde</I>I, whereas the 142-467 fragment was digested with
<I>Bam</I>HI.<SUP> </SUP>The digested DNA fragments were gel-purified again
and ligated<SUP> </SUP>into the <I>Nde</I>I/<I>Bam</I>HI-digested
expression vector pET-3a. The ligation<SUP> </SUP>of these two fragments
introduced an additional valine at the<SUP> </SUP>junction between the amino acids
133&nbsp;and 142.&nbsp;The <IMG SRC="/math/12pt/normal/Delta.gif" ALIGN=BASELINE ALT="Delta ">134-141 construct<SUP> </SUP>was transformed into JM109 and
BL21(DE3). The expression plasmids<SUP> </SUP><IMG SRC="/math/12pt/normal/Delta.gif" ALIGN=BASELINE ALT="Delta ">219-290, <IMG SRC="/math/12pt/normal/Delta.gif" ALIGN=BASELINE ALT="Delta ">290-297, and
<IMG SRC="/math/12pt/normal/Delta.gif" ALIGN=BASELINE ALT="Delta ">290-400, which encode MAP2c lacking regions<SUP> </SUP>219-290, 290-297, and
290-400, respectively, were constructed<SUP> </SUP>using a similar strategy,
except that the PCR products were initially<SUP> </SUP>digested with
<I>Bsi</I>WI prior to digestion with <I>Nde</I>I or
<I>Bam</I>HI. In<SUP> </SUP>making the double deletion construct <IMG SRC="/math/12pt/normal/Delta.gif" ALIGN=BASELINE ALT="Delta ">134-141,
219-290, a PCR<SUP> </SUP>fragment corresponding to the region 142-467,
<IMG SRC="/math/12pt/normal/Delta.gif" ALIGN=BASELINE ALT="Delta ">219-290, was amplified<SUP> </SUP>from the <IMG SRC="/math/12pt/normal/Delta.gif" ALIGN=BASELINE ALT="Delta ">219-290 single deletion plasmid
described above. This<SUP> </SUP>PCR fragment was digested with <I>Bss</I>SI
and <I>Bam</I>HI, gel-purified,<SUP> </SUP>and then ligated to the
<I>Nde</I>I/<I>Bss</I>SI-digested fragment 1-133. For<SUP> </SUP>the
double deletion construct <IMG SRC="/math/12pt/normal/Delta.gif" ALIGN=BASELINE ALT="Delta ">134-141, 290-297, a PCR fragment<SUP> </SUP>corresponding to the region 1-289, <IMG SRC="/math/12pt/normal/Delta.gif" ALIGN=BASELINE ALT="Delta ">134-141, was amplified from<SUP> </SUP>the
<IMG SRC="/math/12pt/normal/Delta.gif" ALIGN=BASELINE ALT="Delta ">134-141 single deletion plasmid described above. This PCR<SUP> </SUP>fragment
was sequentially digested with <I>Bsi</I>WI followed by
<I>Nde</I>I,<SUP> </SUP>gel-purified, and then ligated to the
<I>Bsi</I>WI/<I>Bam</I>HI-digested fragment<SUP> </SUP>298-467. To
construct the <IMG SRC="/math/12pt/normal/Delta.gif" ALIGN=BASELINE ALT="Delta ">219-297 plasmid, the <IMG SRC="/math/12pt/normal/Delta.gif" ALIGN=BASELINE ALT="Delta ">219-290 plasmid<SUP> </SUP>was digested
with <I>Nde</I>I and <I>Bsi</I>WI, and the larger fragment,
which<SUP> </SUP>encodes the region 1-218, was gel-purified. The <IMG SRC="/math/12pt/normal/Delta.gif" ALIGN=BASELINE ALT="Delta ">290-297
plasmid<SUP> </SUP>was digested with <I>Bsi</I>WI and <I>Bam</I>HI, and
the smaller fragment, which<SUP> </SUP>encodes the region 298-467, was
gel-purified. The 1-218 <I>Nde</I>I/<I>Bsi</I>WI<SUP> </SUP>fragment was
ligated to the 298-467 <I>Bsi</I>WI/<I>Bam</I>HI fragment and
then<SUP> </SUP>inserted into an <I>Nde</I>I/<I>Bam</I>HI-digested pET-3a
plasmid. To construct<SUP> </SUP>the triple deletion <IMG SRC="/math/12pt/normal/Delta.gif" ALIGN=BASELINE ALT="Delta ">134-141, 220-297
plasmid, the <IMG SRC="/math/12pt/normal/Delta.gif" ALIGN=BASELINE ALT="Delta ">134-141, 219-290<SUP> </SUP>plasmid was digested with
<I>Nde</I>I and <I>Bsi</I>WI, and the larger fragment,<SUP> </SUP>which
encodes the region 1-218, <IMG SRC="/math/12pt/normal/Delta.gif" ALIGN=BASELINE ALT="Delta ">134-141, was gel-purified. This<SUP> </SUP>fragment
was ligated to the 298-467 <I>Bsi</I>WI/<I>Bam</I>HI fragment
and then<SUP> </SUP>inserted into an <I>Nde</I>I/<I>Bam</I>HI-digested
pET-3a plasmid. These constructs<SUP> </SUP>were transformed into cloning bacteria
<I>E.&nbsp;coli</I> JM109 and expression<SUP> </SUP>host bacteria<SUP> </SUP>BL21(DE3).



    

    


<P><I>Expression and Purification of Recombinant MAP2c
Proteins--</I>

Induction of protein expression in bacteria and
purification of full-length and mutant non-fusion MAP2c proteins were
performed<SUP> </SUP>as described (<A HREF="#B1">1</A>), with modifications. The concentration of<SUP> </SUP>NaCl was reduced to 0.1&nbsp;<font size=-1>M</font>. The boiled supernatant collected
was<SUP> </SUP>further purified by SP</A> ion exchange chromatography. The supernatant<SUP> </SUP>was filtered through a 0.45&nbsp;&#181;m filter and loaded onto a 0.6&nbsp;&#215;<SUP>&nbsp;</SUP>1.0-cm SP</A> column pre-equilibrated with 5&nbsp;ml of 50&nbsp;m<font size=-1>M</font> sodium
acetate,<SUP> </SUP>pH 5.5,&nbsp;1&nbsp;m<font size=-1>M</font> EDTA, 1&nbsp;m<font size=-1>M</font> EGTA, and 5&nbsp;m<font size=-1>M</font> <IMG SRC="/math/12pt/normal/beta.gif" ALIGN=BASELINE ALT="beta ">-mercaptoethanol (Buffer<SUP> </SUP>A) at room temperature. The
rMAP2c</A> bound to the SP</A> column at pH<SUP> </SUP>5.5,&nbsp;whereas most of the bacterial
proteins were found in the<SUP> </SUP>run-through. The column was washed with 20&nbsp;ml of Buffer A containing<SUP> </SUP>0.15&nbsp;<font size=-1>M</font> NaCl, and rMAP2c</A> protein
was step eluted with Buffer A<SUP> </SUP>containing 0.3&nbsp;<font size=-1>M</font> NaCl.
Fractions of 300&nbsp;&#181;l were collected and<SUP> </SUP>analyzed by SDS-PAGE.
Fractions containing rMAP2c</A> were pooled<SUP> </SUP>and dialyzed against 500&nbsp;ml of
50&nbsp;m<font size=-1>M</font> Tris-HCl, pH 8,&nbsp;1&nbsp;m<font size=-1>M</font> EGTA,<SUP> </SUP>1&nbsp;m<font size=-1>M</font> EDTA, 0.02% NaN<SUB>3</SUB>, 1&nbsp;m<font size=-1>M</font>
dithiothreitol, and 150&nbsp;m<font size=-1>M</font> NaCl at<SUP> </SUP>4&nbsp;&#176;C overnight,
divided into small aliquots, and stored at -80<SUP> </SUP>&#176;C. The rMAP2c</A>
proteins obtained were of <IMG SRC="/math/12pt/normal/ge.gif" ALIGN=BASELINE ALT="&gt;= ">90% purity, and the<SUP> </SUP>average yield was 0.7&nbsp;mg/50 ml of bacterial<SUP> </SUP>culture.
<P>
The MAP2c MTBD</A> fragment cloned into the expression plasmid pET-32a was
expressed as a histidine-tagged thioredoxin fusion<SUP> </SUP>protein
(HisTrxMTBD</A>). The induction of HisTrxMTBD</A> and the histidine-tagged<SUP> </SUP>thioredoxin (HisTrx</A>) protein expression from pET-32a was initiated<SUP> </SUP>at
<I>A</I><SUB>600</SUB>&nbsp;=&nbsp;0.6-0.9 with 1&nbsp;m<font size=-1>M</font>
isopropyl-1-thio-<IMG SRC="/math/12pt/normal/beta.gif" ALIGN=BASELINE ALT="beta ">-<font size=-1>D</font>-galactopyranoside.<SUP> </SUP>Protein
expression was allowed to proceed at 30&nbsp;&#176;C for 3&nbsp;h. Bacteria<SUP> </SUP>harvested from a 50-ml culture was lysed in 10&nbsp;ml of 20&nbsp;m<font size=-1>M</font>
Tris-HCl,<SUP> </SUP>pH 8.0,&nbsp;100&nbsp;m<font size=-1>M</font> NaCl (Buffer B) supplemented with
1&nbsp;m<font size=-1>M</font> PMSF</A>, 10<SUP>&nbsp;</SUP>&#181;g/ml aprotinin, 10&nbsp;&#181;g/ml pepstatin A, 10&nbsp;&#181;g/ml leupeptin. Cell<SUP> </SUP>lysis using lysozyme and sonication were
performed as described<SUP> </SUP>(<A HREF="#B1">1</A>). The lysate was centrifuged at 15,000&nbsp;&#215;&nbsp;<I>g</I> for 30&nbsp;min at<SUP> </SUP>4&nbsp;&#176;C. The supernatant was collected,
filtered through a 0.45&nbsp;&#181;m<SUP> </SUP>filter, and loaded onto a 0.5&nbsp;&#215;&nbsp;1.5-cm Talon-NX column that had<SUP> </SUP>been pre-equilibrated with Buffer B.&nbsp;The column was washed with<SUP> </SUP>5&nbsp;ml of Buffer B followed by 5&nbsp;ml of Buffer
B containing 10&nbsp;m<font size=-1>M</font><SUP> </SUP>imidazole, pH 8.0,&nbsp;and the
histidine-tagged protein was step eluted<SUP> </SUP>with Buffer B containing 100&nbsp;m<font size=-1>M</font> imidazole, pH 8.0.&nbsp;Fractions of<SUP> </SUP>500&nbsp;&#181;l were collected
and analyzed by SDS-PAGE. Fractions containing<SUP> </SUP>the histidine-tagged
protein were pooled, dialyzed against 250<SUP>&nbsp;</SUP>ml of Buffer B with 5&nbsp;m<font size=-1>M</font> EDTA and 0.02% NaN<SUB>3</SUB> at 4&nbsp;&#176;C overnight,<SUP> </SUP>divided into small aliquots, and stored at -80&nbsp;&#176;C. The proteins<SUP> </SUP>were of <IMG SRC="/math/12pt/normal/ge.gif" ALIGN=BASELINE ALT="&gt;= ">95% purity, and the average yield was 0.5&nbsp;mg/50 ml of<SUP> </SUP>bacterial<SUP> </SUP>culture.



    

    


<P><I>GST-SH3 Fusion Proteins and Thrombin Cleavage--</I>

Expression
and purification of the GST</A> and GST-SH3 fusion proteins using
glutathione-Sepharose beads were carried out according<SUP> </SUP>to the
manufacturer's procedure. A 5-&#181;l aliquot of a ~50% (v/v)<SUP> </SUP>slurry
of protein-bound glutathione-Sepharose beads contained<SUP> </SUP>about 15-20
&#181;g of GST-SH3 fusion protein as determined by SDS-PAGE.<SUP> </SUP>For the
microtubule binding competition experiments, the GST-SH3<SUP> </SUP>fusion
proteins were eluted from the beads with 10&nbsp;m<font size=-1>M</font> reduced<SUP> </SUP>glutathione in 50&nbsp;m<font size=-1>M</font> Tris-HCl, pH 8.0.&nbsp;The eluted proteins
were<SUP> </SUP>dialyzed against 1&#215; PBS</A> at 4&nbsp;&#176;C overnight and stored at
-80&nbsp;&#176;C.
<P>
To obtain the thrombin-cleaved Grb2-N-SH3 fragment, the GST-Grb2-N-SH3
fusion protein was affinity-purified from 5&nbsp;liters<SUP> </SUP>of bacterial culture
and dialyzed against 4&nbsp;liters of 0.5&#215; PBS</A><SUP> </SUP>at 4&nbsp;&#176;C, with two changes
over a period of 12&nbsp;h, to remove the<SUP> </SUP>reduced glutathione. The
protein concentration of the dialyzed<SUP> </SUP>fusion protein was then diluted
to 0.4&nbsp;mg/ml with 10&nbsp;m<font size=-1>M</font> Tris-HCl,<SUP> </SUP>pH 8.0.&nbsp;Thrombin was
added to a final concentration of 10&nbsp;units/mg<SUP> </SUP>protein, and the
digestion was allowed to proceed at room temperature<SUP> </SUP>for 36&nbsp;h. The
digested mixture was filtered through a 0.22&nbsp;&#181;m<SUP> </SUP>filter and applied
onto a 1.0&nbsp;&#215;&nbsp;5.0-cm glutathione-Sepharose column<SUP> </SUP>to remove the
larger GST</A> fragment. The run-through solution containing<SUP> </SUP>the Grb2-N-SH3
fragment was concentrated using a 3000&nbsp;MWCO membrane<SUP> </SUP>and further
purified by gel filtration on Sephadex 30&nbsp;(2&nbsp;&#215;&nbsp;50<SUP>&nbsp;</SUP>cm)
equilibrated with 25&nbsp;m<font size=-1>M</font> Tris-HCl, pH 8.0,&nbsp;0.1&nbsp;m<font size=-1>M</font> EDTA, pH<SUP> </SUP>8.0,&nbsp;and 1&nbsp;m<font size=-1>M</font> NaN<SUB>3</SUB> at a
flow rate of 90&nbsp;ml/h. Fractions of 3&nbsp;ml<SUP> </SUP>were collected and analyzed by
SDS-PAGE. The Grb2-N-SH3 fragment<SUP> </SUP>eluted at about 125&nbsp;ml. Fractions
containing Grb2-N-SH3 were pooled,<SUP> </SUP>concentrated, and stored at
-80&nbsp;&#176;C.



    

    


<P><I>In Vitro GST-SH3 Binding Assay--</I>

Approximately 5&nbsp;&#181;g of
total rMAP2c</A> bacterial lysate, 1-2 &#181;g of partially purified rMAP2c</A>
(boiled, high salt bacterial supernatant),<SUP> </SUP>1-2 &#181;g of SP</A> purified
rMAP2c</A>, and 1&nbsp;&#181;g of phosphorylated or control-phosphorylated<SUP> </SUP>rMAP2c</A>
were incubated with 5&nbsp;&#181;l of a 50% slurry of protein bound<SUP> </SUP>glutathione-Sepharose beads in a 200-&#181;l volume containing 50&nbsp;m<font size=-1>M</font><SUP> </SUP>Tris-HCl, pH 7.5,&nbsp;75&nbsp;m<font size=-1>M</font> NaCl, 1&nbsp;m<font size=-1>M</font> EDTA, 0.5&nbsp;m<font size=-1>M</font> EGTA, 1&nbsp;m<font size=-1>M</font>
dithiothreitol,<SUP> </SUP>7.5&nbsp;m<font size=-1>M</font> MgCl<SUB>2</SUB>, 5% glycerol, 1&nbsp;m<font size=-1>M</font> Na<SUB>3</SUB>VO<SUB>4</SUB>, 0.5% bovine serum
albumin,<SUP> </SUP>and 1% Nonidet P-40 (Buffer C) at 4&nbsp;&#176;C for 2&nbsp;h with
gentle mixing.<SUP> </SUP>In competition assays using HisTrxMTBD</A> or HisTrx</A>
proteins, 0.4<SUP>&nbsp;</SUP>&#181;g (80&nbsp;n<font size=-1>M</font>) of rMAP2c</A> was mixed with
increasing amounts of HisTrxMTBD</A><SUP> </SUP>or HisTrx</A> protein (0.2-6
&#181;<font size=-1>M</font>) and incubated with 0.8&nbsp;&#181;l of the<SUP> </SUP>50% slurry of
protein bound glutathione-Sepharose beads in a final<SUP> </SUP>volume of 100&nbsp;&#181;l. In the binding assays in which rat brain or<SUP> </SUP>transfected COS-7
cell lysates were used, 0.3-1.0 mg of total<SUP> </SUP>lysate protein were
incubated with 40&nbsp;&#181;l of a 50% slurry of protein<SUP> </SUP>bound
glutathione-Sepharose beads in a final volume of 500&nbsp;&#181;l.<SUP> </SUP>The beads
were collected by centrifugation, washed twice with<SUP> </SUP>1.3&nbsp;ml of Buffer C
containing 150&nbsp;m<font size=-1>M</font> NaCl, and washed a third<SUP> </SUP>time with Buffer
C at 150&nbsp;m<font size=-1>M</font> NaCl but minus Nonidet P-40. The<SUP> </SUP>presence of
rMAP2c</A> bound on the beads was analyzed by SDS-PAGE,<SUP> </SUP>followed by
immunoblotting using monoclonal anti-MAP2 antibody<SUP> </SUP>HM2 (1:1000). The
presence of histidine-tagged protein was detected<SUP> </SUP>by monoclonal
anti-polyhistidine antibody HIS-1 (1:3000).



    

    


<P><I>Cell Cultures, Transfection, and Immunofluorescence
Microscopy--</I>

COS-7 cells were maintained in Dulbecco's modified
Eagle's medium supplemented with 10% fetal bovine serum, 2&nbsp;m<font size=-1>M</font> <font size=-1>L</font>-glutamine,<SUP> </SUP>and 2&nbsp;m<font size=-1>M</font> pyruvate
at 37&nbsp;&#176;C in a humidified 5% CO<SUB>2</SUB> atmosphere.<SUP> </SUP>When the
cultures reached approximately 80% confluency, they were<SUP> </SUP>transfected
with 8.5&nbsp;&#181;g of pEGFP-N1-MAP2c plasmid in the presence<SUP> </SUP>of 42&nbsp;&#181;l of
Superfect reagent for 3&nbsp;h at 37&nbsp;&#176;C according to the<SUP> </SUP>manufacturer's protocol. Approximately 60-70 h posttransfection,<SUP> </SUP>cells were rinsed once in ice-cold 1&#215; PBS</A> and harvested in ice-cold<SUP> </SUP>lysis buffer (150&nbsp;m<font size=-1>M</font> NaCl, 50&nbsp;m<font size=-1>M</font> Tris-HCl, pH
7.5,&nbsp;0.5% sodium<SUP> </SUP>deoxycholate, 1% Nonidet P-40, 1&nbsp;m<font size=-1>M</font>
Na<SUB>3</SUB>VO<SUB>4</SUB>, 1&nbsp;m<font size=-1>M</font> PMSF</A>, 1&nbsp;m<font size=-1>M</font>
NaF,<SUP> </SUP>10&nbsp;&#181;g/ml aprotinin, 10&nbsp;&#181;g/ml pepstatin A, and 10&nbsp;&#181;g/ml
leupeptin).<SUP> </SUP>A pulse of sonication (Fisher Sonifer, setting 6,&nbsp;5&nbsp;s)
was given<SUP> </SUP>to ensure complete cell lysis, and the lysate was centrifuged<SUP> </SUP>at 13,500&nbsp;&#215;&nbsp;<I>g</I> for 30&nbsp;min at 4&nbsp;&#176;C. Expression of the
rMAP2c-GFP<SUP> </SUP>fusion protein in the transfected COS-7 cells was verified
by<SUP> </SUP>immunoblot analysis using monoclonal anti-MAP2 antibody HM2. The<SUP> </SUP>clarified lysate was used in the GST-SH3 binding<SUP> </SUP>assay.
<P>
Preparation of dissociated rat hippocampal cultures from E18 rat
embryos, and growth and maintenance of these cultures were<SUP> </SUP>performed as
described (<A HREF="#B43">43</A>). At 15&nbsp;days <I>in vitro</I>, the cultures<SUP> </SUP>were
double-stained for MAP2</A> (HM2, 1:1000) and Src (sc-18, 1:1000),<SUP> </SUP>or MAP2</A>
and Grb2 (sc-255, 1:1000) as described (<A HREF="#B43">43</A>) using Alexa-568<SUP> </SUP>conjugated
goat anti-rabbit IgG (1:1000) and Alexa-488 conjugated<SUP> </SUP>goat anti-mouse
IgG (1:1000) as secondary antibodies. Digital<SUP> </SUP>images were obtained
using a 60&#215; oil immersion objective on an<SUP> </SUP>Olympus IX-70 microscope
coupled with a Photometrix PXL cooled<SUP> </SUP>CCD camera (<A HREF="#B43">43</A>).



    

    


<P><I>Preparation of Rat Brain Lysates--</I>

Adult rat forebrain or
hippocampus and neonatal (postnatal day 1&nbsp;(P1)) brain tissue were lysed
in 50&nbsp;m<font size=-1>M</font> Tris-HCl, pH 7.5,<SUP>&nbsp;</SUP>50&nbsp;m<font size=-1>M</font> NaCl, 50&nbsp;m<font size=-1>M</font> NaF, 5&nbsp;m<font size=-1>M</font> Na<SUB>3</SUB>VO<SUB>4</SUB>, 5&nbsp;m<font size=-1>M</font> EGTA, 5&nbsp;m<font size=-1>M</font> EDTA, 1&nbsp;m<font size=-1>M</font><SUP> </SUP>dithiothreitol, 1% Nonidet P-40, 1&nbsp;m<font size=-1>M</font> PMSF</A>, 10&nbsp;&#181;g/ml
aprotinin,<SUP> </SUP>10&nbsp;&#181;g/ml pepstatin A, and 10&nbsp;&#181;g/ml leupeptin at 4&nbsp;&#176;C.
The homogenates<SUP> </SUP>were sonicated to ensure complete lysis and centrifuged
as described<SUP> </SUP>above. The clarified brain lysates were used in the
GST-SH3 binding<SUP> </SUP>and co-immunoprecipitation<SUP> </SUP>assays.



    

    


<P><I>Co-immunoprecipitation of MAP2</A> with Src or Grb2 from Rat Brain
Lysate--</I>

Clarified brain lysates (1&nbsp;mg of total protein) were
incubated with 25&nbsp;&#181;g of rabbit anti-Src (sc-18), rabbit anti-Grb2
(sc-255),<SUP> </SUP>rabbit anti-MAP2 (no.
4170),<A HREF="#FN2"><SUP>2</SUP></A> irrelevant IgG,
rabbit preimmune serum (no. 4170pis), or lysis<SUP> </SUP>buffer alone in a final
volume of 400&nbsp;&#181;l at 4&nbsp;&#176;C overnight with<SUP> </SUP>gentle mixing. A 100-&#181;l
aliquot of a 50% slurry of protein A-Sepharose<SUP> </SUP>beads in 1&#215; PBS</A> and
1% bovine serum albumin was added, and the<SUP> </SUP>mixture was incubated at
4&nbsp;&#176;C for 1&nbsp;h with gentle agitation. The<SUP> </SUP>beads were collected by
centrifugation at 4000&nbsp;&#215;&nbsp;<I>g</I> for 5&nbsp;min,<SUP> </SUP>washed six times
briefly with 1&nbsp;ml of 1&#215; PBS</A>, boiled in Laemmli<SUP> </SUP>SDS-sample buffer for 2&nbsp;min (<A HREF="#B44">44</A>), and analyzed by SDS-PAGE and<SUP> </SUP>immunoblotting.



    

    


<P><I>In Vitro Phosphorylation of rMAP2c--</I>

Phosphorylation
reactions were carried out in a 50-&#181;l volume containing 50&nbsp;m<font size=-1>M</font> HEPES, pH 7.4,&nbsp;10&nbsp;m<font size=-1>M</font> MgCl<SUB>2</SUB>, 1&nbsp;m<font size=-1>M</font> EDTA, 1&nbsp;m<font size=-1>M</font><SUP> </SUP>EGTA, 2&nbsp;m<font size=-1>M</font>
dithiothreitol, 0.1&nbsp;m<font size=-1>M</font> Na<SUB>3</SUB>VO<SUB>4</SUB>, 1&nbsp;m<font size=-1>M</font> PMSF</A>, 1&nbsp;m<font size=-1>M</font> benzamidine,<SUP> </SUP>1&nbsp;m<font size=-1>M</font>
benzamide, 5&nbsp;m<font size=-1>M</font> <IMG SRC="/math/12pt/normal/epsilon.gif" ALIGN=BASELINE ALT="epsilon ">-caproic acid, 15&nbsp;&#181;g of rMAP2c</A>, and 45&nbsp;units<SUP> </SUP>of either PKA</A> or MAPK</A> (p42, extracellular signal-regulated kinase<SUP> </SUP>2) or 12&nbsp;units of GSK-3<IMG SRC="/math/12pt/normal/beta.gif" ALIGN=BASELINE ALT="beta ">, and 0.8&nbsp;&#181;<font size=-1>M</font> ATP with or
without 0.5&nbsp;&#181;Ci<SUP> </SUP>of [<IMG SRC="/math/12pt/normal/gamma.gif" ALIGN=BASELINE ALT="gamma ">-<SUP>32</SUP>P]ATP (6000&nbsp;Ci/mmol, NEN Life
Science Products) at 30&nbsp;&#176;C. At the<SUP> </SUP>indicated times, 13&nbsp;&#181;l were
removed and boiled at 95&nbsp;&#176;C for 5<SUP>&nbsp;</SUP>min to terminate the reaction.
Samples were incubated on ice for<SUP> </SUP>10&nbsp;min and centrifuged at 13,500&nbsp;&#215;&nbsp;<I>g</I> for 15&nbsp;min, and the supernatant<SUP> </SUP>collected was used in
the GST-SH3 binding assay described above.<SUP> </SUP>Control reactions were
carried out in the absence of added<SUP> </SUP>ATP.



    

    


<P><I>Microtubule Binding Assay--</I>

Microtubules were polymerized
from tubulin at a concentration of 5&nbsp;mg/ml in assembly buffer (20&nbsp;m<font size=-1>M</font> NaPO<SUB>4</SUB>, 100&nbsp;m<font size=-1>M</font>
<font size=-1>L</font>-glutamate,<SUP> </SUP>1&nbsp;m<font size=-1>M</font> EGTA, 1&nbsp;m<font size=-1>M</font>
MgCl<SUB>2</SUB>, and 1&nbsp;m<font size=-1>M</font> GTP) and stabilized by 20&nbsp;&#181;<font size=-1>M</font> Taxol<SUP> </SUP>as described (<A HREF="#B45">45</A>). Binding assay was performed
according to<SUP> </SUP>Ref. <A HREF="#B46">46</A> using 1&nbsp;&#181;<font size=-1>M</font> full-length rMAP2c</A> and 5&nbsp;&#181;<font size=-1>M</font> tubulin in an<SUP> </SUP>assay volume of 50&nbsp;&#181;l. The SDS-PAGE
gels were stained with Sypro<SUP> </SUP>Orange, and the fluorescence was
quantified on a FluorImager 595<SUP>&nbsp;</SUP>(Molecular Dynamics, Sunnyvale, CA). In
assays in which GST</A>, GST-Src-SH3,<SUP> </SUP>or Grb2-N-SH3 was added to the assay,
a 10&nbsp;&#181;<font size=-1>M</font> concentration of<SUP> </SUP>these proteins were used. Data
are expressed as a percentage of<SUP> </SUP>rMAP2c</A> found in the microtubule pellet
fraction: (rMAP2c</A> in pellet)/(rMAP2c</A><SUP> </SUP>in pellet&nbsp;+&nbsp;supernatant)&nbsp;&#215;&nbsp;100.&nbsp;One-way analysis of variance was<SUP> </SUP>used to assess statistical<SUP> </SUP>significance.



    

    


<P><I>Protein Analytical Methods--</I>

Protein concentrations were
determined by the BCA method (<A HREF="#B47">47</A>) using bovine serum albumin as the
protein standard. Protein<SUP> </SUP>samples were analyzed by SDS-PAGE according
to Ref. <A HREF="#B44">44</A> and visualized<SUP> </SUP>by Coomassie Brilliant Blue or Sypro Orange
staining. Electrotransfer<SUP> </SUP>and immunoblot analyses were performed as
described (<A HREF="#B48">48</A>) using<SUP> </SUP>the respective primary antibodies and alkaline
phosphatase-conjugated<SUP> </SUP>or HRP-conjugated secondary antibodies. When
alkaline phosphatase-conjugated<SUP> </SUP>antibodies were used, the
5-bromo-4-chloro-3-indolyl phosphate/nitro<SUP> </SUP>blue tetrazolium
colorimetric detection method was used. When<SUP> </SUP>HRP-conjugated antibodies
were used, the chemiluminescence detection<SUP> </SUP>method was<SUP> </SUP>used.



    


    


    
    


<BR><BR CLEAR=ALL>


<A NAME="SEC2"><!-- null --></A>
<TABLE WIDTH=100% BGCOLOR=E1E1E1 CELLPADDING=0 CELLSPACING=0>
<TR>
<TD ALIGN=LEFT VALIGN=MIDDLE WIDTH=5% BGCOLOR=FFFFFF>&nbsp;<IMG WIDTH=10 HEIGHT=21 HSPACE=5 ALT="" SRC="/icons/toc/rarrow.gif"></TD>
<TH ALIGN=LEFT VALIGN=MIDDLE WIDTH=95%><FONT SIZE=+2>&nbsp;&nbsp;RESULTS</FONT></TH>
</TR>
</TABLE>

<TABLE ALIGN=RIGHT CELLPADDING=5 BORDER NOWRAP><R>
<TH ALIGN=LEFT>
<FONT SIZE=-1>



<A HREF="#Top"><IMG WIDTH=11 HEIGHT=9 BORDER=0 HSPACE=5 SRC="/icons/toc/uarrow.gif">TOP</A>


<BR>
<A HREF="#Abstract"><IMG WIDTH=11 HEIGHT=9 BORDER=0 HSPACE=5 SRC="/icons/toc/uarrow.gif">ABSTRACT</A>


<BR>
<A HREF="#Introduction"><IMG WIDTH=11 HEIGHT=9 BORDER=0 HSPACE=5 SRC="/icons/toc/uarrow.gif">INTRODUCTION</A>


<BR>
<A HREF="#SEC1"><IMG WIDTH=11 HEIGHT=9 BORDER=0 HSPACE=5 SRC="/icons/toc/uarrow.gif">EXPERIMENTAL PROCEDURES</A>


<BR><IMG WIDTH=11 HEIGHT=9 BORDER=0 HSPACE=5 SRC="/icons/toc/dot.gif"><FONT COLOR=46
4C53>RESULTS</FONT>


<BR>
<A HREF="#SEC3"><IMG WIDTH=11 HEIGHT=9 BORDER=0 HSPACE=5 SRC="/icons/toc/darrow.gif">DISCUSSION</A>


<BR>
<A HREF="#References"><IMG WIDTH=11 HEIGHT=9 BORDER=0 HSPACE=5 SRC="/icons/toc/darrow.gif">REFERENCES</A>
</FONT></TH>
</TR></TABLE>


    



    



    


<P><I>Recombinant MAP2c Interacts Specifically with SH3 Domains of Src
and Grb2--</I>

We hypothesized that other cellular proteins interact
with MAP2</A> in addition to those that have been reported thus far.
Analysis<SUP> </SUP>of the amino acid sequence of rat MAP2c, the smallest known
isoform<SUP> </SUP>of MAP2</A> (also known as juvenile MAP2</A> or embryonic MAP2</A>, due to<SUP> </SUP>its perinatal expression pattern), revealed 13&nbsp;P<I>XX</I>P motifs
distributed<SUP> </SUP>in several clusters in the amino-terminal two-thirds of the
molecule.<SUP> </SUP>These polyproline clusters lie within regions of unknown
function.<SUP> </SUP>Eleven of these P<I>XX</I>P motifs are conserved across
species and across<SUP> </SUP>alternatively spliced isoforms of MAP2</A>. Because
P<I>XX</I>P is a known<SUP> </SUP>consensus motif of SH3 domain ligands (<A HREF="#B49">49</A>),
we investigated<SUP> </SUP>the possibility that MAP2c would interact with a
variety of SH3<SUP> </SUP>domain-containing<SUP> </SUP>proteins.
<P>
Using a GST-SH3 binding assay, we tested the ability of rMAP2c</A> protein
in crude bacterial lysate to co-precipitate with SH3<SUP> </SUP>domains of several
candidate proteins of varying cellular functions<SUP> </SUP>(Fig.
<a href="#F1">1</a>). In this assay, the GST</A> protein alone
did not precipitate<SUP> </SUP>any detectable amount of rMAP2c</A>. However, rMAP2c</A>
reliably co-precipitated<SUP> </SUP>together with GST-SH3 fusion proteins derived
from the protein<SUP> </SUP>tyrosine kinase Src and the adaptor protein Grb2 (Fig.
<a href="#F1">1</a><I>A</I>). In<SUP> </SUP>contrast, the SH3 domains of the protein tyrosine
kinase Abl,<SUP> </SUP>the adaptor protein Nck, and the GTPase activating protein
GAP<SUP> </SUP>exhibited little or no detectable binding to rMAP2c</A>. Although<SUP> </SUP>the
fusion protein containing the second of the three SH3 domains<SUP> </SUP>in Nck,
GST-Nck-SH3(2), reproducibly precipitated trace amounts<SUP> </SUP>of rMAP2c</A>,
neither the fusion protein containing the first SH3<SUP> </SUP>domain Nck-SH3(1)
nor that containing both the second and third<SUP> </SUP>SH3 domains of Nck,
Nck-SH3(2&nbsp;+&nbsp;3), produced detectable binding<SUP> </SUP>with rMAP2c</A>. This suggests
that perhaps the second SH3 domain<SUP> </SUP>of Nck was conformationally
inaccessible in the larger Nck-SH3(2<SUP>&nbsp;</SUP>+&nbsp;3) molecule. In addition to the
intact Grb2, the SH3 domains<SUP> </SUP>located at the amino and carboxyl termini
of Grb2 were analyzed<SUP> </SUP>independently. Grb2-N-SH3 exhibited stronger
binding than Grb2-C-SH3<SUP> </SUP>and may well contribute to the majority of the
SH3 domain interaction<SUP> </SUP>observed for the intact Grb2.



<BR CLEAR=ALL><A NAME="F1"><!-- FIG 1 --></A>

<P><CENTER><TABLE BORDER WIDTH=95% CELLPADDING=0 CELLSPACING=0><TR BGCOLOR=E1E1E1><TD>
<TABLE CELLPADDING=2 CELLSPACING=2><TR BGCOLOR=E1E1E1>
<TD ALIGN=CENTER VALIGN=TOP BGCOLOR=FFFFFF>

<A HREF="/cgi/content/full/275/27/20578/F1">
<IMG HSPACE=10 VSPACE=5 BORDER=2 WIDTH=200 HEIGHT=119 SRC="/content/vol275/issue27/images/small/bc2606780001.gif"></A><BR>

<STRONG>View larger version</STRONG> (39K):
<BR><NOBR><A HREF="/cgi/content/full/275/27/20578/F1">[in this window]</A>
<BR><A HREF="/cgi/content-nw/full/275/27/20578/F1"
onClick="startTarget('F1', 590, 461); this.href='/cgi/content-nw/full/275/27/20578/F1'"
    onMouseOver="window.status='View figure in a separate window'; return true"
    TARGET="F1">[in a new window]</A><BR>&nbsp;
</NOBR>
</TD><TD ALIGN=LEFT VALIGN=TOP>

<STRONG>Fig. 1. &nbsp;</STRONG>

<B>Bacterially expressed recombinant MAP2c
interacts specifically with the SH3 domains of Src and Grb2.</B>
<I>In vitro</I> GST-SH3 binding assay in which non-heat-denatured
rMAP2c</A> bacterial lysate was incubated with GST</A> or various GST-SH3
fusion proteins bound to glutathione-Sepharose beads and analyzed by
SDS-PAGE as described under "Experimental Procedures."
<I>A,</I> nitrocellulose immunoblot probed with anti-MAP2 antibody
HM2 and detected by colorimetric reaction. <I>B,</I> Coomassie
Brilliant Blue-stained 12% SDS-PAGE gel. Positions of rMAP2c</A>, GST</A>, and
assorted GST-SH3 fusion proteins are indicated at the <I>right</I>.
Positions of molecular size markers (in kDa) are indicated at the
<I>left</I>.


<BR CLEAR=LEFT>

</TD></TR></TABLE>
</TD></TR></TABLE>
</CENTER>
<!-- /FIG 1 -->




<P>
Both the recombinant and native forms of MAP2</A> are heat-stable and
retain their microtubule binding properties after being<SUP> </SUP>subjected to
high temperature (90-105&nbsp;&#176;C; 10&nbsp;min) in the presence<SUP> </SUP>of high salt,
a step often used in the purification of MAP2</A> and<SUP> </SUP>other heat-stable
microtubule-associated proteins (<A HREF="#B50">50</A>). As with<SUP> </SUP>microtubule binding, heat
treatment did not abolish the ability<SUP> </SUP>of rMAP2c</A> to interact with SH3
domains. The same SH3 domain selectivity<SUP> </SUP>profile was observed for
rMAP2c</A> partially purified from bacterial<SUP> </SUP>lysates via a high salt
boiling step and also for the subsequently<SUP> </SUP>SP-purified rMAP2c</A> (data not
shown), suggesting that SH3 binding<SUP> </SUP>by MAP2c was not dependent on
heat-labile secondary<SUP> </SUP>structure.



    

    


<P><I>MAP2</A> Isoforms Expressed in Mammalian Cells Interact with SH3
Domains of Src and Grb2--</I>

Among the SH3 domains tested, we observed
that the Src and Grb2 SH3 domains gave specific and strongest binding
signals for<SUP> </SUP>rMAP2c</A> in our GST-SH3 binding assay. We therefore proceeded
to<SUP> </SUP>analyze whether native isoforms of MAP2</A> expressed in eukaryotic<SUP> </SUP>cells behaved likewise. Recombinant MAP2c-GFP expressed in transfected<SUP> </SUP>COS-7 cells as well as both the high and low molecular weight<SUP> </SUP>isoforms
of MAP2</A> present in rat brain lysate specifically bound<SUP> </SUP>the SH3 domains
of Src and Grb2 (Fig. <a href="#F2">2</a>). In transfected
COS-7<SUP> </SUP>cells, the rMAP2c-GFP fusion protein was highly expressed and<SUP> </SUP>was
detected as a doublet or a triplet of ~90-95 kDa (Fig.
<a href="#F2">2</a><I>A</I>),<SUP> </SUP>indicative of post-translational modification of the
GFP</A> fusion<SUP> </SUP>protein in COS-7 cells. These posttranslationally modified
rMAP2c-GFPs<SUP> </SUP>from COS-7 cell lysate bound specifically to GST-Src-SH3
and GST-Grb2-N-SH3<SUP> </SUP>(Fig. <a href="#F2">2</a><I>A</I>). Adult rat brains express
predominantly the high molecular<SUP> </SUP>weight isoforms MAP2a and MAP2b, and
both of these isoforms were<SUP> </SUP>detected bound to GST-Src-SH3 and
GST-Grb2-N-SH3 (Fig. <a href="#F2">2</a><I>B</I>). Insignificant<SUP> </SUP>trace amounts of
MAP2a and MAP2b were detected in GST</A> alone control<SUP> </SUP>conditions.
Similarly, the high molecular weight isoform MAP2b<SUP> </SUP>and the low
molecular weight isoform MAP2c from neonatal rat brain<SUP> </SUP>lysate bound to
GST-Src-SH3, GST-Grb2, GST-Grb2-N-SH3, and GST-Grb2-C-SH3<SUP> </SUP>but not to
GST</A> (Fig. <a href="#F2">2</a><I>C</I>).



<BR CLEAR=ALL><A NAME="F2"><!-- FIG 2 --></A>

<P><CENTER><TABLE BORDER WIDTH=95% CELLPADDING=0 CELLSPACING=0><TR BGCOLOR=E1E1E1><TD>
<TABLE CELLPADDING=2 CELLSPACING=2><TR BGCOLOR=E1E1E1>
<TD ALIGN=CENTER VALIGN=TOP BGCOLOR=FFFFFF>

<A HREF="/cgi/content/full/275/27/20578/F2">
<IMG HSPACE=10 VSPACE=5 BORDER=2 WIDTH=121 HEIGHT=200 SRC="/content/vol275/issue27/images/small/bc2606780002.gif"></A><BR>

<STRONG>View larger version</STRONG> (33K):
<BR><NOBR><A HREF="/cgi/content/full/275/27/20578/F2">[in this window]</A>
<BR><A HREF="/cgi/content-nw/full/275/27/20578/F2"
onClick="startTarget('F2', 417, 640); this.href='/cgi/content-nw/full/275/27/20578/F2'"
    onMouseOver="window.status='View figure in a separate window'; return true"
    TARGET="F2">[in a new window]</A><BR>&nbsp;
</NOBR>
</TD><TD ALIGN=LEFT VALIGN=TOP>

<STRONG>Fig. 2. &nbsp;</STRONG>

<B>Recombinant MAP2c-GFP fusion protein
expressed from transiently transfected COS-7 cells and native </B><B>MAP2</B></A><B>
isoforms from rat brain interact specifically with the SH3 domains of
Src and Grb2.</B> <I>In vitro</I> GST-SH3 binding assays were
performed with transfected COS-7 cell lysates (<I>A</I>), adult rat
hippocampal lysate (<I>B</I>), and neonatal rat brain lysate
(<I>C</I>), as described under "Experimental Procedures." The
immunoblots were stained with anti-MAP2 antibody HM2 and detected by
colorimetric reaction. Positions of rMAP2c-GFP, various rat brain
isoforms of MAP2</A> (MAP2a, MAP2b, and MAP2c) are indicated at the
<I>left</I>. Positions of molecular size markers (in kDa) are
indicated at the <I>right</I>.


<BR CLEAR=LEFT>

</TD></TR></TABLE>
</TD></TR></TABLE>
</CENTER>
<!-- /FIG 2 -->







    

    


<P><I>Co-immunoprecipitation of Endogenous MAP2</A> with Endogenous Src or
Grb2--</I>

Results obtained from the <I>in vitro</I> GST-SH3
binding assay above strongly suggest that the SH3 domains of Src and
Grb2 could<SUP> </SUP>mediate direct association with MAP2</A>. We next asked whether
endogenous<SUP> </SUP>Src or Grb2 also bind to native MAP2</A> by performing
co-immunoprecipitation<SUP> </SUP>experiments (Fig.
<a href="#F3">3</a>). Under non-denaturing conditions,
antibodies<SUP> </SUP>to Src or Grb2 precipitated MAP2</A> isoforms in the immune
complexes.<SUP> </SUP>A fraction of MAP2</A> in the adult and neonatal rat brains
associated<SUP> </SUP>with Src or Grb2 and was found in the immunoprecipitates
(Fig.<SUP> </SUP><a href="#F3">3</a><I>A, lanes 3,&nbsp;4,&nbsp;7,</I>&nbsp;and <I>8</I>). No MAP2</A> was
detected in a control immunoprecipitation<SUP> </SUP>experiment using an
irrelevant IgG (Fig. <a href="#F3">3</a><I>A, lane 5</I>). Similarly,<SUP> </SUP>no detectable
MAP2</A> was bound on the protein A-Sepharose beads<SUP> </SUP>alone (Fig. <a href="#F3">3</a><I>A,
lane 6</I>). In the reciprocal co-immunoprecipitation<SUP> </SUP>experiments
using rabbit anti-MAP2 antibody (no. 4170), Grb2 was<SUP> </SUP>found
co-precipitating with MAP2</A> in the immune complex (Fig. <a href="#F3">3</a><I>B,<SUP> </SUP>lane
2</I>). Preimmune serum IgG precipitated no MAP2</A> and a negligible<SUP> </SUP>amount of Grb2 (Fig. <a href="#F3">3</a><I>B, lane 3</I>). A similar experiment to
detect<SUP> </SUP>co-precipitation of Src using anti-MAP2 antibodies was not
interpretable<SUP> </SUP>due to interference of cross-reacting IgG bands in the
vicinity<SUP> </SUP>of Src on the blot.



<BR CLEAR=ALL><A NAME="F3"><!-- FIG 3 --></A>

<P><CENTER><TABLE BORDER WIDTH=95% CELLPADDING=0 CELLSPACING=0><TR BGCOLOR=E1E1E1><TD>
<TABLE CELLPADDING=2 CELLSPACING=2><TR BGCOLOR=E1E1E1>
<TD ALIGN=CENTER VALIGN=TOP BGCOLOR=FFFFFF>

<A HREF="/cgi/content/full/275/27/20578/F3">
<IMG HSPACE=10 VSPACE=5 BORDER=2 WIDTH=185 HEIGHT=200 SRC="/content/vol275/issue27/images/small/bc2606780003.gif"></A><BR>

<STRONG>View larger version</STRONG> (60K):
<BR><NOBR><A HREF="/cgi/content/full/275/27/20578/F3">[in this window]</A>
<BR><A HREF="/cgi/content-nw/full/275/27/20578/F3"
onClick="startTarget('F3', 556, 640); this.href='/cgi/content-nw/full/275/27/20578/F3'"
    onMouseOver="window.status='View figure in a separate window'; return true"
    TARGET="F3">[in a new window]</A><BR>&nbsp;
</NOBR>
</TD><TD ALIGN=LEFT VALIGN=TOP>

<STRONG>Fig. 3. &nbsp;</STRONG>

<B>Endogenous Src and Grb2 interact with native
</B><B>MAP2</B></A><B> isoforms in rat brain.</B> The interactions of endogenous Src and
Grb2 with native MAP2</A> proteins in rat brains were assessed by
co-immunoprecipitation as described under "Experimental
Procedures." <I>A,</I> the immunoblot was stained for MAP2</A> using
anti-MAP2 antibody HM2 and detected by colorimetric reaction
(<I>lanes 1-6</I>) or by the chemiluminescence method (<I>lanes
7</I>&nbsp;and <I>8</I>). <I>Lane 1,</I>&nbsp;adult rat hippocampal
lysate (20&nbsp;&#181;g); <I>lane 2,</I>&nbsp;neonatal P1 brain lysate (20&nbsp;&#181;g); <I>lane 3,</I>&nbsp;IP</A> of hippocampal lysate with anti-Src
antibody sc-18; <I>lane 4,</I>&nbsp;IP</A> of P1 brain lysate with anti-Src
antibody sc-18; <I>lane 5,</I>&nbsp;IP</A> of hippocampal lysate with an
irrelevant IgG; <I>lane 6,</I>&nbsp;IP</A> of hippocampal lysate without
added antibody; <I>lane 7,</I>&nbsp;IP</A> of hippocampal lysate with rabbit
anti-Grb2 sc-255; <I>lane 8,</I>&nbsp;IP</A> of P1 brain lysate with
anti-Grb2 antibody sc-255. <I>B,</I> reciprocal
co-immunoprecipitation assay of rat forebrain lysate using rabbit
anti-MAP2 (no. 4170) antibody. Immune complexes were separated on a
4-12% gradient gel, electroblotted onto nitrocellulose membrane, and
immunoblotted. The blot was divided and probed separately for MAP2</A> or
Grb2. The <I>top blot</I> was stained for MAP2</A> using anti-MAP2
antibody HM2, and the <I>bottom blot</I> was stained for Grb2 using
mouse anti-Grb2 antibody. Both blots were detected by the
chemiluminescence method. <I>Lane 1,</I>&nbsp;adult forebrain lysate (30&nbsp;&#181;g); <I>lane 2,</I>&nbsp;IP</A> of adult forebrain lysate with rabbit
anti-MAP2 antibody (no. 4170); <I>lane 3,</I>&nbsp;IP</A> of adult forebrain
lysate with rabbit preimmune serum (no. 4170pis). Positions of MAP2</A>
isoforms and Grb2 are indicated at the <I>left</I>. Positions of
molecular size markers (in kDa) are indicated at the
<I>right</I>.


<BR CLEAR=LEFT>

</TD></TR></TABLE>
</TD></TR></TABLE>
</CENTER>
<!-- /FIG 3 -->







    

    


<P><I>Native MAP2</A> from Rat Brains Are Not Phosphorylated on Tyrosine
Residues--</I>

Because MAP2</A> from rat brain bound Src, we asked whether
endogenous MAP2</A> was a natural substrate of Src kinase activity.
<I>In<SUP> </SUP>vitro,</I> MAP2</A> is a substrate of the tyrosine kinases
insulin receptor<SUP> </SUP>kinase, epidermal growth factor receptor kinase, and
Src kinase<SUP> </SUP>(<A HREF="#B29">29-31</A>). Native MAP2</A> and rMAP2c-GFP were immunoprecipitated
from<SUP> </SUP>rat brain and transfected COS-7 cells lysates, respectively, in<SUP> </SUP>the presence of the tyrosine phosphatase inhibitor sodium vanadate.<SUP> </SUP>The
immune complexes were separated on SDS-PAGE gels, transferred<SUP> </SUP>to
nitrocellulose membranes, and immunoblotted with three different<SUP> </SUP>anti-phosphotyrosine antibodies: P-Tyr-100, 4G10, and PY20-derived<SUP> </SUP>RC20-HRP. No detectable signal above background was observed for<SUP> </SUP>any of
these three antibodies (data not shown), suggesting that<SUP> </SUP>under our
experimental conditions, native MAP2</A> is not a substrate<SUP> </SUP>of Src or other
tyrosine kinases <I>in vivo</I>. These results were in<SUP> </SUP>agreement
with experiments in which phosphoamino acid analyses<SUP> </SUP>failed to detect
phosphotyrosine in MAP2</A> isolated from <SUP>32</SUP>P-labeled rat
hippocampal slices (<A HREF="#B16">16</A>).



    

    


<P><I>MAP2</A>, Src, and Grb2 Are Co-expressed in the Dendrites of Cultured
Neurons--</I>

The results of the <I>in vitro</I> GST-SH3 binding
assays and the co-immunoprecipitation experiments strongly suggest that
MAP2</A> could<SUP> </SUP>indeed interact with Src and Grb2 <I>in vivo</I>. For
these interactions<SUP> </SUP>to occur in neurons, these proteins must be in close
proximity<SUP> </SUP>with each other. We proceeded to establish the distribution
and<SUP> </SUP>localization of these proteins in cultured rat hippocampal neurons.<SUP> </SUP>Neurons were double immunostained for MAP2</A> and Src or MAP2</A> and<SUP> </SUP>Grb2. As
described previously (<A HREF="#B51">51</A>), MAP2</A> was found exclusively<SUP> </SUP>in the
somatodendritic compartment and was absent from axons (Fig.<SUP> </SUP><a href="#F4">4</a>). Src immunoreactivity exhibited a
punctate distribution in<SUP> </SUP>the cell body, axons, and dendrites (Fig.
<a href="#F4">4</a><I>A</I>). Grb2 immunoreactivity<SUP> </SUP>was largely concentrated in the
cell body with modest levels throughout<SUP> </SUP>the dendrites; axons were
stained only weakly (Fig. <a href="#F4">4</a><I>B</I>). Thus,<SUP> </SUP>MAP2</A>, Src, and Grb2
were all present in neuronal cell bodies and<SUP> </SUP>dendrites, where they
could potentially engage in functional interactions.<SUP> </SUP>


<BR CLEAR=ALL><A NAME="F4"><!-- FIG 4 --></A>

<P><CENTER><TABLE BORDER WIDTH=95% CELLPADDING=0 CELLSPACING=0><TR BGCOLOR=E1E1E1><TD>
<TABLE CELLPADDING=2 CELLSPACING=2><TR BGCOLOR=E1E1E1>
<TD ALIGN=CENTER VALIGN=TOP BGCOLOR=FFFFFF>

<A HREF="/cgi/content/full/275/27/20578/F4">
<IMG HSPACE=10 VSPACE=5 BORDER=2 WIDTH=200 HEIGHT=153 SRC="/content/vol275/issue27/images/small/bc2606780004.gif"></A><BR>

<STRONG>View larger version</STRONG> (90K):
<BR><NOBR><A HREF="/cgi/content/full/275/27/20578/F4">[in this window]</A>
<BR><A HREF="/cgi/content-nw/full/275/27/20578/F4"
onClick="startTarget('F4', 590, 536); this.href='/cgi/content-nw/full/275/27/20578/F4'"
    onMouseOver="window.status='View figure in a separate window'; return true"
    TARGET="F4">[in a new window]</A><BR>&nbsp;
</NOBR>
</TD><TD ALIGN=LEFT VALIGN=TOP>

<STRONG>Fig. 4. &nbsp;</STRONG>

<B>MAP2</B></A><B>, Src, and Grb2 proteins are co-expressed
in the dendrites of cultured neurons.</B> Rat hippocampal neurons were
cultured at low density for 15&nbsp;days <I>in vitro</I> and
double-stained for MAP2</A> and Src or for MAP2</A> and Grb2, as described
under "Experimental Procedures." <I>A,</I> neuron
double-stained for Src with anti-Src antibody sc-18 (<I>left</I>)
and for MAP2</A> with anti-MAP2 antibody HM2 (<I>right</I>). Src
immunoreactivity is present throughout the cell body, axons, and
dendrites. Axons are labeled only weakly. <I>Arrowheads</I>
indicate processes that are positive for Src immunoreactivity but
negative for MAP2</A>, identifying them as axons. <I>B,</I> neuron
double-stained for Grb2 with anti-Grb2 antibody sc-255
(<I>left</I>) and for MAP2</A> with anti-MAP2 antibody HM2
(<I>right</I>). Grb2 immunoreactivity is strongest in the cell body
and moderate in the dendrites. The <I>scale bar</I> represents 20&nbsp;&#181;m.


<BR CLEAR=LEFT>

</TD></TR></TABLE>
</TD></TR></TABLE>
</CENTER>
<!-- /FIG 4 -->







    

    


<P><I>The MTBD</A> of rMAP2c</A> Binds Src-SH3 and Grb2-N-SH3 Domains--</I>

There
are 13&nbsp;potential SH3 domain binding P<I>XX</I>P motifs arranged in
clusters in the amino-terminal two-thirds of the rat MAP2c,<SUP> </SUP>and 11&nbsp;of
these P<I>XX</I>P motifs are conserved across species (Fig.<SUP> </SUP><a href="#F5">5</a><I>A</I>). To identify the Src-SH3
and Grb2-SH3 binding region in MAP2c,<SUP> </SUP>we created a series of
recombinant, non-fusion MAP2c proteins<SUP> </SUP>that have amino- or
carboxyl-terminal truncations or have internal<SUP> </SUP>deletions at the
polyproline-rich clusters (Fig. <a href="#F5">5</a><I>B</I>). These mutants<SUP> </SUP>were
assessed for SH3 domain binding activity by the GST-SH3 binding<SUP> </SUP>assay.
All mutant MAP2c proteins were expressed at high levels<SUP> </SUP>in expression
host bacteria (10-20% of total protein), remained<SUP> </SUP>primarily in the
soluble fraction, and were purified from the<SUP> </SUP>soluble fraction by the
high salt boiling method (<A HREF="#B1">1</A>) followed<SUP> </SUP>by SP</A> ion exchange
chromatography. The rMAP2c</A> in clarified bacterial<SUP> </SUP>lysates and
SP-purified rMAP2c</A> were used separately in the GST-SH3<SUP> </SUP>binding assays,
and both samples of rMAP2c</A> proteins gave essentially<SUP> </SUP>similar results.
Surprisingly, the carboxyl-terminally truncated<SUP> </SUP>mutants that contained
all 13&nbsp;potential SH3 domain binding P<I>XX</I>P<SUP> </SUP>motifs, 1-299 and
1-310, exhibited no detectable co-precipitation<SUP> </SUP>with GST-Src-SH3 or
GST-Grb2-N-SH3 in the GST</A> binding assay (Fig.<SUP> </SUP><a href="#F5">5</a><I>B</I>). In
contrast, the amino-terminally truncated mutants, 154-476<SUP> </SUP>and
296-467, efficiently co-precipitated with GST-Src-SH3 and<SUP> </SUP>GST-Grb2-N-SH3. The 296-467 MAP2c contains no P<I>XX</I>P motif.
These<SUP> </SUP>data suggest that binding of Src-SH3 and Grb2-N-SH3 domains by<SUP> </SUP>MAP2c did not involve P<I>XX</I>P motifs and that the binding
region<SUP> </SUP>resides elsewhere in the MAP2c molecule, possibly in the
carboxyl<SUP> </SUP>terminus. This hypothesis was verified using mutant MAP2c
proteins<SUP> </SUP>lacking specific polyproline-rich regions, amino acids
134-141,<SUP> </SUP>219-290, and 290-297. All the polyproline-rich deletion
mutants<SUP> </SUP>were still able to bind to GST-Src-SH3 and GST-Grb2-N-SH3, and<SUP> </SUP>the observed binding of the mutant rMAP2c</A> proteins was comparable<SUP> </SUP>to
that of the full-length rMAP2c</A> (Fig. <a href="#F5">5</a><I>B</I>). A candidate region<SUP> </SUP>in the carboxyl terminus of MAP2c that could contribute to SH3<SUP> </SUP>domain
binding is the MTBD</A>. A mutant <IMG SRC="/math/12pt/normal/Delta.gif" ALIGN=BASELINE ALT="Delta ">290-400 rMAP2c</A> lacking both<SUP> </SUP>the
polyproline-rich cluster amino acids 290-297 and the MTBD</A><SUP> </SUP>was
constructed and tested for SH3 domain binding. The <IMG SRC="/math/12pt/normal/Delta.gif" ALIGN=BASELINE ALT="Delta ">290-400<SUP> </SUP>rMAP2c</A>
exhibited no detectable co-precipitation with GST-Src-SH3<SUP> </SUP>and
GST-Grb2-N-SH3 in the GST-SH3 binding assay (Fig.
<a href="#F5">5</a><I>B</I>).



<BR CLEAR=ALL><A NAME="F5"><!-- FIG 5 --></A>

<P><CENTER><TABLE BORDER WIDTH=95% CELLPADDING=0 CELLSPACING=0><TR BGCOLOR=E1E1E1><TD>
<TABLE CELLPADDING=2 CELLSPACING=2><TR BGCOLOR=E1E1E1>
<TD ALIGN=CENTER VALIGN=TOP BGCOLOR=FFFFFF>

<A HREF="/cgi/content/full/275/27/20578/F5">
<IMG HSPACE=10 VSPACE=5 BORDER=2 WIDTH=151 HEIGHT=200 SRC="/content/vol275/issue27/images/small/bc2606780005.gif"></A><BR>

<STRONG>View larger version</STRONG> (34K):
<BR><NOBR><A HREF="/cgi/content/full/275/27/20578/F5">[in this window]</A>
<BR><A HREF="/cgi/content-nw/full/275/27/20578/F5"
onClick="startTarget('F5', 482, 640); this.href='/cgi/content-nw/full/275/27/20578/F5'"
    onMouseOver="window.status='View figure in a separate window'; return true"
    TARGET="F5">[in a new window]</A><BR>&nbsp;
</NOBR>
</TD><TD ALIGN=LEFT VALIGN=TOP>

<STRONG>Fig. 5. &nbsp;</STRONG>

<B>Mapping the region on MAP2c that binds
specifically to the SH3 domain of Src and Grb2.</B> <I>A,</I>
schematic diagram of MAP2c showing the polyproline-rich clusters in the
amino-terminal two-thirds of the molecule. The amino acid sequences of
the 11&nbsp;conserved SH3 domain-binding P<I>XX</I>P motifs are shown.
Other functional domains of MAP2c shown are the binding region for the
type II regulatory subunit (<I>RII</I>) of the
cAMP-dependent protein kinase (<I>black box</I>) near
the amino terminus (<A HREF="#B33">33</A>, <A HREF="#B34">34</A>), and MTBD</A> containing three imperfect
repeats (<I>gray boxes</I>) and inter-repeats near the carboxyl
terminus (<A HREF="#B7">7</A>). <I>B,</I> summary of the interactions of the
full-length and the various mutant rMAP2c</A> proteins with the SH3 domains
of Src and Grb2. The amino- or carboxyl-terminally truncated rMAP2c</A> are
denoted by the amino acid residue positions corresponding to the
specific region on the full-length MAP2c that are present in the mutant
proteins (<I>left</I>). Mutants with internal deletion are denoted
by the region of amino acid residues that are removed. A <I>positive
sign</I> (+) indicates that there was detectable rMAP2
co-precipitation in the GST</A> binding assay and that the amounts
co-precipitated were comparable to those observed for the full-length
rMAP2c</A>. A <I>negative sign</I> (-) indicates that there was no
detectable co-precipitation with GST-SH3 fusion proteins.
<I>ND,</I> not determined. Results were compiled from at least
three independent GST-SH3 binding assays using different batches of
SP-purified rMAP2c</A> proteins.


<BR CLEAR=LEFT>

</TD></TR></TABLE>
</TD></TR></TABLE>
</CENTER>
<!-- /FIG 5 -->




<P>
To further test that only the MTBD</A> in rMAP2c</A> contributes to Src-SH3 and
Grb2-SH3 domain binding, a recombinant MTBD</A>, amino<SUP> </SUP>acids 300-400,
expressed as a histidine-tagged thioredoxin fusion<SUP> </SUP>protein
(HisTrxMTBD</A>), was made. However, the leader fusion protein<SUP> </SUP>HisTrx</A>
consistently gave a background of non-specific binding<SUP> </SUP>to the GST-SH3
fusion proteins used in the GST-SH3 binding assay.<SUP> </SUP>To circumvent this
problem, the purified HisTrxMTBD</A> protein was<SUP> </SUP>used in competition with
the full-length rMAP2c</A> in the GST-SH3<SUP> </SUP>binding assay. If the MTBD</A> in
rMAP2c</A> specifically interacts with<SUP> </SUP>the SH3 domains of Src and Grb2, we
would expect a concentration-dependent<SUP> </SUP>inhibition of
full-length rMAP2c</A> binding to the above-mentioned<SUP> </SUP>SH3 domains in the
presence of excess HisTrxMTBD</A> but not in the<SUP> </SUP>presence of excess HisTrx</A>.
Increasing concentrations of HisTrxMTBD</A><SUP> </SUP>significantly reduced the
amount of full-length rMAP2c</A> co-precipitating<SUP> </SUP>with GST-Src-SH3 (Fig.
<a href="#F6">6</a><I>A</I>) or GST-Grb2-N-SH3 (Fig.
<a href="#F6">6</a><I>B</I>). At the<SUP> </SUP>same time, a corresponding increase of
HisTrxMTBD</A> in the pellet<SUP> </SUP>with GST-Src-SH3 was detected (Fig.
<a href="#F6">6</a><I>A</I>). Increasing amounts of<SUP> </SUP>HisTrx</A> had no effect on the
interaction of the full-length rMAP2c</A><SUP> </SUP>with the SH3 domains of Src (data
not shown) or Grb2 (Fig. <a href="#F6">6</a><I>C</I>)<SUP> </SUP>as the amount of full-length
rMAP2c</A> co-precipitated remained constant.<SUP> </SUP>Therefore, we conclude that
the Src-SH3 and Grb2-SH3 domain-binding<SUP> </SUP>region is localized to the
MTBD</A>, amino acids 300-400 of MAP2c.<SUP> </SUP>


<BR CLEAR=ALL><A NAME="F6"><!-- FIG 6 --></A>

<P><CENTER><TABLE BORDER WIDTH=95% CELLPADDING=0 CELLSPACING=0><TR BGCOLOR=E1E1E1><TD>
<TABLE CELLPADDING=2 CELLSPACING=2><TR BGCOLOR=E1E1E1>
<TD ALIGN=CENTER VALIGN=TOP BGCOLOR=FFFFFF>

<A HREF="/cgi/content/full/275/27/20578/F6">
<IMG HSPACE=10 VSPACE=5 BORDER=2 WIDTH=152 HEIGHT=200 SRC="/content/vol275/issue27/images/small/bc2606780006.gif"></A><BR>

<STRONG>View larger version</STRONG> (39K):
<BR><NOBR><A HREF="/cgi/content/full/275/27/20578/F6">[in this window]</A>
<BR><A HREF="/cgi/content-nw/full/275/27/20578/F6"
onClick="startTarget('F6', 485, 640); this.href='/cgi/content-nw/full/275/27/20578/F6'"
    onMouseOver="window.status='View figure in a separate window'; return true"
    TARGET="F6">[in a new window]</A><BR>&nbsp;
</NOBR>
</TD><TD ALIGN=LEFT VALIGN=TOP>

<STRONG>Fig. 6. &nbsp;</STRONG>

<B>MAP2</B></A><B> microtubule-binding domain binds the SH3
domain of Src and Grb2.</B> Recombinant MAP2c was incubated in the
absence or presence of an increasing amount HisTrxMTBD</A> or HisTrx</A>
proteins in the GST-SH3 binding assay. Proteins on the immunoblots were
detected by the chemiluminescence method. <I>A,</I> immunoblot of
proteins co-precipitated with GST-Src-SH3. The blot was divided and
probed separately for MAP2</A> using anti-MAP2 antibody HM2
(<I>top</I>) or stained for HisTrxMTBD</A> using anti-polyhistidine
HIS-1 (<I>bottom</I>). <I>B</I> and <I>C,</I> immunoblots
of proteins co-precipitated with GST-Grb2-N-SH3. Blots were stained for
MAP2</A> using anti-MAP2 antibody HM2.


<BR CLEAR=LEFT>

</TD></TR></TABLE>
</TD></TR></TABLE>
</CENTER>
<!-- /FIG 6 -->







    

    


<P><I>Src and Grb2 Bind Soluble Non-microtubule-bound MAP2--</I>

Because
microtubules, Src-SH3 domain, and Grb2-SH3 domain bind within the same
region in MAP2c, we were interested to know<SUP> </SUP>whether these associations
were mutually or non-mutually exclusive.<SUP> </SUP>MAP2c-microtubule binding
assays were performed in the absence<SUP> </SUP>and presence of 10-fold molar
excess of GST</A> or GST-Src-SH3. Because<SUP> </SUP>GST</A> is known to form dimers (<A HREF="#B52">52</A>,
<A HREF="#B53">53</A>), the dimeric GST-Src-SH3<SUP> </SUP>might be sterically excluded from the MTBD</A>
in the presence of<SUP> </SUP>microtubules. We attempted to obtained the Src-SH3
and Grb2-N-SH3<SUP> </SUP>fragments from the GST</A> fusion proteins by thrombin
cleavage for<SUP> </SUP>use in this assay. Only the GST-Grb2-N-SH3 protein was
digestible<SUP> </SUP>by thrombin to liberate a larger, 27-kDa GST</A> leader and a
smaller,<SUP> </SUP>6-kDa Grb2-N-SH3 protein. The thrombin cleavage site between
the<SUP> </SUP>GST</A> leader and Src-SH3 domain in GST-Src-SH3 fusion was presumed<SUP> </SUP>to
be eliminated during the construction of the expression vector.<SUP> </SUP>Under
the microtubule-binding experimental conditions, rMAP2c</A><SUP> </SUP>bound to
microtubules with an approximate stoichiometry of 1&nbsp;mol<SUP> </SUP>of rMAP2c</A> to 4&nbsp;mol of tubulin dimer. No significant difference<SUP> </SUP>in rMAP2c</A> binding to
microtubules was observed in the presence<SUP> </SUP>of excess GST</A>, GST-Src-SH3,
or Grb2-N-SH3 (<I>p</I>&nbsp;<IMG SRC="/math/12pt/normal/ge.gif" ALIGN=BASELINE ALT="&gt;= ">&nbsp;0.05; <I>n</I>&nbsp;=&nbsp;5;
one-way<SUP> </SUP>analysis of variance) (Fig. <a href="#F7">7</a>).
We further determined whether<SUP> </SUP>microtubule-bound rMAP2c</A> could mediate
GST-Src-SH3 co-precipitation<SUP> </SUP>in the presence of microtubules.
GST-Src-SH3 alone did not bind<SUP> </SUP>microtubules, and it did not
co-precipitate along with MAP2</A> in<SUP> </SUP>the microtubule pellet (data not
shown). These data indicate that<SUP> </SUP>microtubule binding and SH3 domain
binding are mutually exclusive<SUP> </SUP>and that rMAP2c</A> shows preference to
microtubule binding even in<SUP> </SUP>the presence of excess Src-SH3 or
Grb2-N-SH3 domains. Thus, Src<SUP> </SUP>or Grb2 would interact with soluble,
non-microtubule-associated<SUP> </SUP>MAP2</A>.



<BR CLEAR=ALL><A NAME="F7"><!-- FIG 7 --></A>

<P><CENTER><TABLE BORDER WIDTH=95% CELLPADDING=0 CELLSPACING=0><TR BGCOLOR=E1E1E1><TD>
<TABLE CELLPADDING=2 CELLSPACING=2><TR BGCOLOR=E1E1E1>
<TD ALIGN=CENTER VALIGN=TOP BGCOLOR=FFFFFF>

<A HREF="/cgi/content/full/275/27/20578/F7">
<IMG HSPACE=10 VSPACE=5 BORDER=2 WIDTH=200 HEIGHT=148 SRC="/content/vol275/issue27/images/small/bc2606780007.gif"></A><BR>

<STRONG>View larger version</STRONG> (14K):
<BR><NOBR><A HREF="/cgi/content/full/275/27/20578/F7">[in this window]</A>
<BR><A HREF="/cgi/content-nw/full/275/27/20578/F7"
onClick="startTarget('F7', 590, 525); this.href='/cgi/content-nw/full/275/27/20578/F7'"
    onMouseOver="window.status='View figure in a separate window'; return true"
    TARGET="F7">[in a new window]</A><BR>&nbsp;
</NOBR>
</TD><TD ALIGN=LEFT VALIGN=TOP>

<STRONG>Fig. 7. &nbsp;</STRONG>

<B>Recombinant MAP2c binds microtubules in the
presence of GST-Src-SH3 and Grb2-N-SH3.</B> The microtubule binding
activity of full-length rMAP2c</A> was determined in the absence and
presence of GST</A>, GST-Src-SH3, and Grb2-N-SH3 proteins as described
under "Experimental Procedures." The supernatant and pellet samples
were analyzed by SDS-PAGE. The separated proteins were stained with
Sypro Orange and quantified by fluorometry. The data are presented as a
percentage of microtubule-bound rMAP2c</A> over total rMAP2c</A>. There was no
significant difference in rMAP2c</A> binding to microtubules in the
presence of excess GST</A>, GST-Src-SH3, or Grb2-N-SH3 (<I>p</I>&nbsp;<IMG SRC="/math/12pt/normal/ge.gif" ALIGN=BASELINE ALT="&gt;= ">&nbsp;0.05; <I>n</I>&nbsp;=&nbsp;5; one-way analysis of variance).


<BR CLEAR=LEFT>

</TD></TR></TABLE>
</TD></TR></TABLE>
</CENTER>
<!-- /FIG 7 -->







    

    


<P><I>Phosphorylation of rMAP2c</A> by MAPK</A> Reduces Binding to Src and
Grb2--</I>

MAP2</A> proteins isolated from brain tissues are highly and
differentially phosphorylated (<A HREF="#B16">16</A>, <A HREF="#B54">54</A>, <A HREF="#B55">55</A>). Phosphorylation<SUP> </SUP>regulates
microtubule binding (<A HREF="#B56">56</A>), and activity-dependent
phosphorylation<SUP> </SUP>and dephosphorylation of MAP2</A> have been observed
(<A HREF="#B15">15-20</A>). Therefore,<SUP> </SUP>we tested the effects of phosphorylation on the
SH3 domain binding<SUP> </SUP>activity of MAP2</A>. The cAMP-dependent
protein kinase (PKA</A>) rapidly<SUP> </SUP>phosphorylated rMAP2c</A> within 1&nbsp;h
(Fig. <a href="#F8">8</a><I>A</I>). In contrast, the
mitogen-activated,<SUP> </SUP>proline-directed protein kinase MAPK</A> (extracellular
signal-regulated<SUP> </SUP>kinase 2,&nbsp;p42) achieved comparable phosphorylation of
rMAP2c</A> after<SUP> </SUP>8&nbsp;h under similar kinase reaction conditions <I>in
vitro</I> (Fig. <a href="#F8">8</a><I>A</I>).<SUP> </SUP>Under the conditions used, both
protein kinases maximally phosphorylate<SUP> </SUP>rMAP2c</A> to a stoichiometry of
approximately 3&nbsp;mol of
phosphate/mol.<A HREF="#FN3"><SUP>3</SUP></A> The
PKA-phosphorylated rMAP2c</A> bound SH3 domains of Src and Grb2.<SUP> </SUP>All rMAP2c</A>
incubated in the absence of ATP (as control-phosphorylated<SUP> </SUP>rMAP2c</A>) also
bound SH3 domains of Src and Grb2 (Fig. <a href="#F8">8</a>, <I>B</I> and<SUP> </SUP><I>C</I>). However, rMAP2c</A> phosphorylated by MAPK</A> for 8&nbsp;h
exhibited significantly<SUP> </SUP>reduced interaction with the SH3 domains of
both Src and Grb2<SUP> </SUP>(Fig. <a href="#F8">8</a><I>C</I>).



<BR CLEAR=ALL><A NAME="F8"><!-- FIG 8 --></A>

<P><CENTER><TABLE BORDER WIDTH=95% CELLPADDING=0 CELLSPACING=0><TR BGCOLOR=E1E1E1><TD>
<TABLE CELLPADDING=2 CELLSPACING=2><TR BGCOLOR=E1E1E1>
<TD ALIGN=CENTER VALIGN=TOP BGCOLOR=FFFFFF>

<A HREF="/cgi/content/full/275/27/20578/F8">
<IMG HSPACE=10 VSPACE=5 BORDER=2 WIDTH=85 HEIGHT=200 SRC="/content/vol275/issue27/images/small/bc2606780008.gif"></A><BR>

<STRONG>View larger version</STRONG> (33K):
<BR><NOBR><A HREF="/cgi/content/full/275/27/20578/F8">[in this window]</A>
<BR><A HREF="/cgi/content-nw/full/275/27/20578/F8"
onClick="startTarget('F8', 336, 640); this.href='/cgi/content-nw/full/275/27/20578/F8'"
    onMouseOver="window.status='View figure in a separate window'; return true"
    TARGET="F8">[in a new window]</A><BR>&nbsp;
</NOBR>
</TD><TD ALIGN=LEFT VALIGN=TOP>

<STRONG>Fig. 8. &nbsp;</STRONG>

<B>Phosphorylation by </B><B>MAPK</B></A><B> (extracellular
signal-regulated kinase 2,&nbsp;p42) regulates </B><B>rMAP2c</B></A><B> association with SH3
domains of Src and Grb2.</B> Recombinant MAP2c was incubated in the
absence (control) or presence of ATP and with either PKA</A> or MAPK</A> for
various times, as indicated, prior to use in the GST-SH3 binding.
Immunoblots were stained for MAP2c with anti-MAP2 antibody HM2 and
detected by colorimetric reaction. <I>A,</I> autoradiograph showing
time-dependent phosphorylation of rMAP2c</A> by PKA</A> or MAPK</A>.
<I>B,</I> immunoblots of phosphorylated and mock-phosphorylated
(control) rMAP2c</A> co-precipitated with GST-Src-SH3. <I>C,</I>
immunoblots of phosphorylated and control rMAP2c</A> co-precipitated with
GST-Grb2-N-SH3. Results in <I>B</I> and <I>C</I> are
representative of three independent experiments.


<BR CLEAR=LEFT>

</TD></TR></TABLE>
</TD></TR></TABLE>
</CENTER>
<!-- /FIG 8 -->




<P>
Because MAPK</A> is a proline-directed protein kinase, phosphorylation by
MAPK</A> would theoretically occur at serine and threonine<SUP> </SUP>residues that
are adjacent to proline residues. To investigate<SUP> </SUP>whether
phosphorylation near proline residues is involved in regulating<SUP> </SUP>MAP2/SH3 domain interactions, proline-rich deletion mutant rMAP2c</A><SUP> </SUP>proteins were phosphorylated by MAPK</A> for 8&nbsp;h and then tested for<SUP> </SUP>SH3 domain binding. The SH3 domain binding activities of all
proline-rich<SUP> </SUP>deletion mutant rMAP2c</A> proteins were inhibited by MAPK</A>
phosphorylation<SUP> </SUP>(data not shown). In addition, full-length rMAP2c</A> was
maximally<SUP> </SUP>phosphorylated using another proline-directed kinase GSK-3<IMG SRC="/math/12pt/normal/beta.gif" ALIGN=BASELINE ALT="beta ">
for<SUP> </SUP>24&nbsp;h and analyzed for Src-SH3 and Grb2-N-SH3 domain binding
activity.<SUP> </SUP>The GSK-3<IMG SRC="/math/12pt/normal/beta.gif" ALIGN=BASELINE ALT="beta ">-phosphorylated rMAP2c</A> demonstrated binding to
the selected<SUP> </SUP>SH3 domains that was comparable to that of the
full-length, non-phosphorylated<SUP> </SUP>rMAP2c</A> (data not shown). Together,
these data suggest that the<SUP> </SUP>addition of phosphate groups to regions
other than the proline-rich<SUP> </SUP>areas in MAP2c regulate the interaction
with SH3 domains. Key<SUP> </SUP>phosphorylation sites may lie within the MTBD/SH3
domain-binding<SUP> </SUP>region<SUP> </SUP>itself.
<P>
In MAP2c, the MTBD/SH3 domain-binding region contains three
K<I>X</I>GS motifs that are targets of multiple kinases.
Phosphorylation<SUP> </SUP>of the serine residues in the K<I>X</I>GS motifs
abolishes microtubule<SUP> </SUP>binding <I>in vitro</I> (<A HREF="#B22">22</A>). Furthermore,
full-length rMAP2c</A> containing<SUP> </SUP>serine to glutamate mutations in the
K<I>X</I>GS motifs exhibit greatly<SUP> </SUP>reduced microtubule binding
<I>in vivo</I>.<SUP>2</SUP> We tested the SH3 domain binding
activity of these mutants and<SUP> </SUP>found that they retained SH3 domain
binding activity <I>in vitro</I><SUP> </SUP>that was indistinguishable from
that of the full-length, wild<SUP> </SUP>type rMAP2c</A> (data not shown). This
observation indicates that<SUP> </SUP>phosphorylation at the K<I>X</I>GS
motifs in the MTBD</A> is unlikely to<SUP> </SUP>modulate the binding of Src-SH3 and
Grb2-SH3 domains and that<SUP> </SUP>other candidate residues must be<SUP> </SUP>considered.



    


    


    
    


<BR><BR CLEAR=ALL>


<A NAME="SEC3"><!-- null --></A>
<TABLE WIDTH=100% BGCOLOR=E1E1E1 CELLPADDING=0 CELLSPACING=0>
<TR>
<TD ALIGN=LEFT VALIGN=MIDDLE WIDTH=5% BGCOLOR=FFFFFF>&nbsp;<IMG WIDTH=10 HEIGHT=21 HSPACE=5 ALT="" SRC="/icons/toc/rarrow.gif"></TD>
<TH ALIGN=LEFT VALIGN=MIDDLE WIDTH=95%><FONT SIZE=+2>&nbsp;&nbsp;DISCUSSION</FONT></TH>
</TR>
</TABLE>

<TABLE ALIGN=RIGHT CELLPADDING=5 BORDER NOWRAP><R>
<TH ALIGN=LEFT>
<FONT SIZE=-1>



<A HREF="#Top"><IMG WIDTH=11 HEIGHT=9 BORDER=0 HSPACE=5 SRC="/icons/toc/uarrow.gif">TOP</A>


<BR>
<A HREF="#Abstract"><IMG WIDTH=11 HEIGHT=9 BORDER=0 HSPACE=5 SRC="/icons/toc/uarrow.gif">ABSTRACT</A>


<BR>
<A HREF="#Introduction"><IMG WIDTH=11 HEIGHT=9 BORDER=0 HSPACE=5 SRC="/icons/toc/uarrow.gif">INTRODUCTION</A>


<BR>
<A HREF="#SEC1"><IMG WIDTH=11 HEIGHT=9 BORDER=0 HSPACE=5 SRC="/icons/toc/uarrow.gif">EXPERIMENTAL PROCEDURES</A>


<BR>
<A HREF="#SEC2"><IMG WIDTH=11 HEIGHT=9 BORDER=0 HSPACE=5 SRC="/icons/toc/uarrow.gif">RESULTS</A>


<BR><IMG WIDTH=11 HEIGHT=9 BORDER=0 HSPACE=5 SRC="/icons/toc/dot.gif"><FONT COLOR=46
4C53>DISCUSSION</FONT>


<BR>
<A HREF="#References"><IMG WIDTH=11 HEIGHT=9 BORDER=0 HSPACE=5 SRC="/icons/toc/darrow.gif">REFERENCES</A>
</FONT></TH>
</TR></TABLE>


    
    <P>This study presents evidence of novel and direct interactions of
the intracellular signaling proteins Src and Grb2 with soluble,<SUP> </SUP>non-microtubule-bound MAP2</A> in neurons. These interactions were<SUP> </SUP>mediated
through the SH3 domains of Src and Grb2 and the MTBD</A><SUP> </SUP>of MAP2</A> and were
regulated by MAPK-dependent phosphorylation.<SUP> </SUP>Previous
studies showed that MAP2</A> is a substrate of MAPK</A> <I>in vivo</I><SUP> </SUP>(<A HREF="#B18">18</A>,
<A HREF="#B19">19</A>).
    
    </P>

    
    <P>SH3 domains are well documented protein-protein interaction modules
found in multiple signaling pathways in cells, and they<SUP> </SUP>mediate
formation of large multiprotein complexes (<A HREF="#B57">57</A>). MAP2c<SUP> </SUP>contains eleven
P<I>XX</I>P motifs that are conserved across species.<SUP> </SUP>The
P<I>XX</I>P motif has been demonstrated to be the minimum consensus<SUP> </SUP>ligand motif of SH3 domains (<A HREF="#B58">58</A>). Surprisingly, none of the<SUP> </SUP>P<I>XX</I>P motifs in MAP2c were involved in binding the SH3
domains<SUP> </SUP>of Src or Grb2 under the described experimental conditions.
Instead,<SUP> </SUP>the SH3 domain-binding region in MAP2c was localized to the
same<SUP> </SUP>region responsible for microtubule binding, the MTBD</A>, amino acids<SUP> </SUP>300-400. However, it seems likely that the binding of microtubules<SUP> </SUP><I>versus</I> SH3 domains to this region of MAP2</A> involve distinct
conformations,<SUP> </SUP>because serine to glutamate mutations in the
K<I>X</I>GS motifs that<SUP> </SUP>severely disrupt microtubule binding had no
effect on SH3 domain<SUP> </SUP>binding.
    
    </P>

    
    <P>The related neuronal microtubule-associated protein tau was also
reported to bind the Src family of protein tyrosine kinases,<SUP> </SUP>Fyn, Lck,
and Src, through SH3 domains of the kinases (<A HREF="#B8">8</A>).<SUP> </SUP>The Fyn-SH3
domain-binding site in tau was mapped to a proline-rich<SUP> </SUP>region of amino
acids 170-178, <SUP>170</SUP>VVRTPPKSP<SUP>178</SUP>, in the
352-amino acid isoform of human tau. This region accounted<SUP> </SUP>for ~90%
of the Fyn-SH3 domain binding activity in tau (<A HREF="#B8">8</A>).<SUP> </SUP>A similar sequence,
<SUP>286</SUP>IIRTPPKSP<SUP>294</SUP>, is also found in MAP2c (Fig.
<a href="#F9">9</a>). However, this sequence does<SUP> </SUP>not
appear to participate in binding the SH3 domains of Src and<SUP> </SUP>Grb2,
because the <IMG SRC="/math/12pt/normal/Delta.gif" ALIGN=BASELINE ALT="Delta ">290-297 rMAP2c</A> mutant bound to both Src-SH3<SUP> </SUP>and
Grb2-N-SH3, and the binding activities were indistinguishable<SUP> </SUP>from
those of the full-length rMAP2c</A> protein (Fig. <a href="#F5">5</a><I>B</I>). The amino<SUP> </SUP>acids in the vicinity of <SUP>286</SUP>IIRTPPKSP<SUP>294</SUP> in
MAP2c differ from those of tau (Fig. <a href="#F9">9</a>). It is possible that<SUP> </SUP>neighboring amino acid composition influences the binding specificity<SUP> </SUP>of Src in MAP2c <I>versus</I> tau.



<BR CLEAR=ALL><A NAME="F9"><!-- FIG 9 --></A>

<P><CENTER><TABLE BORDER WIDTH=95% CELLPADDING=0 CELLSPACING=0><TR BGCOLOR=E1E1E1><TD>
<TABLE CELLPADDING=2 CELLSPACING=2><TR BGCOLOR=E1E1E1>
<TD ALIGN=CENTER VALIGN=TOP BGCOLOR=FFFFFF>

<A HREF="/cgi/content/full/275/27/20578/F9">
<IMG HSPACE=10 VSPACE=5 BORDER=2 WIDTH=200 HEIGHT=145 SRC="/content/vol275/issue27/images/small/bc2606780009.gif"></A><BR>

<STRONG>View larger version</STRONG> (46K):
<BR><NOBR><A HREF="/cgi/content/full/275/27/20578/F9">[in this window]</A>
<BR><A HREF="/cgi/content-nw/full/275/27/20578/F9"
onClick="startTarget('F9', 590, 519); this.href='/cgi/content-nw/full/275/27/20578/F9'"
    onMouseOver="window.status='View figure in a separate window'; return true"
    TARGET="F9">[in a new window]</A><BR>&nbsp;
</NOBR>
</TD><TD ALIGN=LEFT VALIGN=TOP>

<STRONG>Fig. 9. &nbsp;</STRONG>

<B>Amino acid sequence similarity between human
tau and rat MAP2c.</B> Sequence alignment of the carboxyl half of the
human tau protein (GenBank<SUP>TM</SUP> accession number <A HREF="/cgi/external_ref?access_num=J03778&link_type=GEN">J03778</A>) (<A HREF="#B73">73</A>)
and rat neonatal MAP2c (GenBank<SUP>TM</SUP> accession number <A HREF="/cgi/external_ref?access_num=X17682&link_type=GEN">X17682</A>)
(<A HREF="#B74">74</A>). The <I>boxed regions</I> denote the imperfect repeats of the
MTBD</A>, and the <I>underlined region</I> denotes the tyrosine kinase
Fyn-SH3 domain-binding region in the human tau protein (352-amino acid
isoform) (<A HREF="#B8">8</A>). Note that this region is similar in both proteins,
although significant differences occur in the flanking regions.


<BR CLEAR=LEFT>

</TD></TR></TABLE>
</TD></TR></TABLE>
</CENTER>
<!-- /FIG 9 -->




    
    </P>

    
    <P>Whereas P<I>XX</I>P motifs are ideal binding ligands for SH3
domains, non-proline-rich ligands have also been documented. For
example,<SUP> </SUP>the Sab protein preferentially associates with Bruton's
tyrosine<SUP> </SUP>kinase through a non-proline-rich sequence (<A HREF="#B59">59</A>). Similarly,<SUP> </SUP>the c-Src-SH3 domain intramolecularly binds a stretch of 14&nbsp;amino<SUP> </SUP>acids
(KPQTQGLAKDAWEI) in the SH2-kinase linker region in the<SUP> </SUP>molecule (<A HREF="#B60">60</A>,
<A HREF="#B61">61</A>). In an ideal P<I>XX</I>P-SH3 interaction, the P<I>XX</I>P<SUP> </SUP>ligand assumes a polyproline II left-handed helix such that the<SUP> </SUP>alternating proline and arginine residues make contact with tyrosine,<SUP> </SUP>tryptophan, and aspartate residues in the hydrophobic pocket of<SUP> </SUP>the SH3
domain (<A HREF="#B62">62</A>, <A HREF="#B63">63</A>). In the inactive c-Src, the SH2-kinase<SUP> </SUP>linker region
adopts a partial pseudo-polyproline II left-handed<SUP> </SUP>helix conformation
and binds the SH3 domain to form an intramolecular<SUP> </SUP>interaction (<A HREF="#B60">60</A>,
<A HREF="#B61">61</A>). It is possible that certain regions<SUP> </SUP>in the MTBD</A> of MAP2c assume a
similar pseudo-polyproline II left-handed<SUP> </SUP>helix conformation that is
capable of interacting with the hydrophobic<SUP> </SUP>pockets of Src-SH3 and
Grb2-SH3. Structural studies of the MTBD/SH3<SUP> </SUP>domain complexes will be
needed to reveal the true nature of these<SUP> </SUP>interactions.
    
    </P>

    
    <P>The fact that microtubules and the SH3 domains of Src and Grb2 bind to
the MTBD</A> of MAP2c has two major implications. First,<SUP> </SUP>the 11&nbsp;conserved
P<I>XX</I>P motifs on MAP2c remain available to interact<SUP> </SUP>with other
cellular proteins. With MAP2</A> acting as a scaffold protein<SUP> </SUP>for Src,
these proteins could become potential substrates for<SUP> </SUP>the MAP2c-bound
Src tyrosine kinase activity. Second, because<SUP> </SUP>microtubule binding and
SH3 domain binding are mutually exclusive,<SUP> </SUP>the non-microtubule-bound
pool of MAP2</A> could assume the additional<SUP> </SUP>physiological role of a
scaffold protein for Src and Grb2. Both<SUP> </SUP>microtubule binding and SH3
domain binding are regulated by phosphorylation.<SUP> </SUP>Differential
phosphorylation could presumably regulate the dual<SUP> </SUP>roles of MAP2</A>
<I>in vivo</I> (scaffold <I>versus</I> cytoskeletal). Our data<SUP> </SUP>suggest that MAPK</A> may regulate MAP2/Src/Grb2 interaction <I>in
vivo</I>.<SUP> </SUP>Phosphorylation within the three K<I>X</I>GS motifs in
the MTBD</A> releases<SUP> </SUP>MAP2</A> from microtubules (<A HREF="#B22">22</A>) and thus is predicted to
increase<SUP> </SUP>the pool of soluble MAP2</A> available for binding and organizing<SUP> </SUP>signaling molecules. In addition, a subpopulation of
non-microtubule-bound<SUP> </SUP>MAP2c is observed to co-localize with
microfilament-based structures<SUP> </SUP>in transfected cells<SUP>2</SUP> and
neurons (<A HREF="#B64">64</A>). This suggests that MAP2</A> could also function<SUP> </SUP>to shuttle
signaling molecules to the microfilament system in<SUP> </SUP>the dendrites of<SUP> </SUP>neurons.
    
    </P>

    
    <P>Although MAP2</A> did not appear to be a target of Src <I>in vivo</I>,
other potential cellular substrates of Src in the dendrites include<SUP> </SUP>the
microtubules, microfilaments, and proteins directly or indirectly<SUP> </SUP>associated with MAP2</A>. Phosphorylation could release Src from MAP2</A><SUP> </SUP>to
augment an increase in membrane-associated Src at the growth<SUP> </SUP>cone. In
this way, MAP2-bound Src in developing dendrites might<SUP> </SUP>be directed to
regions of active microtubule dynamics, such as<SUP> </SUP>nerve growth cones, for
participation in neuronal morphogenesis.<SUP> </SUP><IMG SRC="/math/12pt/normal/alpha.gif" ALIGN=BASELINE ALT="alpha ">- and <IMG SRC="/math/12pt/normal/beta.gif" ALIGN=BASELINE ALT="beta ">-tubulins, the
building blocks of microtubules, have been<SUP> </SUP>reported to be
phosphorylated by pp60<SUP>c-Src</SUP> at nerve growth cone membranes
(<A HREF="#B65">65</A>). Because soluble MAP2</A> associates<SUP> </SUP>with microfilaments, actin and
actin-binding proteins could also<SUP> </SUP>be targeted by Src. For example,
F-actin and the actin-membrane<SUP> </SUP>linker protein ezrin are substrates of
tyrosine kinases <I>in situ</I><SUP> </SUP>(<A HREF="#B66">66-68</A>). Growth factor-induced
increases in protein tyrosine phosphorylation<SUP> </SUP>and a consequent
reorganization of the actin cytoskeleton are<SUP> </SUP>well documented (<A HREF="#B69">69</A>).
Other proteins that bind MAP2</A>, including<SUP> </SUP>the type II regulatory subunit
of PKA</A>, calmodulin, and the mitochondrial<SUP> </SUP>outer membrane protein porin,
might also be substrates of Src.<SUP> </SUP>In addition, a
Src-dependent signaling cascade might affect other<SUP> </SUP>yet-to-be-identified proteins associated with MAP2</A> and the microtubule<SUP> </SUP>cytoskeleton. The adult isoforms of MAP2</A> are large molecules (~200<SUP> </SUP>kDa), and the function of more than 90% of the molecule is still<SUP> </SUP>unknown. The binding of MAP2</A> with other proteins could occur via<SUP> </SUP>P<I>XX</I>P motifs or other regions in the molecule. These proteins
could<SUP> </SUP>then become substrates of Src. The discovery of new MAP2</A> binding<SUP> </SUP>partners would have implications for understanding the relationship<SUP> </SUP>between cell signaling and the neuronal cytoskeleton, particularly<SUP> </SUP>in
areas of neurite growth and synaptic<SUP> </SUP>plasticity.
    
    </P>

    
    <P>Interactions of proteins with the adaptor protein Grb2 on MAP2</A> could
potentially generate multimeric signaling complexes on<SUP> </SUP>soluble MAP2</A>.
Grb2 has two SH3 domains and a SH2 domain and is<SUP> </SUP>known to bridge
receptor and cytoplasmic tyrosine kinases with<SUP> </SUP>the Ras signaling
pathway and MAPK</A> activation (<A HREF="#B70">70</A>, <A HREF="#B71">71</A>). The<SUP> </SUP>amino-terminal SH3 domain
exhibited stronger binding to rMAP2c</A><SUP> </SUP>than the carboxyl-terminal SH3
domain. The amino-terminal SH3<SUP> </SUP>domain could be MAP2-bound, leaving the
carboxyl-terminal SH3<SUP> </SUP>domain free for binding other proteins. In
addition, the SH2 domain<SUP> </SUP>could bind tyrosine-phosphorylated proteins.
Thus, <I>in vivo</I>, Grb2<SUP> </SUP>might mediate the indirect interaction
between MAP2</A> and cellular<SUP> </SUP>proteins, such as activated tyrosine kinases,
tyrosine phosphatases,<SUP> </SUP>and other adaptor molecules (<A HREF="#B57">57</A>, <A HREF="#B70">70-72</A>).
Phosphorylation of MAP2</A><SUP> </SUP>by MAPK</A> would then release the multimeric
signaling complexes<SUP> </SUP>into the cytoplasmic compartment, and the complexes
could then<SUP> </SUP>join up with the appropriate effector molecules downstream
of<SUP> </SUP>the signaling<SUP> </SUP>cascades.
    
    </P>

    
    <P>In conclusion, the demonstrated direct interactions between MAP2/Src
and MAP2/Grb2, and the proposed phosphorylation-dependent<SUP> </SUP>scaffolding role of MAP2</A> provide a plausible mechanism for modulation<SUP> </SUP>of the neuronal cytoskeleton by extracellular signals. The interaction<SUP> </SUP>also provides a mechanism for the local organization of signaling<SUP> </SUP>complexes in the neuron. Signal-dependent modulation of the
neuronal<SUP> </SUP>cytoskeleton is necessary during the growth of dendrites in
order<SUP> </SUP>to form intricate neural circuitry. Future experiments aimed at<SUP> </SUP>uncovering new proteins associated with MAP2</A> and the neuronal<SUP> </SUP>cytoskeleton will no doubt provide further insight into the
physiological<SUP> </SUP>significance of these novel interactions in the
developing nervous<SUP> </SUP>system.
    
    


<BR><BR CLEAR=ALL>

</TXT>



<!-- Stray figures and tables -->


<!-- Acknowledgements -->

<A NAME="ack"><!-- comment for mosaic --></A>

<TABLE WIDTH=100% BGCOLOR=E1E1E1 CELLPADDING=0 CELLSPACING=0>
<TR>
<TD ALIGN=LEFT VALIGN=MIDDLE WIDTH=5% BGCOLOR=FFFFFF>&nbsp;<IMG WIDTH=10 HEIGHT=21 HSPACE=5 ALT="" SRC="/icons/toc/rarrow.gif"></TD>
<TH ALIGN=LEFT VALIGN=MIDDLE WIDTH=95%><FONT SIZE=+2>&nbsp;&nbsp;ACKNOWLEDGEMENTS</FONT></TH>
</TR>
</TABLE>


<P>We thank Dr. Gary M.&nbsp;Bokoch for gifts of
GST-SH3 expression <I>E.&nbsp;coli;</I> Dr. Klaus Hahn for the COS-7
cells; Rachel Ozer for the<SUP> </SUP>pEGFP-N1-MAP2c and the various serine
mutated rMAP2c</A> plasmids;<SUP> </SUP>Arlene Hipolito for help with cell culture,
staining, and fluorescence<SUP> </SUP>microscopy; Amy Batinica for the
construction of the <IMG SRC="/math/12pt/normal/Delta.gif" ALIGN=BASELINE ALT="Delta ">219-290<SUP> </SUP>rMAP2c</A> expression plasmid; Dr. Brian
Ceresa for a gift RC20-HRP;<SUP> </SUP>Dr. Jawdat Al-bassam for bovine tubulin;
Dr. Jennifer Kowalski<SUP> </SUP>for assistance in purification of
Grb2-N-SH3-cleaved fragments;<SUP> </SUP>Camilo Orozco for expert computer
assistance; and Melissa Paine<SUP> </SUP>for help with the purification of
HisTrxMTBD</A><SUP> </SUP>protein.
</P>

<!-- Notes -->


<!-- Footnotes -->

<A NAME="foo"><!-- comment for mosaic --></A>

<TABLE WIDTH=100% BGCOLOR=E1E1E1 CELLPADDING=0 CELLSPACING=0>
<TR>
<TD ALIGN=LEFT VALIGN=MIDDLE WIDTH=5% BGCOLOR=FFFFFF>&nbsp;<IMG WIDTH=10 HEIGHT=21 HSPACE=5 ALT="" SRC="/icons/toc/rarrow.gif"></TD>
<TH ALIGN=LEFT VALIGN=MIDDLE WIDTH=95%><FONT SIZE=+2>&nbsp;&nbsp;FOOTNOTES</FONT></TH>
</TR>
</TABLE>



<P><A NAME="FN150"><SUP>*</SUP></A>
This work was supported by National Institutes of Health
Grant MH 50861&nbsp;(to S.&nbsp;H.).The costs of publication of this
article<SUP> </SUP>were defrayed in part by the
payment of page charges. The article<SUP> </SUP>must therefore be hereby marked
"<I>advertisement</I>" in
accordance<SUP> </SUP>with 18&nbsp;U.S.C. Section
1734&nbsp;solely to indicate this<SUP> </SUP>fact.

<P>
<P><A NAME="FN152"><SUP><IMG SRC="/math/12pt/normal/Dagger.gif" ALIGN=BASELINE ALT="Dagger "></SUP></A>
To whom correspondence should be addressed: Department of Cell
Biology, The Scripps Research Institute, 10550&nbsp;N.&nbsp;Torrey Pines<SUP> </SUP>Rd., La
Jolla, CA&nbsp;92037. Tel.: 858-784-2420; Fax: 858-784-2513;<SUP> </SUP>E-mail:
shelley@scripps.edu.

<P>


<P>
Published, JBC Papers in Press, April 25,&nbsp;2000,&nbsp;DOI 10.1074/jbc.M001887200

<P>
<P><A NAME="FN2"><SUP>2</SUP></A>
R. Ozer and S.&nbsp;Halpain, submitted for<SUP> </SUP>publication.

<P>
<P><A NAME="FN3"><SUP>3</SUP></A>
A. Batinica and S.&nbsp;Halpain, unpublished<SUP> </SUP>data.

<P>

<A NAME="abb"><!-- comment for mosaic --></A>

<TABLE WIDTH=100% BGCOLOR=E1E1E1 CELLPADDING=0 CELLSPACING=0>
<TR>
<TD ALIGN=LEFT VALIGN=MIDDLE WIDTH=5% BGCOLOR=FFFFFF>&nbsp;<IMG WIDTH=10 HEIGHT=21 HSPACE=5 ALT="" SRC="/icons/toc/rarrow.gif"></TD>
<TH ALIGN=LEFT VALIGN=MIDDLE WIDTH=95%><FONT SIZE=+2>&nbsp;&nbsp;ABBREVIATIONS</FONT></TH>
</TR>
</TABLE>

<P>The abbreviations used are: 

MAP2, microtubule-associated protein 2;

MTBD, microtubule-binding domain;

PKA, protein kinase A;

MAPK, mitogen-activated protein kinase;

rMAP2c, recombinant bacterially expressed MAP2c;

SP, sulfopropyl;

HRP, horseradish peroxidase;

PMSF, phenylmethylsulfonyl fluoride;

GSK-3<IMG SRC="/math/12pt/normal/beta.gif" ALIGN=BASELINE ALT="beta ">, glycogen synthase kinase-3<IMG SRC="/math/12pt/normal/beta.gif" ALIGN=BASELINE ALT="beta ">;

PBS, phosphate-buffered saline;

PAGE, polyacrylamide gel electrophoresis;

GST, glutathione
<I>S</I>-transferase;

HisTrx, histidine-tagged thioredoxin fusion
protein;

HisTrxMTBD, histidine-tagged thioredoxin-MTBD fusion protein;

IP, immunoprecipitate;

GFP, green fluorescent protein;

SH, Src
homology.

</P>


<!-- Appendices -->

<!-- Bibliography -->




<A NAME="References"><!-- null --></A>
<TABLE WIDTH=100% BGCOLOR=E1E1E1 CELLPADDING=0 CELLSPACING=0>
<TR>
<TD ALIGN=LEFT VALIGN=MIDDLE WIDTH=5% BGCOLOR=FFFFFF>&nbsp;<IMG WIDTH=10 HEIGHT=21 HSPACE=5 ALT="" SRC="/icons/toc/rarrow.gif"></TD>
<TH ALIGN=LEFT VALIGN=MIDDLE WIDTH=95%><FONT SIZE=+2>&nbsp;&nbsp;REFERENCES</FONT></TH>
</TR>
</TABLE>

<TABLE ALIGN=RIGHT CELLPADDING=5 BORDER NOWRAP><R>
<TH ALIGN=LEFT>
<FONT SIZE=-1>



<A HREF="#Top"><IMG WIDTH=11 HEIGHT=9 BORDER=0 HSPACE=5 SRC="/icons/toc/uarrow.gif">TOP</A>


<BR>
<A HREF="#Abstract"><IMG WIDTH=11 HEIGHT=9 BORDER=0 HSPACE=5 SRC="/icons/toc/uarrow.gif">ABSTRACT</A>


<BR>
<A HREF="#Introduction"><IMG WIDTH=11 HEIGHT=9 BORDER=0 HSPACE=5 SRC="/icons/toc/uarrow.gif">INTRODUCTION</A>


<BR>
<A HREF="#SEC1"><IMG WIDTH=11 HEIGHT=9 BORDER=0 HSPACE=5 SRC="/icons/toc/uarrow.gif">EXPERIMENTAL PROCEDURES</A>


<BR>
<A HREF="#SEC2"><IMG WIDTH=11 HEIGHT=9 BORDER=0 HSPACE=5 SRC="/icons/toc/uarrow.gif">RESULTS</A>


<BR>
<A HREF="#SEC3"><IMG WIDTH=11 HEIGHT=9 BORDER=0 HSPACE=5 SRC="/icons/toc/uarrow.gif">DISCUSSION</A>


<BR><IMG WIDTH=11 HEIGHT=9 BORDER=0 HSPACE=5 SRC="/icons/toc/dot.gif"><FONT COLOR=46
4C53>REFERENCES</FONT>
</FONT></TH>
</TR></TABLE>

<BR CLEAR=ALL>
<TABLE BORDER=0>
<TR><TD VALIGN=TOP><a name="B1">1.</a>
    <TD VALIGN=TOP>

Gamblin, T. C.,
Nachmanoff, K.,
Williams, R. G.,
 and Halpain, S.



(1996)

<I>Biochemistry</I>
<B>35</B>,
12576-12586<!-- HIGHWIRE ID="275:27:20578:1" --><A HREF="/cgi/external_ref?access_num=10.1021/bi961135d&link_type=DOI">[CrossRef]</A><A HREF="/cgi/external_ref?access_num=8823195&link_type=MED">[Medline]</A> 
<A HREF="/cgi/external_ref?access_num=8823195&displayid=77141&link_type=INFOTRIEVE">[Order&nbsp;article&nbsp;via&nbsp;Infotrieve]</A><!-- /HIGHWIRE -->
    </TR>
<TR><TD VALIGN=TOP><a name="B2">2.</a>
    <TD VALIGN=TOP>

Caceres, A.,
Mautino, J.,
 and Kosik, K. S.



(1992)

<I>Neuron</I>
<B>9</B>,
607-618<!-- HIGHWIRE ID="275:27:20578:2" --><A HREF="/cgi/external_ref?access_num=1389180&link_type=MED">[Medline]</A> 
<A HREF="/cgi/external_ref?access_num=1389180&displayid=77141&link_type=INFOTRIEVE">[Order&nbsp;article&nbsp;via&nbsp;Infotrieve]</A><!-- /HIGHWIRE -->
    </TR>
<TR><TD VALIGN=TOP><a name="B3">3.</a>
    <TD VALIGN=TOP>

Johnson, G. V.&nbsp;W.,
 and Jope, R. S.



(1992)

<I>J. Neurosci. Res.</I>
<B>33</B>,
505-512<!-- HIGHWIRE ID="275:27:20578:3" --><A HREF="/cgi/external_ref?access_num=1484385&link_type=MED">[Medline]</A> 
<A HREF="/cgi/external_ref?access_num=1484385&displayid=77141&link_type=INFOTRIEVE">[Order&nbsp;article&nbsp;via&nbsp;Infotrieve]</A><!-- /HIGHWIRE -->
    </TR>
<TR><TD VALIGN=TOP><a name="B4">4.</a>
    <TD VALIGN=TOP>

Lee, G.



(1993)

<I>Curr. Opin. Cell Biol.</I>
<B>5</B>,
88-94<!-- HIGHWIRE ID="275:27:20578:4" --><A HREF="/cgi/external_ref?access_num=8448035&link_type=MED">[Medline]</A> 
<A HREF="/cgi/external_ref?access_num=8448035&displayid=77141&link_type=INFOTRIEVE">[Order&nbsp;article&nbsp;via&nbsp;Infotrieve]</A><!-- /HIGHWIRE -->
    </TR>
<TR><TD VALIGN=TOP><a name="B5">5.</a>
    <TD VALIGN=TOP>

Hirokawa, N.



(1994)

<I>Curr. Opin. Cell Biol.</I>
<B>6</B>,
74-81<!-- HIGHWIRE ID="275:27:20578:5" --><A HREF="/cgi/external_ref?access_num=8167029&link_type=MED">[Medline]</A> 
<A HREF="/cgi/external_ref?access_num=8167029&displayid=77141&link_type=INFOTRIEVE">[Order&nbsp;article&nbsp;via&nbsp;Infotrieve]</A><!-- /HIGHWIRE -->
    </TR>
<TR><TD VALIGN=TOP><a name="B6">6.</a>
    <TD VALIGN=TOP>

Maccioni, R. B.,
 and Cambiazo, V.



(1995)

<I>Physiol. Rev.</I>
<B>75</B>,
835-864<!-- HIGHWIRE ID="275:27:20578:6" --><A HREF="/cgi/ijlink?linkType=ABST&journalCode=physrev&resid=75/4/835"><nobr>[Abstract/<font COLOR="CC0000">Free</font> Full&nbsp;Text]</nobr></A><!-- /HIGHWIRE -->
    </TR>
<TR><TD VALIGN=TOP><a name="B7">7.</a>
    <TD VALIGN=TOP>

Lewis, S. A.,
Wang, D. H.,
 and Cowan, N. J.



(1988)

<I>Science</I>
<B>242</B>,
936-939<!-- HIGHWIRE ID="275:27:20578:7" --><A HREF="/cgi/external_ref?access_num=3142041&link_type=MED">[Medline]</A> 
<A HREF="/cgi/external_ref?access_num=3142041&displayid=77141&link_type=INFOTRIEVE">[Order&nbsp;article&nbsp;via&nbsp;Infotrieve]</A><!-- /HIGHWIRE -->
    </TR>
<TR><TD VALIGN=TOP><a name="B8">8.</a>
    <TD VALIGN=TOP>

Lee, G.,
Newman, T. S.,
Gard, D. L.,
Band, H.,
 and Panchamoorthy, G.



(1998)

<I>J. Cell Sci.</I>
<B>111</B>,
3167-3177<!-- HIGHWIRE ID="275:27:20578:8" --><A HREF="/cgi/ijlink?linkType=ABST&journalCode=joces&resid=111/21/3167"><nobr>[Abstract/<font COLOR="CC0000">Free</font> Full&nbsp;Text]</nobr></A><!-- /HIGHWIRE -->
    </TR>
<TR><TD VALIGN=TOP><a name="B9">9.</a>
    <TD VALIGN=TOP>

Sontag, E.,
Nunbhakdi-Craig, V.,
Lee, G.,
Bloom, G. S.,
 and Mumby, M. C.



(1996)

<I>Neuron</I>
<B>17</B>,
1201-1207<!-- HIGHWIRE ID="275:27:20578:9" --><A HREF="/cgi/external_ref?access_num=8982166&link_type=MED">[Medline]</A> 
<A HREF="/cgi/external_ref?access_num=8982166&displayid=77141&link_type=INFOTRIEVE">[Order&nbsp;article&nbsp;via&nbsp;Infotrieve]</A><!-- /HIGHWIRE -->
    </TR>
<TR><TD VALIGN=TOP><a name="B10">10.</a>
    <TD VALIGN=TOP>

Liao, H.,
Li, Y.,
Brautigan, D. L.,
 and Gundersen, G. G.



(1998)

<I>J. Biol. Chem.</I>
<B>273</B>,
21901-21908<!-- HIGHWIRE ID="275:27:20578:10" --><A HREF="/cgi/ijlink?linkType=ABST&journalCode=jbc&resid=273/34/21901"><nobr>[Abstract/<font COLOR="CC0000">Free</font> Full&nbsp;Text]</nobr></A><!-- /HIGHWIRE -->
    </TR>
<TR><TD VALIGN=TOP><a name="B11">11.</a>
    <TD VALIGN=TOP>

Sontag, E.,
Nunbhakdi-Craig, V.,
Lee, G.,
Brandt, R.,
Kamibayashi, C.,
Kuret, J.,
White III, C. L.,
Mumby, M. C.,
 and Bloom, G. C.



(1999)

<I>J. Biol. Chem.</I>
<B>274</B>,
25490-25498<!-- HIGHWIRE ID="275:27:20578:11" --><A HREF="/cgi/ijlink?linkType=ABST&journalCode=jbc&resid=274/36/25490"><nobr>[Abstract/<font COLOR="CC0000">Free</font> Full&nbsp;Text]</nobr></A><!-- /HIGHWIRE -->
    </TR>
<TR><TD VALIGN=TOP><a name="B12">12.</a>
    <TD VALIGN=TOP>

Goedert, M.,
Jakes, R.,
Spillantini, M. G.,
Crowther, R. A.,
Cohen, P.,
Vanmechelen, E.,
Probst, A.,
Gotz, J.,
 and Burki, K.



(1995)

<I>Biochem. Soc. Trans.</I>
<B>23</B>,
80-85<!-- HIGHWIRE ID="275:27:20578:12" --><A HREF="/cgi/external_ref?access_num=7758799&link_type=MED">[Medline]</A> 
<A HREF="/cgi/external_ref?access_num=7758799&displayid=77141&link_type=INFOTRIEVE">[Order&nbsp;article&nbsp;via&nbsp;Infotrieve]</A><!-- /HIGHWIRE -->
    </TR>
<TR><TD VALIGN=TOP><a name="B13">13.</a>
    <TD VALIGN=TOP>

Goedert, M.,
Crowther, R. A.,
 and Spillantini, M. G.



(1998)

<I>Neuron</I>
<B>21</B>,
955-958<!-- HIGHWIRE ID="275:27:20578:13" --><A HREF="/cgi/external_ref?access_num=9856453&link_type=MED">[Medline]</A> 
<A HREF="/cgi/external_ref?access_num=9856453&displayid=77141&link_type=INFOTRIEVE">[Order&nbsp;article&nbsp;via&nbsp;Infotrieve]</A><!-- /HIGHWIRE -->
    </TR>
<TR><TD VALIGN=TOP><a name="B14">14.</a>
    <TD VALIGN=TOP>

Spillantini, M. G.,
 and Goedert, M.



(1998)

<I>Trends Neurosci.</I>
<B>21</B>,
428-433<!-- HIGHWIRE ID="275:27:20578:14" --><A HREF="/cgi/external_ref?access_num=10.1016/S0166-2236(98)01337-X&link_type=DOI">[CrossRef]</A><A HREF="/cgi/external_ref?access_num=9786340&link_type=MED">[Medline]</A> 
<A HREF="/cgi/external_ref?access_num=9786340&displayid=77141&link_type=INFOTRIEVE">[Order&nbsp;article&nbsp;via&nbsp;Infotrieve]</A><!-- /HIGHWIRE -->
    </TR>
<TR><TD VALIGN=TOP><a name="B15">15.</a>
    <TD VALIGN=TOP>

Aoki, C.,
 and Siekevitz, P.



(1985)

<I>J. Neurosci.</I>
<B>5</B>,
2465-2483<!-- HIGHWIRE ID="275:27:20578:15" --><A HREF="/cgi/ijlink?linkType=ABST&journalCode=jneuro&resid=5/9/2465">[Abstract]</A><!-- /HIGHWIRE -->
    </TR>
<TR><TD VALIGN=TOP><a name="B16">16.</a>
    <TD VALIGN=TOP>

Halpain, S.,
 and Greengard, P.



(1990)

<I>Neuron</I>
<B>5</B>,
237-246<!-- HIGHWIRE ID="275:27:20578:16" --><A HREF="/cgi/external_ref?access_num=2169265&link_type=MED">[Medline]</A> 
<A HREF="/cgi/external_ref?access_num=2169265&displayid=77141&link_type=INFOTRIEVE">[Order&nbsp;article&nbsp;via&nbsp;Infotrieve]</A><!-- /HIGHWIRE -->
    </TR>
<TR><TD VALIGN=TOP><a name="B17">17.</a>
    <TD VALIGN=TOP>

Montoro, R. J.,
Diaz-Nido, J.,
Avila, J.,
 and Lopez-Barneo, J.



(1993)

<I>Neuroscience</I>
<B>54</B>,
859-871<!-- HIGHWIRE ID="275:27:20578:17" --><A HREF="/cgi/external_ref?access_num=8393539&link_type=MED">[Medline]</A> 
<A HREF="/cgi/external_ref?access_num=8393539&displayid=77141&link_type=INFOTRIEVE">[Order&nbsp;article&nbsp;via&nbsp;Infotrieve]</A><!-- /HIGHWIRE -->
    </TR>
<TR><TD VALIGN=TOP><a name="B18">18.</a>
    <TD VALIGN=TOP>

Quinlan, E. M.,
 and Halpain, S.



(1996)

<I>Neuron</I>
<B>16</B>,
357-368<!-- HIGHWIRE ID="275:27:20578:18" --><A HREF="/cgi/external_ref?access_num=8789950&link_type=MED">[Medline]</A> 
<A HREF="/cgi/external_ref?access_num=8789950&displayid=77141&link_type=INFOTRIEVE">[Order&nbsp;article&nbsp;via&nbsp;Infotrieve]</A><!-- /HIGHWIRE -->
    </TR>
<TR><TD VALIGN=TOP><a name="B19">19.</a>
    <TD VALIGN=TOP>

Quinlan, E. M.,
 and Halpain, S.



(1996)

<I>J. Neurosci.</I>
<B>16</B>,
7627-7637<!-- HIGHWIRE ID="275:27:20578:19" --><A HREF="/cgi/ijlink?linkType=ABST&journalCode=jneuro&resid=16/23/7627"><nobr>[Abstract/<font COLOR="CC0000">Free</font> Full&nbsp;Text]</nobr></A><!-- /HIGHWIRE -->
    </TR>
<TR><TD VALIGN=TOP><a name="B20">20.</a>
    <TD VALIGN=TOP>

Philpot, B. D.,
Lim, J. H.,
Halpain, S.,
 and Brunjes, P. C.



(1997)

<I>J. Neurosci.</I>
<B>17</B>,
9596-9604<!-- HIGHWIRE ID="275:27:20578:20" --><A HREF="/cgi/ijlink?linkType=ABST&journalCode=jneuro&resid=17/24/9596"><nobr>[Abstract/<font COLOR="CC0000">Free</font> Full&nbsp;Text]</nobr></A><!-- /HIGHWIRE -->
    </TR>
<TR><TD VALIGN=TOP><a name="B21">21.</a>
    <TD VALIGN=TOP>

Theurkauf, W. E.,
 and Vallee, R. B.



(1983)

<I>J. Biol. Chem.</I>
<B>258</B>,
7883-7886<!-- HIGHWIRE ID="275:27:20578:21" --><A HREF="/cgi/ijlink?linkType=ABST&journalCode=jbc&resid=258/12/7883"><nobr>[Abstract/<font COLOR="CC0000">Free</font> Full&nbsp;Text]</nobr></A><!-- /HIGHWIRE -->
    </TR>
<TR><TD VALIGN=TOP><a name="B22">22.</a>
    <TD VALIGN=TOP>

Illenberger, S.,
Drewes, G.,
Trinczek, B.,
Biernat, J.,
Meyer, H. E.,
Olmsted, J. B.,
Mandelkow, E. M.,
 and Mandelkow, E.



(1996)

<I>J. Biol. Chem.</I>
<B>271</B>,
10834-10843<!-- HIGHWIRE ID="275:27:20578:22" --><A HREF="/cgi/ijlink?linkType=ABST&journalCode=jbc&resid=271/18/10834"><nobr>[Abstract/<font COLOR="CC0000">Free</font> Full&nbsp;Text]</nobr></A><!-- /HIGHWIRE -->
    </TR>
<TR><TD VALIGN=TOP><a name="B23">23.</a>
    <TD VALIGN=TOP>

S&aacute;nchez, C.,
Tompa, P.,
Sz&uuml;cs, K.,
Friedrich, P.,
 and Avila, J.



(1996)

<I>Eur. J.&nbsp;Biochem.</I>
<B>241</B>,
765-771<!-- HIGHWIRE ID="275:27:20578:23" --><A HREF="/cgi/ijlink?linkType=ABST&journalCode=ejbiochem&resid=241/3/765">[Abstract]</A><!-- /HIGHWIRE -->
    </TR>
<TR><TD VALIGN=TOP><a name="B24">24.</a>
    <TD VALIGN=TOP>

Akiyama, T.,
Nishida, E.,
Ishida, J.,
Saji, N.,
Ogawara, H.,
Hoshi, M.,
Miyata, Y.,
 and Sakai, H.



(1986)

<I>J. Biol. Chem.</I>
<B>261</B>,
15648-15651<!-- HIGHWIRE ID="275:27:20578:24" --><A HREF="/cgi/ijlink?linkType=ABST&journalCode=jbc&resid=261/33/15648"><nobr>[Abstract/<font COLOR="CC0000">Free</font> Full&nbsp;Text]</nobr></A><!-- /HIGHWIRE -->
    </TR>
<TR><TD VALIGN=TOP><a name="B25">25.</a>
    <TD VALIGN=TOP>

Berling, B.,
Willie, H.,
Roll, B.,
Mandelkow, E.-M.,
Garner, C.,
 and Mandelkow, E.



(1994)

<I>Eur. J.&nbsp;Cell Biol.</I>
<B>64</B>,
120-130<!-- HIGHWIRE ID="275:27:20578:25" --><A HREF="/cgi/external_ref?access_num=7525290&link_type=MED">[Medline]</A> 
<A HREF="/cgi/external_ref?access_num=7525290&displayid=77141&link_type=INFOTRIEVE">[Order&nbsp;article&nbsp;via&nbsp;Infotrieve]</A><!-- /HIGHWIRE -->
    </TR>
<TR><TD VALIGN=TOP><a name="B26">26.</a>
    <TD VALIGN=TOP>

Goldenring, J. R.,
Vallano, M. L.,
 and deLorenzo, R. J.



(1985)

<I>J. Neurochem.</I>
<B>45</B>,
900-905<!-- HIGHWIRE ID="275:27:20578:26" --><A HREF="/cgi/ijlink?linkType=ABST&journalCode=jneurochem&resid=45/3/900">[Abstract]</A><!-- /HIGHWIRE -->
    </TR>
<TR><TD VALIGN=TOP><a name="B27">27.</a>
    <TD VALIGN=TOP>

Goto, S.,
Yamamoto, H.,
Fukunaga, K.,
Iwasa, I.,
Matsukado, Y.,
 and Miyamoto, E.



(1985)

<I>J. Neurochem.</I>
<B>4</B>,
276-283<!-- HIGHWIRE ID="275:27:20578:27" --><!-- /HIGHWIRE -->
    </TR>
<TR><TD VALIGN=TOP><a name="B28">28.</a>
    <TD VALIGN=TOP>

Yamamoto, H.,
Saitoh, Y.,
Fukunaga, K.,
Nishimura, H.,
 and Miyamoto, E.



(1988)

<I>J. Neurochem.</I>
<B>50</B>,
1614-1623<!-- HIGHWIRE ID="275:27:20578:28" --><A HREF="/cgi/ijlink?linkType=ABST&journalCode=jneurochem&resid=50/5/1614">[Abstract]</A><!-- /HIGHWIRE -->
    </TR>
<TR><TD VALIGN=TOP><a name="B29">29.</a>
    <TD VALIGN=TOP>

Kadowaki, T.,
Fujita-Yamaguchi, Y.,
Nishida, E.,
Takaku, F.,
Akiyama, T.,
Kathuria, S.,
Akanuma, Y.,
 and Kasuga, M.



(1985)

<I>J. Biol. Chem.</I>
<B>260</B>,
4016-4020<!-- HIGHWIRE ID="275:27:20578:29" --><A HREF="/cgi/ijlink?linkType=ABST&journalCode=jbc&resid=260/7/4016">[Abstract]</A><!-- /HIGHWIRE -->
    </TR>
<TR><TD VALIGN=TOP><a name="B30">30.</a>
    <TD VALIGN=TOP>

Akiyama, T.,
Kadowaki, T.,
Nishida, E.,
Kadooka, T.,
Ogawara, H.,
Fukami, Y.,
Sakai, H.,
Takaku, F.,
 and Kasuga, M.



(1986)

<I>J. Biol. Chem.</I>
<B>261</B>,
14797-14803<!-- HIGHWIRE ID="275:27:20578:30" --><A HREF="/cgi/ijlink?linkType=ABST&journalCode=jbc&resid=261/31/14797"><nobr>[Abstract/<font COLOR="CC0000">Free</font> Full&nbsp;Text]</nobr></A><!-- /HIGHWIRE -->
    </TR>
<TR><TD VALIGN=TOP><a name="B31">31.</a>
    <TD VALIGN=TOP>

Nishida, E.,
Hoshi, M.,
Miyata, Y.,
Sakai, H.,
Kadowaki, T.,
Kasuga, M.,
Saijo, S.,
Ogawara, H.,
 and Akiyama, T.



(1987)

<I>J. Biol. Chem.</I>
<B>262</B>,
16200-16204<!-- HIGHWIRE ID="275:27:20578:31" --><A HREF="/cgi/ijlink?linkType=ABST&journalCode=jbc&resid=262/33/16200"><nobr>[Abstract/<font COLOR="CC0000">Free</font> Full&nbsp;Text]</nobr></A><!-- /HIGHWIRE -->
    </TR>
<TR><TD VALIGN=TOP><a name="B32">32.</a>
    <TD VALIGN=TOP>

Vallee, R. B.,
DiBartolomeis, M. J.,
 and Theukauf, W. E.



(1981)

<I>J. Cell Biol.</I>
<B>90</B>,
568-576<!-- HIGHWIRE ID="275:27:20578:32" --><A HREF="/cgi/external_ref?access_num=6270156&link_type=MED">[Medline]</A> 
<A HREF="/cgi/external_ref?access_num=6270156&displayid=77141&link_type=INFOTRIEVE">[Order&nbsp;article&nbsp;via&nbsp;Infotrieve]</A><!-- /HIGHWIRE -->
    </TR>
<TR><TD VALIGN=TOP><a name="B33">33.</a>
    <TD VALIGN=TOP>

Obar, R. A.,
Dingus, J.,
Bayley, H.,
 and Vallee, R. B.



(1989)

<I>Neuron</I>
<B>3</B>,
639-645<!-- HIGHWIRE ID="275:27:20578:33" --><A HREF="/cgi/external_ref?access_num=2561973&link_type=MED">[Medline]</A> 
<A HREF="/cgi/external_ref?access_num=2561973&displayid=77141&link_type=INFOTRIEVE">[Order&nbsp;article&nbsp;via&nbsp;Infotrieve]</A><!-- /HIGHWIRE -->
    </TR>
<TR><TD VALIGN=TOP><a name="B34">34.</a>
    <TD VALIGN=TOP>

Rubino, H. M.,
Damerman, M.,
Shafit-Zagardo, B.,
 and Erlichman, J.



(1989)

<I>Neuron</I>
<B>3</B>,
631-638<!-- HIGHWIRE ID="275:27:20578:34" --><A HREF="/cgi/external_ref?access_num=2701845&link_type=MED">[Medline]</A> 
<A HREF="/cgi/external_ref?access_num=2701845&displayid=77141&link_type=INFOTRIEVE">[Order&nbsp;article&nbsp;via&nbsp;Infotrieve]</A><!-- /HIGHWIRE -->
    </TR>
<TR><TD VALIGN=TOP><a name="B35">35.</a>
    <TD VALIGN=TOP>

Griffith, L. M.,
 and Pollard, T. D.



(1978)

<I>J. Cell Biol.</I>
<B>78</B>,
958-965<!-- HIGHWIRE ID="275:27:20578:35" --><A HREF="/cgi/ijlink?linkType=ABST&journalCode=jcb&resid=78/3/958">[Abstract]</A><!-- /HIGHWIRE -->
    </TR>
<TR><TD VALIGN=TOP><a name="B36">36.</a>
    <TD VALIGN=TOP>

Griffith, L. M.,
 and Pollard, T. D.



(1982)

<I>J. Biol. Chem.</I>
<B>257</B>,
9143-9151<!-- HIGHWIRE ID="275:27:20578:36" --><A HREF="/cgi/ijlink?linkType=ABST&journalCode=jbc&resid=257/15/9143"><nobr>[Abstract/<font COLOR="CC0000">Free</font> Full&nbsp;Text]</nobr></A><!-- /HIGHWIRE -->
    </TR>
<TR><TD VALIGN=TOP><a name="B37">37.</a>
    <TD VALIGN=TOP>

Sattilaro, R.



(1986)

<I>Biochemistry</I>
<B>25</B>,
2003-2009<!-- HIGHWIRE ID="275:27:20578:37" --><A HREF="/cgi/external_ref?access_num=3707926&link_type=MED">[Medline]</A> 
<A HREF="/cgi/external_ref?access_num=3707926&displayid=77141&link_type=INFOTRIEVE">[Order&nbsp;article&nbsp;via&nbsp;Infotrieve]</A><!-- /HIGHWIRE -->
    </TR>
<TR><TD VALIGN=TOP><a name="B38">38.</a>
    <TD VALIGN=TOP>

Cunningham, C. C.,
Leclerc, N.,
Flanagan, L. A.,
Lu, M.,
Janmey, P. A.,
 and Kosik, K. S.



(1997)

<I>J. Cell Biol.</I>
<B>136</B>,
845-857<!-- HIGHWIRE ID="275:27:20578:38" --><A HREF="/cgi/ijlink?linkType=ABST&journalCode=jcb&resid=136/4/845"><nobr>[Abstract/<font COLOR="CC0000">Free</font> Full&nbsp;Text]</nobr></A><!-- /HIGHWIRE -->
    </TR>
<TR><TD VALIGN=TOP><a name="B39">39.</a>
    <TD VALIGN=TOP>

Kotani, S.,
Nishida, E.,
Kumaga, i. H.,
 and Sakai, H.



(1985)

<I>J. Biol. Chem.</I>
<B>260</B>,
10779-10783<!-- HIGHWIRE ID="275:27:20578:39" --><A HREF="/cgi/ijlink?linkType=ABST&journalCode=jbc&resid=260/19/10779"><nobr>[Abstract/<font COLOR="CC0000">Free</font> Full&nbsp;Text]</nobr></A><!-- /HIGHWIRE -->
    </TR>
<TR><TD VALIGN=TOP><a name="B40">40.</a>
    <TD VALIGN=TOP>

Lind&eacute;n, M.,
 and Karlsson, G.



(1996)

<I>Biochem. Biophys. Res. Commun.</I>
<B>218</B>,
833-836<!-- HIGHWIRE ID="275:27:20578:40" --><A HREF="/cgi/external_ref?access_num=10.1006/bbrc.1996.0148&link_type=DOI">[CrossRef]</A><A HREF="/cgi/external_ref?access_num=8579600&link_type=MED">[Medline]</A> 
<A HREF="/cgi/external_ref?access_num=8579600&displayid=77141&link_type=INFOTRIEVE">[Order&nbsp;article&nbsp;via&nbsp;Infotrieve]</A><!-- /HIGHWIRE -->
    </TR>
<TR><TD VALIGN=TOP><a name="B41">41.</a>
    <TD VALIGN=TOP>

Sambrook, J.,
Fritsch, E. F.,
 and Maniatis, T.



(1989)


<I>Molecular Cloning: A Laboratory Manual</I>
, 2nd Ed.
, Cold Spring Harbor Laboratory, Cold Spring Harbor, NY
    <!-- HIGHWIRE ID="275:27:20578:41" --><!-- /HIGHWIRE --></TR>
<TR><TD VALIGN=TOP><a name="B42">42.</a>
    <TD VALIGN=TOP>

Sanger, F.,
Nicklen, S.,
 and Coulson, A. R.



(1977)

<I>Proc. Natl. Acad. Sci. U.&nbsp;S.&nbsp;A.</I>
<B>74</B>,
5463-5467<!-- HIGHWIRE ID="275:27:20578:42" --><A HREF="/cgi/external_ref?access_num=271968&link_type=MED">[Medline]</A> 
<A HREF="/cgi/external_ref?access_num=271968&displayid=77141&link_type=INFOTRIEVE">[Order&nbsp;article&nbsp;via&nbsp;Infotrieve]</A><!-- /HIGHWIRE -->
    </TR>
<TR><TD VALIGN=TOP><a name="B43">43.</a>
    <TD VALIGN=TOP>

Halpain, S.,
Hipolito, A.,
 and Saffer, L.



(1998)

<I>J. Neurosci.</I>
<B>18</B>,
9835-9844<!-- HIGHWIRE ID="275:27:20578:43" --><A HREF="/cgi/ijlink?linkType=ABST&journalCode=jneuro&resid=18/23/9835"><nobr>[Abstract/<font COLOR="CC0000">Free</font> Full&nbsp;Text]</nobr></A><!-- /HIGHWIRE -->
    </TR>
<TR><TD VALIGN=TOP><a name="B44">44.</a>
    <TD VALIGN=TOP>

Laemmli, U. K.



(1970)

<I>Nature</I>
<B>227</B>,
680-685<!-- HIGHWIRE ID="275:27:20578:44" --><A HREF="/cgi/external_ref?access_num=5432063&link_type=MED">[Medline]</A> 
<A HREF="/cgi/external_ref?access_num=5432063&displayid=77141&link_type=INFOTRIEVE">[Order&nbsp;article&nbsp;via&nbsp;Infotrieve]</A><!-- /HIGHWIRE -->
    </TR>
<TR><TD VALIGN=TOP><a name="B45">45.</a>
    <TD VALIGN=TOP>

Butner, K. A.,
 and Kirschner, M. W.



(1991)

<I>J. Cell Biol.</I>
<B>115</B>,
717-730<!-- HIGHWIRE ID="275:27:20578:45" --><A HREF="/cgi/ijlink?linkType=ABST&journalCode=jcb&resid=115/3/717">[Abstract]</A><!-- /HIGHWIRE -->
    </TR>
<TR><TD VALIGN=TOP><a name="B46">46.</a>
    <TD VALIGN=TOP>

Goode, B. L.,
 and Feinstein, S. C.



(1994)

<I>J. Cell Biol.</I>
<B>124</B>,
769-782<!-- HIGHWIRE ID="275:27:20578:46" --><A HREF="/cgi/ijlink?linkType=ABST&journalCode=jcb&resid=124/5/769">[Abstract]</A><!-- /HIGHWIRE -->
    </TR>
<TR><TD VALIGN=TOP><a name="B47">47.</a>
    <TD VALIGN=TOP>

Smith, P. K.,
Krohn, R. I.,
Hermanson, G. T.,
Mallia, A. K.,
Gartner, F. H.,
Provenzano, M. D.,
Fujimoto, E. K.,
Goeke, N. M.,
Olson, B. J.,
 and Klenk, D. C.



(1985)

<I>Anal. Biochem.</I>
<B>150</B>,
76-85<!-- HIGHWIRE ID="275:27:20578:47" --><A HREF="/cgi/external_ref?access_num=3843705&link_type=MED">[Medline]</A> 
<A HREF="/cgi/external_ref?access_num=3843705&displayid=77141&link_type=INFOTRIEVE">[Order&nbsp;article&nbsp;via&nbsp;Infotrieve]</A><!-- /HIGHWIRE -->
    </TR>
<TR><TD VALIGN=TOP><a name="B48">48.</a>
    <TD VALIGN=TOP>

Towbin, H.,
Staehelin, T.,
 and Gordon, J.



(1979)

<I>Proc. Natl. Acad. Sci. U.&nbsp;S.&nbsp;A.</I>
<B>76</B>,
4350-4354<!-- HIGHWIRE ID="275:27:20578:48" --><A HREF="/cgi/external_ref?access_num=388439&link_type=MED">[Medline]</A> 
<A HREF="/cgi/external_ref?access_num=388439&displayid=77141&link_type=INFOTRIEVE">[Order&nbsp;article&nbsp;via&nbsp;Infotrieve]</A><!-- /HIGHWIRE -->
    </TR>
<TR><TD VALIGN=TOP><a name="B49">49.</a>
    <TD VALIGN=TOP>

Luttrell, L. M.,
Della Rocca, G. J.,
van Biesen, T.,
Luttrell, D. K.,
 and Lefkowitz, R. J.



(1997)

<I>J. Biol. Chem.</I>
<B>272</B>,
4637-4644<!-- HIGHWIRE ID="275:27:20578:49" --><A HREF="/cgi/ijlink?linkType=ABST&journalCode=jbc&resid=272/7/4637"><nobr>[Abstract/<font COLOR="CC0000">Free</font> Full&nbsp;Text]</nobr></A><!-- /HIGHWIRE -->
    </TR>
<TR><TD VALIGN=TOP><a name="B50">50.</a>
    <TD VALIGN=TOP>

Sloboda, R. D.,
 and Rosenbaum, J. L.



(1982)

<I>Methods Enzymol.</I>
<B>85</B>,
409-417<!-- HIGHWIRE ID="275:27:20578:50" --><A HREF="/cgi/external_ref?access_num=7121279&link_type=MED">[Medline]</A> 
<A HREF="/cgi/external_ref?access_num=7121279&displayid=77141&link_type=INFOTRIEVE">[Order&nbsp;article&nbsp;via&nbsp;Infotrieve]</A><!-- /HIGHWIRE -->
    </TR>
<TR><TD VALIGN=TOP><a name="B51">51.</a>
    <TD VALIGN=TOP>

Matus, A.,
Bernhardt, R.,
Bodmer, R.,
 and Alaimo, D.



(1986)

<I>Neuroscience</I>
<B>17</B>,
371-389<!-- HIGHWIRE ID="275:27:20578:51" --><A HREF="/cgi/external_ref?access_num=3517689&link_type=MED">[Medline]</A> 
<A HREF="/cgi/external_ref?access_num=3517689&displayid=77141&link_type=INFOTRIEVE">[Order&nbsp;article&nbsp;via&nbsp;Infotrieve]</A><!-- /HIGHWIRE -->
    </TR>
<TR><TD VALIGN=TOP><a name="B52">52.</a>
    <TD VALIGN=TOP>

Mannervik, B.,
 and Danielson, U. H.



(1988)

<I>CRC Crit. Rev. Biochem.</I>
<B>23</B>,
283-337<!-- HIGHWIRE ID="275:27:20578:52" --><A HREF="/cgi/external_ref?access_num=3069329&link_type=MED">[Medline]</A> 
<A HREF="/cgi/external_ref?access_num=3069329&displayid=77141&link_type=INFOTRIEVE">[Order&nbsp;article&nbsp;via&nbsp;Infotrieve]</A><!-- /HIGHWIRE -->
    </TR>
<TR><TD VALIGN=TOP><a name="B53">53.</a>
    <TD VALIGN=TOP>

Lundqvist, G.,
Yucel-Lindberg, T.,
 and Morgenstern, R.



(1992)

<I>Biochim. Biophys. Acta</I>
<B>1159</B>,
103-108<!-- HIGHWIRE ID="275:27:20578:53" --><A HREF="/cgi/external_ref?access_num=1390907&link_type=MED">[Medline]</A> 
<A HREF="/cgi/external_ref?access_num=1390907&displayid=77141&link_type=INFOTRIEVE">[Order&nbsp;article&nbsp;via&nbsp;Infotrieve]</A><!-- /HIGHWIRE -->
    </TR>
<TR><TD VALIGN=TOP><a name="B54">54.</a>
    <TD VALIGN=TOP>

Tsuyama, S.,
Terayama, Y.,
 and Matsuyama, S.



(1987)

<I>J. Biol. Chem.</I>
<B>262</B>,
10886-10892<!-- HIGHWIRE ID="275:27:20578:54" --><A HREF="/cgi/ijlink?linkType=ABST&journalCode=jbc&resid=262/22/10886"><nobr>[Abstract/<font COLOR="CC0000">Free</font> Full&nbsp;Text]</nobr></A><!-- /HIGHWIRE -->
    </TR>
<TR><TD VALIGN=TOP><a name="B55">55.</a>
    <TD VALIGN=TOP>

Walaas, S. I.,
 and Nairn, A. C.



(1989)

<I>J. Mol. Neurosci.</I>
<B>1</B>,
117-127<!-- HIGHWIRE ID="275:27:20578:55" --><A HREF="/cgi/external_ref?access_num=2561875&link_type=MED">[Medline]</A> 
<A HREF="/cgi/external_ref?access_num=2561875&displayid=77141&link_type=INFOTRIEVE">[Order&nbsp;article&nbsp;via&nbsp;Infotrieve]</A><!-- /HIGHWIRE -->
    </TR>
<TR><TD VALIGN=TOP><a name="B56">56.</a>
    <TD VALIGN=TOP>

Brugg, B.,
 and Matus, A.



(1991)

<I>J. Cell Biol.</I>
<B>114</B>,
735-743<!-- HIGHWIRE ID="275:27:20578:56" --><A HREF="/cgi/ijlink?linkType=ABST&journalCode=jcb&resid=114/4/735">[Abstract]</A><!-- /HIGHWIRE -->
    </TR>
<TR><TD VALIGN=TOP><a name="B57">57.</a>
    <TD VALIGN=TOP>

Birge, R. B.,
Knudsen, B. S.,
Besser, D.,
 and Hanafusa, H.



(1996)

<I>Genes Cells</I>
<B>1</B>,
595-613<!-- HIGHWIRE ID="275:27:20578:57" --><A HREF="/cgi/external_ref?access_num=9078388&link_type=MED">[Medline]</A> 
<A HREF="/cgi/external_ref?access_num=9078388&displayid=77141&link_type=INFOTRIEVE">[Order&nbsp;article&nbsp;via&nbsp;Infotrieve]</A><!-- /HIGHWIRE -->
    </TR>
<TR><TD VALIGN=TOP><a name="B58">58.</a>
    <TD VALIGN=TOP>

Cheadle, C.,
Ivashchenko, Y.,
South, V.,
Searfoss, G. H.,
French, S.,
Howk, K.,
Ricca, G. A.,
 and Jaye, M.



(1994)

<I>J. Biol. Chem.</I>
<B>269</B>,
24034-24039<!-- HIGHWIRE ID="275:27:20578:58" --><A HREF="/cgi/ijlink?linkType=ABST&journalCode=jbc&resid=269/39/24034"><nobr>[Abstract/<font COLOR="CC0000">Free</font> Full&nbsp;Text]</nobr></A><!-- /HIGHWIRE -->
    </TR>
<TR><TD VALIGN=TOP><a name="B59">59.</a>
    <TD VALIGN=TOP>

Mutsushita, M.,
Yamadori, T.,
Kato, S.,
Takemoto, Y.,
Inazawa, J.,
Baba, Y.,
Hashimoto, S.,
Sekine, S.,
Arai, S.,
Kunikata, T.,
Kurimoto, M.,
Kishimoto, T.,
 and Tsukada, S.



(1998)

<I>Biochem. Biophys. Res. Commun.</I>
<B>245</B>,
337-343<!-- HIGHWIRE ID="275:27:20578:59" --><A HREF="/cgi/external_ref?access_num=10.1006/bbrc.1998.8420&link_type=DOI">[CrossRef]</A><A HREF="/cgi/external_ref?access_num=9571151&link_type=MED">[Medline]</A> 
<A HREF="/cgi/external_ref?access_num=9571151&displayid=77141&link_type=INFOTRIEVE">[Order&nbsp;article&nbsp;via&nbsp;Infotrieve]</A><!-- /HIGHWIRE -->
    </TR>
<TR><TD VALIGN=TOP><a name="B60">60.</a>
    <TD VALIGN=TOP>

Xu, W.,
Harrison, S. C.,
 and Eck, M. J.



(1997)

<I>Nature</I>
<B>385</B>,
595-602<!-- HIGHWIRE ID="275:27:20578:60" --><A HREF="/cgi/external_ref?access_num=9024657&link_type=MED">[Medline]</A> 
<A HREF="/cgi/external_ref?access_num=9024657&displayid=77141&link_type=INFOTRIEVE">[Order&nbsp;article&nbsp;via&nbsp;Infotrieve]</A><!-- /HIGHWIRE -->
    </TR>
<TR><TD VALIGN=TOP><a name="B61">61.</a>
    <TD VALIGN=TOP>

Sicheri, F.,
Moarefi, I.,
 and Kuriyan, J.



(1997)

<I>Nature</I>
<B>385</B>,
602-609<!-- HIGHWIRE ID="275:27:20578:61" --><A HREF="/cgi/external_ref?access_num=9024658&link_type=MED">[Medline]</A> 
<A HREF="/cgi/external_ref?access_num=9024658&displayid=77141&link_type=INFOTRIEVE">[Order&nbsp;article&nbsp;via&nbsp;Infotrieve]</A><!-- /HIGHWIRE -->
    </TR>
<TR><TD VALIGN=TOP><a name="B62">62.</a>
    <TD VALIGN=TOP>

Goode, B. L.,
Denis, P. E.,
Panda, D.,
Radeke, M. J.,
Miller, H. P.,
Wilson, L.,
 and Feinstein, S. C.



(1997)

<I>Mol. Biol. Cell</I>
<B>8</B>,
353-365<!-- HIGHWIRE ID="275:27:20578:62" --><A HREF="/cgi/ijlink?linkType=ABST&journalCode=molbiolcell&resid=8/2/353">[Abstract]</A><!-- /HIGHWIRE -->
    </TR>
<TR><TD VALIGN=TOP><a name="B63">63.</a>
    <TD VALIGN=TOP>

Dalgarno, D. C.,
Botfield, M. C.,
 and Rickles, R. J.



(1997)

<I>Biopolymers</I>
<B>43</B>,
383-400<!-- HIGHWIRE ID="275:27:20578:63" --><A HREF="/cgi/external_ref?access_num=10.1002/(SICI)1097-0282(1997)43:5<383::AID-BIP4>3.3.CO;2-G&link_type=DOI">[CrossRef]</A><A HREF="/cgi/external_ref?access_num=9566119&link_type=MED">[Medline]</A> 
<A HREF="/cgi/external_ref?access_num=9566119&displayid=77141&link_type=INFOTRIEVE">[Order&nbsp;article&nbsp;via&nbsp;Infotrieve]</A><!-- /HIGHWIRE -->
    </TR>
<TR><TD VALIGN=TOP><a name="B64">64.</a>
    <TD VALIGN=TOP>

Morales, M.,
 and Fifkova, E.



(1989)

<I>Cell Tissue Res.</I>
<B>256</B>,
447-456<!-- HIGHWIRE ID="275:27:20578:64" --><A HREF="/cgi/external_ref?access_num=2743390&link_type=MED">[Medline]</A> 
<A HREF="/cgi/external_ref?access_num=2743390&displayid=77141&link_type=INFOTRIEVE">[Order&nbsp;article&nbsp;via&nbsp;Infotrieve]</A><!-- /HIGHWIRE -->
    </TR>
<TR><TD VALIGN=TOP><a name="B65">65.</a>
    <TD VALIGN=TOP>

Matten, W. T.,
Aubry, M.,
West, J.,
 and Maness, P. F.



(1990)

<I>J. Cell Biol.</I>
<B>111</B>,
1959-1970<!-- HIGHWIRE ID="275:27:20578:65" --><A HREF="/cgi/ijlink?linkType=ABST&journalCode=jcb&resid=111/5/1959">[Abstract]</A><!-- /HIGHWIRE -->
    </TR>
<TR><TD VALIGN=TOP><a name="B66">66.</a>
    <TD VALIGN=TOP>

Schweiger, A.,
Mihalache, O.,
Ecke, M.,
 and Gerisch, G.



(1992)

<I>J. Cell Sci.</I>
<B>102</B>,
601-609<!-- HIGHWIRE ID="275:27:20578:66" --><A HREF="/cgi/ijlink?linkType=ABST&journalCode=joces&resid=102/3/601">[Abstract]</A><!-- /HIGHWIRE -->
    </TR>
<TR><TD VALIGN=TOP><a name="B67">67.</a>
    <TD VALIGN=TOP>

Howard, P. K.,
Sefton, B. M.,
 and Firtel, R. A.



(1993)

<I>Science</I>
<B>259</B>,
241-244<!-- HIGHWIRE ID="275:27:20578:67" --><A HREF="/cgi/external_ref?access_num=7678470&link_type=MED">[Medline]</A> 
<A HREF="/cgi/external_ref?access_num=7678470&displayid=77141&link_type=INFOTRIEVE">[Order&nbsp;article&nbsp;via&nbsp;Infotrieve]</A><!-- /HIGHWIRE -->
    </TR>
<TR><TD VALIGN=TOP><a name="B68">68.</a>
    <TD VALIGN=TOP>

Jiang, W. G.,
Hiscox, S.,
Singhrao, S. K.,
Puntis, M. C.,
Nakamura, T.,
Mansel, R. E.,
 and Hallett, M. B.



(1995)

<I>Biochem. Biophys. Res. Commun.</I>
<B>217</B>,
1062-1069<!-- HIGHWIRE ID="275:27:20578:68" --><A HREF="/cgi/external_ref?access_num=10.1006/bbrc.1995.2877&link_type=DOI">[CrossRef]</A><A HREF="/cgi/external_ref?access_num=8554558&link_type=MED">[Medline]</A> 
<A HREF="/cgi/external_ref?access_num=8554558&displayid=77141&link_type=INFOTRIEVE">[Order&nbsp;article&nbsp;via&nbsp;Infotrieve]</A><!-- /HIGHWIRE -->
    </TR>
<TR><TD VALIGN=TOP><a name="B69">69.</a>
    <TD VALIGN=TOP>

Yin, H. L.,
 and Stull, J. T.



(1999)

<I>J. Biol. Chem.</I>
<B>274</B>,
32529-32530<!-- HIGHWIRE ID="275:27:20578:69" --><A HREF="/cgi/ijlink?linkType=FULL&journalCode=jbc&resid=274/46/32529"><nobr>[<font COLOR="CC0000">Free</font> Full&nbsp;Text]</nobr></A><!-- /HIGHWIRE -->
    </TR>
<TR><TD VALIGN=TOP><a name="B70">70.</a>
    <TD VALIGN=TOP>

Chardin, P.,
Cussac, D.,
Maignan, S.,
 and Ducruix, A.



(1995)

<I>FEBS Lett.</I>
<B>369</B>,
47-51<!-- HIGHWIRE ID="275:27:20578:70" --><A HREF="/cgi/external_ref?access_num=10.1016/0014-5793(95)00578-W&link_type=DOI">[CrossRef]</A><A HREF="/cgi/external_ref?access_num=7641883&link_type=MED">[Medline]</A> 
<A HREF="/cgi/external_ref?access_num=7641883&displayid=77141&link_type=INFOTRIEVE">[Order&nbsp;article&nbsp;via&nbsp;Infotrieve]</A><!-- /HIGHWIRE -->
    </TR>
<TR><TD VALIGN=TOP><a name="B71">71.</a>
    <TD VALIGN=TOP>

Schlessinger, J.



(1994)

<I>Curr. Opin. Genet. Dev.</I>
<B>4</B>,
25-30<!-- HIGHWIRE ID="275:27:20578:71" --><A HREF="/cgi/external_ref?access_num=8193536&link_type=MED">[Medline]</A> 
<A HREF="/cgi/external_ref?access_num=8193536&displayid=77141&link_type=INFOTRIEVE">[Order&nbsp;article&nbsp;via&nbsp;Infotrieve]</A><!-- /HIGHWIRE -->
    </TR>
<TR><TD VALIGN=TOP><a name="B72">72.</a>
    <TD VALIGN=TOP>

den Hertog, J.,
 and Hunter, T.



(1996)

<I>EMBO J.</I>
<B>15</B>,
3016-3027<!-- HIGHWIRE ID="275:27:20578:72" --><A HREF="/cgi/ijlink?linkType=ABST&journalCode=emboj&resid=15/12/3016">[Abstract]</A><!-- /HIGHWIRE -->
    </TR>
<TR><TD VALIGN=TOP><a name="B73">73.</a>
    <TD VALIGN=TOP>

Goedert, M.,
Wischik, C. M.,
Crowther, R. A.,
Walker, J. E.,
 and Klug, A.



(1988)

<I>Proc. Natl. Acad. Sci. U.&nbsp;S.&nbsp;A.</I>
<B>85</B>,
4051-4055<!-- HIGHWIRE ID="275:27:20578:73" --><A HREF="/cgi/external_ref?access_num=3131773&link_type=MED">[Medline]</A> 
<A HREF="/cgi/external_ref?access_num=3131773&displayid=77141&link_type=INFOTRIEVE">[Order&nbsp;article&nbsp;via&nbsp;Infotrieve]</A><!-- /HIGHWIRE -->
    </TR>
<TR><TD VALIGN=TOP><a name="B74">74.</a>
    <TD VALIGN=TOP>

Doll, T.,
Papandrikopoulou, A.,
 and Matus, A.



(1990)

<I>Nucleic Acids Res.</I>
<B>18</B>,
361<!-- HIGHWIRE ID="275:27:20578:74" --><A HREF="/cgi/external_ref?access_num=2326166&link_type=MED">[Medline]</A> 
<A HREF="/cgi/external_ref?access_num=2326166&displayid=77141&link_type=INFOTRIEVE">[Order&nbsp;article&nbsp;via&nbsp;Infotrieve]</A><!-- /HIGHWIRE -->
    </TR>

</TABLE>



<P><HR>
<FONT SIZE=-1><A HREF="/misc/terms.shtml"><B>Copyright &#169; 2000 by The American Society for Biochemistry and Molecular Biology, Inc.</B></A></FONT>




<!--

   Pages created by the Electronic Press Engine from

		Atypon Systems, Inc.

   Visit http://www.atypon.com/

-->


</vardef>

		
        
        
	
	
	



 




<BR CLEAR=ALL>




<A NAME="otherarticles"><!-- null --></A>
<P>
<FONT SIZE=+2><STRONG>This article has been cited by other articles:</STRONG></FONT>
<P>




<TABLE CELLPADDING="0" CELLSPACING="0" BORDER="0" WIDTH="600">

<TR BGCOLOR="#D9D4C5">

<TD ALIGN="CENTER" VALIGN="TOP" WIDTH="25"><IMG ALT="" WIDTH="25" HEIGHT="5" SRC="/icons/spacer.gif"><BR>

</TD>
<TD WIDTH="5"><IMG ALT="" WIDTH="5" HEIGHT="5" SRC="/icons/spacer.gif"></TD>

<TD ALIGN="CENTER" VALIGN="TOP" WIDTH="61" NOWRAP>
	<A HREF="http://www.jneurochem.org"><IMG ALT="Home page" WIDTH="59" HEIGHT="80" VSPACE="5" BORDER="1" SRC="http://www.jneurochem.org/portal_imgs/search_result.gif"></A><BR>
</TD>
<TD WIDTH="5"><IMG ALT="" WIDTH="5" HEIGHT="5" SRC="/icons/spacer.gif"></TD>
<TD VALIGN="TOP" WIDTH="499" NOWRAP>
<A HREF="http://www.jneurochem.org"><IMG ALT="J Neurochem" WIDTH="450" HEIGHT="16" VSPACE="5" BORDER="0" SRC="/icons/shared/searchall/journals/jneurochem.gif"><IMG ALT="Home page" BORDER="0" WIDTH="49" HEIGHT="16" VSPACE="5" SRC="/icons/shared/search/homelink.gif"></A><BR>
<FONT SIZE="-1" FACE="verdana,arial,helvetica">
	C. A. Farah, M. D. Nguyen, J. P. Julien, and N. Leclerc<BR>
	<STRONG>Altered levels and distribution of microtubule-associated proteins before disease onset in a mouse model of amyotrophic lateral sclerosis</STRONG><BR>
	
		
			J. Neurochem.,
		
	
        
        January&nbsp;1,&nbsp;2003;
	84(1):
	77 - 86.
<BR>
<A HREF="http://www.jneurochem.org/cgi/content/abstract/84/1/77">[Abstract]</A>

<A HREF="http://www.jneurochem.org/cgi/content/full/84/1/77">[Full Text]</A>
<A HREF="http://www.jneurochem.org/cgi/reprint/84/1/77">[PDF]</A>
<BR><IMG ALT="" WIDTH="499" HEIGHT="1" SRC="/icons/spacer.gif">
</FONT></TD>
<TD WIDTH="5"><IMG ALT="" WIDTH="5" HEIGHT="5" SRC="/icons/spacer.gif"></TD>
</TR>
<TR><TD COLSPAN="6"><HR NOSHADE SIZE="1" WIDTH="600" COLOR="#828282" ALIGN="LEFT"></TD></TR>
</TABLE>



<TABLE CELLPADDING="0" CELLSPACING="0" BORDER="0" WIDTH="600">

<TR BGCOLOR="#D9D4C5">

<TD ALIGN="CENTER" VALIGN="TOP" WIDTH="25"><IMG ALT="" WIDTH="25" HEIGHT="5" SRC="/icons/spacer.gif"><BR>

</TD>
<TD WIDTH="5"><IMG ALT="" WIDTH="5" HEIGHT="5" SRC="/icons/spacer.gif"></TD>

<TD ALIGN="CENTER" VALIGN="TOP" WIDTH="61" NOWRAP>
	<A HREF="http://www.jcb.org"><IMG ALT="Home page" WIDTH="59" HEIGHT="80" VSPACE="5" BORDER="1" SRC="http://www.jcb.org/portal_imgs/search_result.gif"></A><BR>
</TD>
<TD WIDTH="5"><IMG ALT="" WIDTH="5" HEIGHT="5" SRC="/icons/spacer.gif"></TD>
<TD VALIGN="TOP" WIDTH="499" NOWRAP>
<A HREF="http://www.jcb.org"><IMG ALT="J. Cell Biol." WIDTH="450" HEIGHT="16" VSPACE="5" BORDER="0" SRC="/icons/shared/searchall/journals/jcb.gif"><IMG ALT="Home page" BORDER="0" WIDTH="49" HEIGHT="16" VSPACE="5" SRC="/icons/shared/search/homelink.gif"></A><BR>
<FONT SIZE="-1" FACE="verdana,arial,helvetica">
	J. Al-Bassam, R. S. Ozer, D. Safer, S. Halpain, and R. A. Milligan<BR>
	<STRONG>MAP2 and tau bind longitudinally along the outer ridges of microtubule protofilaments</STRONG><BR>
	
		
			J. Cell Biol.,
		
	
        
        June&nbsp;24,&nbsp;2002;
	157(7):
	1187 - 1196.
<BR>
<A HREF="http://www.jcb.org/cgi/content/abstract/157/7/1187">[Abstract]</A>

<A HREF="http://www.jcb.org/cgi/content/full/157/7/1187">[Full Text]</A>
<A HREF="http://www.jcb.org/cgi/reprint/157/7/1187">[PDF]</A>
<BR><IMG ALT="" WIDTH="499" HEIGHT="1" SRC="/icons/spacer.gif">
</FONT></TD>
<TD WIDTH="5"><IMG ALT="" WIDTH="5" HEIGHT="5" SRC="/icons/spacer.gif"></TD>
</TR>
<TR><TD COLSPAN="6"><HR NOSHADE SIZE="1" WIDTH="600" COLOR="#828282" ALIGN="LEFT"></TD></TR>
</TABLE>



<TABLE CELLPADDING="0" CELLSPACING="0" BORDER="0" WIDTH="600">

<TR BGCOLOR="#D9D4C5">

<TD ALIGN="CENTER" VALIGN="TOP" WIDTH="25"><IMG ALT="" WIDTH="25" HEIGHT="5" SRC="/icons/spacer.gif"><BR>

</TD>
<TD WIDTH="5"><IMG ALT="" WIDTH="5" HEIGHT="5" SRC="/icons/spacer.gif"></TD>

<TD ALIGN="CENTER" VALIGN="TOP" WIDTH="61" NOWRAP>
	<A HREF="http://jcs.biologists.org"><IMG ALT="Home page" WIDTH="59" HEIGHT="80" VSPACE="5" BORDER="1" SRC="http://jcs.biologists.org/portal_imgs/search_result.gif"></A><BR>
</TD>
<TD WIDTH="5"><IMG ALT="" WIDTH="5" HEIGHT="5" SRC="/icons/spacer.gif"></TD>
<TD VALIGN="TOP" WIDTH="499" NOWRAP>
<A HREF="http://jcs.biologists.org"><IMG ALT="J Cell Sci" WIDTH="450" HEIGHT="16" VSPACE="5" BORDER="0" SRC="/icons/shared/searchall/journals/joces.gif"><IMG ALT="Home page" BORDER="0" WIDTH="49" HEIGHT="16" VSPACE="5" SRC="/icons/shared/search/homelink.gif"></A><BR>
<FONT SIZE="-1" FACE="verdana,arial,helvetica">
	D. Belanger, C. A. Farah, M. D. Nguyen, M. Lauzon, S. Cornibert, and N. Leclerc<BR>
	<STRONG>The projection domain of MAP2b regulates microtubule protrusion and process formation in Sf9 cells</STRONG><BR>
	
		
			J. Cell Sci.,
		
	
        
        January&nbsp;4,&nbsp;2002;
	115(7):
	1523 - 1539.
<BR>
<A HREF="http://jcs.biologists.org/cgi/content/abstract/115/7/1523">[Abstract]</A>

<A HREF="http://jcs.biologists.org/cgi/content/full/115/7/1523">[Full Text]</A>
<A HREF="http://jcs.biologists.org/cgi/reprint/115/7/1523">[PDF]</A>
<BR><IMG ALT="" WIDTH="499" HEIGHT="1" SRC="/icons/spacer.gif">
</FONT></TD>
<TD WIDTH="5"><IMG ALT="" WIDTH="5" HEIGHT="5" SRC="/icons/spacer.gif"></TD>
</TR>
<TR><TD COLSPAN="6"><HR NOSHADE SIZE="1" WIDTH="600" COLOR="#828282" ALIGN="LEFT"></TD></TR>
</TABLE>



<TABLE CELLPADDING="0" CELLSPACING="0" BORDER="0" WIDTH="600">

<TR BGCOLOR="#D9D4C5">

<TD ALIGN="CENTER" VALIGN="TOP" WIDTH="25"><IMG ALT="" WIDTH="25" HEIGHT="5" SRC="/icons/spacer.gif"><BR>

</TD>
<TD WIDTH="5"><IMG ALT="" WIDTH="5" HEIGHT="5" SRC="/icons/spacer.gif"></TD>

<TD ALIGN="CENTER" VALIGN="TOP" WIDTH="61" NOWRAP>
	<A HREF="http://www.jbc.org"><IMG ALT="Home page" WIDTH="59" HEIGHT="80" VSPACE="5" BORDER="1" SRC="http://www.jbc.org/portal_imgs/search_result.gif"></A><BR>
</TD>
<TD WIDTH="5"><IMG ALT="" WIDTH="5" HEIGHT="5" SRC="/icons/spacer.gif"></TD>
<TD VALIGN="TOP" WIDTH="499" NOWRAP>
<A HREF="http://www.jbc.org"><IMG ALT="J. Biol. Chem." WIDTH="450" HEIGHT="16" VSPACE="5" BORDER="0" SRC="/icons/shared/searchall/journals/jbc.gif"><IMG ALT="Home page" BORDER="0" WIDTH="49" HEIGHT="16" VSPACE="5" SRC="/icons/shared/search/homelink.gif"></A><BR>
<FONT SIZE="-1" FACE="verdana,arial,helvetica">
	S. P. Zamora-Leon, G. Lee, P. Davies, and B. Shafit-Zagardo<BR>
	<STRONG>Binding of Fyn to MAP-2c through an SH3 Binding Domain. REGULATION OF THE INTERACTION BY ERK2</STRONG><BR>
	
		
			J. Biol. Chem.,
		
	
        
        October&nbsp;26,&nbsp;2001;
	276(43):
	39950 - 39958.
<BR>
<A HREF="http://www.jbc.org/cgi/content/abstract/276/43/39950">[Abstract]</A>

<A HREF="http://www.jbc.org/cgi/content/full/276/43/39950">[Full Text]</A>
<A HREF="http://www.jbc.org/cgi/reprint/276/43/39950">[PDF]</A>
<BR><IMG ALT="" WIDTH="499" HEIGHT="1" SRC="/icons/spacer.gif">
</FONT></TD>
<TD WIDTH="5"><IMG ALT="" WIDTH="5" HEIGHT="5" SRC="/icons/spacer.gif"></TD>
</TR>
<TR><TD COLSPAN="6"><HR NOSHADE SIZE="1" WIDTH="600" COLOR="#828282" ALIGN="LEFT"></TD></TR>
</TABLE>



<TABLE CELLPADDING="0" CELLSPACING="0" BORDER="0" WIDTH="600">

<TR BGCOLOR="#D9D4C5">

<TD ALIGN="CENTER" VALIGN="TOP" WIDTH="25"><IMG ALT="" WIDTH="25" HEIGHT="5" SRC="/icons/spacer.gif"><BR>

</TD>
<TD WIDTH="5"><IMG ALT="" WIDTH="5" HEIGHT="5" SRC="/icons/spacer.gif"></TD>

<TD ALIGN="CENTER" VALIGN="TOP" WIDTH="61" NOWRAP>
	<A HREF="http://ajp.amjpathol.org"><IMG ALT="Home page" WIDTH="59" HEIGHT="80" VSPACE="5" BORDER="1" SRC="http://ajp.amjpathol.org/portal_imgs/search_result.gif"></A><BR>
</TD>
<TD WIDTH="5"><IMG ALT="" WIDTH="5" HEIGHT="5" SRC="/icons/spacer.gif"></TD>
<TD VALIGN="TOP" WIDTH="499" NOWRAP>
<A HREF="http://ajp.amjpathol.org"><IMG ALT="Am J Pathol" WIDTH="450" HEIGHT="16" VSPACE="5" BORDER="0" SRC="/icons/shared/searchall/journals/amjpathol.gif"><IMG ALT="Home page" BORDER="0" WIDTH="49" HEIGHT="16" VSPACE="5" SRC="/icons/shared/search/homelink.gif"></A><BR>
<FONT SIZE="-1" FACE="verdana,arial,helvetica">
	D. Fang, J. Hallman, N. Sangha, T. E. Kute, J. A. Hammarback, W. L. White, and V. Setaluri<BR>
	<STRONG>Expression of Microtubule-Associated Protein 2 in Benign and Malignant Melanocytes : Implications for Differentiation and Progression of Cutaneous Melanoma</STRONG><BR>
	
		
			Am. J. Pathol.,
		
	
        
        June&nbsp;1,&nbsp;2001;
	158(6):
	2107 - 2115.
<BR>
<A HREF="http://ajp.amjpathol.org/cgi/content/abstract/158/6/2107">[Abstract]</A>

<A HREF="http://ajp.amjpathol.org/cgi/content/full/158/6/2107">[Full Text]</A>
<A HREF="http://ajp.amjpathol.org/cgi/reprint/158/6/2107">[PDF]</A>
<BR><IMG ALT="" WIDTH="499" HEIGHT="1" SRC="/icons/spacer.gif">
</FONT></TD>
<TD WIDTH="5"><IMG ALT="" WIDTH="5" HEIGHT="5" SRC="/icons/spacer.gif"></TD>
</TR>
<TR><TD COLSPAN="6"><HR NOSHADE SIZE="1" WIDTH="600" COLOR="#828282" ALIGN="LEFT"></TD></TR>
</TABLE>



<TABLE CELLPADDING="0" CELLSPACING="0" BORDER="0" WIDTH="600">

<TR BGCOLOR="#D9D4C5">

<TD ALIGN="CENTER" VALIGN="TOP" WIDTH="25"><IMG ALT="" WIDTH="25" HEIGHT="5" SRC="/icons/spacer.gif"><BR>

</TD>
<TD WIDTH="5"><IMG ALT="" WIDTH="5" HEIGHT="5" SRC="/icons/spacer.gif"></TD>

<TD ALIGN="CENTER" VALIGN="TOP" WIDTH="61" NOWRAP>
	<A HREF="http://www.molbiolcell.org"><IMG ALT="Home page" WIDTH="59" HEIGHT="80" VSPACE="5" BORDER="1" SRC="http://www.molbiolcell.org/portal_imgs/search_result.gif"></A><BR>
</TD>
<TD WIDTH="5"><IMG ALT="" WIDTH="5" HEIGHT="5" SRC="/icons/spacer.gif"></TD>
<TD VALIGN="TOP" WIDTH="499" NOWRAP>
<A HREF="http://www.molbiolcell.org"><IMG ALT="Mol. Biol. Cell" WIDTH="450" HEIGHT="16" VSPACE="5" BORDER="0" SRC="/icons/shared/searchall/journals/molbiolcell.gif"><IMG ALT="Home page" BORDER="0" WIDTH="49" HEIGHT="16" VSPACE="5" SRC="/icons/shared/search/homelink.gif"></A><BR>
<FONT SIZE="-1" FACE="verdana,arial,helvetica">
	R. S. Ozer and S. Halpain<BR>
	<STRONG>Phosphorylation-dependent Localization of Microtubule-associated Protein MAP2c to the Actin Cytoskeleton</STRONG><BR>
	
		
			Mol. Biol. Cell,
		
	
        
        October&nbsp;1,&nbsp;2000;
	11(10):
	3573 - 3587.
<BR>
<A HREF="http://www.molbiolcell.org/cgi/content/abstract/11/10/3573">[Abstract]</A>

<A HREF="http://www.molbiolcell.org/cgi/content/full/11/10/3573">[Full Text]</A>

<BR><IMG ALT="" WIDTH="499" HEIGHT="1" SRC="/icons/spacer.gif">
</FONT></TD>
<TD WIDTH="5"><IMG ALT="" WIDTH="5" HEIGHT="5" SRC="/icons/spacer.gif"></TD>
</TR>
<TR><TD COLSPAN="6"><HR NOSHADE SIZE="1" WIDTH="600" COLOR="#828282" ALIGN="LEFT"></TD></TR>
</TABLE>










<BR CLEAR=ALL>
<BR CLEAR=ALL>


	
	 
		  
			                    
	


	
		<!-- beginning of outer tables -->
               
                
                <TABLE CELLPADDING=4
                ALIGN=RIGHT
                
                CELLPADDING=5 HSPACE=4 VSPACE=4 BORDER NOWRAP>
               
		<TR><TD ALIGN=LEFT NOWRAP>

		<!-- beginning of inner table -->
				<TABLE BORDER="0" CELLPADDING="0" CELLSPACING="1" NOWRAP>
		
		
	 
	
		
			<TR><TD><IMG WIDTH=9 HEIGHT=11 ALIGN=BASELINE ALT="" BORDER=0 SRC="/icons/toc/rarrowsm.gif"><FONT SIZE="-1">
				<A HREF="/cgi/content/abstract/275/27/20578">Abstract of this Article</A>  (<img src="/icons/shared/toc/free.gif">)
			</FONT></TD></TR>
		
		
	


		
		
		

		
		
		


		
		
		

		
		
		

		
		
		

		
                
                

		
	


		

		
	

		
		

		
		 

		
		 
	 		<TR><TD><IMG WIDTH=9 HEIGHT=11 ALIGN=BASELINE ALT="" BORDER=0 SRC="/icons/toc/rarrowsm.gif"><FONT SIZE="-1">

			
			 

			
			

						
				
				<STRONG><A HREF="/cgi/reprint/275/27/20578">Reprint (PDF) Version of this Article</A>
 				
			

			

		</FONT></TD></TR>
		
		


                
                


		


	



		



		
		

		
		
		
	


		


		

		

		
		
			
			
			
				<TR><TD><IMG WIDTH=9 HEIGHT=11 ALIGN=BASELINE ALT="" BORDER=0 SRC="/icons/toc/rarrowsm.gif"><FONT SIZE="-1">
				<A HREF="/cgi/citemap?id=jbc;275/27/20578">Citation Map</A>
				</FONT></TD></TR>
		


		
		<TR><TD><IMG WIDTH=9 HEIGHT=11 ALIGN=BASELINE ALT="" BORDER=0 SRC="/icons/toc/rarrowsm.gif"><FONT SIZE="-1">
		<A HREF="/cgi/mailafriend?url=http://www.jbc.org/cgi/content/full/275/27/20578&title=Regulated+Association+of+Microtubule-associated+Protein+2+%28MAP2%29+with+Src+and+Grb2%3A+Evidence+for+MAP2+as+a+Scaffolding+Protein">Email this article to a friend</A>
		</FONT></TD></TR>
		


		 
		
		


	
	
	
	
		
		
		
		
	
		
		<TR><TD><IMG WIDTH=9 HEIGHT=11 ALIGN=BASELINE ALT="" BORDER=0 SRC="/icons/toc/rarrowsm.gif"><FONT SIZE="-1">
			Similar articles found in:<BR><IMG WIDTH=9 HEIGHT=9 ALIGN=BASELINE ALT="" BORDER=0 SRC="/icons/spacer.gif"><IMG WIDTH=9 HEIGHT=9 ALIGN=BASELINE ALT="" BORDER=0 SRC="/icons/spacer.gif"><A HREF="/cgi/search?qbe=jbc;M001887200&journalcode=jbc&minscore=5000"><FONT SIZE="-1">JBC Online</FONT></A>
				
				
				<BR><IMG WIDTH=9 HEIGHT=9 ALIGN=BASELINE ALT="" BORDER=0 SRC="/icons/spacer.gif"><IMG WIDTH=9 HEIGHT=9 ALIGN=BASELINE ALT="" BORDER=0 SRC="/icons/spacer.gif"><A HREF="/cgi/external_ref?access_num=10781592&link_type=MED_NBRS"><FONT SIZE="-1">PubMed</FONT></A>
		</FONT></TD></TR>
		
		
	
	
	<TR><TD><IMG WIDTH=9 HEIGHT=11 ALIGN=BASELINE ALT="" BORDER=0 SRC="/icons/toc/rarrowsm.gif"><FONT SIZE="-1">
		<A HREF="/cgi/external_ref?access_num=10781592&link_type=PUBMED">PubMed Citation</A>
	</FONT></TD></TR>
	
	

		
		
		
		
		
		
		
			<TR><TD><IMG WIDTH=9 HEIGHT=11 ALIGN=BASELINE ALT="" BORDER=0 SRC="/icons/toc/rarrowsm.gif"><FONT SIZE="-1">
				This Article has been cited by:
			</FONT></TD></TR>
		

		
		
		


                
                


                
                

		
		
		
		
		
		<TR><TD><IMG WIDTH=9 HEIGHT=11 ALIGN=BASELINE ALT="" BORDER=0 SRC="/icons/toc/rarrowsm.gif"><FONT SIZE="-1">
		
			Search PubMed for articles by:<BR>
			<IMG WIDTH=9 HEIGHT=9 ALIGN=BASELINE ALT="" BORDER=0 SRC="/icons/spacer.gif"><IMG WIDTH=9 HEIGHT=9 ALIGN=BASELINE ALT="" BORDER=0 SRC="/icons/spacer.gif">
			
				<A HREF="/cgi/external_ref?access_num=Lim+RW&link_type=AUTHORSEARCH">Lim, R. W. L.</A>
				 || 
			
				<A HREF="/cgi/external_ref?access_num=Halpain+S&link_type=AUTHORSEARCH">Halpain, S.</A>
				
			
		
		</FONT></TD></TR>
		

		
		



		



		


		
		
		
		<TR><TD><IMG WIDTH=9 HEIGHT=11 ALIGN=BASELINE ALT="" BORDER=0 SRC="/icons/toc/rarrowsm.gif"><FONT SIZE="-1">
		Alert me when:
		<BR><IMG WIDTH=9 HEIGHT=9 ALIGN=BASELINE ALT="" BORDER=0 SRC="/icons/spacer.gif"><IMG WIDTH=9 HEIGHT=9 ALIGN=BASELINE ALT="" BORDER=0 SRC="/icons/spacer.gif"><A HREF="/cgi/alerts/ctalert?alertType=citedby&addAlert=cited_by&saveAlert=no&cited_by_criteria_resid=jbc;275/27/20578&return_type=article&return_url=%2Fcgi%2Fcontent%2Ffull%2F275%2F27%2F20578"><FONT SIZE="-1">new articles cite this article</FONT></A><BR>
		</FONT></TD></TR>


		


		
		


		
		
		
			<TR><TD><IMG WIDTH=9 HEIGHT=11 ALIGN=BASELINE ALT="" BORDER=0 SRC="/icons/toc/rarrowsm.gif"><FONT SIZE="-1">
			<A HREF="/cgi/citmgr?gca=jbc;275/27/20578">Download to Citation Manager</A>
		</FONT></TD></TR>
		
		

		
		</TABLE> 
		</TD></TR> <!-- end of inner content box -->




		
		
		
		


		
        <!-- 1xxx --><!-- 2xxx --><!-- 3xxx --><!-- 4xxx --><!-- 5xxx -->         
	        <TR><TD ALIGN=LEFT NOWRAP>   <!-- revisions table -->
		

	
	<TABLE BORDER="0" CELLPADDING="0" CELLSPACING="1" NOWRAP>
	<TR><TD><IMG WIDTH=9 HEIGHT=11 ALIGN=BASELINE ALT="" BORDER=0 SRC="/icons/toc/rarrowsm.gif"><FONT SIZE="-1">
	All Versions of this Article:	
	</FONT></TD></TR>
	         
	<TR><TD ALIGN=LEFT NOWRAP><FONT SIZE=-1><IMG WIDTH=27 HEIGHT=11 ALIGN=BASELINE ALT="" BORDER=0 SRC="/icons/spacer.gif">275/27/20578<FONT COLOR="#A70716"> (most recent)</FONT></FONT></TD></TR>
	         

	         
	<TR><TD ALIGN=LEFT NOWRAP><FONT SIZE=-1><IMG WIDTH=27 HEIGHT=11 ALIGN=BASELINE ALT="" BORDER=0 SRC="/icons/spacer.gif"><A HREF="/cgi/content/short/M001887200v1">M001887200v1</A></FONT></TD></TR>
	
	</TABLE>
	         

    
		</TD></TR>
        
        
	</TABLE>	 <!-- end of outer content box -->
	


					
	

<BR CLEAR=ALL>

        
	<P>
	<HR NOSHADE ALIGN="LEFT" WIDTH="450" SIZE="1">
	
	<TABLE CELLPADDING="2" CELLSPACING="2" WIDTH="450">
	<TR>
		<TD ALIGN="CENTER" VALIGN="MIDDLE" BGCOLOR="#006699" NOWRAP><A TARGET="_top" STYLE="text-decoration: none" HREF="/"><FONT SIZE="-2" COLOR="#FFFFFF" FACE="arial,helvetica">HOME</FONT></A></TD>
	    <TD ALIGN="CENTER" VALIGN="MIDDLE" BGCOLOR="#006699" NOWRAP><A TARGET="_top" STYLE="text-decoration: none" HREF="/help/"><FONT SIZE="-2" COLOR="#FFFFFF" FACE="arial,helvetica">HELP</FONT></A></TD>
	    <TD ALIGN="CENTER" VALIGN="MIDDLE" BGCOLOR="#006699" NOWRAP><A TARGET="_top" STYLE="text-decoration: none" HREF="/cgi/feedback"><FONT SIZE="-2" COLOR="#FFFFFF" FACE="arial,helvetica">FEEDBACK</FONT></A></TD>
	    <TD ALIGN="CENTER" VALIGN="MIDDLE" BGCOLOR="#006699" NOWRAP><A TARGET="_top" STYLE="text-decoration: none" HREF="/subscriptions/"><FONT SIZE="-2" COLOR="#FFFFFF" FACE="arial,helvetica">SUBSCRIPTIONS</FONT></A></TD>
	    <TD ALIGN="CENTER" VALIGN="MIDDLE" BGCOLOR="#006699" NOWRAP><A TARGET="_top" STYLE="text-decoration: none" HREF="/contents-by-date.0.shtml"><FONT SIZE="-2" COLOR="#FFFFFF" FACE="arial,helvetica">ARCHIVE</FONT></A></TD>
	    <TD ALIGN="CENTER" VALIGN="MIDDLE" BGCOLOR="#006699" NOWRAP><A TARGET="_top" STYLE="text-decoration: none" HREF="/search.dtl"><FONT SIZE="-2" COLOR="#FFFFFF" FACE="arial,helvetica">SEARCH</FONT></A></TD>
	    


<TD ALIGN="CENTER" VALIGN="MIDDLE" BGCOLOR="#006699" NOWRAP><A TARGET="_top" STYLE="text-decoration: none" HREF="/content/vol275/issue27/index.shtml"><FONT SIZE="-2" COLOR="#FFFFFF" FACE="arial,helvetica">TABLE OF CONTENTS</FONT></A></TD>




	</TR>
	</TABLE>
	
	


	
<TABLE CELLPADDING="2" CELLSPACING="2" BORDER="0">
<TR>
        <TD BGCOLOR="#006699" ALIGN="CENTER" NOWRAP>&nbsp;<A STYLE="text-decoration:none" HREF="http://www.asbmb.org/"><STRONG><FONT SIZE="-2" FACE="verdana,arial,helvetica" COLOR="#FFFFFF">All ASBMB Journals</A></STRONG>&nbsp;</TD>
        <TD BGCOLOR="#CC0000" ALIGN="CENTER" NOWRAP>&nbsp;<A STYLE="text-decoration:none" HREF="http://www.mcponline.org/"><STRONG><FONT SIZE="-2" FACE="verdana,arial,helvetica" COLOR="#FFFFFF">Molecular and Cellular Proteomics</A></STRONG>&nbsp;</TD>
</TR><TR>
        <TD BGCOLOR="#0099FF" ALIGN="CENTER" NOWRAP>&nbsp;<A STYLE="text-decoration:none" HREF="http://www.jlr.org/"><STRONG><FONT SIZE="-2" FACE="verdana,arial,helvetica" COLOR="#FFFFFF">Journal of Lipid Research</A></STRONG>&nbsp;</TD>
        <TD BGCOLOR="#003F77" ALIGN="CENTER" NOWRAP>&nbsp;<A STYLE="text-decoration:none" HREF="http://www.bambed.org/"><STRONG><FONT SIZE="-2" FACE="verdana,arial,helvetica" COLOR="#FFFFFF">Biochemistry and Molecular Biology Education</A></STRONG>&nbsp;</TD>
</TR>
</TABLE>

	<FONT SIZE="-1"><STRONG><A TARGET="_top" HREF="/misc/terms.shtml">Copyright &copy; 2000 by the American Society for Biochemistry and Molecular Biology.</A></STRONG></FONT>
	


</BODY>
</HTML>
