<HTML>
<HEAD>
<TITLE>
Characterization of the Mutant Visual Pigment Responsible for Congenital Night Blindness: A Biochemical and Fourier-Transform Infrared Spectroscopy Study
</TITLE>
</HEAD>
<BODY BGCOLOR="FFFFFF">
<IMG SRC="/images/v3/acs_small_logo.gif" width="170" height="26" border="0" alt="ACS Publications Division">
<BR CLEAR=all>
<A HREF="/journals/bichaw/index.html">[Journal Home Page]</A>
<A HREF="/journals/query/subscriberSearch.jsp">[Search the Journals]</A>
<A HREF="http://pubs3.acs.org/acs/journals/toc.page?incoden=bichaw&indecade=0&involume=35&inissue=23">[Table of Contents]</A>
<A onClick='sa_onclick(this.href)' onKeyPress='sa_onclick(this.href)' HREF="/cgi-bin/archive.cgi/bichaw/1996/35/i23/pdf/bi960391n.pdf">[PDF version of this article]</A>
<BR>
<!--CKBICHAw-35-23-7536/CK-->
<P><I><TI>Biochemistry</TI></I>, <B><VOL>35</VOL></B>
(<ISS>23</ISS>), <SPN>7536</SPN>
-<EPN>7545</EPN>, <PUBYR>1996</PUBYR>.


<JID>10.1021/bi960391n</JID>
<PII>S0006-2960(96)00391-1</PII>
</P><FONT SIZE=-1><B>Copyright &copy; 1996 American Chemical Society</B></FONT>
<ATL><P><FONT SIZE=+2><B>Characterization of the Mutant Visual Pigment Responsible for Congenital Night
Blindness: A Biochemical and Fourier-Transform Infrared Spectroscopy Study<A HREF="#bi960391nAF2"><IMG SRC="/images/entities/dagger.gif"></A>
</B></FONT></P></ATL>
<AUI AUINM="Zvyaga, T. A.">
<AUI AUINM="Fahmy, K.">
<AUI AUINM="Siebert, F.">
<AUI AUINM="Sakmar, T. P.">
<AUG><AUL><P>
<AU><B>Tatyana A. Zvyaga,<A HREF="#bi960391nAF3"><IMG SRC="/images/entities/Dagger.gif"></A><A HREF="#bi960391nAF5"><IMG SRC="/images/entities/Verbar.gif"></A> </B></AU>
<AU><B>Karim Fahmy,<A HREF="#bi960391nAF4"><IMG SRC="/images/entities/sect.gif"></A><A HREF="#bi960391nAF5"><IMG SRC="/images/entities/Verbar.gif"></A></B></AU>
<AU><B>Friedrich Siebert,<A HREF="#bi960391nAF4"><IMG SRC="/images/entities/sect.gif"></A> and</B></AU>
<AU><B>Thomas P. Sakmar<A HREF="#bi960391nAF1">*</A><A HREF="#bi960391nAF3"><IMG SRC="/images/entities/Dagger.gif"></A></B></AU>
</P></AUL><AFF><P><I>The Howard Hughes Medical Institute, Laboratory of Molecular Biology and Biochemistry, Rockefeller University, 1230 York
Avenue, New York, New York 10021, and Institut f&uuml;r Biophysik und Strahlenbiologie, Albert-Ludwigs-Universit&auml;t, Albertstrasse
23, 79104 Freiburg, Federal Republic of Germany
</I></P></AFF>
</AUG>
<P><I>Received February 19, 1996</I></P>
<P><I>Revised Manuscript Received April 9, 1996<A HREF="#bi960391nAF7"><IMG SRC="/images/entities/cross.gif"></A></I></P>
<ABS><P><B>Abstract:</B></P>
<P>A mutation in the gene for the rod photoreceptor molecule
rhodopsin causes congenital night
blindness. The mutation results in a replacement of
Gly<SUP>90</SUP> by an aspartic acid residue. Two
molecular
mechanisms have been proposed to explain the physiology of affected rod
cells. One involves constitutive
activity of the G90D mutant opsin [Rao, V. R., Cohen, G. B., &amp; Oprian,
D. D. (1994) <I>Nature 367</I>, 639-642]. A second involves increased photoreceptor noise caused by
thermal isomerization of the G90D
pigment chromophore [Sieving, P. A., Richards, J. E., Naarendorp, F.,
Bingham, E. L., Scott, K., &amp;
Alpern, M. (1995) <I>Proc. Natl. Acad. Sci.
</I><I>U.S.</I><I>A. 92</I>, 880-884]. Based on
existing models of rhodopsin
and <I>in vitro</I> biochemical studies of site-directed mutants,
it appears likely that Gly<SUP>90</SUP> is in the
immediate
proximity of the Schiff base chromophore linkage. We have studied
in detail the mutant pigments G90D
and G90D/E113A using biochemical and Fourier-transform infrared (FTIR)
spectroscopic methods. The
photoproduct of mutant pigment G90D, which absorbs maximally at 468 nm
and contains a protonated
Schiff base linkage, can activate transducin. However, the active
photoproduct decays rapidly to opsin
and free <I>all-trans</I>-retinal. FTIR studies of mutant G90D
show that the dark state of the pigment has
several structural features of metarhodopsin II, the active form of
rhodopsin. These include a protonated
carboxylic acid group at position Glu<SUP>113</SUP> and increased
hydrogen-bond strength of Asp<SUP>83</SUP>. Additional
results,
which relate to the structure of the active G90D photoproduct, are also
reported. Taken together, these
results may be relevant to understanding the molecular mechanism of
congenital night blindness caused
by the G90D mutation in human rhodopsin.
</P>
<HR></ABS>
<BDY>
<H4></H4>
<P>Rhodopsin is the photoreceptor molecule of the retinal rod
cell, which is responsible for dim-light vision.
Rhodopsin
contains an 11-<I>cis</I>-retinal chromophore, which is
covalently
bound to Lys<SUP>296</SUP> of the opsin via a protonated Schiff
base
linkage. The positive charge of the retinal Schiff base
is
stabilized by a negative counterion provided by the side
chain
of Glu<SUP>113</SUP> (Sakmar et al., 1989; Zhukovsky &amp; Oprian,
1989;
Nathans, 1990). Numerous mutations in the gene for
rhodopsin have been linked to the disease autosomal
dominant retinitis pigmentosa (ADRP) (Dryja et al., 1991;
Humphries et al., 1992; Sung et al., 1991). A
prominent
symptom of ADRP is night blindness, which generally
correlates with the degree of retinal degeneration and rod
cell loss. One particular rhodopsin mutation, which
results
in a Gly<SUP>90</SUP> to Asp replacement, has been shown to
cause
congenital night blindness in the absence of significant
retinal
degeneration (Sieving et al., 1995). The detailed
elucidation
of the molecular mechanism underlying congenital night
blindness may lead to a better understanding of how the
active conformation of rhodopsin is formed.
</P>
<P>In a previous <I>in vitro</I> study, mutant pigment G90D
was
shown to activate transducin under conditions where a
visible-absorbing photoproduct, i.e., a species with a
protonated Schiff base, was predominantly formed (Rao et al.,
1994). In the absence of chromophore, the mutant
opsin
also displayed constitutive activity. It was proposed
that
constitutive activity in the mutant opsin might be due to
neutralization of Glu<SUP>113</SUP> in favor of a new
electrostatic
interaction between a negative Asp<SUP>90</SUP> carboxylate and
the
protonated Lys<SUP>296</SUP> <IMG SRC="/images/gifchars/epsilon.gif" BORDER=0 >-amino group. Asp<SUP>90</SUP>
was shown to
stabilize a protonated Schiff base in the mutant pigment
G90D/E113Q (Rao et al., 1994). However, the identity
of
the counterion in mutant G90D was not determined. In
addition, it was not determined whether or not the
counterion
was identical in both the dark state and the photoproduct
of
mutant pigment G90D.
</P>
<P>We have carried out a detailed study of the Schiff base
environments of the dark and active states of mutant
pigment
G90D. It is demonstrated that the protonated Schiff
base
photoproduct of G90D is compatible with a MII-like
receptor
conformation. The ability of the photoproduct to
activate
transducin decays rapidly and with the same time course as
its Schiff base linkage. FTIR<A HREF="#bi960391nb00001"><SUP>1</SUP></A> -difference
spectroscopy was
employed to look for infrared absorption changes that
could
be assigned to Glu<SUP>113</SUP> or Asp<SUP>90</SUP> in mutant G90D.
The key
results show that the dark pigment has features of the
active
MII conformation, including a neutral
residue at position 113.
Therefore, our data support the hypothesis of an
electrostatic
competition between Asp<SUP>90</SUP> and Glu<SUP>113</SUP> in mutant
pigment
G90D as a possible cause of congenital stationary night
blindness.
</P>
<H4>Materials and Methods</H4>
<P><I>Construction and Preparation of Rhodopsin
Mutants.</I>
Site-directed mutagenesis was performed using
restriction
fragment replacement (Lo et al., 1984) in a synthetic gene
of rhodopsin (Ferretti et al., 1986), which had been
cloned
into the expression vector as described previously (Franke
et al., 1988). Mutant G90D was prepared by
replacement
of a 70-bp <I>Bgl</I>II-<I>Rsr</I>II restriction fragment
with a synthetic
duplex containing the desired codon alteration. The
construction of mutant E113A was previously described (Sakmar
et al., 1991). The double mutant G90D/E113A was made
by combining the 1371-bp <I>Avr</I>II-<I>Rsr</I>II
restriction fragment
of the plasmid containing the G90D mutation with the 4817-bp <I>Rsr</I>II-<I>Avr</I>II restriction fragment containing
the E113A
mutation. The nucleotide sequences of all cloned
synthetic
duplexes were confirmed by the chain terminator method
for DNA sequencing of purified plasmid DNA using
[<SUP>35</SUP>S]dATP<IMG SRC="/images/gifchars/alpha.gif" BORDER=0>S. Opsin genes were expressed in COS-1 cells,
and
the corresponding pigments were reconstituted with
11-<I>cis</I>-retinal and purified following the procedures described
(Franke et al., 1988, 1992; Zvyaga et al., 1994).
Pigments
were generally prepared in 100 mM NaCl/0.1% dodecyl
maltoside detergent. In order to carry out experiments
at
different pH values, concentrated buffer solutions (10&#215;)
were
added to give the desired final pH.
</P>
<P><I>UV</I><I>-</I><I>Visible Absorption Spectroscopy and
Photolysis of
Mutant Pigments.</I> Spectroscopy was performed on a
Lambda-19 Perkin-Elmer spectrophotometer at 15 <IMG SRC="/images/entities/deg.gif">C on purified
samples as previously described (Zvyaga et al., 1994).
Spectroscopy was also carried out under conditions
identical
to those used in the FTIR experiments described below
using
a <IMG SRC="/images/gifchars/lambda.gif" BORDER=0 >-17 Perkin-Elmer instrument.
</P>
<P><I>Transducin Activation Assay and Measurement of
Activity
Decay Rates.</I> A radionucleotide filter-binding assay
which
monitors the light-dependent guanine nucleotide exchange
by transducin was carried out as described previously (Min
et al., 1993). Transducin activation rates were
determined
by linear regression analysis of the increasing amount of
radioactivity retained on successive filters. The data
acquired
were used to calculate the corresponding decay rates using
single exponential functions.
</P>
<P><I>Measurement of Schiff Base Hydrolysis Rates.</I> An
acid
denaturation method was used to monitor the rate of Schiff
base hydrolysis after illumination of the recombinant pigments (Sakmar et al., 1989). After acid denaturation,
all
Schiff base-containing pigments are converted to a state
that
absorbs at 440 nm irrespective of the initial protonation
state
and absorption maximum.
</P>
<P><I>Reaction of Mutant Pigments with Hydroxylamine</I>.
The
rates of hydroxylamine reaction with mutant pigments were
determined in darkness at pH 7.0 as previously described
(Zvyaga et al., 1993).
</P>
<P><I>Sample Preparation for FTIR Spectroscopy.</I> Con
A-purified ROS rhodopsin was prepared in 0.02% dodecyl
maltoside as described (K&ouml;nig et al., 1989). Mutant
pigments
with the amino acid replacements G90D and G90D/E113A
were purified for FTIR studies as reported (Fahmy et al.,
1993). About 2 nmol of recombinant or con A-purified
rhodopsin was dried under a stream of nitrogen on a
CaF<SUB>2</SUB>
window for experiments at 0 <IMG SRC="/images/entities/deg.gif">C. Control
measurements
showed that the hygroscopic nature of the concentrated
detergent-solubilized sample ensured sufficient hydration
for
unperturbed MII formation. All samples contained 200
nmol
of potassium buffer of pH 5. H<SUB>2</SUB>O/D<SUB>2</SUB>O
exchange was carried
out as reported (Fahmy et al., 1993).
</P>
<P><I>FTIR-Difference Spectroscopy</I>. FTIR-difference
spectra
were obtained with a Bruker IFS-28 instrument with a
liquid
nitrogen-cooled MCT detector. Spectral resolution was
2
cm<SUP>-</SUP><SUP>1</SUP>. Interferograms were averaged
(<I>n</I> = 1024) before and
after illumination of the sample, and the difference
spectra
were calculated from the transformed single channel
spectra
(Ganter et al., 1990). Illumination at 0 <IMG SRC="/images/entities/deg.gif">C was carried
out
by irradiation with wavelengths above 515 nm (OG-515
filter, Schott). Under these conditions, con
A-purified
rhodopsin forms MII.
</P>
<H4>Results</H4>
<P>Rhodopsin mutant G90D, which contains an amino acid
replacement corresponding to the mutation identified to
cause
congenital night blindness, was constructed by
site-directed
mutagenesis, expressed, and purified in a detergent
solution
for biochemical and spectroscopic analysis. Mutant
G90D/E113A was prepared in parallel as a control. Both
mutants
regenerated with 11-<I>cis</I>-retinal to form stable
visible-absorbing pigments. The absorption maxima of the dark states
of
G90D and G90D/E113A were blue shifted versus native
rhodopsin by 16 and 26 nm, respectively (Figure 1,
panels
A and B). Upon illumination, the mutant pigments G90D
and G90D/E113A formed photoproducts that absorbed
maximally at 468 and 454 nm, respectively (Figure 2,
panels
A and B), indicative of protonated Schiff base pigments
similar to MI of rhodopsin. Illumination of con
A-purified
rhodopsin in detergent causes an absorption shift from 500
to 380 nm as a consequence of Schiff base deprotonation
upon formation of the active receptor state MII (not
shown).
The <IMG SRC="/images/gifchars/lambda.gif" BORDER=0 ><SUB>max</SUB> values of the dark states of both mutants
and their
photoproducts were independent of pH over the range of
5-8
(not shown). Both pigment photoproducts activated
transducin in response to light, although to lesser degrees than
rhodopsin (Figure 3).
The relative activities with respect to
rhodopsin were 0.59 and 0.43 for G90D and G90D/E113A,
respectively. These data are evaluated and summarized
in
Table 1<A HREF="#bi960391nt00001"><IMG SRC="/images/gifchars/smsun.gif"></A>.
</P><HR><TABLE><TR><TD><A HREF="/isubscribe/journals/bichaw/35/i23/figures/bi960391nf00001.html"><IMG SRC="/isubscribe/journals/bichaw/35/i23/thumb/tbi960391nf00001.gif" ALIGN="left" HSPACE=5></A></TD><TD>
Figure 1 UV-visible absorption spectra of mutant pigments.
Spectra of pigments G90D (A) and G90D/E113A (B) after
regeneration with 11-<I>cis</I>-retinal and purification in dodecyl
maltoside detergent buffer in darkness are shown. The <IMG SRC="/images/gifchars/lambda.gif" BORDER=0 ><SUB>max</SUB>
values were
484 and 474 nm, respectively.
</TD></TR><TR><TD><A HREF="/isubscribe/journals/bichaw/35/i23/figures/bi960391nf00002.html"><IMG SRC="/isubscribe/journals/bichaw/35/i23/thumb/tbi960391nf00002.gif" ALIGN="left" HSPACE=5></A></TD><TD>
Figure 2 Photobleaching spectra of mutant pigments. Spectra of
pigments G90D (A) and G90D/E113A (B) are shown before and
after illumination for 10 s at 15 <IMG SRC="/images/entities/deg.gif">C. A photobleaching
difference
spectrum is shown as an inset for each pigment. The light
minus
dark absorption differences are shown. The mutants formed
primarily photoproducts with visible-absorbing <IMG SRC="/images/gifchars/lambda.gif" BORDER=0 ><SUB>max</SUB>
values of 468
nm for G90D and 454 nm for G90D/E113A. A second smaller
spectral component absorbing at about 380 nm was also
detected.
This 380 nm-absorbing species cannot be distinguished from
free
<I>all</I>-<I>trans</I>-retinal caused by Schiff base
hydrolysis. Both mutant
photoproducts eventually decayed to 380 nm due to Schiff
base
hydrolysis (see Table 2).
</TD></TR><TR><TD><A HREF="/isubscribe/journals/bichaw/35/i23/figures/bi960391nf00003.html"><IMG SRC="/isubscribe/journals/bichaw/35/i23/thumb/tbi960391nf00003.gif" ALIGN="left" HSPACE=5></A></TD><TD>
Figure 3 Transducin activation by mutant pigments. The ability
of the mutant photoproducts to activate transducin was
evaluated
by a GTP<IMG SRC="/images/gifchars/gamma.gif" BORDER=0 >S filter-binding assay. The amount of GTP<IMG SRC="/images/gifchars/gamma.gif" BORDER=0 >S
bound
per pmol of pigment is plotted as a function of time. Closed
symbols
represent binding in the dark, while open symbols correspond
to
binding after illumination. Mutant G90D activated transducin at
a
slightly reduced level when compared with rhodopsin.
G90D/E113A
showed a more significant impairment in its ability to
activate
transducin. The spectral and biochemical properties of both
mutants
are presented quantitatively in Table 1.
</TD></TR></TABLE><HR>
<P>The protonated Schiff base photoproducts of the mutants
decayed to 380 nm-absorbing species. The results of
simultaneous measurements of visible absorbance changes,
transducin activation rates, and Schiff base hydrolysis
are
presented in Table 2<A HREF="#bi960391nt00002"><IMG SRC="/images/gifchars/smsun.gif"></A>. The 468 and 454 nm
absorptions of
the photoproducts of G90D and G90D/E113A, respectively,
decay essentially in parallel with the Schiff base as
measured
by acid denaturation. These data show that the 380 nm
absorbance is due to generation of opsin and free retinal
rather than to additional short wavelength-absorbing photoproducts with unprotonated Schiff bases. The decay
time
courses of transducin activation rates also correlate well
with
the decays of the visible-absorbing photoproducts of both
mutants. Table 2 summarizes the time constants for
the
decays. For each mutant photoproduct, the decay half-lifetimes for the visible absorbance, the rate of
transducin
activation, and the Schiff base agree within 10-15% and
are statistically identical. In contrast, a significant
difference
between the two mutant pigments is the nearly 4 times
faster
decay of the transducin activation rate of mutant G90D
(<I>t</I><SUB>1&#47;2</SUB>
= 3.9 min) versus that of mutant G90D/E113A
(<I>t</I><SUB>1&#47;2</SUB> = 14.5
min) (see Table 2). The decay of mutant G90D activity
is
also significantly faster than that of rhodopsin measured
under nearly identical conditions (<I>t</I><SUB>1&#47;2</SUB> = 18
min) (Sakmar et
al., 1989).
</P>
<P>The reactivity of hydroxylamine with the mutant pigments
is shown in Figure 4. The experiment was
carried out in
darkness in order to probe the Schiff base accessibility
to
hydroxylamine. Under the conditions of the assay,
rhodopsin
does not react with hydroxylamine (Zvyaga et al., 1993).
However, both mutants reacted with hydroxylamine in
darkness. The decay of visible absorption followed
first-order kinetics. The <I>t</I><SUB>1&#47;2</SUB> of the decay was
2.7 min for each
mutant pigment.
</P><HR><TABLE><TR><TD><A HREF="/isubscribe/journals/bichaw/35/i23/figures/bi960391nf00004.html"><IMG SRC="/isubscribe/journals/bichaw/35/i23/thumb/tbi960391nf00004.gif" ALIGN="left" HSPACE=5></A></TD><TD>
Figure 4 Rates of hydroxylamine reaction with mutant pigments
in darkness. Arbitrary units of absorbance at 484 nm for
mutant
G90D and 474 nm for mutant G90D/E113A are scaled to equal
1.0 at time equals zero. The abscissa values represent the time
after
addition of hydroxylamine. The experiment was carried out
with
100 mM hydroxylamine (pH 8.0) at 15 <IMG SRC="/images/entities/deg.gif">C. The decay curves
were
identical for the two mutants and fit well to a single
exponential
decay function. The half-time of decay (<I>t</I><SUB>1&#47;2</SUB>)
was 2.7 min for each
mutant. Rhodopsin did not react with hydroxylamine under
these
conditions in darkness.
</TD></TR></TABLE><HR>
<P>The UV-visible photobleaching-difference spectra of
hydrated films of the two mutants prepared for FTIR
studies
were measured (Figure 5). The difference
spectrum of
mutant G90D exhibits a positive lobe at 428 nm and a
negative lobe corresponding to depletion of the dark-state
absorbance at 513 nm. The difference spectrum of
mutant
G90D/E113A exhibits a positive lobe at 442 nm and a
negative lobe at 515 nm. Taking the broadness of the
absorption difference bands into account, the actual absorption maxima of the mutant photoproducts under the conditions of the FTIR measurements are in general agreement
with the spectral changes recorded in solution (Figure 2).
The shorter wavelength of maximal absorption increase
upon
formation of the G90D photoproduct in spite of its
red-shifted
absorption versus G90D/E113A in the dark (Figure 1) may
be due to a small amount of a 380 nm-absorbing photoproduct as is also observed in the spectrum taken in solution
(Figure 2). The lack of a significant absorption change
at
380 nm upon photoproduct formation for both mutants
demonstrates that, under the conditions of the FTIR experiment, predominantly protonated Schiff base photoproducts
were stabilized.
</P><HR><TABLE><TR><TD><A HREF="/isubscribe/journals/bichaw/35/i23/figures/bi960391nf00005.html"><IMG SRC="/isubscribe/journals/bichaw/35/i23/thumb/tbi960391nf00005.gif" ALIGN="left" HSPACE=5></A></TD><TD>
Figure 5 UV-visible absorption photobleaching difference spectra
of hydrated films of mutant G90D (solid line) and
G90D/E113A
(dotted line). The samples were prepared as for FTIR
spectroscopy
experiments with the exception that half the usual amounts
of
pigment and phosphate buffer (pH 5) were used. The films
were
cooled to 0 <IMG SRC="/images/entities/deg.gif">C in the spectrophotometer in the dark, and
absorption
spectra were recorded. After illumination (<IMG SRC="/images/gifchars/lambda.gif" BORDER=0 > &gt; 495 nm) for 90
s,
spectra were again recorded. The light minus dark
absorption
differences are shown. Each of the mutant spectra was stable
for
at least 15 min after illumination.
</TD></TR></TABLE><HR>
<P>FTIR-difference spectroscopy was used to probe the
protein conformational state of each mutant.
FTIR-difference
spectra obtained in H<SUB>2</SUB>O are
shown in Figure 6 for mutant
G90D (trace A), G90D/E113A (trace B), and con A-purified
ROS rhodopsin (trace C). The most prominent band of
the
MII state (positive bands) is an amide I vibration
(peptide
carbonyl stretching mode) at 1644 cm<SUP>-</SUP><SUP>1</SUP>.
It has not been
assigned to any particular peptide bond(s) but is
generally
displayed by the photoproducts of pigments that activate
transducin (see Fahmy et al., 1995). In contrast, this
band
is not displayed by a mutant photoproduct which has been
shown to be incapable of transducin activation (Fahmy et
al., 1994), nor does this absorption change occur during
formation of any earlier trapped photoproduct of
rhodopsin.
Therefore, this band is a highly sensitive marker band
for
the formation of the transducin-activating conformation of
rhodopsin, or rhodopsin mutants. Clearly, this
absorption
is observed in the protonated Schiff base photoproducts of
mutants G90D and G90D/E113A.
</P><HR><TABLE><TR><TD><A HREF="/isubscribe/journals/bichaw/35/i23/figures/bi960391nf00006.html"><IMG SRC="/isubscribe/journals/bichaw/35/i23/thumb/tbi960391nf00006.gif" ALIGN="left" HSPACE=5></A></TD><TD>
Figure 6 FTIR-difference spectra obtained at 0 <IMG SRC="/images/entities/deg.gif">C after illumination. (A) Mutant G90D. (B) Mutant G90D/E113A. (C) Native
rhodopsin. About 2 nmol of pigment was used in each
experiment.
All samples contained 200 nmol of phosphate buffer (pH 5)
and
were measured in the presence of H<SUB>2</SUB>O. Under these
conditions,
rhodopsin forms the deprotonated Schiff base photoproduct
MII.
Photoproduct absorption bands point upward, whereas bands of
the
dark states point downward. The tick interval on the
ordinate
corresponds to 0.005 absorption units.
</TD></TR></TABLE><HR>
<P>A negative absorption band of the dark state at 1626
cm<SUP>-</SUP><SUP>1</SUP>
is also reproduced in the mutant difference spectra, but
is
weakened in favor of a negative absorption at 1618
cm<SUP>-</SUP><SUP>1</SUP>,
which in rhodopsin only occurs as a shoulder (Figure 6).
More pronounced deviations from the amide I
absorption
changes occur between 1690 and 1650 cm<SUP>-</SUP><SUP>1</SUP>.
Dark rhodopsin has an intense absorption at 1656 cm<SUP>-</SUP><SUP>1</SUP>.
The lack of
this band in the difference spectra of both mutants may in
part be due to an altered C=N stretching frequency which
in the dark state of rhodopsin absorbs at this position.
However, altered amide I modes of helical segments
are
likely to contribute to absorption changes at this
frequency
as well. A similar mutational effect has been described
for
mutant E113A (J&auml;ger et al., 1994). In addition, the
positive
part of a characteristic Rho/MII-difference band at 1695
cm<SUP>-</SUP><SUP>1</SUP>
(negative)/1687 cm<SUP>-</SUP><SUP>1</SUP> (positive) is almost
abolished in the
mutant spectra. Therefore, the observed deviations
from
rhodopsin must be due to the G90D
replacement, which both
pigments have in common, and not to the second replacement
at Glu<SUP>113</SUP>, which does not induce appreciable
additional
changes. Apparently, the Gly<SUP>90</SUP> to Asp mutation affects
the
vibrational frequency of peptide carbonyl groups which are
not part of an <IMG SRC="/images/gifchars/alpha.gif" BORDER=0>-helical structure for which a C=O
stretching
mode at about 1650 cm<SUP>-</SUP><SUP>1</SUP> would be expected.
In the amide
II spectral range (1540-1560 cm<SUP>-</SUP><SUP>1</SUP>) both
recombinant
pigments show a 1550 cm<SUP>-</SUP><SUP>1</SUP> vibration not
present in the MII-difference spectrum of rhodopsin. There is also a more
structured appearance of the difference bands between 1530
and 1560 cm<SUP>-</SUP><SUP>1</SUP> for both mutants. The
predominant alterations from rhodopsin are caused by the Gly<SUP>90</SUP> to
Asp
replacement alone, since the additional Glu<SUP>113</SUP> to Ala
mutation
affects relative peak intensities but does not induce
distinct
deviations from the difference spectrum of the single
mutant.
This is also the case for the negative band at 1516
cm<SUP>-</SUP><SUP>1</SUP>
seen in the dark states of both mutants, but not in dark
rhodopsin.
</P>
<P>Outside the frequency range of the prominent amide II
vibrations, a good correspondence of peak positions can be
observed. Thus, a group of unassigned vibrations of
amino
acid side chains in the 1350-1500 cm<SUP>-</SUP><SUP>1</SUP>
range are well
reproduced among all the difference spectra. All dark
states
exhibit distinct absorptions at 1438 and 1392
cm<SUP>-</SUP><SUP>1</SUP>. The
latter is flanked by absorption increases at about 1420
and
1375 cm<SUP>-</SUP><SUP>1</SUP>. This band pattern has been
observed in the
active protonated Schiff base photoproduct of mutant E113A/A117E, but not in its inactive state (Fahmy et al., 1994).
The same holds for the doublet at 1448
cm<SUP>-</SUP><SUP>1</SUP>/1462
cm<SUP>-</SUP><SUP>1</SUP>,
which is well reproduced in both mutant photoproducts.
</P>
<P>Chromophore modes are observed between 1050 and 1300
cm<SUP>-</SUP><SUP>1</SUP> where the coupled C-C stretches and
C-H bending
vibrations of the retinal polyene chain absorb. Based
on
FTIR studies (Ganter et al., 1988a,b) and resonance Raman
experiments of retinal analogue pigments (Palings et al.,
1987), it is known that the prominent negative band at
1238
cm<SUP>-</SUP><SUP>1</SUP> in rhodopsin is a highly delocalized
mode. It contains
predominantly C<SUB>12</SUB>-C<SUB>13</SUB> stretching character
but also the C<SUB>10</SUB>-C<SUB>11</SUB> stretching mode and the
C<SUB>14</SUB>-C<SUB>15</SUB> stretch coupled to the
C<SUB>15</SUB>-H and N-H bending mode (Palings et al., 1987;
Ganter
et al., 1988a,b). This so-called
"C<SUB>12</SUB>-C<SUB>13</SUB>" mode at 1238
cm<SUP>-</SUP><SUP>1</SUP> is sensitive to perturbations at the
terminal part of the
chromophore between C<SUB>10</SUB> and the Schiff base
nitrogen.
Interestingly, the "C<SUB>12</SUB>-C<SUB>13</SUB>" stretch
in mutant G90D/E113A,
<I>as well as</I> in the single mutant G90D, is downshifted by
8
cm<SUP>-</SUP><SUP>1</SUP> as has been observed for mutants in
which the
counterion Glu<SUP>113</SUP> had been replaced by Ala (J&auml;ger et
al.,
1994; Fahmy et al., 1994).
</P>
<P>In spite of the frequency shift of the
"C<SUB>12</SUB>-C<SUB>13</SUB>" stretch,
the delocalized nature of this mode, and particularly the
involvement of the N-H bending vibration, is retained in
the mutant pigments as shown by the distinct effect of H/D
exchange on the fingerprint vibrations (Figure 7).
The H/D
exchange causes a loss of coupling between the N-H
bending mode above 1300 cm<SUP>-</SUP><SUP>1</SUP> and the C-C
stretches
below 1300 cm<SUP>-</SUP><SUP>1</SUP>. The resulting typical
upshift of the "C<SUB>12</SUB>-C<SUB>13</SUB>" stretch to 1257 cm<SUP>-</SUP><SUP>1</SUP> in
D<SUB>2</SUB>O indicates that the coupling
between the involved coordinates has essentially not
changed
in the mutants versus rhodopsin, as has also been reported
for mutant E113A (J&auml;ger et al., 1994). In H<SUB>2</SUB>O,
the lower
frequency of the "C<SUB>12</SUB>-C<SUB>13</SUB>" stretch in the
mutants uncovers
a small negative band at 1247 cm<SUP>-</SUP><SUP>1</SUP>, which in
rhodopsin is
discernible as a shoulder on the 1238 cm<SUP>-</SUP><SUP>1</SUP>
band. Therefore,
this absorption is probably not caused by
the mutations but
is merely better resolved due to less overlap with the
"C<SUB>12</SUB>-C<SUB>13</SUB>" stretch. Again, the additional
Glu<SUP>113</SUP> to Ala replacement
in the G90D background alters the relative band
intensities
but does not cause additional band shifts.
</P><HR><TABLE><TR><TD><A HREF="/isubscribe/journals/bichaw/35/i23/figures/bi960391nf00007.html"><IMG SRC="/isubscribe/journals/bichaw/35/i23/thumb/tbi960391nf00007.gif" ALIGN="left" HSPACE=5></A></TD><TD>
Figure 7 Comparison of FTIR-difference spectra of mutants G90D
and G90D/E113A recorded at 0 <IMG SRC="/images/entities/deg.gif">C in H<SUB>2</SUB>O or
D<SUB>2</SUB>O. Conditions
and conventions are as described in Figure 6.
</TD></TR></TABLE><HR>
<P>Other fingerprint vibrations are hardly affected at all by
the additional Glu<SUP>113</SUP> to Ala mutation. In both
mutant
photoproducts, the intensities of the positive absorption
bands
at 1200 cm<SUP>-</SUP><SUP>1</SUP> are increased to levels usually
observed only
in MI. Due to the protonated Schiff bases in the
mutant
photoproducts (as in MI), such a similarity is expected
and
can be explained by the positive charge of the chromophore
which increases the infrared absorption (Siebert &amp;
M&auml;ntele,
1980). The 1263 cm<SUP>-</SUP><SUP>1</SUP> (negative)/1273
cm<SUP>-</SUP><SUP>1</SUP> (positive)
difference bands are well reproduced in the spectra of
both
mutant pigments.
</P>
<P>In addition to the fingerprint modes, the retinal chromophore causes hydrogen-out-of-plane (HOOP) vibrations
below 1000 cm<SUP>-</SUP><SUP>1</SUP>. In the MII difference
spectra, however,
only the coupled C<SUB>11</SUB>-C<SUB>12</SUB> HOOP of dark
rhodopsin at 967
cm<SUP>-</SUP><SUP>1</SUP> has appreciable intensity (not shown).
This band is
downshifted in G90D by 6 cm<SUP>-</SUP><SUP>1</SUP>, and again no
further
influence on this band is exerted by the additional Glu<SUP>113</SUP>
to
Ala replacement (Figure 7).
</P>
<P>The C=O stretching modes of protonated carboxylic acid
groups absorb between 1690 and 1800 cm<SUP>-</SUP><SUP>1</SUP>.
This spectral
range is shown for mutants G90D and G90D/E113A in
Figure 8 for measurements in H<SUB>2</SUB>O and
D<SUB>2</SUB>O. It has been
shown that Asp<SUP>83</SUP> is protonated but very weakly
hydrogen
bonded in dark rhodopsin and that hydrogen bonding
increases slightly upon MII formation causing the 1767
cm<SUP>-</SUP><SUP>1</SUP>
(negative)/1752 cm<SUP>-</SUP><SUP>1</SUP> (positive) band shift of
the carbonyl
stretching vibration (Fahmy et al., 1993; Rath et al.,
1993).
The appearance of this difference band is also a marker
for
the formation of a MII-like protein conformation (Klinger
&amp; Braiman, 1992). In the difference spectra of both
mutants,
a slightly downshifted negative band of reduced intensity
is
found at 1765 cm<SUP>-</SUP><SUP>1</SUP>. It is likely that,
as in rhodopsin, the
band is caused by Asp<SUP>83</SUP>. Possible reasons for the
reduced
intensity of the C=O stretching mode of this residue
will
be discussed below. Since the photoproducts of both
recombinant pigments also show the typical 1752
cm<SUP>-</SUP><SUP>1</SUP> band,
part of this band can be assigned to protonated Asp<SUP>83</SUP> in
the
mutant photoproducts. In MII, another contribution to
the
1752 cm<SUP>-</SUP><SUP>1</SUP> absorption comes from
Glu<SUP>122</SUP>, which in dark
rhodopsin absorbs at 1734 cm<SUP>-</SUP><SUP>1</SUP> and shifts to
1752 cm<SUP>-</SUP><SUP>1</SUP> in
MII (Fahmy et al., 1993). The same band shift can be
discerned in the difference spectra of the mutants,
arguing
for unperturbed changes in the hydrogen-bond strength of
Glu<SUP>122</SUP>. These IR absorption changes have been
identified
in detergent-solubilized recombinant rhodopsin. In
reconstituted lipids, however, no upshift of the C=O
stretching
frequency of Glu<SUP>122</SUP> has been observed (Rath et al.,
1993;
De Caluw&eacute; et al., 1995). Finally, a positive band at
1711
cm<SUP>-</SUP><SUP>1</SUP> is present in the photoproduct of G90D.
It has been
shown that the C=O stretching vibration of protonated
Glu<SUP>113</SUP>
in MII absorbs at this position (J&auml;ger et al., 1994).
Therefore,
Glu<SUP>113</SUP> seems to be protonated in the photoproduct of
G90D.
</P><HR><TABLE><TR><TD><A HREF="/isubscribe/journals/bichaw/35/i23/figures/bi960391nf00008.html"><IMG SRC="/isubscribe/journals/bichaw/35/i23/thumb/tbi960391nf00008.gif" ALIGN="left" HSPACE=5></A></TD><TD>
Figure 8 FTIR-absorption changes in the spectral range of the
carbonyl-stretching vibrations of protonated carboxylic acid
groups.
Data are identical with those shown in Figure 7. Spectra
are
normalized with respect to the 1263 cm<SUP>-</SUP><SUP>1</SUP>/1270
cm<SUP>-</SUP><SUP>1</SUP> difference
band (Figure 6).
</TD></TR></TABLE><HR>
<P>The main deviations from the MII-difference spectrum are
the much more pronounced negative band in the 1725-1735
cm<SUP>-</SUP><SUP>1</SUP> range and the presence of a band at 1693
cm<SUP>-</SUP><SUP>1</SUP>. This
indicates that additional C=O stretching vibrations of
the
dark state of mutant G90D contribute to the spectrum.
The
existence of C=O stretching modes of protonated
carboxylic
acid groups different from Asp<SUP>83</SUP> and Glu<SUP>122</SUP> is
also supported
by the band shifts in D<SUB>2</SUB>O. For rhodopsin, the effects
of H/D
exchange on these bands have been described in detail
(Ganter et al., 1988a, 1989). The 1734
cm<SUP>-</SUP><SUP>1</SUP> band shifts
into a negative band at 1727 cm<SUP>-</SUP><SUP>1</SUP>, which has
been assigned
to an unusually high frequency peptide carbonyl stretching
mode insensitive to isotope exchange. In mutant G90D,
however, a shoulder at 1720 cm<SUP>-</SUP><SUP>1</SUP> is
discernible in the 1727
cm<SUP>-</SUP><SUP>1</SUP> band measured in D<SUB>2</SUB>O.
Glu<SUP>113</SUP> or Asp<SUP>90</SUP> may cause
the 1725-1735 cm<SUP>-</SUP><SUP>1</SUP> (H<SUB>2</SUB>O) and
the 1720 cm<SUP>-</SUP><SUP>1</SUP> (D<SUB>2</SUB>O)
absorption of the dark state, assuming that the D<SUB>2</SUB>O
sensitivity
of the neighboring band of Glu<SUP>122</SUP> at 1734
cm<SUP>-</SUP><SUP>1</SUP> has not
changed in the mutant. This assumption can be
validated
by inspecting the difference spectra of the double mutant
G90D/E113A. The major influence of the additional
amino
acid replacement at position 113 is indeed a distinct
reduction
of the negative 1725-1735 cm<SUP>-</SUP><SUP>1</SUP> absorption
and a concomitant decrease of the 1711 cm<SUP>-</SUP><SUP>1</SUP> band. This
shows that Glu<SUP>113</SUP>
causes a difference band at 1725-1735 cm<SUP>-</SUP><SUP>1</SUP>
(negative)/1711
cm<SUP>-</SUP><SUP>1</SUP> (positive). Thus,
Glu<SUP>113</SUP> is at least partly protonated
in the dark state as well as in the photoproduct of G90D.
</P>
<P>When measured in D<SUB>2</SUB>O, the shoulder at 1720
cm<SUP>-</SUP><SUP>1</SUP> in the
negative band at 1727 cm<SUP>-</SUP><SUP>1</SUP> is absent in
mutant G90D/E113A. It can be concluded that the 1720
cm<SUP>-</SUP><SUP>1</SUP> absorption
is indeed solely due to deuterated Glu<SUP>113</SUP> and not caused
by
a mutation-induced increase of the isotope shift of
Glu<SUP>122</SUP>
from 1734 to 1720 cm<SUP>-</SUP><SUP>1</SUP>. The C=O
stretch of deuterated
Glu<SUP>122</SUP> in the dark state of G90D/E113A is
superimposed
with the isotope-insensitive band at 1727
cm<SUP>-</SUP><SUP>1</SUP> as known
from rhodopsin. As in the difference spectra of other
mutants
that lack a counterion at position 113, a residual
absorption
band around 1710 cm<SUP>-</SUP><SUP>1</SUP> is seen in the
photoproduct of the
double mutant. However, in contrast to previously
studied
recombinant pigments, a slight sensitivity to H/D exchange
is still observed in the broad positive
absorption at 1710
cm<SUP>-</SUP><SUP>1</SUP>. Since Glu<SUP>113</SUP> is absent
in mutant G90D/E113A, we
assign this absorption to the partial protonation of Asp<SUP>90</SUP>
upon
photoactivation of G90D/E113A.
</P>
<P>In addition to the unusual 1725-1735 cm<SUP>-</SUP><SUP>1</SUP>
(negative)/1711 cm<SUP>-</SUP><SUP>1</SUP> (positive) difference band of
protonated Glu<SUP>113</SUP>,
the single amino acid replacement in G90D causes a
negative
band at 1693 cm<SUP>-</SUP><SUP>1</SUP> which is not observed in
rhodopsin. With
the given assignments of Asp<SUP>83</SUP>, Glu<SUP>113</SUP>, and
Glu<SUP>122</SUP> to bands
above 1700 cm<SUP>-</SUP><SUP>1</SUP>, we consider this band to be
caused either
by Asp<SUP>90</SUP> (which in this case must be partly protonated
in
the dark state) or by an unusually high absorbing amide I
mode. The D<SUB>2</SUB>O sensitivity of this band argues for the
Asp<SUP>90</SUP>
assignment. However, the isotope effect may as well
be
caused by overlap with the positive band at 1705
cm<SUP>-</SUP><SUP>1</SUP>
downshifted from 1711 cm<SUP>-</SUP><SUP>1</SUP>. In the
double mutant, not only
a predominant effect above 1700 cm<SUP>-</SUP><SUP>1</SUP> is
observed, but the
1693 cm<SUP>-</SUP><SUP>1</SUP> absorption is reduced as well and
has lost isotope
sensitivity.
</P>
<P>To summarize the FTIR spectroscopy results presented in
Figure 8, mutant G90D shows increased absorption of the
dark state at 1725-1735 cm<SUP>-</SUP><SUP>1</SUP>, which can be
assigned to
the C=O stretching mode of an at least partially
protonated
Glu<SUP>113</SUP>. Glu<SUP>113</SUP> stays protonated after
illumination and causes
the absorption increase at 1711 cm<SUP>-</SUP><SUP>1</SUP>. In
addition, an isotope-sensitive difference band at 1693 cm<SUP>-</SUP><SUP>1</SUP>
(negative)/1711 cm<SUP>-</SUP><SUP>1</SUP>
(positive) not present in difference spectra of rhodopsin
is
observed and may be caused by partially protonated
Asp<SUP>90</SUP>.
In the double replacement mutant G90D/E113A, only a
small
positive band around 1710 cm<SUP>-</SUP><SUP>1</SUP> can be
assigned to Asp<SUP>90</SUP>.
This residue is entirely ionized in the dark and
undergoes
only partial light-induced protonation.
</P>
<H4>Discussion</H4>
<P>The Gly<SUP>90</SUP> to Asp mutation in human rhodopsin has
been
identified as the cause of congenital night blindness.
The
molecular pathophysiology underlying the trait is unclear.
The aim of the current study was to subject the
mutant
pigment G90D to a detailed biochemical and spectroscopic
characterization. Studies of mutant pigment
G90D/E113A
were carried out in parallel as a control. When
regenerated
with chromophore, mutant G90D formed a stable pigment
with a <IMG SRC="/images/gifchars/lambda.gif" BORDER=0 ><SUB>max</SUB> value of 484 nm (Figure 1). This value
is in
good agreement with values of 480 and 483 nm reported
previously (Kaushel &amp; Khorana, 1994; Rao et al., 1994).
The <IMG SRC="/images/gifchars/lambda.gif" BORDER=0 ><SUB>max</SUB> value of mutant pigment G90D/E113A (474
nm)
also agrees well with that of a similar mutant, G90D/E113Q
(472 nm) (Rao et al., 1994).
</P>
<P>Both mutant pigments G90D and G90D/E113A formed
relatively stable photoproducts at 15 <IMG SRC="/images/entities/deg.gif">C with <IMG SRC="/images/gifchars/lambda.gif" BORDER=0 ><SUB>max</SUB>
values of
468 and 454 nm, respectively (Figure 2). These <IMG SRC="/images/gifchars/lambda.gif" BORDER=0 ><SUB>max</SUB>
values
suggest the presence of a protonated Schiff base linkage,
which decays to form opsin and free retinal without appreciable formation of a 380 nm-absorbing MII-like pigment
(Table 2). The mutants were somewhat defective in
their
ability to activate transducin after illumination (Figure
3).
For example, the rate of transducin activation by
mutant
pigment G90D was 59% of that of rhodopsin under identical
conditions (Table 1). The ability of pigment G90D to
activate transducin after illumination decayed over time.
The
rate of this activity decay was essentially identical to
the
decay of the photoproduct at 468 nm, and to the decay of
the Schiff base bond (Table 2). Thus, the 468-nm
photoproduct of mutant G90D, which contains a protonated Schiff
base chromophore linkage, activates transducin as was
previously suggested (Rao et al., 1994). The fact that
a
protonated Schiff base photoproduct can activate
transducin
contrasts with the situation found in rhodopsin (Kibelbek
et
al., 1991), but parallels previous reports on
light-dependent
transducin activation by the mutant pigment E113A/A117E
(Fahmy et al., 1994; Zvyaga et al., 1994).
</P>
<P>It can be concluded that the 454-nm photoproduct of the
control mutant G90D/E113A also activates transducin (Table
2). The major biochemical difference between the G90D
and G90D/E113A photoproducts is the nearly 4 times slower
decay of activity for G90D/E113A as compared to G90D
(Table 2). However, the salient result is that the G90D
active
photoproduct is significantly less stable
(<I>t</I><SUB>1&#47;</SUB><SUB>2</SUB> = 3.9 min)
than
that of either G90D/E113A (<I>t</I><SUB>1&#47;2</SUB> = 14.5 min)
or rhodopsin
(<I>t</I><SUB>1&#47;2</SUB> = 18 min) (Table 2; Sakmar et al.,
1989).
</P>
<P>Previous studies of mutant E113A/A117E helped to
identify marker bands in the FTIR-difference spectrum of
rhodopsin that appear to be characteristic of the active
conformation, independent of the protonation state of the
Schiff base (Fahmy et al., 1994). Based on these
marker
bands, it is suggested above that the light-induced
absorption
changes in the mutant pigments are consistent with the
formation of MII-like photoproducts. The deviations
from
a typical MII-difference spectrum, which are caused by the
introduction of an additional internal carboxylic acid
group
at position 90 in G90D, can be divided into two classes:
(1) effects on absorption bands in the C=O stretching
frequency range between 1800 and 1690 cm<SUP>-</SUP><SUP>1</SUP>,
and (2)
indirect mutational effects on peptide carbonyl and chromophore vibrations.
</P>
<P><I>Effects on Absorption Bands in the </I><I>C</I>=<I>O
Stretching
Frequency Range between </I><I>1800 and 1690
cm</I><I><SUP>-</SUP></I><I><SUP>1</SUP></I>.
The
proposed competition between Asp<SUP>90</SUP> and Glu<SUP>113</SUP>
for electrostatic interaction with the protonated Schiff base in
mutant
G90D (Rao et al., 1994) would require Glu<SUP>113</SUP> to exist in
the
protonated form. The protonation states of carboxylic
acid
groups can be investigated directly by inspection of the
C=O
stretching modes absorbing between 1690 and 1800
cm<SUP>-</SUP><SUP>1</SUP>.
The major effect of the G90D replacement is the enhancement of an absorption decrease at 1725-1730
cm<SUP>-</SUP><SUP>1</SUP>, indicative of a protonated carboxylic acid group in the dark
state
of the mutant that is not present in rhodopsin.
Furthermore,
this group becomes perturbed after photoactivation of the
mutant. A straightforward interpretation would be to
assign
this additional band to the new carboxylic acid introduced
in the mutant, i.e., Asp<SUP>90</SUP>. However, we have shown
that
this band, as well as the positive band at 1711
cm<SUP>-</SUP><SUP>1</SUP>, is largely
reduced by the replacement of Glu<SUP>113</SUP> by Ala. It will
be
shown below also that well characterized chromophore
vibrations in the dark states of mutants G90D and G90D/E113A are indicative of the protonated state of
Glu<SUP>113</SUP>.
Therefore, we assign the bands at 1725-1730
cm<SUP>-</SUP><SUP>1</SUP> (negative) and 1711 cm<SUP>-</SUP><SUP>1</SUP> (positive) to protonated
Glu<SUP>113</SUP> in the
dark state and the photoproduct of G90D, respectively.
Thus
Asp<SUP>90</SUP> must compete with Glu<SUP>113</SUP> for
electrostatic interaction
with the protonated Schiff base, which allows Glu<SUP>113</SUP> to
be
protonated.
</P>
<P>In general, a replacement of Glu<SUP>113</SUP> by a neutral
amino
acid residue greatly enhances the hydroxylamine reactivity
of the mutant pigment in darkness (Sakmar et al., 1989;
Zvyaga et al., 1993, 1994). The fact
that both mutants G90D
and G90D/E113A react with hydroxylamine in darkness
(Figure 4) also supports the conclusion that Glu<SUP>113</SUP> is at
least
partially protonated in the dark state of mutant G90D.
However, we cannot exclude that the electrostatic
competition is incomplete. In the FTIR-difference spectrum of
mutant G90D, Asp<SUP>90</SUP> may be partly protonated and cause
a
difference band at 1693 cm<SUP>-</SUP><SUP>1</SUP> (negative)/1711
cm<SUP>-</SUP><SUP>1</SUP> (positive)
as well. If this assignment is accepted, the
disappearance
of the 1693 cm<SUP>-</SUP><SUP>1</SUP> absorption in the double
mutant can be
explained by the complete replacement of the counterion
function by Asp<SUP>90</SUP> in the dark due to the lack of
Glu<SUP>113</SUP>. In
the photoproduct of the double mutant, a small fraction of
Asp<SUP>90</SUP> becomes protonated and causes the small
positive
C=O stretching absorption at 1710 cm<SUP>-</SUP><SUP>1</SUP>.
This band shows
that, in addition to substituting for the counterion
function
of Glu<SUP>113</SUP>, a small fraction of the Asp<SUP>90</SUP>
carboxylic acid side
chains may also function as proton acceptor for the Schiff
base proton in the G90D/E113A photoproduct. This
interpretation is consistent with the light-induced formation of
a
small amount of 380-nm absorbance detected in solution and
under conditions of the FTIR experiments.
</P>
<P>Our interpretation of the spectral consequences of the
amino acid replacements is based on the assumption that
the
carboxylic acid groups of Asp<SUP>83</SUP> and Glu<SUP>122</SUP> have
not changed
their previously identified frequencies measured for
rhodopsin. This assumption seems valid because the peak
positions
of these residues, as well as the magnitude of their
frequency
shifts after H/D exchange, are retained in the difference
spectrum of G90D after H/D exchange. However, the
absorption decrease of Asp<SUP>83</SUP> at 1767
cm<SUP>-</SUP><SUP>1</SUP> (negative) is less
pronounced than usual. Difference bands are only
observed
if the physical environment of a residue is changed upon
photoactivation. Therefore, the reduction of the
Asp<SUP>83</SUP>
difference band indicates that the dark state of G90D
already
has some local conformational similarities with the light-activated form, which reduces the amplitude of further
observable changes during complete formation of the active
conformation. The fact that Asp<SUP>83</SUP> lies on the same
helix as
the replaced glycine residue (helix 2) may support the
view
of a local mutationally induced conformational change.
Furthermore, the normal appearance of the difference
band
of Glu<SUP>122</SUP> on helix 3 argues for the absence of
"long-range"
effects of the G90D exchange.
</P>
<P><I>Indirect Mutational Effects on Peptide Carbonyl and
Chromophore Vibrations.</I> Pronounced deviations from
the
normal MII-difference spectrum are found at 1656
cm<SUP>-</SUP><SUP>1</SUP>,
where both mutants exhibit reduced negative bands.
The
C=N stretch of dark rhodopsin is found at this
frequency,
and the expected change in the Schiff base environment in
the mutants may in part account for the difference.
However,
the effect is too large to be entirely due to an altered
C=N
stretching mode (for an estimate of the relative intensity
of
the C=N stretching mode of rhodopsin, see Siebert et
al.,
1983). Therefore, an amide I difference band must also
be
reduced. This may again reflect a partial anticipation of
a
local active conformation as proposed for the Asp<SUP>83</SUP>
environment and would support the concept of weighted additivity
of local "off-on" transitions during photoactivation
of
rhodopsin (Fahmy et al., 1995).
</P>
<P>The lack of a peptide carbonyl band of unusual high
frequency at 1687 cm<SUP>-</SUP><SUP>1</SUP> parallels a situation
described for
Glu<SUP>122</SUP> replacement (Fahmy et al., 1993), where a band
at
1727 cm<SUP>-</SUP><SUP>1</SUP> was affected. The mutation was
located adjacent
to consecutive glycine residues, which are weak helix
stabilizers, on transmembrane helix 3. Gly<SUP>90</SUP> is also
one out
of a pair of consecutive glycine residues on transmembrane
helix 2, and the same argument may apply for mutants G90D
and G90D/E113A. The actual secondary structure may be
less defined by intrahelical C=O&#183;&#183;&#183;H-N hydrogen
bonds
than by neighboring amino acids. Thus, the
corresponding
peptide conformations may result in C=O stretching frequencies which differ from the usually observed 1650
cm<SUP>-</SUP><SUP>1</SUP>
in <IMG SRC="/images/gifchars/alpha.gif" BORDER=0>-helices.
</P>
<P>The amide II spectral range between 1540 and 1560
cm<SUP>-</SUP><SUP>1</SUP>
cannot be used as a monitor of the protein conformational
change because of the overlap with the C=C stretching
difference bands of the chromophore. Since protonated
Schiff base photoproducts are formed by the mutant pigments, the C=C stretches are expected to have higher
intensities and different frequencies when compared with
the
native MII state (Doukas et al., 1978; Rimai et al., 1973;
Siebert &amp; M&auml;ntele, 1980). In particular, the changes in
the
C=C stretching modes are expected to be similar to those
usually observed upon MI formation because this photoproduct also has a protonated Schiff base. This explains
the
more structured appearance of the difference bands between
1530 and 1560 cm<SUP>-</SUP><SUP>1</SUP> for both mutant pigments,
which exhibit
a 1550 cm<SUP>-</SUP><SUP>1</SUP> vibration not present in the
MII-difference
spectrum of rhodopsin. To the low frequency side of
the
amide II spectral range, a negative band at 1516
cm<SUP>-</SUP><SUP>1</SUP> not
seen in rhodopsin is observed in the difference spectra of
both mutants G90D and G90D/E113A. Such a band has
previously been observed in the MII-difference spectrum of
the single amino acid replacement mutant E113A (J&auml;ger
et
al., 1994). Thus, the replacement of Gly<SUP>90</SUP> by Asp
appears
to induce spectral changes which resemble those caused by
neutralization of Glu<SUP>113</SUP>.
</P>
<P>Chromophore vibrations are also affected by the G90D
replacement. The frequencies of the coupled C-C
stretching
and C-H bending vibrations between 1100 and 1250
cm<SUP>-</SUP><SUP>1</SUP>
are downshifted by 2-7 cm<SUP>-</SUP><SUP>1</SUP>. However,
the fingerprint
difference bands appear generally similar to those of
rhodopsin and show that the chromophore isomerization in both
mutants is specifically 11-<I>cis</I> to
<I>all</I>-<I>trans</I>. The small
frequency shifts can be explained by the reduced delocalization of the retinal Schiff base <IMG SRC="/images/gifchars/pi.gif" BORDER=0 >-electron system as
manifested in the blue-shifted visible absorption maximum
of mutant G90D, as well as that of mutant G90D/E113A.
Hence, the alteration of bond strength between single
and
double bonds in the retinal polyene chain is expected to
be
more pronounced in the mutants than in rhodopsin. Correspondingly, single bond and double bond stretching
frequencies should be decreased and increased,
respectively.
</P>
<P>The most pronounced effect found for the
"C<SUB>12</SUB>-C<SUB>13</SUB>"
stretching frequency of the dark states of the mutants
merits
special discussion. In rhodopsin, this delocalized mode
is
found at 1238 cm<SUP>-</SUP><SUP>1</SUP> and is sensitive to
perturbations between
C<SUB>10</SUB> and the Schiff base, including H/D exchange of the
Schiff
base proton (Palings et al., 1987; Ganter et al.,
1988a,b).
The fact that the isotope sensitivity is conserved in
both
mutants shows that the normal mode character of the 1230
cm<SUP>-</SUP><SUP>1</SUP> vibration has essentially not changed,
although the
frequency has dropped by 8 cm<SUP>-</SUP><SUP>1</SUP>. Since
the downshift is
about 3 times larger than that of the other fingerprint
modes,
it seems likely to be caused by perturbations involving
the
terminal half of the chromophore in
addition to the reduced
<IMG SRC="/images/gifchars/pi.gif" BORDER=0 >-electron delocalization discussed above.
</P>
<P>As to the nature of possible chromophore perturbations, a
comparison with previously studied mutants is most revealing. The dark state of the mutant E113A, in which
chloride
served as inorganic counterion, showed a shift of the 1238
cm<SUP>-</SUP><SUP>1</SUP> absorption to 1230
cm<SUP>-</SUP><SUP>1</SUP> (J&auml;ger et al., 1994; Lin et
al.,
1992), whereas the E113D replacement did not cause any
band shift in the fingerprint range (J&auml;ger et al.,
1994).
Therefore, the "C<SUB>12</SUB>-C<SUB>13</SUB>" frequency
seems to be largely
determined by the electrostatic chromophore environment,
rather than by specific steric interactions. Likewise, a
1230
cm<SUP>-</SUP><SUP>1</SUP> band has been observed in mutants E113Q
(Lin et al.,
1992) and E113A/A117E (Fahmy et al., 1994). The 1238
to 1230 cm<SUP>-</SUP><SUP>1</SUP> band shift is consistent with a
model of the
chromophore-binding pocket in which the carboxylate of
Glu<SUP>113</SUP> is close to C<SUB>12</SUB> of the chromophore
(Birge et al., 1990;
Han et al., 1993). An effect of the G90D replacement
on
the chromophore near C<SUB>12</SUB> is further supported by
the
downshift of the C<SUB>11</SUB>-C<SUB>12</SUB> HOOP from 967 to 961
cm<SUP>-</SUP><SUP>1</SUP>.
Again, the double mutant exhibits the same spectral
feature,
suggesting that the effect in mutant G90D is exerted
through
neutralization of Glu<SUP>113</SUP>. In summary, the most
pronounced
indirect effect of the G90D replacement on the chromophore
fingerprint vibrations is on a band (1238
cm<SUP>-</SUP><SUP>1</SUP>), which is
highly sensitive to the existence of a negative charge at
position 113. This band is found in the dark states of
mutants
G90D and G90D/E113A at a frequency indicative of a
neutral side chain at position 113.
</P>
<P><I>Relevance to Congenital Night Blindness.</I> A
molecular
mechanism was proposed for congenital night blindness
based on the observation that the mutant G90D opsin
displayed constitutive activity <I>in vitro</I> (Rao et al.,
1994).
According to the model, the Asp<SUP>90</SUP> residue destabilizes
a salt
bridge between Glu<SUP>113</SUP> and Lys<SUP>296</SUP> (Cohen et al.,
1993). The
resulting opsin activity causes the rod cell to adapt to
an
"on" signal in darkness so that an increased
dark-adapted
threshold is observed. However, it was pointed out that
this
explanation may not be totally consistent with the finding
of normal pigment density and a lack of significant
retinal
degeneration <I>in vivo</I> in individuals with night
blindness
caused by the G90D mutation (Sieving et al., 1995).
The
physiology of the rod function deficit was proposed to be
most consistent with increased basal signaling from
thermal
isomerization of the mutant pigment (Sieving et al.,
1995).
Photoreceptor noise has been previously correlated to
thermal
isomerization of an unprotonated Schiff base chromophore
(Barlow et al., 1993; Birge &amp; Barlow, 1995).
</P>
<P>Several biochemical and spectroscopic features of the
G90D pigment from the present study are relevant. The
468-nm photoproduct of pigment G90D, which is slightly
defective in its ability to activate transducin, decays
rapidly.
The ability of the photoproduct to activate transducin
decays
with the same time course. Although the primary photochemical isomerization was shown to be 11-<I>cis</I> to
<I>all</I>-<I>trans</I>,
secondary photochemical inactivation of the photoproduct
by visible light could theoretically occur. FTIR studies
show
that the dark pigment has key features of the active MII
conformation, including a neutral residue at position 113.
In addition, Asp<SUP>83</SUP> seems to be exposed to a
protein
environment in the dark which is similar to that in the
active
MII conformation (Fahmy et al., 1993; Rath et al., 1993).
Similarly, an amide I absorption change usually
occurring
upon MII formation seems to be anticipated by the
mutation.
Thus, the pigment has some structural features of the
"on
state" even in darkness.
</P>
<P>Although the Schiff base p<I>K</I><SUB>a</SUB> of pigment G90D is
above
8, it may be significantly lower than that of rhodopsin
because of the altered electrostatic environment of the
Schiff
base demonstrated by the FTIR studies. Thus, rate of
thermal
isomerization of the chromophore, and the resulting photoreceptor noise, might be expected to be increased for the
G90D pigment. However, the FTIR evidence for electrostatic competition between Asp<SUP>90</SUP> and Glu<SUP>113</SUP>
strongly argues
for a neutral chromophore binding site as in rhodopsin,
rather
than a mutational introduction of an additional negative
charge (Sieving et al., 1995). Low temperature FTIR
studies
of early G90D photoproducts will be helpful to elucidate
relevant chromophore-opsin interactions in greater
detail.
</P>
</BDY>
<BM>
<FNGR>
<PAGEREF PAGERNUM="bi003502307536">
<A NAME="bi960391nAF2"><P><IMG SRC="/images/entities/dagger.gif">
 T.P.S. is an Associate Investigator and T.A.Z. is
an Associate of
the Howard Hughes Medical Institute. K.F. is supported by DFG
Grant
Fa 248/2-1.</P>
</A>
<A NAME="bi960391nAF1"><P>*
 Address correspondence to this author at: Box 284,
Rockefeller
University, 1230 York Ave., New York, NY 10021. Tel:
212-327-8288. FAX: 212-327-8370. E mail:
sakmar@rockvax.rockefeller.edu.
</P>
</A>
<A NAME="bi960391nAF3"><P><IMG SRC="/images/entities/Dagger.gif">
 The Howard Hughes Medical Institute, Rockefeller
University.</P>
</A>
<A NAME="bi960391nAF4"><P><IMG SRC="/images/entities/sect.gif">
 Albert-Ludwigs-Universit&auml;t.</P>
</A>
<A NAME="bi960391nAF5"><P><IMG SRC="/images/entities/Verbar.gif">
 These authors contributed equally to the work
presented.
</P>
</A>
<A NAME="bi960391nAF7"><P><IMG SRC="/images/entities/cross.gif">
 Abstract published in <I>Advance ACS
Abstracts,</I> May 15, 1996.</P>
</A>
</FNGR>
<BBGR>
<PAGEREF PAGERNUM="bi003502307544">
<P>Barlow, R. B., Birge, R. R., Kaplan, E., &amp; Tallent, J. R.
(1993)
<I>Nature 336</I>, 64-66.<citation reflink="journal=Nature&year=1993&volume=336&page=64&reference=     T.P.S. is an Associate Investigator and T.A.Z. is an Associate of the Howard Hughes Medical Institute. K.F. is supported by DFG Grant Fa 248/2-1.  *  Address correspondence to this author at: Box 284, Rockefeller University, 1230 York Ave., New York, NY 10021. Tel: 212-327-8288. FAX: 212-327-8370. E mail:      The Howard Hughes Medical Institute, Rockefeller University.    Albert-Ludwigs-Universit&auml;t.    These authors contributed equally to the work presented.     Abstract published in  May 15, 1996.     Barlow, R. B., Birge, R. R., Kaplan, E., &amp; Tallent, J. R. (1993) Nature 336, 64-66.">
</P>
</A>
<P>Birge, R. R. (1990) <I>Biochim. Biophys. Acta 1016</I>,
293-327.<citation reflink="journal=Biochim.+Biophys.+Acta&year=1990&volume=1016&page=293&reference=   Birge, R. R. (1990) Biochim. Biophys. Acta 1016, 293-327."><A HREF="/cgi-bin/chemport/version=0.0&coi=1:CAS:528:DyaK3cXktF2msLw%253D&journal=Biochim.%2BBiophys.%2BActa&pubyear=1990&volume=1016&startpage=293&reference=%20%20%20Birge%2C%20R.%20R.%201990%20Biochim.%20Biophys.%20Acta%201016%2C%20293-327.&md5=562b5be26bd1d12ae8ce8eaec8d63056">[ChemPort]</A><A HREF="/cgi-bin/pubmed/db=m&form=6&uid=2184895&Dopt=r">[Medline]</A>
</P>
</A>
<P>Birge, R. R., &amp; Barlow, R. B. (1995) <I>Biophys. Chem. 55</I>,
115-126.<citation reflink="journal=Biophys.+Chem.&year=1995&volume=55&page=115&reference=   Birge, R. R., &amp; Barlow, R. B. (1995) Biophys. Chem. 55, 115-126."><A HREF="/cgi-bin/chemport/version=0.0&coi=1:CAS:528:DyaK2MXmsFGntbk%253D&journal=Biophys.%2BChem.&pubyear=1995&volume=55&startpage=115&reference=%20%20%20Birge%2C%20R.%20R.%2C%20a%20Barlow%2C%20R.%20B.%201995%20Biophys.%20Chem.%2055%2C%20115-126.&md5=8816dda4130f653d9d4c01986c50bdf8">[ChemPort]</A>
</P>
</A>
<P>Chan, T., Lee, M., &amp; Sakmar, T. P. (1991) <I>J. Biol. Chem.
267,</I>
9478-9480.<citation reflink="journal=J.+Biol.+Chem.&year=1991&volume=267&page=9478&reference=   Chan, T., Lee, M., &amp; Sakmar, T. P. (1991) J. Biol. Chem. 267, 9478-9480."><A HREF="/cgi-bin/pubmed/db=m&form=6&uid=1577792&Dopt=r">[Medline]</A>
</P>
</A>
<P>Cohen, G. B., Oprian, D. D., &amp; Robinson, P. R. (1992)
<I>Biochemistry
31</I>, 12592-12601.<citation reflink="journal=Biochemistry&year=1992&volume=31&page=12592&reference=   Cohen, G. B., Oprian, D. D., &amp; Robinson, P. R. (1992) Biochemistry 31, 12592-12601."><A HREF="/cgi-bin/chemport/version=0.0&coi=1:CAS:528:DyaK38XmsVOgurs%253D&journal=Biochemistry&pubyear=1992&volume=31&startpage=12592&reference=%20%20%20Cohen%2C%20G.%20B.%2C%20Oprian%2C%20D.%20D.%2C%20a%20Robinson%2C%20P.%20R.%201992%20Biochemistry%2031%2C%2012592-12601.&md5=30235baf178c9b8314840d6326286515">[ChemPort]</A><A HREF="/cgi-bin/pubmed/db=m&form=6&uid=1472495&Dopt=r">[Medline]</A>
</P>
</A>
<P>De Caluw&eacute;, G. L. J., Bovee-Geurts, P. H. M., Rath, P.,
Rothschild,
K. J., &amp; DeGrip, W. J. (1995) <I>Biophys. Chem.</I> <I>56</I>,
79-87.<citation reflink="journal=Biophys.+Chem.&year=1995&volume=56&page=79&reference=   De Caluw&eacute;, G. L. J., Bovee-Geurts, P. H. M., Rath, P., Rothschild, K. J., &amp; DeGrip, W. J. (1995) Biophys. Chem. 56, 79-87."><A HREF="/cgi-bin/chemport/version=0.0&coi=1:CAS:528:DyaK2MXntFCltbw%253D&journal=Biophys.%2BChem.&pubyear=1995&volume=56&startpage=79&reference=%20%20%20De%20Caluwe%2C%20G.%20L.%20J.%2C%20Bovee-Geurts%2C%20P.%20H.%20M.%2C%20Rath%2C%20P.%2C%20Rothschild%2C%20K.%20J.%2C%20a%20DeGrip%2C%20W.%20J.%201995%20Biophys.%20Chem.%2056%2C%2079-87.&md5=ac8d97fc0f799ff7510de683f73f0a55">[ChemPort]</A><A HREF="/cgi-bin/pubmed/db=m&form=6&uid=7662872&Dopt=r">[Medline]</A>
</P>
</A>
<P>Doukas, A. G., Aton, B., Callender, R. H., &amp; Ebrey, T. G.
(1978)
<I>Biochemistry 17</I>, 2430-2435.<citation reflink="journal=Biochemistry&year=1978&volume=17&page=2430&reference=   Doukas, A. G., Aton, B., Callender, R. H., &amp; Ebrey, T. G. (1978) Biochemistry 17, 2430-2435."><A HREF="/cgi-bin/chemport/version=0.0&coi=1:CAS:528:DyaE1cXksFGis7c%253D&journal=Biochemistry&pubyear=1978&volume=17&startpage=2430&reference=%20%20%20Doukas%2C%20A.%20G.%2C%20Aton%2C%20B.%2C%20Callender%2C%20R.%20H.%2C%20a%20Ebrey%2C%20T.%20G.%201978%20Biochemistry%2017%2C%202430-2435.&md5=311fe257ff372a1231be8c9e3fba5577">[ChemPort]</A><A HREF="/cgi-bin/pubmed/db=m&form=6&uid=678522&Dopt=r">[Medline]</A>
</P>
</A>
<P>Dryja, T. P., Hahn, L. B., Cowley, G. S., McGee, T. L., &amp;
Berson,
E. L. (1991) <I>Proc. Natl. Acad. </I><I>Sci.
</I><I>U.S.</I><I>A. 88</I>, 9370-9374.<citation reflink="journal=Proc.+Natl.+Acad.+Sci.+U.S.A.&year=1991&volume=88&page=9370&reference=   Dryja, T. P., Hahn, L. B., Cowley, G. S., McGee, T. L., &amp; Berson, E. L. (1991) Proc. Natl. Acad. Sci. U.S.A. 88, 9370-9374."><A HREF="/cgi-bin/chemport/version=0.0&coi=1:CAS:528:DyaK3MXmslahtLg%253D&journal=Proc.%2BNatl.%2BAcad.%2BSci.%2BU.S.A.&pubyear=1991&volume=88&startpage=9370&reference=%20%20%20Dryja%2C%20T.%20P.%2C%20Hahn%2C%20L.%20B.%2C%20Cowley%2C%20G.%20S.%2C%20McGee%2C%20T.%20L.%2C%20a%20Berson%2C%20E.%20L.%201991%20Proc.%20Natl.%20Acad.%20Sci.%20U.S.A.%2088%2C%209370-9374.&md5=59da7d3be272928ed84dcf64982fb978">[ChemPort]</A>
</P>
</A>
<P>Fahmy, K., J&auml;ger, F., Beck, M., Sakmar, T. P., &amp; Siebert, F.
(1993)
<I>Proc. Natl. Acad. Sci. </I><I>U.S.</I><I>A.
</I><I>90</I>, 10206-10210.<citation reflink="journal=Proc.+Natl.+Acad.+Sci.+U.S.A.&year=1993&volume=90&page=10206&reference=   Fahmy, K., J&auml;ger, F., Beck, M., Sakmar, T. P., &amp; Siebert, F. (1993) Proc. Natl. Acad. Sci. U.S.A. 90, 10206-10210."><A HREF="/cgi-bin/chemport/version=0.0&coi=1:CAS:528:DyaK2cXkvVantw%253D%253D&journal=Proc.%2BNatl.%2BAcad.%2BSci.%2BU.S.A.&pubyear=1993&volume=90&startpage=10206&reference=%20%20%20Fahmy%2C%20K.%2C%20Jager%2C%20F.%2C%20Beck%2C%20M.%2C%20Sakmar%2C%20T.%20P.%2C%20a%20Siebert%2C%20F.%201993%20Proc.%20Natl.%20Acad.%20Sci.%20U.S.A.%2090%2C%2010206-10210.&md5=b0302d8444edb9264ac902ec4142cfa2">[ChemPort]</A><A HREF="/cgi-bin/pubmed/db=m&form=6&uid=7901852&Dopt=r">[Medline]</A>
</P>
</A>
<P>Fahmy, K., Siebert, F., &amp; Sakmar, T. P. (1994) <I>Biochemistry
33</I>,
13700-13705.<citation reflink="journal=Biochemistry&year=1994&volume=33&page=13700&reference=   Fahmy, K., Siebert, F., &amp; Sakmar, T. P. (1994) Biochemistry 33, 13700-13705."><A HREF="/cgi-bin/chemport/version=0.0&coi=1:CAS:528:DyaK2cXmslCkt7Y%253D&journal=Biochemistry&pubyear=1994&volume=33&startpage=13700&reference=%20%20%20Fahmy%2C%20K.%2C%20Siebert%2C%20F.%2C%20a%20Sakmar%2C%20T.%20P.%201994%20Biochemistry%2033%2C%2013700-13705.&md5=5fba53a043f38fd9885cba724f2f9465">[ChemPort]</A><A HREF="/cgi-bin/pubmed/db=m&form=6&uid=7947779&Dopt=r">[Medline]</A>
</P>
</A>
<P>Fahmy, K., Siebert, F., &amp; Sakmar, T. P. (1995) <I>Biophys.
Chem.
56</I>, 171-181.<citation reflink="journal=Biophys.+Chem.&year=1995&volume=56&page=171&reference=   Fahmy, K., Siebert, F., &amp; Sakmar, T. P. (1995) Biophys. Chem. 56, 171-181."><A HREF="/cgi-bin/chemport/version=0.0&coi=1:CAS:528:DyaK2MXntFCluro%253D&journal=Biophys.%2BChem.&pubyear=1995&volume=56&startpage=171&reference=%20%20%20Fahmy%2C%20K.%2C%20Siebert%2C%20F.%2C%20a%20Sakmar%2C%20T.%20P.%201995%20Biophys.%20Chem.%2056%2C%20171-181.&md5=659c5fb2114ceafb90545c6b8f5e6339">[ChemPort]</A><A HREF="/cgi-bin/pubmed/db=m&form=6&uid=7662864&Dopt=r">[Medline]</A>
</P>
</A>
<P>Ferretti, L., Karnik, S. S., Khorana, H. G., Nassal, M., &amp;
Oprian,
D. D. (1986)<I> Proc. Natl. Acad. </I><I>Sci.
</I><I>U.S.</I><I>A. 83</I>, 599-603.<citation reflink="journal=Proc.+Natl.+Acad.+Sci.+U.S.A.&year=1986&volume=83&page=599&reference=   Ferretti, L., Karnik, S. S., Khorana, H. G., Nassal, M., &amp; Oprian, D. D. (1986) Proc. Natl. Acad. Sci. U.S.A. 83, 599-603."><A HREF="/cgi-bin/chemport/version=0.0&coi=1:CAS:528:DyaL28XitVWgtL4%253D&journal=Proc.%2BNatl.%2BAcad.%2BSci.%2BU.S.A.&pubyear=1986&volume=83&startpage=599&reference=%20%20%20Ferretti%2C%20L.%2C%20Karnik%2C%20S.%20S.%2C%20Khorana%2C%20H.%20G.%2C%20Nassal%2C%20M.%2C%20a%20Oprian%2C%20D.%20D.%201986%20Proc.%20Natl.%20Acad.%20Sci.%20U.S.A.%2083%2C%20599-603.&md5=ba985fa67deae628971badc3d11cec2a">[ChemPort]</A><A HREF="/cgi-bin/pubmed/db=m&form=6&uid=3456156&Dopt=r">[Medline]</A>
</P>
</A>
<P>Franke, R. R., Sakmar, T. P., Oprian, D. D., &amp; Khorana. H.
G.
(1988) <I>J. Biol. Chem. 263</I>, 2119-2122.<citation reflink="journal=J.+Biol.+Chem.&year=1988&volume=263&page=2119&reference=   Franke, R. R., Sakmar, T. P., Oprian, D. D., &amp; Khorana. H. G. (1988) J. Biol. Chem. 263, 2119-2122."><A HREF="/cgi-bin/chemport/version=0.0&coi=1:CAS:528:DyaL1cXhsVGks7o%253D&journal=J.%2BBiol.%2BChem.&pubyear=1988&volume=263&startpage=2119&reference=%20%20%20Franke%2C%20R.%20R.%2C%20Sakmar%2C%20T.%20P.%2C%20Oprian%2C%20D.%20D.%2C%20a%20Khorana.%20H.%20G.%201988%20J.%20Biol.%20Chem.%20263%2C%202119-2122.&md5=e07cd63263ae8405c02adc8541156622">[ChemPort]</A><A HREF="/cgi-bin/pubmed/db=m&form=6&uid=3123487&Dopt=r">[Medline]</A>
</P>
</A>
<P>Franke, R. R., Sakmar, T. P., Graham, R. M., &amp; Khorana. H.
G.
(1992) <I>J. Biol. Chem. 267</I>, 14767-14774.<citation reflink="journal=J.+Biol.+Chem.&year=1992&volume=267&page=14767&reference=   Franke, R. R., Sakmar, T. P., Graham, R. M., &amp; Khorana. H. G. (1992) J. Biol. Chem. 267, 14767-14774."><A HREF="/cgi-bin/chemport/version=0.0&coi=1:CAS:528:DyaK38XltVKktrs%253D&journal=J.%2BBiol.%2BChem.&pubyear=1992&volume=267&startpage=14767&reference=%20%20%20Franke%2C%20R.%20R.%2C%20Sakmar%2C%20T.%20P.%2C%20Graham%2C%20R.%20M.%2C%20a%20Khorana.%20H.%20G.%201992%20J.%20Biol.%20Chem.%20267%2C%2014767-14774.&md5=49180b2bda08fb2891ac85406d84f3c1">[ChemPort]</A><A HREF="/cgi-bin/pubmed/db=m&form=6&uid=1634520&Dopt=r">[Medline]</A>
</P>
</A>
<P>Ganter, U. M., G&auml;rtner, W., &amp; Siebert, F. (1988a)
<I>Biochemistry
27</I>, 7480-7488.<citation reflink="journal=Biochemistry&year=1988&volume=27&page=7480&reference=   Ganter, U. M., G&auml;rtner, W., &amp; Siebert, F. (1988a) Biochemistry 27, 7480-7488."><A HREF="/cgi-bin/chemport/version=0.0&coi=1:CAS:528:DyaL1cXlt1Cmurc%253D&journal=Biochemistry&pubyear=1988&volume=27&startpage=7480&reference=%20%20%20Ganter%2C%20U.%20M.%2C%20Gartner%2C%20W.%2C%20a%20Siebert%2C%20F.%201988a%20Biochemistry%2027%2C%207480-7488.&md5=2b5f592c18b0d88b3ba324c05c02a2f6">[ChemPort]</A><A HREF="/cgi-bin/pubmed/db=m&form=6&uid=3207686&Dopt=r">[Medline]</A>
</P>
</A>
<P>Ganter, U. M., Schmid, E. D., &amp; Siebert, F. (1988b) <I>J.
Photochem.
Photobiol. 2</I>, 417-426.<citation reflink="journal=J.+Photochem.+Photobiol.&year=1988&volume=2&page=417&reference=   Ganter, U. M., Schmid, E. D., &amp; Siebert, F. (1988b) J. Photochem. Photobiol. 2, 417-426.">
</P>
</A>
<P>Ganter, U. M., Schmid E. D., Perez-Sala, D., Rando, R. R.,
&amp;
Siebert, F. (1989) <I>Biochemistry 28</I>,
5954-5962.<citation reflink="journal=Biochemistry&year=1989&volume=28&page=5954&reference=   Ganter, U. M., Schmid E. D., Perez-Sala, D., Rando, R. R., &amp; Siebert, F. (1989) Biochemistry 28, 5954-5962."><A HREF="/cgi-bin/chemport/version=0.0&coi=1:CAS:528:DyaL1MXktlGhurg%253D&journal=Biochemistry&pubyear=1989&volume=28&startpage=5954&reference=%20%20%20Ganter%2C%20U.%20M.%2C%20Schmid%20E.%20D.%2C%20Perez-Sala%2C%20D.%2C%20Rando%2C%20R.%20R.%2C%20a%20Siebert%2C%20F.%201989%20Biochemistry%2028%2C%205954-5962.&md5=9241f92ddeb5760db879db27dd701d01">[ChemPort]</A><A HREF="/cgi-bin/pubmed/db=m&form=6&uid=2505843&Dopt=r">[Medline]</A>
</P>
</A>
<P>Ganter, U. M., G&auml;rtner, W., &amp; Siebert, F. (1990) <I>Eur.
Biophys. J.
18</I>, 295-299.<citation reflink="journal=Eur.+Biophys.+J.&year=1990&volume=18&page=295&reference=   Ganter, U. M., G&auml;rtner, W., &amp; Siebert, F. (1990) Eur. Biophys. J. 18, 295-299."><A HREF="/cgi-bin/chemport/version=0.0&coi=1:CAS:528:DyaK3cXlsVOntLo%253D&journal=Eur.%2BBiophys.%2BJ.&pubyear=1990&volume=18&startpage=295&reference=%20%20%20Ganter%2C%20U.%20M.%2C%20Gartner%2C%20W.%2C%20a%20Siebert%2C%20F.%201990%20Eur.%20Biophys.%20J.%2018%2C%20295-299.&md5=787e5093b3bb3802cc34a1cf63b05408">[ChemPort]</A>
</P>
</A>
<P>Humphries, P., Kenna, P., &amp; Farrar, G. J. (1992)<I> Science
256</I>, 804-808.<citation reflink="journal=Science&year=1992&volume=256&page=804&reference=   Humphries, P., Kenna, P., &amp; Farrar, G. J. (1992) Science 256, 804-808.">
</P>
</A>
<P>J&auml;ger, F., Fahmy, K., Sakmar, T. P., &amp; Siebert, F.
(1994)<I>
Biochemistry 33</I>, 10878-10882.<citation reflink="journal=Biochemistry&year=1994&volume=33&page=10878&reference=   J&auml;ger, F., Fahmy, K., Sakmar, T. P., &amp; Siebert, F. (1994) Biochemistry 33, 10878-10882."><A HREF="/cgi-bin/chemport/version=0.0&coi=1:CAS:528:DyaK2cXlsleiu7c%253D&journal=Biochemistry&pubyear=1994&volume=33&startpage=10878&reference=%20%20%20Jager%2C%20F.%2C%20Fahmy%2C%20K.%2C%20Sakmar%2C%20T.%20P.%2C%20a%20Siebert%2C%20F.%201994%20Biochemistry%2033%2C%2010878-10882.&md5=05b336c24892639e683bf654cd28d25a">[ChemPort]</A><A HREF="/cgi-bin/pubmed/db=m&form=6&uid=7916209&Dopt=r">[Medline]</A>
</P>
</A>
<P>Kaushal, S., &amp; Khorana, H. G. (1994)<I> Biochemistry 33</I>,
10878-10882.<citation reflink="journal=Biochemistry&year=1994&volume=33&page=10878&reference=   Kaushal, S., &amp; Khorana, H. G. (1994) Biochemistry 33, 10878-10882."><A HREF="/cgi-bin/chemport/version=0.0&coi=1:CAS:528:DyaK2cXlsleiu7c%253D&journal=Biochemistry&pubyear=1994&volume=33&startpage=10878&reference=%20%20%20Kaushal%2C%20S.%2C%20a%20Khorana%2C%20H.%20G.%201994%20Biochemistry%2033%2C%2010878-10882.&md5=9129f108ccb05da4740517a2ac23155d">[ChemPort]</A><A HREF="/cgi-bin/pubmed/db=m&form=6&uid=7916209&Dopt=r">[Medline]</A>
</P>
</A>
<P>Kibelbek, J., Mitchell, D. C., Beach, J. M., &amp; Litman, B. L.
(1991)
<I>Biochemistry 30</I>, 6761-6768.<citation reflink="journal=Biochemistry&year=1991&volume=30&page=6761&reference=   Kibelbek, J., Mitchell, D. C., Beach, J. M., &amp; Litman, B. L. (1991) Biochemistry 30, 6761-6768."><A HREF="/cgi-bin/chemport/version=0.0&coi=1:CAS:528:DyaK3MXkt1Cmsb4%253D&journal=Biochemistry&pubyear=1991&volume=30&startpage=6761&reference=%20%20%20Kibelbek%2C%20J.%2C%20Mitchell%2C%20D.%20C.%2C%20Beach%2C%20J.%20M.%2C%20a%20Litman%2C%20B.%20L.%201991%20Biochemistry%2030%2C%206761-6768.&md5=aeefb6ccb6d7f59689f380d98df0d1e3">[ChemPort]</A><A HREF="/cgi-bin/pubmed/db=m&form=6&uid=1905955&Dopt=r">[Medline]</A>
</P>
</A>
<P>Klinger, A. L., &amp; Braiman, M. S. (1992) <I>Biophys. J. 63</I>,
1244-1255.<citation reflink="journal=Biophys.+J.&year=1992&volume=63&page=1244&reference=   Klinger, A. L., &amp; Braiman, M. S. (1992) Biophys. J. 63, 1244-1255."><A HREF="/cgi-bin/chemport/version=0.0&coi=1:CAS:528:DyaK3sXps1Gjtg%253D%253D&journal=Biophys.%2BJ.&pubyear=1992&volume=63&startpage=1244&reference=%20%20%20Klinger%2C%20A.%20L.%2C%20a%20Braiman%2C%20M.%20S.%201992%20Biophys.%20J.%2063%2C%201244-1255.&md5=28e5428fc2ffb9ad58034bf13197d599">[ChemPort]</A><A HREF="/cgi-bin/pubmed/db=m&form=6&uid=1477276&Dopt=r">[Medline]</A>
</P>
</A>
<P>K&ouml;nig, B., Arendt, A, McDowell, J. H., Kahlert, M., Hargrave,
P.
A., &amp; Hofmann, K. P. (1989) <I>Proc. Natl. Acad. Sci.
</I><I>U.S.</I><I>A. 86</I>,
6878-6882.<citation reflink="journal=Proc.+Natl.+Acad.+Sci.+U.S.A.&year=1989&volume=86&page=6878&reference=   K&ouml;nig, B., Arendt, A, McDowell, J. H., Kahlert, M., Hargrave, P. A., &amp; Hofmann, K. P. (1989) Proc. Natl. Acad. Sci. U.S.A. 86, 6878-6882."><A HREF="/cgi-bin/chemport/version=0.0&coi=1:CAS:528:DyaL1MXlvV2rsbs%253D&journal=Proc.%2BNatl.%2BAcad.%2BSci.%2BU.S.A.&pubyear=1989&volume=86&startpage=6878&reference=%20%20%20Konig%2C%20B.%2C%20Arendt%2C%20A%2C%20McDowell%2C%20J.%20H.%2C%20Kahlert%2C%20M.%2C%20Hargrave%2C%20P.%20A.%2C%20a%20Hofmann%2C%20K.%20P.%201989%20Proc.%20Natl.%20Acad.%20Sci.%20U.S.A.%2086%2C%206878-6882.&md5=41ea0cadc47eb014e527fbabeec3e03d">[ChemPort]</A><A HREF="/cgi-bin/pubmed/db=m&form=6&uid=2780545&Dopt=r">[Medline]</A>
</P>
</A>
<P>Lin, S. W., Sakmar, T. P., Franke, R. R., Khorana, H. G.,
&amp;
Mathies, R. A. (1992) <I>Biochemistry </I><I>31</I>,
5105-5111.<citation reflink="journal=Biochemistry&year=1992&volume=31&page=5105&reference=   Lin, S. W., Sakmar, T. P., Franke, R. R., Khorana, H. G., &amp; Mathies, R. A. (1992) Biochemistry 31, 5105-5111."><A HREF="/cgi-bin/chemport/version=0.0&coi=1:CAS:528:DyaK38XisFWns7g%253D&journal=Biochemistry&pubyear=1992&volume=31&startpage=5105&reference=%20%20%20Lin%2C%20S.%20W.%2C%20Sakmar%2C%20T.%20P.%2C%20Franke%2C%20R.%20R.%2C%20Khorana%2C%20H.%20G.%2C%20a%20Mathies%2C%20R.%20A.%201992%20Biochemistry%2031%2C%205105-5111.&md5=7632e164d7cd7defadb554f4b6fdb7d8">[ChemPort]</A>
</P>
</A>
<P>Lo, K.-M., Jones, S. S., Hackett, N. R., &amp; Khorana, H. G.
(1984)
<I>Proc. Natl. Acad. Sci. </I><I>U.S.</I><I>A. 81,</I>
2285-2289.<citation reflink="journal=Proc.+Natl.+Acad.+Sci.+U.S.A.&year=1984&volume=81&page=2285&reference=   Lo, K.-M., Jones, S. S., Hackett, N. R., &amp; Khorana, H. G. (1984) Proc. Natl. Acad. Sci. U.S.A. 81, 2285-2289."><A HREF="/cgi-bin/chemport/version=0.0&coi=1:CAS:528:DyaL2cXktFygsbk%253D&journal=Proc.%2BNatl.%2BAcad.%2BSci.%2BU.S.A.&pubyear=1984&volume=81&startpage=2285&reference=%20%20%20Lo%2C%20K.-M.%2C%20Jones%2C%20S.%20S.%2C%20Hackett%2C%20N.%20R.%2C%20a%20Khorana%2C%20H.%20G.%201984%20Proc.%20Natl.%20Acad.%20Sci.%20U.S.A.%2081%2C%202285-2289.&md5=381eeca0a2ebeadebe9a129ad0fa08bd">[ChemPort]</A>
</P>
</A>
<PAGEREF PAGERNUM="bi003502307545">
<P>Min, K. C., Zvyaga, T. A., Cypess, A. M., &amp; Sakmar, T. P.
(1993)
<I>J. Biol. Chem. 268</I>, 9400-9404.<citation reflink="journal=J.+Biol.+Chem.&year=1993&volume=268&page=9400&reference=    Min, K. C., Zvyaga, T. A., Cypess, A. M., &amp; Sakmar, T. P. (1993) J. Biol. Chem. 268, 9400-9404."><A HREF="/cgi-bin/chemport/version=0.0&coi=1:CAS:528:DyaK3sXkt1Grurc%253D&journal=J.%2BBiol.%2BChem.&pubyear=1993&volume=268&startpage=9400&reference=%20%20%20%20Min%2C%20K.%20C.%2C%20Zvyaga%2C%20T.%20A.%2C%20Cypess%2C%20A.%20M.%2C%20a%20Sakmar%2C%20T.%20P.%201993%20J.%20Biol.%20Chem.%20268%2C%209400-9404.&md5=67635efa9578c6d9de0667bf89a73b45">[ChemPort]</A><A HREF="/cgi-bin/pubmed/db=m&form=6&uid=8486634&Dopt=r">[Medline]</A>
</P>
</A>
<P>Nathans, J. (1990) <I>Biochemistry 29</I>,
9746-9752.<citation reflink="journal=Biochemistry&year=1990&volume=29&page=9746&reference=   Nathans, J. (1990) Biochemistry 29, 9746-9752."><A HREF="/cgi-bin/chemport/version=0.0&coi=1:CAS:528:DyaK3cXls1Olsro%253D&journal=Biochemistry&pubyear=1990&volume=29&startpage=9746&reference=%20%20%20Nathans%2C%20J.%201990%20Biochemistry%2029%2C%209746-9752.&md5=15d6c5055326502a07f661604fd40ae7">[ChemPort]</A><A HREF="/cgi-bin/pubmed/db=m&form=6&uid=1980212&Dopt=r">[Medline]</A>
</P>
</A>
<P>Palings, I., Pardoen, J. A., van den Berg, E., Winkel, C.,
Lugtenburg,
J., &amp; Mathies, R. A. (1987) <I>Biochemistry 26</I>,
2544-2556.<citation reflink="journal=Biochemistry&year=1987&volume=26&page=2544&reference=   Palings, I., Pardoen, J. A., van den Berg, E., Winkel, C., Lugtenburg, J., &amp; Mathies, R. A. (1987) Biochemistry 26, 2544-2556."><A HREF="/cgi-bin/chemport/version=0.0&coi=1:CAS:528:DyaL2sXhslWnsb8%253D&journal=Biochemistry&pubyear=1987&volume=26&startpage=2544&reference=%20%20%20Palings%2C%20I.%2C%20Pardoen%2C%20J.%20A.%2C%20van%20den%20Berg%2C%20E.%2C%20Winkel%2C%20C.%2C%20Lugtenburg%2C%20J.%2C%20a%20Mathies%2C%20R.%20A.%201987%20Biochemistry%2026%2C%202544-2556.&md5=68499921b682405dfebb19ed45657186">[ChemPort]</A><A HREF="/cgi-bin/pubmed/db=m&form=6&uid=3607032&Dopt=r">[Medline]</A>
</P>
</A>
<P>Rao, V. R., Cohen, G. B., &amp; Oprian, D. D. (1994) <I>Nature
367</I>,
639-642.<citation reflink="journal=Nature&year=1994&volume=367&page=639&reference=   Rao, V. R., Cohen, G. B., &amp; Oprian, D. D. (1994) Nature 367, 639-642."><A HREF="/cgi-bin/chemport/version=0.0&coi=1:CAS:528:DyaK2cXhs1SltLg%253D&journal=Nature&pubyear=1994&volume=367&startpage=639&reference=%20%20%20Rao%2C%20V.%20R.%2C%20Cohen%2C%20G.%20B.%2C%20a%20Oprian%2C%20D.%20D.%201994%20Nature%20367%2C%20639-642.&md5=ce391ccf7feca0a241694acc315d2898">[ChemPort]</A><A HREF="/cgi-bin/pubmed/db=m&form=6&uid=8107847&Dopt=r">[Medline]</A>
</P>
</A>
<P>Rath, P., DeCaluw&eacute;, L. L. J., Bovee-Geurts, P. H. M., DeGrip,
W.
J., &amp; Rothschild, K. J. (1993) <I>Biochemistry 32</I>,
10277-10282.<citation reflink="journal=Biochemistry&year=1993&volume=32&page=10277&reference=   Rath, P., DeCaluw&eacute;, L. L. J., Bovee-Geurts, P. H. M., DeGrip, W. J., &amp; Rothschild, K. J. (1993) Biochemistry 32, 10277-10282."><A HREF="/cgi-bin/chemport/version=0.0&coi=1:CAS:528:DyaK3sXlvFOqu7c%253D&journal=Biochemistry&pubyear=1993&volume=32&startpage=10277&reference=%20%20%20Rath%2C%20P.%2C%20DeCaluwe%2C%20L.%20L.%20J.%2C%20Bovee-Geurts%2C%20P.%20H.%20M.%2C%20DeGrip%2C%20W.%20J.%2C%20a%20Rothschild%2C%20K.%20J.%201993%20Biochemistry%2032%2C%2010277-10282.&md5=aa675295f93b31df0c78fe873a700ced">[ChemPort]</A><A HREF="/cgi-bin/pubmed/db=m&form=6&uid=8399169&Dopt=r">[Medline]</A>
</P>
</A>
<P>Rimai, L., Heyde, M. E., &amp; Gill, D. (1973) <I>J. Am. Chem. Soc.
95</I>,
4493-4501.<citation reflink="journal=J.+Am.+Chem.+Soc.&year=1973&volume=95&page=4493&reference=   Rimai, L., Heyde, M. E., &amp; Gill, D. (1973) J. Am. Chem. Soc. 95, 4493-4501."><A HREF="/cgi-bin/chemport/version=0.0&coi=1:CAS:528:DyaE3sXks1Ggtr4%253D&journal=J.%2BAm.%2BChem.%2BSoc.&pubyear=1973&volume=95&startpage=4493&reference=%20%20%20Rimai%2C%20L.%2C%20Heyde%2C%20M.%20E.%2C%20a%20Gill%2C%20D.%201973%20J.%20Am.%20Chem.%20Soc.%2095%2C%204493-4501.&md5=aac1c9b353687595f87e19a8894f2299">[ChemPort]</A><A HREF="/cgi-bin/pubmed/db=m&form=6&uid=4782645&Dopt=r">[Medline]</A>
</P>
</A>
<P>Robinson, P. R., Cohen, G. B., Zhukovsky, E. A., &amp; Oprian, D.
D.
(1992) <I>Neuron 9</I>, 719-725.<citation reflink="journal=Neuron&year=1992&volume=9&page=719&reference=   Robinson, P. R., Cohen, G. B., Zhukovsky, E. A., &amp; Oprian, D. D. (1992) Neuron 9, 719-725."><A HREF="/cgi-bin/chemport/version=0.0&coi=1:CAS:528:DyaK3sXps1Crsw%253D%253D&journal=Neuron&pubyear=1992&volume=9&startpage=719&reference=%20%20%20Robinson%2C%20P.%20R.%2C%20Cohen%2C%20G.%20B.%2C%20Zhukovsky%2C%20E.%20A.%2C%20a%20Oprian%2C%20D.%20D.%201992%20Neuron%209%2C%20719-725.&md5=91d18ceb33e7fc75ddb0e779aa8cb6ea">[ChemPort]</A><A HREF="/cgi-bin/pubmed/db=m&form=6&uid=1356370&Dopt=r">[Medline]</A>
</P>
</A>
<P>Sakmar, T. P., Franke, R. R., &amp; Khorana, H. G. (1989) <I>Proc.
Natl.
Acad. Sci. </I><I>U.S.</I><I>A. 86</I>,
8309-8313.<citation reflink="journal=Proc.+Natl.+Acad.+Sci.+U.S.A.&year=1989&volume=86&page=8309&reference=   Sakmar, T. P., Franke, R. R., &amp; Khorana, H. G. (1989) Proc. Natl. Acad. Sci. U.S.A. 86, 8309-8313."><A HREF="/cgi-bin/chemport/version=0.0&coi=1:CAS:528:DyaK3cXnt1CltA%253D%253D&journal=Proc.%2BNatl.%2BAcad.%2BSci.%2BU.S.A.&pubyear=1989&volume=86&startpage=8309&reference=%20%20%20Sakmar%2C%20T.%20P.%2C%20Franke%2C%20R.%20R.%2C%20a%20Khorana%2C%20H.%20G.%201989%20Proc.%20Natl.%20Acad.%20Sci.%20U.S.A.%2086%2C%208309-8313.&md5=ee83dbe3bdb4df6c4855fdb3d70cc962">[ChemPort]</A><A HREF="/cgi-bin/pubmed/db=m&form=6&uid=2573063&Dopt=r">[Medline]</A>
</P>
</A>
<P>Sakmar, T. P., Franke, R. R., &amp; Khorana, H. G. (1991) <I>Proc.
Natl.
Acad. Sci. </I><I>U.S.</I><I>A. 88</I>,
3079-3083.<citation reflink="journal=Proc.+Natl.+Acad.+Sci.+U.S.A.&year=1991&volume=88&page=3079&reference=   Sakmar, T. P., Franke, R. R., &amp; Khorana, H. G. (1991) Proc. Natl. Acad. Sci. U.S.A. 88, 3079-3083."><A HREF="/cgi-bin/chemport/version=0.0&coi=1:CAS:528:DyaK3MXisVejuro%253D&journal=Proc.%2BNatl.%2BAcad.%2BSci.%2BU.S.A.&pubyear=1991&volume=88&startpage=3079&reference=%20%20%20Sakmar%2C%20T.%20P.%2C%20Franke%2C%20R.%20R.%2C%20a%20Khorana%2C%20H.%20G.%201991%20Proc.%20Natl.%20Acad.%20Sci.%20U.S.A.%2088%2C%203079-3083.&md5=fd3099c99c1064efaaad91126806ec1b">[ChemPort]</A><A HREF="/cgi-bin/pubmed/db=m&form=6&uid=2014228&Dopt=r">[Medline]</A>
</P>
</A>
<P>Siebert, F., &amp; M&auml;ntele, W. (1980) <I>Biophys. Struct. Mech.
6</I>, 147-164.<citation reflink="journal=Biophys.+Struct.+Mech.&year=1980&volume=6&page=147&reference=   Siebert, F., &amp; M&auml;ntele, W. (1980) Biophys. Struct. Mech. 6, 147-164."><A HREF="/cgi-bin/chemport/version=0.0&coi=1:CAS:528:DyaL3cXltV2hsLw%253D&journal=Biophys.%2BStruct.%2BMech.&pubyear=1980&volume=6&startpage=147&reference=%20%20%20Siebert%2C%20F.%2C%20a%20Mantele%2C%20W.%201980%20Biophys.%20Struct.%20Mech.%206%2C%20147-164.&md5=e0ba6715789cdc1338709f59262f1c7e">[ChemPort]</A>
</P>
</A>
<P>Siebert, F., M&auml;ntele, W., &amp; Gerwert, K. (1983) <I>Eur. J.
Biochem.
136</I>, 119-127.<citation reflink="journal=Eur.+J.+Biochem.&year=1983&volume=136&page=119&reference=   Siebert, F., M&auml;ntele, W., &amp; Gerwert, K. (1983) Eur. J. Biochem. 136, 119-127."><A HREF="/cgi-bin/chemport/version=0.0&coi=1:CAS:528:DyaL3sXlvFSjtb0%253D&journal=Eur.%2BJ.%2BBiochem.&pubyear=1983&volume=136&startpage=119&reference=%20%20%20Siebert%2C%20F.%2C%20Mantele%2C%20W.%2C%20a%20Gerwert%2C%20K.%201983%20Eur.%20J.%20Biochem.%20136%2C%20119-127.&md5=b7a1b492a8647ce12d17c7373e2d8d82">[ChemPort]</A><A HREF="/cgi-bin/pubmed/db=m&form=6&uid=6311543&Dopt=r">[Medline]</A>
</P>
</A>
<P>Sieving, P. A., Richards, J. E., Naarendorp, F., Bingham, E.
L.,
Scott, K., Alpern, M. (1995) <I>Proc. </I><I>Natl. Acad. Sci.
</I><I>U.S.</I><I>A. 92</I>,
880-884.<citation reflink="journal=Proc.+Natl.+Acad.+Sci.+U.S.A.&year=1995&volume=92&page=880&reference=   Sieving, P. A., Richards, J. E., Naarendorp, F., Bingham, E. L., Scott, K., Alpern, M. (1995) Proc. Natl. Acad. Sci. U.S.A. 92, 880-884."><A HREF="/cgi-bin/chemport/version=0.0&coi=1:CAS:528:DyaK2MXjsVKiu7Y%253D&journal=Proc.%2BNatl.%2BAcad.%2BSci.%2BU.S.A.&pubyear=1995&volume=92&startpage=880&reference=%20%20%20Sieving%2C%20P.%20A.%2C%20Richards%2C%20J.%20E.%2C%20Naarendorp%2C%20F.%2C%20Bingham%2C%20E.%20L.%2C%20Scott%2C%20K.%2C%20Alpern%2C%20M.%201995%20Proc.%20Natl.%20Acad.%20Sci.%20U.S.A.%2092%2C%20880-884.&md5=15d6ae74ecd890ef350ca2a487d65274">[ChemPort]</A><A HREF="/cgi-bin/pubmed/db=m&form=6&uid=7846071&Dopt=r">[Medline]</A>
</P>
</A>
<P>Sung, C.-H., Davenport, C. M., Hennessey, J. C., Maumenee,
I.
H., Jacobson, S. G., Heckenlively, J. R., Howakowski, R.,
Gishman, G., Gouras, P., &amp; Nathans, J. (1991) <I>Proc. Natl.
Acad.
Sci. </I><I>U.S.</I><I>A. 88</I>, 6481-6485.<citation reflink="journal=Proc.+Natl.+Acad.+Sci.+U.S.A.&year=1991&volume=88&page=6481&reference=   Sung, C.-H., Davenport, C. M., Hennessey, J. C., Maumenee, I. H., Jacobson, S. G., Heckenlively, J. R., Howakowski, R., Gishman, G., Gouras, P., &amp; Nathans, J. (1991) Proc. Natl. Acad. Sci. U.S.A. 88, 6481-6485."><A HREF="/cgi-bin/chemport/version=0.0&coi=1:CAS:528:DyaK3MXlsFKgur4%253D&journal=Proc.%2BNatl.%2BAcad.%2BSci.%2BU.S.A.&pubyear=1991&volume=88&startpage=6481&reference=%20%20%20Sung%2C%20C.-H.%2C%20Davenport%2C%20C.%20M.%2C%20Hennessey%2C%20J.%20C.%2C%20Maumenee%2C%20I.%20H.%2C%20Jacobson%2C%20S.%20G.%2C%20Heckenlively%2C%20J.%20R.%2C%20Howakowski%2C%20R.%2C%20Gishman%2C%20G.%2C%20Gouras%2C%20P.%2C%20a%20Nathans%2C%20J.%201991%20Proc.%20Natl.%20Acad.%20Sci.%20U.S.A.%2088%2C%206481-6485.&md5=fab7576daf48bb37410453eab84ca988">[ChemPort]</A><A HREF="/cgi-bin/pubmed/db=m&form=6&uid=1862076&Dopt=r">[Medline]</A>
</P>
</A>
<P>Zhukovsky, E. A., &amp; Oprian, D. D. (1989) <I>Science 246,</I>
928-930.<citation reflink="journal=Science&year=1989&volume=246&page=928&reference=   Zhukovsky, E. A., &amp; Oprian, D. D. (1989) Science 246, 928-930."><A HREF="/cgi-bin/chemport/version=0.0&coi=1:CAS:528:DyaK3cXltVyntw%253D%253D&journal=Science&pubyear=1989&volume=246&startpage=928&reference=%20%20%20Zhukovsky%2C%20E.%20A.%2C%20a%20Oprian%2C%20D.%20D.%201989%20Science%20246%2C%20928-930.&md5=69b33b3cdc2989a8875f5f9e664df1ce">[ChemPort]</A><A HREF="/cgi-bin/pubmed/db=m&form=6&uid=2573154&Dopt=r">[Medline]</A>
</P>
</A>
<P>Zvyaga, T. A., Min, K. C., Beck, M., &amp; Sakmar, T. P. (1993)
<I>J.
Biol. Chem. 268</I>, 4661-4667.<citation reflink="journal=J.+Biol.+Chem.&year=1993&volume=268&page=4661&reference=   Zvyaga, T. A., Min, K. C., Beck, M., &amp; Sakmar, T. P. (1993) J. Biol. Chem. 268, 4661-4667."><A HREF="/cgi-bin/chemport/version=0.0&coi=1:CAS:528:DyaK3sXitVOqt7g%253D&journal=J.%2BBiol.%2BChem.&pubyear=1993&volume=268&startpage=4661&reference=%20%20%20Zvyaga%2C%20T.%20A.%2C%20Min%2C%20K.%20C.%2C%20Beck%2C%20M.%2C%20a%20Sakmar%2C%20T.%20P.%201993%20J.%20Biol.%20Chem.%20268%2C%204661-4667.&md5=e4b2f560a5d2ef77c2d1d213e55ca8c8">[ChemPort]</A> Correction:<I> J. Biol. Chem.
269,</I>
13056 (1994).

</A>
<P>Zvyaga, T. A., Fahmy, K., &amp; Sakmar, T. P. (1994)
<I>Biochemistry
33, </I>9753-9761.<citation reflink="journal=Biochemistry&year=1994&volume=33&page=9753&reference=  Zvyaga, T. A., Fahmy, K., &amp; Sakmar, T. P. (1994) Biochemistry 33, 9753-9761."><A HREF="/cgi-bin/chemport/version=0.0&coi=1:CAS:528:DyaK2cXltVagu7c%253D&journal=Biochemistry&pubyear=1994&volume=33&startpage=9753&reference=%20%20Zvyaga%2C%20T.%20A.%2C%20Fahmy%2C%20K.%2C%20a%20Sakmar%2C%20T.%20P.%201994%20Biochemistry%2033%2C%209753-9761.&md5=cdffeb48a499314746c0424fedccc799">[ChemPort]</A><A HREF="/cgi-bin/pubmed/db=m&form=6&uid=8068654&Dopt=r">[Medline]</A>
<PAGEREF PAGERNUM="bi003502307536">
<A NAME="bi960391nb00001"><P>1. Abbreviations: bp, base pairs; con A, concanavalin A;
FTIR,
Fourier-transform infrared; HOOP, hydrogen-out-of-plane; MII,
metarhodopsin II; ROS, rod outer segment.</P>
</A>
</P>
</A>
</BBGR>
<PAGEREF PAGERNUM="bi003502307537">
<PAGEREF PAGERNUM="bi003502307538">
<PAGEREF PAGERNUM="bi003502307539">
<PAGEREF PAGERNUM="bi003502307540">
<PAGEREF PAGERNUM="bi003502307541">
<PAGEREF PAGERNUM="bi003502307538">
<A NAME="bi960391nt00001">
<HR><TABLE BORDER>
<CAPTION ALIGN="top">
<P>
Table 1: Spectral and Biochemical Properties of Recombinant
Pigments</P>
</CAPTION>
<TR>
<TD>&nbsp;</TD>
<TD COLSPAN="3">
<P><IMG SRC="/images/gifchars/lambda.gif" BORDER=0 ><SUB>max</SUB> value (nm)</P>
</TD>
</TR>
<TR>
<TD>&nbsp;</TD>
<TD>&nbsp;</TD>
<TD COLSPAN="2">
<P>photoproduct<I><SUP>b</SUP></I></P>
</TD>
</TR>
<TR>
<TD>
<P>pigment</P>
</TD>
<TD>
<P>dark</P>
</TD>
<TD>
<P>1</P>
</TD>
<TD>
<P>2</P>
</TD>
<TD>
<P><IMG SRC="/images/gifchars/epsilon.gif" BORDER=0 ><I><SUP>c</SUP></I> (M<SUP>-</SUP><SUP>1</SUP> cm<SUP>-</SUP><SUP>1</SUP>)</P>
</TD>
<TD>
<P>activation rate<I><SUP>a</SUP></I> (pmol/(pmol&#183;min))
(mean &#177; SE (<I>n</I>))</P>
</TD>
<TD>
<P>rel act.
</P>
</TD>
</TR>
<TR>
<TD>
<P>rhodopsin
</P>
</TD>
<TD>
<P>500
</P>
</TD>
<TD>
<P>380
</P>
</TD>
<TD>
<P>&nbsp;</P>
</TD>
<TD>
<P>42,700
</P>
</TD>
<TD>
<P>70 &#177; 1.8 (4)
</P>
</TD>
<TD>
<P>1.00
</P>
</TD>
</TR>
<TR>
<TD>
<P>G90D
</P>
</TD>
<TD>
<P>484
</P>
</TD>
<TD>
<P>468
</P>
</TD>
<TD>
<P>380
</P>
</TD>
<TD>
<P>37,000
</P>
</TD>
<TD>
<P>41 &#177; 1.2 (4)
</P>
</TD>
<TD>
<P>0.59
</P>
</TD>
</TR>
<TR>
<TD>
<P>G90D/E113A
</P>
</TD>
<TD>
<P>474
</P>
</TD>
<TD>
<P>454
</P>
</TD>
<TD>
<P>380
</P>
</TD>
<TD>
<P>32,000
</P>
</TD>
<TD>
<P>30 &#177; 0.8 (4)
</P>
</TD>
<TD>
<P>0.43</P>
</TD>
</TR>
</TABLE>
<P><I><SUP>a</SUP></I> Figure 3 shows the results of a single GTP<IMG SRC="/images/gifchars/gamma.gif" BORDER=0 >S
filter-binding experiment.<I><SUP>b</SUP></I> The
<IMG SRC="/images/gifchars/lambda.gif" BORDER=0 ><SUB>max</SUB> value (nm) of the photoproduct after illumination as
described
in Materials and Methods is given. For the mutant pigments, a
small amount of 380 nm-absorbing species cannot be distinguished from
free
<I>all</I>-<I>trans</I>-retinal caused by Schiff base
hydrolysis.<I><SUP>c</SUP></I> Extinction coefficients
(<IMG SRC="/images/gifchars/epsilon.gif" BORDER=0 >) were measured by the acid denaturation method (Sakmar et
al.,
1989).</P>
<HR>
</TABLE>
<A NAME="bi960391nt00002">
<HR><TABLE BORDER>
<CAPTION ALIGN="top">
<P>
Table 2: Comparison of Decay Kinetics of Mutant
Pigments</P>
</CAPTION>
<TR>
<TD>&nbsp;</TD>
<TD COLSPAN="3">
<P>half-time of decay (<I>t</I><SUB>1&#47;2</SUB>) (min)<I><SUP>a</SUP></I><I><SUP>
</SUP></I>(mean &#177; SE (<I>n</I>))</P>
</TD>
</TR>
<TR>
<TD>
<P>pigment</P>
</TD>
<TD>
<P>photoproduct<I><SUP>b</SUP></I></P>
</TD>
<TD>
<P>activity<I><SUP>c</SUP></I></P>
</TD>
<TD>
<P>Schiff base<I><SUP>d</SUP></I>
</P>
</TD>
</TR>
<TR>
<TD>
<P>G90D
</P>
</TD>
<TD>
<P>3.9 &#177; 1.3 (3)
</P>
</TD>
<TD>
<P>3.9 &#177; 0.9 (3)
</P>
</TD>
<TD>
<P>4.2 &#177; 0.1 (2)
</P>
</TD>
</TR>
<TR>
<TD>
<P>G90D/E113A
</P>
</TD>
<TD>
<P>12.5 &#177; 0.4 (3)
</P>
</TD>
<TD>
<P>14.5 &#177; 2.3 (5)
</P>
</TD>
<TD>
<P>13.5 &#177; 1.7 (2)</P>
</TD>
</TR>
</TABLE>
<P><I><SUP>a</SUP></I> Decay experiments were carried out in which
the photoproduct
spectrum, the transducin-activating ability, and the presence of a
Schiff
base chromophore linkage were monitored simultaneously at pH
8.0
at 15 <IMG SRC="/images/entities/deg.gif">C. A half-time of decay was extracted from the function
that
best fit the data for each independent experiment (see Materials
and
Methods). The average values of multiple experiments are
presented.<I><SUP>b</SUP></I> The half-times of disappearance of the
primary photoproducts are
given: G90D - <I>A</I><SUB>468</SUB>; G90D/E113A -
<I>A</I><SUB>454</SUB>.<I><SUP>c</SUP></I> Activity
refers to the
ability of the light-activated pigment to catalyze GTP<IMG SRC="/images/gifchars/gamma.gif" BORDER=0 >S binding
by
transducin. The half-times of decay are for activation rates
as
determined in filter-binding experiments as shown in Figure
3.<I><SUP>d</SUP></I> Schiff
base decay was measured by the acid denaturation method (Sakmar
et
al., 1989).</P>
<HR>
</TABLE>
</BM>
<!-- Copyright IBM Corporation, 2001-2004 -->

<script language="JavaScript">
<!--

function SymError()
{
  return true;
}

window.onerror = SymError;

var SymRealWinOpen = window.open;

function SymWinOpen(url, name, attributes)
{
  return (new Object());
}

window.open = SymWinOpen;

//-->
</script>

<script language='Javascript1.1' type='text/javascript'>var SA_ID='acspix;acspix';</script>
<script language='Javascript1.1' type='text/javascript' src='//stats.surfaid.ihost.com/sacdcg3p_acspix_acspix.js'></script>
<noscript><img src='//stats.surfaid.ihost.com/crc/images/uc.GIF?1.20&acspix&acspix&noscript' height='1' width='1' alt='' border='0'/></noscript>
</BODY></HTML>

<script language="JavaScript">
<!--
var SymRealOnLoad;
var SymRealOnUnload;

function SymOnUnload()
{
  window.open = SymWinOpen;
  if(SymRealOnUnload != null)
     SymRealOnUnload();
}

function SymOnLoad()
{
  if(SymRealOnLoad != null)
     SymRealOnLoad();
  window.open = SymRealWinOpen;
  SymRealOnUnload = window.onunload;
  window.onunload = SymOnUnload;
}

SymRealOnLoad = window.onload;
window.onload = SymOnLoad;

//-->
</script>

